// DO NOT EDIT THIS FILE!
// This file is autogenerated from the internet-sourced OBO files.
// Any edits made will be lost when it is recreated.

// ReSharper disable InconsistentNaming

namespace PSI_Interface.CV
{
    public static partial class CV
    {
        /// <summary>
        /// A full enumeration of the Controlled Vocabularies PSI-MS, UNIMOD, and the vocabularies they depend on
        /// </summary>
        public enum CVID : int
        {
            /// <summary>CVID_Unknown</summary>
            CVID_Unknown = -1,

            /// <summary>Proteomics Standards Initiative Mass Spectrometry Vocabularies.</summary>
            MS_Proteomics_Standards_Initiative_Mass_Spectrometry_Vocabularies = 0,

            /// <summary>A reference number relevant to the sample under study.</summary>
            MS_sample_number = 1000001,

            /// <summary>A reference string relevant to the sample under study.</summary>
            MS_sample_name = 1000002,

            /// <summary>The chemical phase of a pure sample, or the state of a mixed sample.</summary>
            MS_sample_state = 1000003,

            /// <summary>Total mass of sample used.</summary>
            MS_sample_mass = 1000004,

            /// <summary>Total volume of solution used.</summary>
            MS_sample_volume = 1000005,

            /// <summary>Concentration of sample in picomol/ul, femtomol/ul or attomol/ul solution used.</summary>
            MS_sample_concentration = 1000006,

            /// <summary>The nature of the sample inlet.</summary>
            MS_inlet_type = 1000007,

            /// <summary>The method by which gas phase ions are generated from the sample.</summary>
            MS_ionization_type = 1000008,

            /// <summary>OBSOLETE Whether positive or negative ions are selected for analysis by the spectrometer.</summary>
            MS_ionization_mode_OBSOLETE = 1000009,

            /// <summary>OBSOLETE The common name of the particular analyzer stage being described. Synonym of mass analyzer, should be obsoleted.</summary>
            MS_analyzer_type_OBSOLETE = 1000010,

            /// <summary>Smallest mass difference between two equal magnitude peaks so that the valley between them is a specified fraction of the peak height.</summary>
            MS_mass_resolution = 1000011,

            /// <summary>Which of the available standard measures is used to define whether two peaks are separate.</summary>
            MS_resolution_measurement_method = 1000012,

            /// <summary>OBSOLETE Specify the nature of resolution for the mass analyzer. Resolution is usually either constant with respect to m/z or proportional to m/z.</summary>
            MS_resolution_type_OBSOLETE = 1000013,

            /// <summary>Accuracy is the degree of conformity of a measured mass to its actual value.</summary>
            MS_accuracy = 1000014,

            /// <summary>Rate in Th/sec for scanning analyzers.</summary>
            MS_scan_rate = 1000015,

            /// <summary>The time that an analyzer started a scan, relative to the start of the MS run.</summary>
            MS_scan_start_time = 1000016,

            /// <summary>OBSOLETE Describes the type of mass analysis being performed. Two primary modes are: typical acquisition over a range of masses (Mass Scan), and Selected Ion Detection. The primary difference is that Selected Ion Detection produces a single value for the signal at the selected mass rather than producing a mass spectrum.</summary>
            MS_Scan_Function_OBSOLETE = 1000017,

            /// <summary>Direction in terms of m/z of the scan for scanning analyzers (low to high, or high to low).</summary>
            MS_scan_direction = 1000018,

            /// <summary>Describes the function in control of the m/z scan (for scanning instruments). Commonly the scan function is linear, but in principle any function can be used.</summary>
            MS_scan_law = 1000019,

            /// <summary>Describes the acquisition data type produced by a tandem mass spectrometry experiment.</summary>
            MS_scanning_method_OBSOLETE = 1000020,

            /// <summary>Status of the reflectron, turned on or off.</summary>
            MS_reflectron_state = 1000021,

            /// <summary>The length of the field free drift space in a time of flight mass spectrometer.</summary>
            MS_TOF_Total_Path_Length = 1000022,

            /// <summary>OBSOLETE The total width (i.e. not half for plus-or-minus) of the gate applied around a selected precursor ion.</summary>
            MS_isolation_width_OBSOLETE = 1000023,

            /// <summary>Final MS level achieved when performing PFF with the ion trap (e.g. MS E10).</summary>
            MS_final_MS_exponent = 1000024,

            /// <summary>A property of space that produces a force on a charged particle equal to qv x B where q is the particle charge and v its velocity.</summary>
            MS_magnetic_field_strength = 1000025,

            /// <summary>Type of detector used in the mass spectrometer.</summary>
            MS_detector_type = 1000026,

            /// <summary>Method by which detector signal is acquired by the data system.</summary>
            MS_detector_acquisition_mode = 1000027,

            /// <summary>The resolving power of the detector to detect the smallest difference between two ions so that the valley between them is a specified fraction of the peak height.</summary>
            MS_detector_resolution = 1000028,

            /// <summary>The rate of signal sampling (measurement) with respect to time.</summary>
            MS_sampling_frequency = 1000029,

            /// <summary>OBSOLETE Name of instrument vendor.</summary>
            MS_vendor_OBSOLETE = 1000030,

            /// <summary>Instrument model name not including the vendor's name.</summary>
            MS_instrument_model = 1000031,

            /// <summary>Free text description of a single customization made to the instrument; for several modifications, use several entries.</summary>
            MS_customization = 1000032,

            /// <summary>The removal of isotope peaks to represent the fragment ion as one data point and is commonly done to reduce complexity. It is done in conjunction with the charge state deconvolution.</summary>
            MS_deisotoping = 1000033,

            /// <summary>The determination of the mass of an ion based on the mass spectral peaks that represent multiple-charge ions.</summary>
            MS_charge_deconvolution = 1000034,

            /// <summary>Spectral peak processing conducted on the acquired data to convert profile data to centroided data.</summary>
            MS_peak_picking = 1000035,

            /// <summary>OBSOLETE.</summary>
            MS_scan_mode_OBSOLETE = 1000036,

            /// <summary>OBSOLETE Terms to describe the polarity setting of the instrument.</summary>
            MS_polarity_OBSOLETE = 1000037,

            /// <summary>OBSOLETE Acquisition time in minutes.</summary>
            MS_minute_OBSOLETE = 1000038,

            /// <summary>OBSOLETE Acquisition time in seconds.</summary>
            MS_second_OBSOLETE = 1000039,

            /// <summary>Three-character symbol m/z is used to denote the quantity formed by dividing the mass of an ion in unified atomic mass units by its charge number (regardless of sign). The symbol is written in italicized lower case letters with no spaces. Note 1: The term mass-to-charge-ratio is deprecated. Mass-to-charge ratio has been used for the abscissa of a mass spectrum, although the quantity measured is not the quotient of the ion's mass to its electric charge. The three-character symbol m/z is recommended for the quantity that is the independent variable in a mass spectrum Note 2: The proposed unit thomson (Th) is deprecated.</summary>
            MS_m_z = 1000040,

            /// <summary>Number of net charges, positive or negative, on an ion.</summary>
            MS_charge_state = 1000041,

            /// <summary>Intensity of ions as measured by the height or area of a peak in a mass spectrum.</summary>
            MS_peak_intensity = 1000042,

            /// <summary>Intensity units are commonly arbitrary. Detected in counts per second (cps) when using counting detectors, but measured in volts when using analog detectors.</summary>
            MS_intensity_unit = 1000043,

            /// <summary>Fragmentation method used for dissociation or fragmentation.</summary>
            MS_dissociation_method = 1000044,

            /// <summary>Energy for an ion experiencing collision with a stationary gas particle resulting in dissociation of the ion.</summary>
            MS_collision_energy = 1000045,

            /// <summary>OBSOLETE Energy units are represented in either eV or Joules.</summary>
            MS_energy_unit_OBSOLETE = 1000046,

            /// <summary>State if the sample is in emulsion form.</summary>
            MS_emulsion = 1000047,

            /// <summary>State if the sample is in gaseous form.</summary>
            MS_gaseous_sample_state = 1000048,

            /// <summary>State if the sample is in liquid form.</summary>
            MS_liquid_sample_state = 1000049,

            /// <summary>State if the sample is in solid form.</summary>
            MS_solid_sample_state = 1000050,

            /// <summary>State if the sample is in solution form.</summary>
            MS_solution = 1000051,

            /// <summary>State if the sample is in suspension form.</summary>
            MS_suspension = 1000052,

            /// <summary>Sample batch lot identifier.</summary>
            MS_sample_batch = 1000053,

            /// <summary>OBSOLETE Chromatographic conditions used to obtain the sample.</summary>
            MS_chromatography_OBSOLETE = 1000054,

            /// <summary>Fast atom bombardment ionization in which the analyte in solution is entrained in a flowing liquid matrix.</summary>
            MS_continuous_flow_fast_atom_bombardment = 1000055,

            /// <summary>The sample is directly inserted into the ion source, usually on the end of a heatable probe.</summary>
            MS_direct_inlet = 1000056,

            /// <summary>Inlet used for introducing the liquid sample into an electrospray ionization source.</summary>
            MS_electrospray_inlet = 1000057,

            /// <summary>Sample is directly injected or infused into the ionization source.</summary>
            MS_flow_injection_analysis = 1000058,

            /// <summary>A gas discharge ion source in which the energy to the plasma is supplied by electromagnetic induction.</summary>
            MS_inductively_coupled_plasma = 1000059,

            /// <summary>The continuous flow of solution of a sample into the ionization source.</summary>
            MS_infusion = 1000060,

            /// <summary>A device that separates carrier gas from gaseous analyte molecules on the basis of diffusivity.</summary>
            MS_jet_separator = 1000061,

            /// <summary>A device to separate carrier molecules from analyte molecules on the basis of ease of diffusion across a semipermeable membrane.</summary>
            MS_membrane_separator = 1000062,

            /// <summary>Continuous moving surface in the form of a belt which passes through an ion source carrying analyte molecules.</summary>
            MS_moving_belt = 1000063,

            /// <summary>Continuous moving surface in the form of a wire which passes through an ion source carrying analyte molecules.</summary>
            MS_moving_wire = 1000064,

            /// <summary>A division of flowing stream of liquid into two streams.</summary>
            MS_open_split = 1000065,

            /// <summary>Method for generating ions from a solution of an analyte.</summary>
            MS_particle_beam = 1000066,

            /// <summary>A sample inlet method involving a reservoir.</summary>
            MS_reservoir = 1000067,

            /// <summary>A disc composed of a flexible material that seals the entrance to the reservoir. Can also be entrance to the vacuum chamber.</summary>
            MS_septum = 1000068,

            /// <summary>A method for generating gas phase ions from a solution of an analyte by rapid heating of the sample.</summary>
            MS_thermospray_inlet = 1000069,

            /// <summary>Chemical ionization that takes place at atmospheric pressure as opposed to the reduced pressure is normally used for chemical ionization.</summary>
            MS_atmospheric_pressure_chemical_ionization = 1000070,

            /// <summary>The formation of a new ion by the reaction of a neutral species with an ion. The process may involve transfer of an electron, a proton or other charged species between the reactants. When a positive ion results from chemical ionization the term may be used without qualification. When a negative ion results the term negative ion chemical ionization should be used. Note that this term is not synonymous with chemi-ionization.</summary>
            MS_chemical_ionization = 1000071,

            /// <summary>OBSOLETE The ionization of an atom or molecule by electrons that are typically accelerated to energies between 50 and 150 eV. Usually 70 eV electrons are used to produce positive ions. The term 'electron impact' is not recommended.</summary>
            MS_Electronic_Ionization_OBSOLETE = 1000072,

            /// <summary>A process in which ionized species in the gas phase are produced from an analyte-containing solution via highly charged fine droplets, by means of spraying the solution from a narrow-bore needle tip at atmospheric pressure in the presence of a high electric field. When a pressurized gas is used to aid in the formation of a stable spray, the term pneumatically assisted electrospray ionization is used. The term ion spray is not recommended.</summary>
            MS_electrospray_ionization = 1000073,

            /// <summary>The ionization of any species by the interaction of a focused beam of neutral atoms having a translational energy of several thousand eV with a sample that is typically dissolved in a solvent matrix. See also secondary ionization.</summary>
            MS_fast_atom_bombardment_ionization = 1000074,

            /// <summary>The formation of gas-phase ions from molecules that are present in a solid or solvent matrix that is irradiated with a pulsed laser. See also laser desorption/ionization.</summary>
            MS_matrix_assisted_laser_desorption_ionization = 1000075,

            /// <summary>OBSOLETE.</summary>
            MS_negative_ion_mode_OBSOLETE = 1000076,

            /// <summary>OBSOLETE.</summary>
            MS_positive_ion_mode_OBSOLETE = 1000077,

            /// <summary>A linear ion trap mass spectrometer where ions are ejected along the axis of the analyzer.</summary>
            MS_axial_ejection_linear_ion_trap = 1000078,

            /// <summary>A mass spectrometer based on the principle of ion cyclotron resonance in which an ion in a magnetic field moves in a circular orbit at a frequency characteristic of its m/z value. Ions are coherently excited to a larger radius orbit using a pulse of radio frequency energy and their image charge is detected on receiver plates as a time domain signal. Fourier transformation of the time domain signal results in a frequency domain signal which is converted to a mass spectrum based in the inverse relationship between frequency and m/z.</summary>
            MS_fourier_transform_ion_cyclotron_resonance_mass_spectrometer = 1000079,

            /// <summary>A device that produces a magnetic field perpendicular to a charged particle beam that deflects the beam to an extent that is proportional to the particle momentum per unit charge. For a monoenergetic beam, the deflection is proportional to m/z.</summary>
            MS_magnetic_sector = 1000080,

            /// <summary>A mass spectrometer that consists of four parallel rods whose centers form the corners of a square and whose opposing poles are connected. The voltage applied to the rods is a superposition of a static potential and a sinusoidal radio frequency potential. The motion of an ion in the x and y dimensions is described by the Matthieu equation whose solutions show that ions in a particular m/z range can be transmitted along the z axis.</summary>
            MS_quadrupole = 1000081,

            /// <summary>Quadrupole Ion Trap mass analyzer captures the ions in a three dimensional ion trap and then selectively ejects them by varying the RF and DC potentials.</summary>
            MS_quadrupole_ion_trap = 1000082,

            /// <summary>A linear ion trap mass spectrometer where ions are ejected along the radius of the analyzer.</summary>
            MS_radial_ejection_linear_ion_trap = 1000083,

            /// <summary>Instrument that separates ions by m/z in a field-free region after acceleration to a fixed acceleration energy.</summary>
            MS_time_of_flight = 1000084,

            /// <summary>An attribute of resolution when recording the detector response in absence of the analyte.</summary>
            MS_baseline = 1000085,

            /// <summary>A measure of resolution represented as width of the peak at half peak height.</summary>
            MS_full_width_at_half_maximum = 1000086,

            /// <summary>An attribute of resolution when the ratio between adjacent signals is 10% of the signal height.</summary>
            MS_ten_percent_valley = 1000087,

            /// <summary>OBSOLETE When resolution is constant with respect to m/z.</summary>
            MS_constant_OBSOLETE = 1000088,

            /// <summary>OBSOLETE When resolution is proportional with respect to m/z.</summary>
            MS_proportional_OBSOLETE = 1000089,

            /// <summary>OBSOLETE A variation of instrument where a selected mass is scanned.</summary>
            MS_mass_scan_OBSOLETE = 1000090,

            /// <summary>OBSOLETE Please see Single Ion Monitoring.</summary>
            MS_selected_ion_detection_OBSOLETE = 1000091,

            /// <summary>High to low direction in terms of m/z of the scan for scanning analyzers.</summary>
            MS_decreasing_m_z_scan = 1000092,

            /// <summary>Low to high direction in terms of m/z of the scan for scanning analyzers.</summary>
            MS_increasing_m_z_scan = 1000093,

            /// <summary>The mass scan is done in exponential mode.</summary>
            MS_exponential = 1000094,

            /// <summary>OBSOLETE The mass scan is done in linear mode.</summary>
            MS_linear_OBSOLETE = 1000095,

            /// <summary>The mass scan is done in quadratic mode.</summary>
            MS_quadratic = 1000096,

            /// <summary>OBSOLETE A spectrum formed of all product ions that have been produced with a selected m/z decrement from any precursor ions. The spectrum shown correlates to the precursor ion spectrum. See also neutral loss spectrum.</summary>
            MS_constant_neutral_mass_loss_OBSOLETE = 1000097,

            /// <summary>OBSOLETE Data acquired when monitoring the ion current of a few specific m/z values. Remap to MS:1000205 -Selected Ion Monitoring.</summary>
            MS_multiple_ion_monitoring_OBSOLETE = 1000098,

            /// <summary>OBSOLETE This term is not recommended. See Selected Reaction Monitoring.</summary>
            MS_multiple_reaction_monitoring_OBSOLETE = 1000099,

            /// <summary>OBSOLETE The specific scan function or process that will record a precursor ion spectrum.</summary>
            MS_precursor_ion_scan_OBSOLETE = 1000100,

            /// <summary>OBSOLETE The specific scan function or process that records product ion spectrum.</summary>
            MS_product_ion_scan_OBSOLETE = 1000101,

            /// <summary>OBSOLETE The operation of a mass spectrometer to monitor a single ion rather than scanning entire mass spectrum.</summary>
            MS_single_ion_monitoring_OBSOLETE = 1000102,

            /// <summary>OBSOLETE This term is not recommended. See Selected Reaction Monitoring.</summary>
            MS_single_reaction_monitoring_OBSOLETE = 1000103,

            /// <summary>OBSOLETE None.</summary>
            MS_None____OBSOLETE = 1000104,

            /// <summary>Reflectron is off.</summary>
            MS_reflectron_off = 1000105,

            /// <summary>Reflectron is on.</summary>
            MS_reflectron_on = 1000106,

            /// <summary>A horn-shaped (or cone-shaped) continuous dynode particle multiplier. The ion strikes the inner surface of the device and induces the production of secondary electrons that in turn impinge on the inner surfaces to produce more secondary electrons. This avalanche effect produces an increase in signal in the final measured current pulse.</summary>
            MS_channeltron = 1000107,

            /// <summary>A surface that is held at high potential so that ions striking the surface produce electrons that are subsequently detected.</summary>
            MS_conversion_dynode_electron_multiplier = 1000108,

            /// <summary>A detector in which ions strike a conversion dynode to produce electrons that in turn generate photons through a phosphorescent screen that are detected by a photomultiplier.</summary>
            MS_conversion_dynode_photomultiplier = 1000109,

            /// <summary>Detector consisting of a conversion dynode, scintillator and photomultiplier. The metal knob at high potential emits secondary electrons when ions impinge on the surface. The secondary electrons are accelerated onto the scintillator that produces light that is then detected by the photomultiplier detector.</summary>
            MS_daly_detector = 1000110,

            /// <summary>A device to amplify the current of a beam or packet of charged particles or photons by incidence upon the surface of an electrode to produce secondary electrons.</summary>
            MS_electron_multiplier_tube = 1000111,

            /// <summary>A conducting cup or chamber that intercepts a charged particle beam and is electrically connected to a current measuring device.</summary>
            MS_faraday_cup = 1000112,

            /// <summary>An array of detectors for spatially disperse ion beams in which all ions simultaneously impinge on the detector plane.</summary>
            MS_focal_plane_array = 1000113,

            /// <summary>A thin plate that contains a closely spaced array of channels that each act as a continuous dynode particle multiplier. A charged particle, fast neutral particle, or photon striking the plate causes a cascade of secondary electrons that ultimately exits the opposite side of the plate.</summary>
            MS_microchannel_plate_detector = 1000114,

            /// <summary>A detector system commonly used in inductively coupled plasma mass spectrometers.</summary>
            MS_multi_collector = 1000115,

            /// <summary>A detector for conversion of the ion/electron signal into photon(s) which are then amplified and detected.</summary>
            MS_photomultiplier = 1000116,

            /// <summary>Analog-to-digital converter (abbreviated ADC, A/D or A to D) is an electronic integrated circuit (i/c) that converts continuous signals to discrete digital numbers.</summary>
            MS_analog_digital_converter = 1000117,

            /// <summary>Definition to do.</summary>
            MS_pulse_counting = 1000118,

            /// <summary>A device for converting a signal of sporadic pluses into a digital representation of their time indices.</summary>
            MS_time_digital_converter = 1000119,

            /// <summary>A detector acquisition mode used for detecting transient signals.</summary>
            MS_transient_recorder = 1000120,

            /// <summary>The brand of instruments from the joint venture between Applied Biosystems and MDS Analytical Technologies (formerly MDS SCIEX). Previously branded as \ Applied Biosystems|MDS SCIEX\ .</summary>
            MS_SCIEX_instrument_model = 1000121,

            /// <summary>Bruker Daltonics' instrument model.</summary>
            MS_Bruker_Daltonics_instrument_model = 1000122,

            /// <summary>IonSpec corporation instrument model.</summary>
            MS_IonSpec_instrument_model = 1000123,

            /// <summary>Shimadzu corporation instrument model.</summary>
            MS_Shimadzu_instrument_model = 1000124,

            /// <summary>ThermoFinnigan from Thermo Electron Corporation instrument model.</summary>
            MS_Thermo_Finnigan_instrument_model = 1000125,

            /// <summary>Waters Corporation instrument model.</summary>
            MS_Waters_instrument_model = 1000126,

            /// <summary>Processing of profile data to produce spectra that contains discrete peaks of zero width. Often used to reduce the size of dataset.</summary>
            MS_centroid_spectrum = 1000127,

            /// <summary>A profile mass spectrum is created when data is recorded with ion current (counts per second) on one axis and mass/charge ratio on another axis.</summary>
            MS_profile_spectrum = 1000128,

            /// <summary>Polarity of the scan is negative.</summary>
            MS_negative_scan = 1000129,

            /// <summary>Polarity of the scan is positive.</summary>
            MS_positive_scan = 1000130,

            /// <summary>The number of counted events observed in one or a group of elements of a detector.</summary>
            MS_number_of_detector_counts = 1000131,

            /// <summary>The magnitude of a peak or measurement element expressed in terms of the percentage of the magnitude of the base peak intensity.</summary>
            MS_percent_of_base_peak = 1000132,

            /// <summary>The dissociation of an ion after collisional excitation. The term collisional-activated dissociation is not recommended.</summary>
            MS_collision_induced_dissociation = 1000133,

            /// <summary>The ionization of material in a solid sample by bombarding it with ionic or neutral atoms formed as a result of the fission of a suitable nuclide, typically 252Cf. Synonymous with fission fragment ionization.</summary>
            MS_plasma_desorption = 1000134,

            /// <summary>A technique specific to reflectron time-of-flight mass spectrometers where product ions of metastable transitions or collision-induced dissociations generated in the drift tube prior to entering the reflectron are m/z separated to yield product ion spectra.</summary>
            MS_post_source_decay = 1000135,

            /// <summary>Fragmentation that results from the collision of an ion with a surface.</summary>
            MS_surface_induced_dissociation = 1000136,

            /// <summary>OBSOLETE A non-SI unit of energy (eV) defined as the energy acquired by a particle containing one unit of charge through a potential difference of one volt. An electron-volt is equal to 1.602 176 53(14) x 10^-19 J.</summary>
            MS_electron_volt_OBSOLETE = 1000137,

            /// <summary>Instrument setting, expressed in percent, for adjusting collisional energies of ions in an effort to provide equivalent excitation of all ions.</summary>
            MS_normalized_collision_energy = 1000138,

            /// <summary>Applied Biosystems/MDS SCIEX Q 4000 TRAP MS.</summary>
            MS_4000_QTRAP = 1000139,

            /// <summary>Applied Biosystems/MDS SCIEX 4700 Proteomics Analyzer MS.</summary>
            MS_4700_Proteomics_Analyzer = 1000140,

            /// <summary>Bruker Daltonics' apex IV: ESI, MALDI, Nanospray, APCI, APPI, Qh-FT_ICR.</summary>
            MS_apex_IV = 1000141,

            /// <summary>Bruker Daltonics' apex Q: ESI, MALDI, Nanospray, APCI, APPI, Qh-FT_ICR.</summary>
            MS_apex_Q = 1000142,

            /// <summary>Applied Biosystems/MDS SCIEX API 150EX MS.</summary>
            MS_API_150EX = 1000143,

            /// <summary>Applied Biosystems/MDS SCIEX API 150EX Prep MS.</summary>
            MS_API_150EX_Prep = 1000144,

            /// <summary>Applied Biosystems/MDS SCIEX API 2000 MS.</summary>
            MS_API_2000 = 1000145,

            /// <summary>Applied Biosystems/MDS SCIEX API 3000 MS.</summary>
            MS_API_3000 = 1000146,

            /// <summary>Applied Biosystems/MDS SCIEX API 4000 MS.</summary>
            MS_API_4000 = 1000147,

            /// <summary>Bruker Daltonics' autoflex II: MALDI TOF.</summary>
            MS_autoflex_II = 1000148,

            /// <summary>Bruker Daltonics' autoflex TOF/TOF MS: MALDI TOF.</summary>
            MS_autoflex_TOF_TOF = 1000149,

            /// <summary>Waters magnetic sector based AutoSpec Ultima NT MS.</summary>
            MS_Auto_Spec_Ultima_NT = 1000150,

            /// <summary>Bruker Daltonics' BioTOF II: ESI TOF.</summary>
            MS_BioTOF_II = 1000151,

            /// <summary>Bruker Daltonics' BioTOF-Q: ESI Q-TOF.</summary>
            MS_BioTOF_Q = 1000152,

            /// <summary>ThermoFinnigan DELTA plusAdvantage MS.</summary>
            MS_DELTA_plusAdvantage = 1000153,

            /// <summary>ThermoFinnigan DELTAplusXP MS.</summary>
            MS_DELTAplusXP = 1000154,

            /// <summary>OBSOLETE ThermoFinnigan ELEMENT2 MS.</summary>
            MS_ELEMENT2_OBSOLETE = 1000155,

            /// <summary>Bruker Daltonics' esquire 4000: linear ion trap, ESI, MALDI, Nanospray, APCI, APPI.</summary>
            MS_esquire_4000 = 1000156,

            /// <summary>Bruker Daltonics' esquire 6000: linear ion trap, ESI, MALDI, Nanospray, APCI, APPI.</summary>
            MS_esquire_6000 = 1000157,

            /// <summary>IonSpec Explorer MS.</summary>
            MS_explorer = 1000158,

            /// <summary>Waters oa-ToF based GCT.</summary>
            MS_GCT = 1000159,

            /// <summary>Bruker Daltonics' HCT: ESI Q-TOF, Nanospray, APCI, APPI.</summary>
            MS_HCT = 1000160,

            /// <summary>Bruker Daltonics' HCTplus: ESI Q-TOF, Nanospray, APCI, APPI.</summary>
            MS_HCTplus = 1000161,

            /// <summary>IonSpec HiResESI MS.</summary>
            MS_HiRes_ESI = 1000162,

            /// <summary>IonSpec HiResMALDI MS.</summary>
            MS_HiRes_MALDI = 1000163,

            /// <summary>Waters IsoPrime MS.</summary>
            MS_IsoPrime = 1000164,

            /// <summary>Waters IsoProbe MS.</summary>
            MS_IsoProbe = 1000165,

            /// <summary>Waters IsoProbe T MS.</summary>
            MS_IsoProbe_T = 1000166,

            /// <summary>ThermoFinnigan LCQ Advantage MS.</summary>
            MS_LCQ_Advantage = 1000167,

            /// <summary>ThermoFinnigan LCQ Classic MS.</summary>
            MS_LCQ_Classic = 1000168,

            /// <summary>ThermoFinnigan LCQ Deca XP Plus MS.</summary>
            MS_LCQ_Deca_XP_Plus = 1000169,

            /// <summary>Waters oa-ToF based MALDI L.</summary>
            MS_M_LDI_L = 1000170,

            /// <summary>Waters oa-ToF based MALDI LR.</summary>
            MS_M_LDI_LR = 1000171,

            /// <summary>ThermoFinnigan MAT253 MS.</summary>
            MS_MAT253 = 1000172,

            /// <summary>ThermoFinnigan MAT900XP MS.</summary>
            MS_MAT900XP = 1000173,

            /// <summary>ThermoFinnigan MAT900XP Trap MS.</summary>
            MS_MAT900XP_Trap = 1000174,

            /// <summary>ThermoFinnigan MAT95XP MS.</summary>
            MS_MAT95XP = 1000175,

            /// <summary>ThermoFinnigan MAT95XP Trap MS.</summary>
            MS_MAT95XP_Trap = 1000176,

            /// <summary>Bruker Daltonics' microflex: MALDI TOF.</summary>
            MS_microflex = 1000177,

            /// <summary>Bruker Daltonics' microTOF LC: ESI TOF, Nanospray, APCI, APPI.</summary>
            MS_microTOF_LC = 1000178,

            /// <summary>ThermoFinnigan NEPTUNE MS.</summary>
            MS_neptune = 1000179,

            /// <summary>Waters NG-5400 MS.</summary>
            MS_NG_5400 = 1000180,

            /// <summary>IonSpec OMEGA MS.</summary>
            MS_OMEGA = 1000181,

            /// <summary>IonSpec OMEGA-2001 MS.</summary>
            MS_OMEGA_2001 = 1000182,

            /// <summary>Bruker Daltonics' OmniFlex: MALDI TOF.</summary>
            MS_OmniFlex = 1000183,

            /// <summary>Waters Platform ICP MS.</summary>
            MS_Platform_ICP = 1000184,

            /// <summary>ThermoFinnigan PolarisQ MS.</summary>
            MS_PolarisQ = 1000185,

            /// <summary>Applied Biosystems/MDS SCIEX Proteomics Solution 1 MS.</summary>
            MS_proteomics_solution_1 = 1000186,

            /// <summary>Applied Biosystems/MDS SCIEX Q TRAP MS.</summary>
            MS_Q_TRAP = 1000187,

            /// <summary>Waters oa-ToF based Q-Tof micro.</summary>
            MS_Q_Tof_micro = 1000188,

            /// <summary>Waters oa-ToF based Q-Tof Ultima.</summary>
            MS_Q_Tof_Ultima = 1000189,

            /// <summary>Applied Biosystems/MDS SCIEX QSTAR MS.</summary>
            MS_QSTAR = 1000190,

            /// <summary>Waters (triple) quadrupole based micro.</summary>
            MS_quattro_micro = 1000191,

            /// <summary>Waters (triple) quadrupole based Ultima.</summary>
            MS_Quattro_Ultima = 1000192,

            /// <summary>ThermoFinnigan Surveyor MSQ MS.</summary>
            MS_Surveyor_MSQ = 1000193,

            /// <summary>Applied Biosystems/MDS SCIEX SymBiot I MS.</summary>
            MS_SymBiot_I = 1000194,

            /// <summary>Applied Biosystems/MDS SCIEX SymBiot XVI MS.</summary>
            MS_SymBiot_XVI = 1000195,

            /// <summary>ThermoFinnigan TEMPUS TOF MS.</summary>
            MS_TEMPUS_TOF = 1000196,

            /// <summary>ThermoFinnigan TRACE DSQ MS.</summary>
            MS_TRACE_DSQ = 1000197,

            /// <summary>ThermoFinnigan TRITON MS.</summary>
            MS_TRITON = 1000198,

            /// <summary>ThermoFinnigan TSQ Quantum MS.</summary>
            MS_TSQ_Quantum = 1000199,

            /// <summary>IonSpec Ultima MS.</summary>
            MS_ultima = 1000200,

            /// <summary>Bruker Daltonics' ultraflex: MALDI TOF.</summary>
            MS_ultraflex = 1000201,

            /// <summary>Bruker Daltonics' ultraflex TOF/TOF: MALDI TOF.</summary>
            MS_ultraflex_TOF_TOF = 1000202,

            /// <summary>Applied Biosystems/MDS SCIEX Voyager-DE PRO MS.</summary>
            MS_Voyager_DE_PRO = 1000203,

            /// <summary>Applied Biosystems/MDS SCIEX Voyager-DE STR MS.</summary>
            MS_Voyager_DE_STR = 1000204,

            /// <summary>The operation of a mass spectrometer in which the intensities of several specific m/z values are recorded rather than the entire mass spectrum.</summary>
            MS_selected_ion_monitoring = 1000205,

            /// <summary>Data acquired from specific product ions corresponding to m/z selected precursor ions recorded via multiple stages of mass spectrometry. Selected reaction monitoring can be performed in time or in space.</summary>
            MS_selected_reaction_monitoring = 1000206,

            /// <summary>OBSOLETE An experimentally determined mass that is can be to determine a unique elemental formula. For ions less than 200 u, a measurement with 5 ppm accuracy is sufficient to determine the elemental composition.</summary>
            MS_accurate_mass_OBSOLETE = 1000207,

            /// <summary>OBSOLETE The mass of an ion or molecule calculated using the average mass of each element weighted for its natural isotopic abundance.</summary>
            MS_average_mass_OBSOLETE = 1000208,

            /// <summary>OBSOLETE The minimum energy that must be imparted to an atom or molecule to produce a specified ion. The term appearance potential is not recommended.</summary>
            MS_appearance_energy_OBSOLETE = 1000209,

            /// <summary>The peak in a mass spectrum that has the greatest intensity. This term may be applied to the spectra of pure substances or mixtures.</summary>
            MS_base_peak = 1000210,

            /// <summary>OBSOLETE The total charge on an ion divided by the electron charge e. OBSOLETED 2009-10-27 since this was viewed as a duplication of 00041 charge state.</summary>
            MS_OBSOLETE_charge_number_OBSOLETE = 1000211,

            /// <summary>OBSOLETE A non-SI unit of mass (symbol Da) that is equal to the unified atomic mass unit: 1.660 538 86(28) x 10^-27 kg.</summary>
            MS_dalton_OBSOLETE = 1000212,

            /// <summary>OBSOLETE The electron affinity of M is the minimum energy required for the process M- ? M + e where M- and M are in their ground rotational, vibrational and electronic states and the electron has zero kinetic energy.</summary>
            MS_electron_affinity_OBSOLETE = 1000213,

            /// <summary>OBSOLETE The potential difference through which electrons are accelerated before they are used to bring about electron ionization.</summary>
            MS_electron_energy_obsolete_OBSOLETE = 1000214,

            /// <summary>OBSOLETE The calculated mass of an ion or molecule containing a single isotope of each atom.</summary>
            MS_exact_mass_OBSOLETE = 1000215,

            /// <summary>A section of a mass spectrometer in which there are no electric or magnetic fields.</summary>
            MS_field_free_region = 1000216,

            /// <summary>OBSOLETE A measure of the probability that a given ionization process will occur when an atom or molecule interacts with a photon, electron, atom or molecule.</summary>
            MS_ionization_cross_section_OBSOLETE = 1000217,

            /// <summary>OBSOLETE The ratio of the number of ions formed to the number of electrons, molecules or photons used.</summary>
            MS_ionization_efficiency_OBSOLETE = 1000218,

            /// <summary>OBSOLETE The minimum energy required to remove an electron from an atom or molecule to produce a positive ion.</summary>
            MS_ionization_energy_OBSOLETE = 1000219,

            /// <summary>OBSOLETE A quantitative mass spectrometry technique in which an isotopically enriched compound is used as an internal standard.</summary>
            MS_isotope_dilution_mass_spectrometry_OBSOLETE = 1000220,

            /// <summary>The deflection of charged particles in a magnetic field due to a force equal to qvxB where q is the particle charge, v its velocity and B the magnetic field. Magnetic deflection of an ion beam is used for m/z separation in a magnetic sector mass spectrometer.</summary>
            MS_magnetic_deflection = 1000221,

            /// <summary>OBSOLETE The difference between the monoisotopic and nominal mass of a molecule or atom.</summary>
            MS_mass_defect_OBSOLETE = 1000222,

            /// <summary>OBSOLETE The sum of the protons and neutrons in an atom, molecule or ion.</summary>
            MS_mass_number_OBSOLETE = 1000223,

            /// <summary>Mass of a molecule measured in unified atomic mass units (u or Da).</summary>
            MS_molecular_mass = 1000224,

            /// <summary>OBSOLETE The mass of an ion or molecule calculated using the mass of the most abundant isotope of each element.</summary>
            MS_monoisotopic_mass_OBSOLETE = 1000225,

            /// <summary>OBSOLETE A mass spectrometry technique in which the sample is introduced into the mass spectrometer as a molecular beam.</summary>
            MS_molecular_beam_mass_spectrometry_OBSOLETE = 1000226,

            /// <summary>Photoionization of an atom or molecule in which in two or more photons are absorbed.</summary>
            MS_multiphoton_ionization = 1000227,

            /// <summary>OBSOLETE An organic molecule containing the elements C, H, O, S, P, or halogen has an odd nominal mass if it contains an odd number of nitrogen atoms.</summary>
            MS_nitrogen_rule_OBSOLETE = 1000228,

            /// <summary>OBSOLETE The mass of an ion or molecule calculated using the mass of the most abundant isotope of each element rounded to the nearest integer value.</summary>
            MS_nominal_mass_OBSOLETE = 1000229,

            /// <summary>OBSOLETE Odd-electron ions may dissociate to form either odd or even-electron ions, whereas even-electron ions generally form even-electron fragment ions.</summary>
            MS_odd_electron_rule_OBSOLETE = 1000230,

            /// <summary>A localized region of relatively large ion signal in a mass spectrum. Although peaks are often associated with particular ions, the terms peak and ion should not be used interchangeably.</summary>
            MS_peak = 1000231,

            /// <summary>OBSOLETE The height or area of a peak in a mass spectrum.</summary>
            MS_peak_intensity_OBSOLETE = 1000232,

            /// <summary>OBSOLETE The proton affinity of a species M is defined as the negative of the enthalpy change for the reaction M + H+ -&gt;</summary>
            MS_proton_affinity_OBSOLETE = 1000233,

            /// <summary>OBSOLETE In a mass spectrum, the observed mass divided by the difference between two masses that can be separated. The method by which delta m was obtained and the mass at which the measurement was made should be reported.</summary>
            MS_mass_resolving_power_OBSOLETE = 1000234,

            /// <summary>Representation of the total ion current detected in each of a series of mass spectra versus time.</summary>
            MS_total_ion_current_chromatogram = 1000235,

            /// <summary>The ratio of the number of ions leaving a region of a mass spectrometer to the number entering that region.</summary>
            MS_transmission = 1000236,

            /// <summary>OBSOLETE A non-SI unit of mass (u) defined as one twelfth of ^12 C in its ground state and equal to 1.660 538 86(28) x 10^-27 kg.</summary>
            MS_unified_atomic_mass_unit_OBSOLETE = 1000237,

            /// <summary>OBSOLETE A mass spectrometry technique in which atoms extracted from a sample are ionized, accelerated to MeV energies and separated according to their momentum, charge and energy.</summary>
            MS_accelerator_mass_spectrometry_OBSOLETE = 1000238,

            /// <summary>Matrix-assisted laser desorption ionization in which the sample target is at atmospheric pressure and the ions formed by the pulsed laser are sampled through a small aperture into the mass spectrometer.</summary>
            MS_atmospheric_pressure_matrix_assisted_laser_desorption_ionization = 1000239,

            /// <summary>Any ionization process in which ions are formed in the gas phase at atmospheric pressure.</summary>
            MS_atmospheric_pressure_ionization = 1000240,

            /// <summary>OBSOLETE Atmospheric pressure chemical ionization in which the reactant ions are generated by photo-ionization.</summary>
            MS_Atmostpheric_Pressure_Photoionization_OBSOLETE = 1000241,

            /// <summary>A special case of infrared multiphoton dissociation wherein excitation of the reactant ion is caused by absorption of infrared photons radiating from heated blackbody surroundings, which are usually the walls of a vacuum chamber. See also infrared multiphoton dissociation.</summary>
            MS_blackbody_infrared_radiative_dissociation = 1000242,

            /// <summary>OBSOLETE A fragmentation of an even-electron ion in which the cleaved bond is not adjacent to the apparent charge site.</summary>
            MS_charge_remote_fragmentation_OBSOLETE = 1000243,

            /// <summary>OBSOLETE MSn experiment with three or more stages of m/z separation and in which a particular multi-step reaction path is monitored.</summary>
            MS_consecutive_reaction_monitoring_OBSOLETE = 1000244,

            /// <summary>The reaction of a positive ion with an atom or molecule that results in the removal of one or more electrons from the ion.</summary>
            MS_charge_stripping = 1000245,

            /// <summary>The application of the accelerating voltage pulse after a time delay in desorption ionization from a surface. The extraction delay can produce energy focusing in a time-of-flight mass spectrometer.</summary>
            MS_delayed_extraction = 1000246,

            /// <summary>The formation of ions from a solid or liquid material after the rapid vaporization of that sample.</summary>
            MS_desorption_ionization = 1000247,

            /// <summary>A device for introducing a solid or liquid sample into a mass spectrometer ion source for desorption ionization.</summary>
            MS_direct_insertion_probe = 1000248,

            /// <summary>The delivery of a liquid sample into a mass spectrometer for spray or desorption ionization.</summary>
            MS_direct_liquid_introduction = 1000249,

            /// <summary>A process in which a multiply protonated molecules interacts with a low energy electrons. Capture of the electron leads the liberation of energy and a reduction in charge state of the ion with the production of the (M + nH) (n-1)+ odd electron ion, which readily fragments.</summary>
            MS_electron_capture_dissociation = 1000250,

            /// <summary>OBSOLETE An ion containing no unpaired electrons in its ground electronic state, e.g. CH3+ in its ground state.</summary>
            MS_even_electron_ion_OBSOLETE = 1000251,

            /// <summary>OBSOLETE The reaction of an ion with an electron in which the translational energy of the collision is converted into internal energy of the ion.</summary>
            MS_electron_induced_excitation_in_organics_OBSOLETE = 1000252,

            /// <summary>A device to amplify the current of a beam or packet of charged particles or photons by incidence upon the surface of an electrode to produce secondary electrons. The secondary electrons are then accelerated to other electrodes or parts of a continuous electrode to produce further secondary electrons.</summary>
            MS_electron_multiplier = 1000253,

            /// <summary>A device consisting of conducting parallel plates, concentric cylinders or concentric spheres that separates charged particles according to their kinetic energy by means of an electric field that is constant in time.</summary>
            MS_electrostatic_energy_analyzer = 1000254,

            /// <summary>An ion source immersed in a flow of helium or other inert buffer gas that carries the ions through a meter-long reactor at pressures around 100 Pa.</summary>
            MS_flowing_afterglow = 1000255,

            /// <summary>OBSOLETE The separation of ions between two concentric cylindrical electrodes due to application of a high voltage asymmetric waveform whereby ions migrate towards one of the two electrodes depending on the ratio of the high- to low-field mobility of the ion.</summary>
            MS_high_field_asymmetric_waveform_ion_mobility_spectrometry_OBSOLETE = 1000256,

            /// <summary>The formation of gas-phase ions from a material deposited on a solid surface in the presence of a high electric field. Because this process may encompass ionization by field ionization or other mechanisms, it is not recommended as a synonym for field desorption ionization.</summary>
            MS_field_desorption = 1000257,

            /// <summary>The removal of electrons from any species by interaction with a high electric field.</summary>
            MS_field_ionization = 1000258,

            /// <summary>The formation of ions in the gas phase and from solid samples at the cathode by application of a voltage to a low pressure gas.</summary>
            MS_glow_discharge_ionization = 1000259,

            /// <summary>OBSOLETE A method of analysis in which a beam of ions is separated according to the ratio of its translational energy to charge.</summary>
            MS_ion_kinetic_energy_spectrometry_OBSOLETE = 1000260,

            /// <summary>OBSOLETE The separation of ions according to their velocity through a buffer gas under the influence of an electric field.</summary>
            MS_ion_mobility_spectrometry_OBSOLETE = 1000261,

            /// <summary>Multiphoton ionization where the reactant ion dissociates as a result of the absorption of multiple infrared photons.</summary>
            MS_infrared_multiphoton_dissociation = 1000262,

            /// <summary>OBSOLETE The measurement of the relative quantity of the different isotopes of an element in a material with a mass spectrometer.</summary>
            MS_isotope_ratio_mass_spectrometry_OBSOLETE = 1000263,

            /// <summary>A device for spatially confining ions using electric and magnetic fields alone or in combination.</summary>
            MS_ion_trap = 1000264,

            /// <summary>OBSOLETE Distribution of values of translational kinetic energy release for an ensemble of metastable ions undergoing a specific dissociation reaction.</summary>
            MS_kinetic_energy_release_distribution_OBSOLETE = 1000265,

            /// <summary>OBSOLETE The formation of ions through the interaction of a laser with a material or with gas-phase ions or molecules.</summary>
            MS_Laser_Desorption_OBSOLETE = 1000266,

            /// <summary>OBSOLETE Spectra that are obtained from a sector mass spectrometer that incorporates at least one magnetic sector plus one electric sector in reverse geometry. The accelerating voltage, V, and the magnetic sector field, B, are set at fixed values to select the precursor ions, which are then allowed to dissociate or to react in a field free region between the two sectors. The kinetic energy product ions of m/z selected precursor ions is analyzed by scanning the electric sector field, E. The width of the product ion spectrum peaks is related to the kinetic energy release distribution (KERD) for the dissociation process.</summary>
            MS_mass_analyzed_ion_kinetic_energy_spectrometry_OBSOLETE = 1000267,

            /// <summary>OBSOLETE The branch of science that deals with all aspects of mass spectrometers and the results obtained with these instruments.</summary>
            MS_mass_spectrometry_OBSOLETE = 1000268,

            /// <summary>OBSOLETE The acquisition, study and spectra of the electrically charged products or precursors of a m/z selected ion or ions.</summary>
            MS_mass_spectrometry_mass_spectrometry_OBSOLETE = 1000269,

            /// <summary>OBSOLETE Multiple stages of precursor ion m/z selection followed by product ion detection for successive progeny ions.</summary>
            MS_multiple_stage_mass_spectrometry_OBSOLETE = 1000270,

            /// <summary>Chemical ionization that results in the formation of negative ions.</summary>
            MS_Negative_Ion_chemical_ionization = 1000271,

            /// <summary>With this technique, m/z selected ions form neutrals by charge transfer to a collision gas or by dissociation. The neutrals are separated from the remaining ions and ionized in collisions with a second gas. This method is used to investigate reaction intermediates and other unstable species.</summary>
            MS_neutralization_reionization_mass_spectrometry = 1000272,

            /// <summary>The ionization of an atom or molecule by a photon, written M + h? ? M^+ + e. The term photon impact is not recommended.</summary>
            MS_photoionization = 1000273,

            /// <summary>A mass spectrometry technique in which the sample is heated to the point of decomposition and the gaseous decomposition products are introduced into the ion source.</summary>
            MS_pyrolysis_mass_spectrometry = 1000274,

            /// <summary>A transmission quadrupole to which an oscillating potential is applied so as to focus a beam of ions through a collision gas with no m/z separation.</summary>
            MS_collision_quadrupole = 1000275,

            /// <summary>Multiphoton ionization in which the ionization cross section is significantly enhanced because the energy of the incident photons is resonant with an intermediate excited state of the neutral species.</summary>
            MS_resonance_enhanced_multiphoton_ionization = 1000276,

            /// <summary>OBSOLETE A mass spectrometer used to measure the composition and pressure of gasses in an evacuated chamber.</summary>
            MS_residual_gas_analyzer_OBSOLETE = 1000277,

            /// <summary>The formation of ionized species in the gas phase from analytes deposited on a particular surface substrate which is irradiated with a laser beam of which wavelength is absorbed by the surface. See also desorption/ionization on silicon and laser desorption/ionization.</summary>
            MS_surface_enhanced_laser_desorption_ionization = 1000278,

            /// <summary>Matrix-assisted laser desorption ionization in which the matrix is covalently linked to the target surface.</summary>
            MS_surface_enhanced_neat_desorption = 1000279,

            /// <summary>OBSOLETE The ionization of a neutral species when it interacts with a solid surface with an appropriate work function and temperature.</summary>
            MS_suface_ionization_OBSOLETE = 1000280,

            /// <summary>A device in which m/z selected ions are entrained in an inert carrier gas and undergo ion-molecule reactions.</summary>
            MS_selected_ion_flow_tube = 1000281,

            /// <summary>A technique associated with Fourier transform ion cyclotron resonance (FT-ICR) mass spectrometry to carry out ion/neutral reactions such as low-energy collision-induced dissociation. A radio-frequency electric field of slightly off-resonance to the cyclotron frequency of the reactant ion cyclically accelerates and decelerates the reactant ion that is confined in the Penning ion trap. The ion's orbit does not exceed the dimensions of ion trap while the ion undergoes an ion/neutral species process that produces a high average translational energy for an extended time.</summary>
            MS_sustained_off_resonance_irradiation = 1000282,

            /// <summary>OBSOLETE Mass spectrometry using spark ionization.</summary>
            MS_Spark_Source_Mass_Spectrometry_OBSOLETE = 1000283,

            /// <summary>A technique to create excitation waveforms for ions in FT-ICR mass spectrometer or Paul ion trap. An excitation waveform in the time-domain is generated by taking the inverse Fourier transform of an appropriate frequency-domain programmed excitation spectrum, in which the resonance frequencies of ions to be excited are included. This technique may be used for selection of precursor ions in MS2 experiments.</summary>
            MS_stored_waveform_inverse_fourier_transform = 1000284,

            /// <summary>The sum of all the separate ion currents carried by the ions of different m/z contributing to a complete mass spectrum or in a specified m/z range of a mass spectrum.</summary>
            MS_total_ion_current = 1000285,

            /// <summary>Energy focusing in a time-of-flight mass spectrometer that is accomplished by introducing a time delay between the formation of the ions and the application of the accelerating voltage pulse.</summary>
            MS_time_lag_focusing = 1000286,

            /// <summary>OBSOLETE An instrument that separates ions by m/z in a field-free region after acceleration to a fixed kinetic energy.</summary>
            MS_time_of_flight_mass_spectrometer_OBSOLETE = 1000287,

            /// <summary>A device that uses an oscillating electric field and magnetic field to accelerate charged particles.</summary>
            MS_cyclotron = 1000288,

            /// <summary>OBSOLETE A mass spectrometer that uses a magnetic sector for m/z focusing and an electric sector for energy focusing of an ion beam.</summary>
            MS_double_focusing_mass_spectrometer_OBSOLETE = 1000289,

            /// <summary>OBSOLETE A mass spectrometer that combines m/z analyzers of different types to perform tandem mass spectrometry.</summary>
            MS_hybrid_mass_spectrometer_OBSOLETE = 1000290,

            /// <summary>A two dimensional Paul ion trap in which ions are confined in the axial dimension by means of an electric field at the ends of the trap.</summary>
            MS_linear_ion_trap = 1000291,

            /// <summary>OBSOLETE An instrument that separates a beam of ions according to their mass-to-charge ratio in which the ions are directed onto a focal plane detector such as a photographic plate.</summary>
            MS_mass_spectrograph_obsolete_OBSOLETE = 1000292,

            /// <summary>OBSOLETE An instrument that measures the mass-to-charge ratio and relative abundances of ions.</summary>
            MS_mass_spectrometer_OBSOLETE = 1000293,

            /// <summary>A plot of the relative abundance of a beam or other collection of ions as a function of the mass-to-charge ratio (m/z).</summary>
            MS_mass_spectrum = 1000294,

            /// <summary>OBSOLETE An arrangement for a double-focusing mass spectrometer in which a deflection of ?/(4 ?(2)) radians in a radial electric field is followed by a magnetic deflection of ?/2 radians.</summary>
            MS_mattauch_herzog_geometry_OBSOLETE = 1000295,

            /// <summary>OBSOLETE An arrangement for a double-focusing mass spectrometer in which a deflection of ?/2 radians in a radial electric field analyzer is followed by a magnetic deflection of ?/3 radians.</summary>
            MS_nier_johnson_geometry_OBSOLETE = 1000296,

            /// <summary>OBSOLETE A device that permits the trapping of ions by means of an alternating current voltage. The ejection of ions with a m/z less than a prescribed value and retention of those with higher mass depends on the application of radio frequency voltages between a ring electrode and two end-cap electrodes to confine the ions in a circular path. The choice of these voltages determines the m/z below which ions are ejected.</summary>
            MS_paul_ion_trap_OBSOLETE = 1000297,

            /// <summary>OBSOLETE A mass spectrometer in which the ions of different m/z are separated by means of crossed electric and magnetic fields in such a way that the selected ions follow a prolate trochoidal path.</summary>
            MS_prolate_traochoidal_mass_spectrometer_OBSOLETE = 1000298,

            /// <summary>OBSOLETE An abbreviation of quadrupole ion storage trap. This term is synonymous with Paul Ion Trap. If so then add a synonym to paul and obsolete this term.</summary>
            MS_quistor_OBSOLETE = 1000299,

            /// <summary>A time-of-flight mass spectrometer that uses a static electric field to reverse the direction of travel of the ions entering it. A reflectron improves mass resolution by assuring that ions of the same m/z but different kinetic energy arrive at the detector at the same time.</summary>
            MS_reflectron = 1000300,

            /// <summary>OBSOLETE A mass spectrometer consisting of one or more magnetic sectors for m/z selection in a beam of ions. Such instruments may also have one or more electric sectors for energy selection.</summary>
            MS_sector_mass_spectrometer_OBSOLETE = 1000301,

            /// <summary>OBSOLETE A mass spectrometer designed for mass spectrometry/mass spectrometry.</summary>
            MS_tandem_mass_spectrometer_OBSOLETE = 1000302,

            /// <summary>OBSOLETE A mass spectrometer that consists of four parallel rods whose centers form the corners of a square and whose opposing poles are connected. The voltage applied to the rods is a superposition of a static potential and a sinusoidal radio frequency potential. The motion of an ion in the x and y dimensions is described by the Matthieu equation whose solutions show that ions in a particular m/z range can be transmitted along the z axis.</summary>
            MS_transmission_quadrupole_mass_spectrometer_OBSOLETE = 1000303,

            /// <summary>The electrical potential used to impart kinetic energy to ions in a mass spectrometer.</summary>
            MS_accelerating_voltage = 1000304,

            /// <summary>OBSOLETE The circular motion of a charged particle moving at velocity v in a magnetic field B that results from the force qvB.</summary>
            MS_cyclotron_motion_OBSOLETE = 1000305,

            /// <summary>OBSOLETE A mass spectrometer in which m/z separation using one or more electric fields that vary with time.</summary>
            MS_dynamic_mass_spectrometry_OBSOLETE = 1000306,

            /// <summary>Three element charged particle lens in which the first and third elements are held at the same voltage. Such a lens produces focusing without changing the translational energy of the particle.</summary>
            MS_einzel_lens = 1000307,

            /// <summary>The magnitude of the force per unit charge at a given point in space.</summary>
            MS_electric_field_strength = 1000308,

            /// <summary>The region of a Mathieu stability diagram closest to the origin. Ions within this region can traverse the full length of a transmission quadrupole.</summary>
            MS_first_stability_region = 1000309,

            /// <summary>The electric or magnetic field that extends from the edge of a sector, lens or other ion optics element.</summary>
            MS_fringing_field = 1000310,

            /// <summary>A device for measuring the kinetic energy of charged particles using a retarding field, time-of-flight, or the extent of deflection in an electric or magnetic field.</summary>
            MS_kinetic_energy_analyzer = 1000311,

            /// <summary>OBSOLETE The m/z value above which ions cannot be detected in a mass spectrometer.</summary>
            MS_mass_limit_OBSOLETE = 1000312,

            /// <summary>OBSOLETE The limit of m/z over which a mass spectrometer can detect ions.</summary>
            MS_scan_m_z_range__OBSOLETE = 1000313,

            /// <summary>OBSOLETE The use of mass selective instability to eject ions of selected m/z values from an ion trap.</summary>
            MS_mass_selective_axial_ejection_OBSOLETE = 1000314,

            /// <summary>OBSOLETE A method for selective ejection of ions according to their m/z value in an ion trap.</summary>
            MS_mass_selective_instability_OBSOLETE = 1000315,

            /// <summary>OBSOLETE A graphical representation expressed in terms of reduced coordinates that describes charged particle motion in a quadrupole mass filter or quadrupole ion trap mass spectrometer.</summary>
            MS_mathieu_stability_diagram_OBSOLETE = 1000316,

            /// <summary>OBSOLETE The pulsed acceleration of ions perpendicular to their direction of travel into a time-of-flight mass spectrometer. Ions may be extracted from a directional ion source, drift tube or m/z separation stage.</summary>
            MS_orthogonal_extraction_OBSOLETE = 1000317,

            /// <summary>OBSOLETE A mode of ion ejection in a quadrupole ion trap that relies on a auxiliary radio frequency voltage that is applied to the end-cap electrodes. The voltage is tuned to the secular frequency of a particular ion to eject it.</summary>
            MS_resonance_ion_ejection_OBSOLETE = 1000318,

            /// <summary>The mutual repulsion of particles of like charge that limits the current in a charged-particle beam and causes beams or packets of charged particles to expand radially over time.</summary>
            MS_space_charge_effect = 1000319,

            /// <summary>An electric or magnetic field that does not change in time.</summary>
            MS_static_field = 1000320,

            /// <summary>OBSOLETE A mass spectrum obtained by setting the electric sector field E to twice the value required to transmit the main ion-beam thereby allowing ions with a kinetic energy-to-charge ratio twice that of the main ion-beam to be transmitted. Product ions resulting from partial charge transfer reactions such as m^2+ + N ? m^+ + N^+ that occur in a collision cell (containing a gas, N) located in a field-free region preceding a magnetic and electric sector combination are detected. When the magnetic sector field B is scanned, a mass spectrum of singly charged product ions of doubly charged precursor ions is obtained.</summary>
            MS_2E_Mass_Spectrum_OBSOLETE = 1000321,

            /// <summary>The measurement of the relative abundance of ions that result from a charge inversion reaction as a function of m/z.</summary>
            MS_charge_inversion_mass_spectrum = 1000322,

            /// <summary>OBSOLETE Spectrum of all precursor ions that undergo a selected m/z decrement.</summary>
            MS_constant_neutral_loss_scan_OBSOLETE = 1000323,

            /// <summary>OBSOLETE Spectrum of all precursor ions that undergo a selected m/z increment.</summary>
            MS_constant_neutral_gain_scan_OBSOLETE = 1000324,

            /// <summary>A spectrum formed of all product ions that have been produced by gain of a pre-selected neutral mass following the reaction with and addition of the gas in a collision cell.</summary>
            MS_constant_neutral_gain_spectrum = 1000325,

            /// <summary>A spectrum formed of all product ions that have been produced with a selected m/z decrement from any precursor ions. The spectrum shown correlates to the precursor ion spectrum. See also neutral loss spectrum.</summary>
            MS_constant_neutral_loss_spectrum = 1000326,

            /// <summary>OBSOLETE A type of MS2 experiments with three or more stages of m/z separation and in which a particular multi-step reaction path is monitored.</summary>
            MS_consecutive_reaction_monitoring_OBSOLETE1 = 1000327,

            /// <summary>A mass spectrum obtained using a sector mass spectrometer in which the electric sector field E is set to half the value required to transmit the main ion-beam. This spectrum records the signal from doubly charged product ions of charge-stripping reactions.</summary>
            MS_e_2_mass_spectrum = 1000328,

            /// <summary>OBSOLETE A scan in an instrument with two or more m/z analysers or in a sector mass spectrometer that incorporates at least one magnetic sector and one electric sector. Two or more of the analyzers are scanned simultaneously so as to preserve a predetermined relationship between scan parameters to produce a product ion, precursor ion or constant neutral loss spectrum.</summary>
            MS_linked_scan_OBSOLETE = 1000329,

            /// <summary>OBSOLETE A linked scan at constant B/E may be performed on a sector mass spectrometer that incorporates at least one magnetic sector plus one electric sector. The magnetic field B and the electric field E are scanned simultaneously while the accelerating voltage V is held constant, so as to maintain the ratio of the two fields constant. This linked scan may record a product ion spectrum of dissociation or other reactions occurring in a field free region preceding the two sectors.</summary>
            MS_linked_scan_at_constant_b_e_OBSOLETE = 1000330,

            /// <summary>OBSOLETE A linked scan performed on a sector instrument that incorporates at least one electric sector plus one magnetic sector. The electric sector field, E, and the accelerating voltage, V, are scanned simultaneously, so as to maintain the ratio E2/V at a constant value. This linked scan recordss a product ion spectrum of dissociation or other reactions occurring in a field free region (FFR) preceding the two sectors.</summary>
            MS_Linked_Scan_at_Constant_E2_V_OBSOLETE = 1000331,

            /// <summary>OBSOLETE A linked scan performed on a sector mass spectrometer that incorporates at least one electric sector plus one magnetic sector in either order. The accelerating voltage is fixed and the magnetic field, B, and the electric field, E, are scanned simultaneously so as to maintain the ratio B2/E at a constant value. This linked scan records a precursor ion spectrum of dissociation or other reactions occurring in the field free region preceding the two sectors. The term B2/E linked scan is not recommended.</summary>
            MS_Linked_Scan_at_Constant_B2_E_OBSOLETE = 1000332,

            /// <summary>OBSOLETE A linked scan performed on a sector instrument that incorporates at least one electric sector plus one magnetic sector placed in either order. The accelerating voltage is fixed while scanning the magnetic field, B, and electric field, E, simultaneously, so as to maintain the quantity B</summary>
            MS_Linked_Scan_at_Constant_B_1__E_E0___1_2___E_OBSOLETE = 1000333,

            /// <summary>OBSOLETE A tandem mass spectrometry method in which product ion spectra are recorded in a single m/z analyzer (such as a Paul Ion Trap or FTMS) in discreet steps over time. Ions in a specific m/z range are selected, dissociated, and the product ions analyzed sequentially in time.</summary>
            MS_MS_MS_in_Time_OBSOLETE = 1000334,

            /// <summary>OBSOLETE A tandem mass spectrometry method in which product ion spectra are recorded in m/z analyzers separated in space. Specific m/z separation functions are designed such that in one section of the instrument ions are selected, dissociated in an intermediate region, and the product ions are then transmitted to another analyser for m/z separation and data acquisition.</summary>
            MS_MS_MS_in_Space_OBSOLETE = 1000335,

            /// <summary>The loss of an uncharged species during a rearrangement process. The value slot holds the molecular formula in Hill notation of the neutral loss molecule, see PMID: 21182243. This term must be used in conjunction with a child of the term MS:1002307 (fragmentation ion type).</summary>
            MS_neutral_loss = 1000336,

            /// <summary>OBSOLETE Serial product ions from dissociation of selected precursor ions where n refers to the number of stages of dissociation. The term granddaughter ion is deprecated.</summary>
            MS_nth_generation_product_ion_OBSOLETE = 1000337,

            /// <summary>OBSOLETE The specific scan functions or processes that record the appropriate generation of product ion or ions of any m/z selected precursor ions.</summary>
            MS_nth_generation_product_ion_scan_OBSOLETE = 1000338,

            /// <summary>OBSOLETE The mass spectrum recorded from any mass spectrometer in which the appropriate scan function can be set to record the appropriate generation product ion or ions of m/z selected precursor ions.</summary>
            MS_nth_generation_product_ion_spectrum_OBSOLETE = 1000339,

            /// <summary>OBSOLETE An ion that reacts to form particular product ions. The reaction can be unimolecular dissociation, ion/molecule reaction, isomerization, or change in charge state. The term parent ion is deprecated.</summary>
            MS_precursor_ion_OBSOLETE = 1000340,

            /// <summary>Spectrum generated by scanning precursor m/z while monitoring a fixed product m/z.</summary>
            MS_precursor_ion_spectrum = 1000341,

            /// <summary>OBSOLETE An ion formed as the product of a reaction involving a particular precursor ion. The reaction can be unimolecular dissociation to form fragment ions, an ion/molecule reaction, or simply involve a change in the number of charges. The term fragment ion is deprecated. The term daughter ion is deprecated.</summary>
            MS_product_ion_OBSOLETE = 1000342,

            /// <summary>OBSOLETE A mass spectrum recorded from any spectrometer in which the appropriate m/z separation scan function is set to record the product ion or ions of selected precursor ions.</summary>
            MS_product_ion_spectrum_OBSOLETE = 1000343,

            /// <summary>OBSOLETE A charged product of a series of consecutive reactions that includes product ions, 1st generation product ions, 2nd generation product ions, etc. Given the sequential fragmentation scheme: M1+ -&gt; M2+ -&gt; M3+ -&gt; M4+ -&gt; M5+. M4+ is the precursor ion of M5+, a 1st generation product ion of M3+, a 2nd generation product ion of M2+ and a 3rd generation product ion of M1+.</summary>
            MS_progeny_ion_OBSOLETE = 1000344,

            /// <summary>Detector comprising several ion collection elements, arranged in a line or grid where each element is an individual detector.</summary>
            MS_array_detector = 1000345,

            /// <summary>A surface that is held at high potential such that ions striking the surface produce electrons that are subsequently detected.</summary>
            MS_conversion_dynode = 1000346,

            /// <summary>One of a series of electrodes in a photomultiplier tube. Such an arrangement is able to amplify the current emitted by the photocathode.</summary>
            MS_dynode = 1000347,

            /// <summary>A detector for spatially disperse ion beams in which all ions simultaneously impinge on the detector plane.</summary>
            MS_focal_plane_collector = 1000348,

            /// <summary>A detector in which ions strike a conversion dynode to produce electrons that in turn strike a phosphor and the resulting photons are detected by a photomultiplier.</summary>
            MS_ion_to_photon_detector = 1000349,

            /// <summary>A detector in which the ion beam is focused onto a point and the individual ions arrive sequentially.</summary>
            MS_point_collector = 1000350,

            /// <summary>A detector in which the charged particles are accelerated to a high velocity and impinge on a conversion dynode, emitting secondary electrons. The electrons are accelerated onto a phosphor screen, which emits photons that are in turn detected using a photomultiplier or other photon detector.</summary>
            MS_postacceleration_detector = 1000351,

            /// <summary>OBSOLETE Electrons that are ejected from a sample surface as a result of bombardment by a primary beam of atoms, ions or photons. WAS IN DETECTOR TYPE. Where should it go.</summary>
            MS_secondary_electron_OBSOLETE = 1000352,

            /// <summary>Ion formed by the interaction of an ion with one or more atoms or molecules to form an ion containing all the constituent atoms of the precursor ion as well as the additional atoms from the associated atoms or molecules.</summary>
            MS_adduct_ion = 1000353,

            /// <summary>OBSOLETE A planar cyclic ion that obeys the Hueckel (4n + 2) rule where n is a positive integer representing the number of conjugated Pi electrons. Charge delocalization leads to greater stability compared to a hypothetical localized structure.</summary>
            MS_aromatic_ion_OBSOLETE = 1000354,

            /// <summary>OBSOLETE Ions that have similar chemical valence, for example the acetyl cation CH3-CO+ and the thioacetyl cation CH3-CS+.</summary>
            MS_analog_ion_OBSOLETE = 1000355,

            /// <summary>OBSOLETE A planar cyclic ion with 4n ? electrons and is therefore not aromatic.</summary>
            MS_anti_aromatic_ion_OBSOLETE = 1000356,

            /// <summary>OBSOLETE An ion formed by the association of a cation with a neutral molecule, M, for example</summary>
            MS_cationized_molecule_OBSOLETE = 1000357,

            /// <summary>OBSOLETE An ion formed by a multi-component atomic or molecular assembly of one or more ions with atoms or molecules, such as</summary>
            MS_cluster_ion_OBSOLETE = 1000358,

            /// <summary>OBSOLETE A radical cation or anion in which the charge site and the unpaired electron spin are both formally located in the same atom or group of atoms, as opposed to the spatially separate electronic configuration of distonic ions. The radical cation of methanol, CH3OH+, in which the charge and spin sites are formally located at the O atom is an example of a conventional ion, whereas .CH2-OH2+ is a distonic ion.</summary>
            MS_Conventional_ion_OBSOLETE = 1000359,

            /// <summary>OBSOLETE A product ion whose formation reveals structural or compositional information of its precursor. For instance, the phenyl cation in an electron ionization mass spectrum is a diagnostic ion for benzene and derivatives.</summary>
            MS_diagnostic_ion_OBSOLETE = 1000360,

            /// <summary>OBSOLETE An ion formed by ionization of a dimer or by the association of an ion with its neutral counterpart such as</summary>
            MS_dimeric_ion_OBSOLETE = 1000361,

            /// <summary>OBSOLETE A radical cation or anion in which the charge site and the unpaired electron spin cannot be both formally located in the same atom or group of atoms as it can be with a conventional ion. For example, CH2-OH2+ is a distonic ion whereas the radical cation of methanol, CH3OH+ is a conventional ion.</summary>
            MS_distonic_ion_OBSOLETE = 1000362,

            /// <summary>OBSOLETE A positively charged lower-valency ion of the nonmetallic elements. The methenium ion is CH3+. Other examples are the oxenium, sulfenium, nitrenium, phosphenium, and halenium ions.</summary>
            MS_enium_ion_OBSOLETE = 1000363,

            /// <summary>OBSOLETE A product ion that results from the dissociation of a precursor ion.</summary>
            MS_fragment_ion_OBSOLETE = 1000364,

            /// <summary>OBSOLETE An atomic or molecular species having a net positive or negative electric charge.</summary>
            MS_ion__OBSOLETE = 1000365,

            /// <summary>OBSOLETE An ion that differs only in the isotopic composition of one or more of its constituent atoms. For example CH4+ and CH3D+ or 10BF3 and 11BF3. The term isotopologue is a contraction of isotopic homologue.</summary>
            MS_Isotopologue_ion_OBSOLETE = 1000366,

            /// <summary>OBSOLETE Isomeric ion having the same numbers of each isotopic atom but differing in their positions. Isotopomeric ions can be either configurational isomers in which two atomic isotopes exchange positions or isotopic stereoisomers. The term isotopomer is a shortening of isotopic isomer.</summary>
            MS_Isotopomeric_ion_OBSOLETE = 1000367,

            /// <summary>OBSOLETE An ion that is formed with internal energy higher than the threshold for dissociation but with a lifetime great enough to allow it to exit the ion source and enter the mass spectrometer where it dissociates before detection.</summary>
            MS_metastable_ion_OBSOLETE = 1000368,

            /// <summary>OBSOLETE An ion formed by the removal of one or more electrons to form a positive ion or the addition off one or more electrons to form a negative ion.</summary>
            MS_molecular_ion_OBSOLETE = 1000369,

            /// <summary>OBSOLETE An atomic or molecular species having a net negative electric charge.</summary>
            MS_negative_ion_OBSOLETE = 1000370,

            /// <summary>OBSOLETE Hyper-coordinated carbonium ion such as the penta-coordinated norbornyl cation. Note: Tri-coordinated carbenium ions are termed classical ions.</summary>
            MS_non_classical_ion_OBSOLETE = 1000371,

            /// <summary>OBSOLETE A positively charged hypervalent ion of the nonmetallic elements. Examples are the methonium ion CH5+, the hydrogenonium ion H3+ and the hydronium ion H3O+. Other examples are the carbonium, oxonium, sulfonium, nitronium, diazonium, phosphonium, and halonium ions. Onium ions are not limited to monopositive ions; multiply-charged onium ions exist such as the gitonic (proximal) oxonium dication H4O2+ and the distonic oxonium dication H2O+-CH2-CH2-OH2+.</summary>
            MS_onium_ion_OBSOLETE = 1000372,

            /// <summary>OBSOLETE Most abundant ion of an isotope cluster, such as the 11B79Br2 81Br+ ion of m/z 250 of the cluster of isotopologue molecular ions of BBr3. The term principal ion has also been used to describe ions that have been artificially isotopically enriched in one or more positions such as CH3 13CH3+ or CH2D2 +, but those are best defined as isotopologue ions.</summary>
            MS_principal_ion_OBSOLETE = 1000373,

            /// <summary>OBSOLETE An atomic or molecular species having a net positive electric charge.</summary>
            MS_positive_ion_OBSOLETE = 1000374,

            /// <summary>OBSOLETE An ion formed by interaction of a neutral molecule with a proton and represented by the symbol</summary>
            MS_protonated_molecule_OBSOLETE = 1000375,

            /// <summary>OBSOLETE An ion, either a cation or anion, containing unpaired electrons in its ground state. The unpaired electron is denoted by a superscript dot alongside the superscript symbol for charge, such as for the molecular ion of a molecule M, that is, M+. Radical ions with more than one charge and/or more than one unpaired electron are denoted such as M(2+)(2). Unless the positions of the unpaired electron and charge can be associated with specific atoms, superscript charge designation should be placed before the superscript dot designation.</summary>
            MS_radical_ion_OBSOLETE = 1000376,

            /// <summary>OBSOLETE A stable ion whose structure is known with certainty. These ions are usually formed by direct ionization of a neutral molecule of known structure and are used to verify by comparison the structure of an unknown ion.</summary>
            MS_reference_ion_OBSOLETE = 1000377,

            /// <summary>OBSOLETE An ion with internal energy sufficiently low that it does not rearrange or dissociate prior to detection in a mass spectrometer.</summary>
            MS_stable_ion_OBSOLETE = 1000378,

            /// <summary>OBSOLETE An ion with sufficient energy to dissociate within the ion source.</summary>
            MS_unstable_ion_OBSOLETE = 1000379,

            /// <summary>A process whereby an electron is removed from an atom, ion, or molecule to produce an ion in its lowest energy state.</summary>
            MS_adiabatic_ionization = 1000380,

            /// <summary>An ionization process in which two excited atoms or molecules react to form a single positive ion and an electron.</summary>
            MS_associative_ionization = 1000381,

            /// <summary>Atmospheric pressure chemical ionization in which the reactant ions are generated by photo-ionization.</summary>
            MS_atmospheric_pressure_photoionization = 1000382,

            /// <summary>The formation of a neutral when a negative ion in a discrete state with an energy greater than the detachment threshold loses an electron spontaneously without further interaction with an energy source.</summary>
            MS_autodetachment = 1000383,

            /// <summary>The formation of an ion when an atom or molecule in a discrete state with an energy greater than the ionization threshold loses an electron spontaneously without further interaction with an energy source.</summary>
            MS_autoionization = 1000384,

            /// <summary>The interaction of an ion with an atom or molecule in which the charge on the ion is transferred to the neutral without the dissociation of either. Synonymous with charge transfer ionization.</summary>
            MS_charge_exchange_ionization = 1000385,

            /// <summary>The reaction of a neutral molecule with an internally excited molecule to form an ion. Note that this term is not synonymous with chemical ionization.</summary>
            MS_chemi_ionization = 1000386,

            /// <summary>The formation of ions by laser desorption ionization of a sample deposited on a porous silicon surface.</summary>
            MS_desorption_ionization_on_silicon = 1000387,

            /// <summary>The reaction of a gas-phase molecule that results in its decomposition to form products, one of which is an ion.</summary>
            MS_dissociative_ionization = 1000388,

            /// <summary>The ionization of an atom or molecule by electrons that are typically accelerated to energies between 50 and 150 eV. Usually 70 eV electrons are used to produce positive ions. The term 'electron impact' is not recommended.</summary>
            MS_electron_ionization = 1000389,

            /// <summary>OBSOLETE The removal of solvent molecules clustered around a gas-phase ion by means of heating and/or collisions with gas molecules.</summary>
            MS_ion_desolvation_OBSOLETE = 1000390,

            /// <summary>OBSOLETE The reaction of a molecule to form both a positive ion and negative ion fragment among the products.</summary>
            MS_ion_pair_formation_OBSOLETE = 1000391,

            /// <summary>The ratio of the number of ions formed to the number of electrons, molecules or photons used.</summary>
            MS_ionization_efficiency = 1000392,

            /// <summary>The formation of gas-phase ions by the interaction of a pulsed laser with a solid or liquid material.</summary>
            MS_laser_desorption_ionization = 1000393,

            /// <summary>The ionization of any species by the interaction of a focused beam of ions with a sample that is dissolved in a solvent matrix. See also fast atom bombardment and secondary ionization.</summary>
            MS_liquid_secondary_ionization = 1000395,

            /// <summary>A semi-permeable membrane separator that permits the passage of gas sample directly to the mass spectrometer ion source.</summary>
            MS_membrane_inlet = 1000396,

            /// <summary>Electrospray ionization at a solvent flow rate of 300-800 nL/min where the flow is a result of a mechanical pump. See nanoelectrospray.</summary>
            MS_microelectrospray = 1000397,

            /// <summary>Electrospray ionization at a flow rate less than ~25 nL/min. Nanoelectrospray is synonymous with nanospray. The flow is dependent on the potenial on the tip of the electrospray needle and/or a gas presure to push the sample through the needle. See also electrospray ionization and microelectrospray.</summary>
            MS_nanoelectrospray = 1000398,

            /// <summary>Ionization that occurs through the interaction of two or more neutral gaseous species, at least one of which is internally excited.</summary>
            MS_penning_ionization = 1000399,

            /// <summary>The ionization of material in a solid sample by bombarding it with ionic or neutral atoms formed as a result of the fission of a suitable nuclide, typically 252Cf. Synonymous with fission fragment ionization.</summary>
            MS_plasma_desorption_ionization = 1000400,

            /// <summary>OBSOLETE An electronic state capable of undergoing auto-Ionization.</summary>
            MS_pre_ionization_state_OBSOLETE = 1000401,

            /// <summary>The process in which ions are ejected from a sample surface as a result of bombardment by a primary beam of atoms or ions.</summary>
            MS_secondary_ionization = 1000402,

            /// <summary>The formation of gas-phase ions without extensive fragmentation.</summary>
            MS_soft_ionization = 1000403,

            /// <summary>The formation of ions from a solid material by an intermittent electrical discharge.</summary>
            MS_spark_ionization = 1000404,

            /// <summary>The formation of gas-phase ions from molecules that are deposited on a particular surface substrate that is irradiated with a pulsed laser. See also matrix-assisted laser desorption ionization.</summary>
            MS_surface_assisted_laser_desorption_ionization = 1000405,

            /// <summary>The ionization of a neutral species when it interacts with a solid surface with an appropriate work function and temperature.</summary>
            MS_surface_ionization = 1000406,

            /// <summary>The ionization of a neutral species through contact with a high temperature surface.</summary>
            MS_thermal_ionization = 1000407,

            /// <summary>A process in which an electron is removed from or added to a molecule without a change in the positions of the atoms. The resulting ion is typically in an excited vibrational state.</summary>
            MS_vertical_ionization = 1000408,

            /// <summary>OBSOLETE The reaction of an ion with a neutral species in which the reactants combine to form a single ion.</summary>
            MS_association_reaction_OBSOLETE = 1000409,

            /// <summary>OBSOLETE A homolytic cleavage where the bond fission occurs between at the atom adjacent to the atom at the apparent charge site and an atom removed from the aparent charge site by two bonds.</summary>
            MS_alpha_cleavage_OBSOLETE = 1000410,

            /// <summary>OBSOLETE A homolytic cleavage where the bond fission occurs between at an atom removed from the apparent charge site atom by two bonds and an atom adjacent to that atom and removed from the aparent charge site by three bonds.</summary>
            MS_beta_cleavage_OBSOLETE = 1000411,

            /// <summary>An inert gas used for collisional deactivation of internally excited ions.</summary>
            MS_buffer_gas = 1000412,

            /// <summary>OBSOLETE Fragmentation of an odd electron ion in which the cleaved bond is adjacent to the apparent charge site. Synonymous with charge mediated fragmentation.</summary>
            MS_charge_induced_fragmentation_OBSOLETE = 1000413,

            /// <summary>OBSOLETE Reaction of an ion with a neutral species in which the charge on the product ion is reversed in sign with respect to the reactant ion.</summary>
            MS_charge_inversion_reaction_OBSOLETE = 1000414,

            /// <summary>OBSOLETE The reaction of an ion with a neutral species with a resulting change in the magnitude or sign of the charge on the reactant ion.</summary>
            MS_charge_permutation_reaction_OBSOLETE = 1000415,

            /// <summary>OBSOLETE Reaction of a positive ion with a neutral species in which the positive charge on the product ion is greater than that on the reactant ion.</summary>
            MS_charge_stripping_reaction_OBSOLETE = 1000416,

            /// <summary>OBSOLETE The reaction of an ion with a neutral species in which some or all of the charge of the reactant ion is transferred to the neutral species.</summary>
            MS_charge_transfer_reaction_OBSOLETE = 1000417,

            /// <summary>OBSOLETE The reaction of an ion with a neutral species in which the translational energy of the collision is converted into internal energy of the ion.</summary>
            MS_collisional_excitation_OBSOLETE = 1000418,

            /// <summary>An inert gas used for collisional excitation. The term target gas is not recommended.</summary>
            MS_collision_gas = 1000419,

            /// <summary>OBSOLETE Fragmentation of a molecule or ion in which both electrons forming the single bond that is broken remain on one of the atoms that were originally bonded. This term is synonymous with heterolysis.</summary>
            MS_heterolytic_cleavage_OBSOLETE = 1000420,

            /// <summary>OBSOLETE Collision-induced dissociation process wherein the projectile ion has laboratory-frame translational energy higher than 1 keV.</summary>
            MS_high_energy_collision_OBSOLETE = 1000421,

            /// <summary>A collision-induced dissociation process that occurs in a beam-type collision cell.</summary>
            MS_beam_type_collision_induced_dissociation = 1000422,

            /// <summary>OBSOLETE Fragmentation of an odd electron ion that results from one of a pair of electrons that form a bond between two atoms moving to form a pair with the odd electron on the atom at the apparent charge site. Fragmentation results in the formation of an even electron ion and a radical. This reaction involves the movement of a single electron and is symbolized by a single-barbed arrow. Synonymous with Homolysis.</summary>
            MS_homolytic_cleavage_OBSOLETE = 1000423,

            /// <summary>OBSOLETE Exchange of hydrogen atoms with deuterium atoms in a molecule or pre-formed ion in solution prior to introduction into a mass spectrometer, or by reaction of an ion with a deuterated collision gas inside a mass spectrometer.</summary>
            MS_hydrogen_deuterium_exchange_OBSOLETE = 1000424,

            /// <summary>OBSOLETE A plot of the relative abundance of a beam or other collection of ions as a function their loss of translational energy in reactions with neutral species.</summary>
            MS_ion_energy_loss_spectrum_OBSOLETE = 1000425,

            /// <summary>OBSOLETE The reaction of an ion with a neutral species in which one or more electrons are removed from either the ion or neutral.</summary>
            MS_ionizing_collision_OBSOLETE = 1000426,

            /// <summary>OBSOLETE The reaction of an ion with a neutral molecule. The term ion-molecule reaction is not recommended because the hyphen suggests a single species that is that is both an ion and a molecule.</summary>
            MS_ion_molecule_reaction_OBSOLETE = 1000427,

            /// <summary>OBSOLETE A particular type of transition state that lies between precursor and product ions on the reaction coordinate of some ion reactions.</summary>
            MS_ion_neutral_complex_OBSOLETE = 1000428,

            /// <summary>OBSOLETE A process wherein a charged species interacts with a neutral reactant to produce either chemically different species or changes in the internal energy of one or both of the reactants.</summary>
            MS_ion_neutral_species_reaction_OBSOLETE = 1000429,

            /// <summary>OBSOLETE In this reaction an association reaction is accompanied by the subsequent or simultaneous liberation of a different neutral species as a product.</summary>
            MS_ion_neutral_species_exchange_reaction_OBSOLETE = 1000430,

            /// <summary>OBSOLETE An approach to determination of ion thermodynamic quantities by a bracketing procedure in which the relative probabilities of competing ion fragmentations are measured via the relative abundances of the reaction products. The extended kinetic method takes the associated entropy changes into account.</summary>
            MS_kinetic_method_OBSOLETE = 1000431,

            /// <summary>OBSOLETE A collision between an ion and neutral species with translational energy approximately 1000 eV or lower.</summary>
            MS_low_energy_collisions_OBSOLETE = 1000432,

            /// <summary>A collision-induced dissociation process wherein the precursor ion has the translational energy lower than approximately 1000 eV. This process typically requires multiple collisions and the collisional excitation is cumulative.</summary>
            MS_low_energy_collision_induced_dissociation = 1000433,

            /// <summary>OBSOLETE A dissociation reaction triggered by transfer of a hydrogen atom via a 6-member transition state to the formal radical/charge site from a carbon atom four atoms removed from the charge/radical site (the gamma-carbon); subsequent rearrangement of electron density leads to expulsion of an olefin molecule. This term was originally applied to ketone ions where the charge/radical site is the carbonyl oxygen, but it is now more widely applied.</summary>
            MS_McLafferty_Rearrangement_OBSOLETE = 1000434,

            /// <summary>A process wherein the reactant ion is dissociated as a result of absorption of one or more photons.</summary>
            MS_photodissociation = 1000435,

            /// <summary>OBSOLETE Reaction of an ion with a neutral species in which some but not all of the ion charge is transferred to the neutral.</summary>
            MS_partial_charge_transfer_reaction_OBSOLETE = 1000436,

            /// <summary>OBSOLETE Chemical transformation involving an ion.</summary>
            MS_ion_reaction_OBSOLETE = 1000437,

            /// <summary>OBSOLETE Collision in which the translational energy of the fast-moving collision partner is increased at the expense of internal energy of one or both collision partners.</summary>
            MS_superelastic_collision_OBSOLETE = 1000438,

            /// <summary>OBSOLETE A process wherein a reactant ion interacts with a surface to produce either chemically different species or a change in the internal energy of the reactant ion.</summary>
            MS_surface_induced_reaction_OBSOLETE = 1000439,

            /// <summary>OBSOLETE Fragmentation reaction in which the molecularity is treated as one, irrespective of whether the dissociative state is that of a metastable ion produced in the ion source or results from collisional excitation of a stable ion.</summary>
            MS_unimolecular_dissociation_OBSOLETE = 1000440,

            /// <summary>Function or process of the mass spectrometer where it records a spectrum.</summary>
            MS_scan = 1000441,

            /// <summary>Representation of intensity values corresponding to a range of measurement space.</summary>
            MS_spectrum = 1000442,

            /// <summary>Mass analyzer separates the ions according to their mass-to-charge ratio.</summary>
            MS_mass_analyzer_type = 1000443,

            /// <summary>OBSOLETE Mass/charge separation Method.</summary>
            MS_m_z_Separation_Method_OBSOLETE = 1000444,

            /// <summary>OBSOLETE Sequential m/z separation method.</summary>
            MS_sequential_m_z_separation_method_OBSOLETE = 1000445,

            /// <summary>The ionization of any species by the interaction of a focused beam of ions having a translational energy of several thousand eV with a solid sample.</summary>
            MS_fast_ion_bombardment = 1000446,

            /// <summary>Finnigan LTQ MS.</summary>
            MS_LTQ = 1000447,

            /// <summary>Finnigan LTQ FT MS.</summary>
            MS_LTQ_FT = 1000448,

            /// <summary>Finnigan LTQ Orbitrap MS.</summary>
            MS_LTQ_Orbitrap = 1000449,

            /// <summary>Finnigan LXQ MS.</summary>
            MS_LXQ = 1000450,

            /// <summary>Terms used to describe the Analyzer.</summary>
            MS_mass_analyzer = 1000451,

            /// <summary>Terms used to describe types of data processing.</summary>
            MS_data_transformation = 1000452,

            /// <summary>The device that detects ions.</summary>
            MS_detector = 1000453,

            /// <summary>OBSOLETE Additional terms to describe the instrument as outlined in the mass spec doc, Appendix 1, section 1.5.</summary>
            MS_instrument_additional_description_OBSOLETE = 1000454,

            /// <summary>Ion selection properties that are associated with a value.</summary>
            MS_ion_selection_attribute = 1000455,

            /// <summary>Terms to describe the precursor activation.</summary>
            MS_precursor_activation = 1000456,

            /// <summary>Terms to describe the sample.</summary>
            MS_sample = 1000457,

            /// <summary>Terms to describe the source.</summary>
            MS_source = 1000458,

            /// <summary>OBSOLETE Terms used to describe the spectrum.</summary>
            MS_spectrum_instrument_description_OBSOLETE = 1000459,

            /// <summary>OBSOLETE Terms to describe units.</summary>
            MS_unit_OBSOLETE = 1000460,

            /// <summary>OBSOLETE Terms to describe Additional.</summary>
            MS_additional_description_OBSOLETE = 1000461,

            /// <summary>Device used in the construction of a mass spectrometer to focus, contain or otherwise manipulate ions.</summary>
            MS_ion_optics = 1000462,

            /// <summary>Description of the instrument or the mass spectrometer.</summary>
            MS_instrument = 1000463,

            /// <summary>OBSOLETE A unit of measurement for mass.</summary>
            MS_mass_unit_OBSOLETE = 1000464,

            /// <summary>Relative orientation of the electromagnetic field during the selection and detection of ions in the mass spectrometer.</summary>
            MS_scan_polarity = 1000465,

            /// <summary>OBSOLETE Alternating.</summary>
            MS_alternating_OBSOLETE = 1000466,

            /// <summary>The 1200 Series LC/MSD SL ion trap belongs to the Agilent LC/MSD ion trap family. It provides fast polarity switching and multisignal data acquisition capabilities in a single run while also providing 5 stages of automated data dependent MS2 and 11 stages of manual MS2.</summary>
            MS_1200_series_LC_MSD_SL = 1000467,

            /// <summary>The 6110 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with an entry level single quadrupole mass spectrometer from the 6100 Series of Agilent quadrupole mass spectrometers. 6110 Quadrupole mass spectrometer has m/z range of 10-1500 and 2500 u/s scan speed. It proves useful for wide range of SIM quantitative applications.</summary>
            MS_6110_Quadrupole_LC_MS = 1000468,

            /// <summary>The 6120A Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a single quadrupole mass spectrometer from the 6100 Series of Agilent mass spectrometers. 6120 quadrupole mass spectrometer has m/z range of 10-1500, 2500 u/s scan speed and utilizes multiple signal acquisition.</summary>
            MS_6120A_Quadrupole_LC_MS = 1000469,

            /// <summary>The 6130 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a single quadrupole mass spectrometer from the 6100 series of Agilent mass spectrometers. The 6130 quadrupole mass spectrometer has m/z range of 2-3000, 2500 u/s scan speed in standard mode and 5250 u/s speed in fast-scan mode. It also uses multiple signal acquisition.</summary>
            MS_6130_Quadrupole_LC_MS = 1000470,

            /// <summary>The 6140 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a single quadrupole mass spectrometer from the 6100 Series of Agilent quadrupole mass spectrometers. 6140 Quadrupole mass spectrometer has m/z range of 10-1350, 2500 u/s scan speed in standard mode and 10000 u/s speed in fast-scan mode. It also uses multiple signal acquisition.</summary>
            MS_6140_Quadrupole_LC_MS = 1000471,

            /// <summary>The 6210 Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer. This time of flight mass spectrometer has a m/z range of 50-12000, mass accuracy of less than 2 ppm and resolution greater than 13,000 at m/z 2722. It has multiple ion sources and can be used with multimode ion sources.</summary>
            MS_6210_Time_of_Flight_LC_MS = 1000472,

            /// <summary>The 6310 Ion Trap LC/MS is a Agilent liquid chromatography instrument combined with a 6300 series Agilent ion trap. It has a mass range of 50-2200 between 0.6 to 0.35 resolution and mass range of 200-4000 with resolution of 3-4. The scan speed varies from 1650-27000 for the respective mass ranges.</summary>
            MS_6310_Ion_Trap_LC_MS = 1000473,

            /// <summary>The 6320 Ion Trap LC/MS is a Agilent liquid chromatography instrument combined with a 6300 series Agilent ion trap. It has a mass range of 50-2200 between 0.6 to 0.25 resolution and mass range of 200-4000 with resolution of less than 3. The scan speed varies from 1650-27000 for the respective mass ranges.</summary>
            MS_6320_Ion_Trap_LC_MS = 1000474,

            /// <summary>The 6330 Ion Trap LC/MS is a Agilent liquid chromatography instrument combined with a 6300 series Agilent ion trap. It has a mass range of 50-2200 between 0.6 to 0.25 resolution and mass range of 200-4000 with resolution of less than 3. The scan speed varies from 1650-27000 for the respective mass ranges.</summary>
            MS_6330_Ion_Trap_LC_MS = 1000475,

            /// <summary>The 6340 Ion Trap LC/MS is a Agilent liquid chromatography instrument combined with a 6300 series Agilent ion trap. It has a mass range of 50-2200 between 0.6 to 0.25 resolution and mass range of 200-4000 with resolution of less than 3. The scan speed varies from 1650-27000 for the respective mass ranges.</summary>
            MS_6340_Ion_Trap_LC_MS = 1000476,

            /// <summary>The 6410 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer. Mass range of the mass spectrometer is 15-1650 m/z, resolution is at three settings of 0.7 u (unit), 1.2 u (wide) and 2.5 u (widest). The mass accuracy for 6410 mass spectrometer is 0.1 across the mass range. The collision cell is a hexapole with linear acceleration.</summary>
            MS_6410_Triple_Quadrupole_LC_MS = 1000477,

            /// <summary>The LC/MSD VL ion trap is part of the family of Agilent ion trap mass spectrometers. It has ESI, APCI and APPI ion sources and is a useful ion trap when the amount of sample is not the limiting factor.</summary>
            MS_1200_series_LC_MSD_VL = 1000478,

            /// <summary>OBSOLETE Terms that will likely become obsolete unless there are wails of dissent.</summary>
            MS_purgatory_OBSOLETE = 1000479,

            /// <summary>Analyzer properties that are associated with a value.</summary>
            MS_mass_analyzer_attribute = 1000480,

            /// <summary>Detector attribute recognized as a value.</summary>
            MS_detector_attribute = 1000481,

            /// <summary>Property of a source device that need a value.</summary>
            MS_source_attribute = 1000482,

            /// <summary>Thermo Fisher Scientific instrument model. The company has gone through several names including Thermo Finnigan, Thermo Scientific.</summary>
            MS_Thermo_Fisher_Scientific_instrument_model = 1000483,

            /// <summary>An ion trapping device that consists of an outer barrel-like electrode and a coaxial inner spindle-like electrode that form an electrostatic field with quadro-logarithmic potential distribution. The frequency of harmonic oscillations of the orbitally trapped ions along the axis of the electrostatic field is independent of the ion velocity and is inversely proportional to the square root of m/z so that the trap can be used as a mass analyzer.</summary>
            MS_orbitrap = 1000484,

            /// <summary>Nanospray Inlet.</summary>
            MS_nanospray_inlet = 1000485,

            /// <summary>Potential difference at the MS source in volts.</summary>
            MS_source_potential = 1000486,

            /// <summary>Ion optics involves components that help focus ion streams in mass spectrometry.</summary>
            MS_ion_optics_attribute = 1000487,

            /// <summary>Hitachi instrument model.</summary>
            MS_Hitachi_instrument_model = 1000488,

            /// <summary>Varian instrument model.</summary>
            MS_Varian_instrument_model = 1000489,

            /// <summary>Agilent instrument model.</summary>
            MS_Agilent_instrument_model = 1000490,

            /// <summary>Dionex instrument model.</summary>
            MS_Dionex_instrument_model = 1000491,

            /// <summary>Thermo Electron Corporation instrument model.</summary>
            MS_Thermo_Electron_instrument_model = 1000492,

            /// <summary>Finnigan MAT instrument model.</summary>
            MS_Finnigan_MAT_instrument_model = 1000493,

            /// <summary>Thermo Scientific instrument model.</summary>
            MS_Thermo_Scientific_instrument_model = 1000494,

            /// <summary>Applied Biosystems instrument model.</summary>
            MS_Applied_Biosystems_instrument_model = 1000495,

            /// <summary>Instrument properties that are associated with a value.</summary>
            MS_instrument_attribute = 1000496,

            /// <summary>Special scan mode where data with improved resolution is acquired. This is typically achieved by scanning a more narrow m/z window or scanning with a lower scan rate.</summary>
            MS_zoom_scan = 1000497,

            /// <summary>OBSOLETE Feature of the ion trap mass spectrometer where MS data is acquired over a mass range.</summary>
            MS_full_scan_OBSOLETE = 1000498,

            /// <summary>Nonphysical characteristic attributed to a spectrum.</summary>
            MS_spectrum_attribute = 1000499,

            /// <summary>The lower m/z bound of a mass spectrometer scan window.</summary>
            MS_scan_window_upper_limit = 1000500,

            /// <summary>The upper m/z bound of a mass spectrometer scan window.</summary>
            MS_scan_window_lower_limit = 1000501,

            /// <summary>The time spent gathering data across a peak.</summary>
            MS_dwell_time = 1000502,

            /// <summary>Nonphysical characteristic attributed to a spectrum acquisition scan.</summary>
            MS_scan_attribute = 1000503,

            /// <summary>M/z value of the signal of highest intensity in the mass spectrum.</summary>
            MS_base_peak_m_z = 1000504,

            /// <summary>The intensity of the greatest peak in the mass spectrum.</summary>
            MS_base_peak_intensity = 1000505,

            /// <summary>OBSOLETE Ion Role.</summary>
            MS_ion_role_OBSOLETE = 1000506,

            /// <summary>Nonphysical characteristic attributed to an ion.</summary>
            MS_ion_property = 1000507,

            /// <summary>OBSOLETE Ion Type.</summary>
            MS_ion_chemical_type_OBSOLETE = 1000508,

            /// <summary>Activation Energy.</summary>
            MS_activation_energy = 1000509,

            /// <summary>Precursor Activation Attribute.</summary>
            MS_precursor_activation_attribute = 1000510,

            /// <summary>Stage number achieved in a multi stage mass spectrometry acquisition.</summary>
            MS_ms_level = 1000511,

            /// <summary>A string unique to Thermo instrument describing instrument settings for the scan.</summary>
            MS_filter_string = 1000512,

            /// <summary>A data array of values.</summary>
            MS_binary_data_array = 1000513,

            /// <summary>A data array of m/z values.</summary>
            MS_m_z_array = 1000514,

            /// <summary>A data array of intensity values.</summary>
            MS_intensity_array = 1000515,

            /// <summary>A data array of charge values.</summary>
            MS_charge_array = 1000516,

            /// <summary>A data array of signal-to-noise values.</summary>
            MS_signal_to_noise_array = 1000517,

            /// <summary>Encoding type of binary data specifying the binary representation and precision, e.g. 64-bit float.</summary>
            MS_binary_data_type = 1000518,

            /// <summary>Signed 32-bit little-endian integer.</summary>
            MS_32_bit_integer = 1000519,

            /// <summary>OBSOLETE Signed 16-bit float.</summary>
            MS_16_bit_float_OBSOLETE = 1000520,

            /// <summary>32-bit precision little-endian floating point conforming to IEEE-754.</summary>
            MS_32_bit_float = 1000521,

            /// <summary>Signed 64-bit little-endian integer.</summary>
            MS_64_bit_integer = 1000522,

            /// <summary>64-bit precision little-endian floating point conforming to IEEE-754.</summary>
            MS_64_bit_float = 1000523,

            /// <summary>Describes the data content on the file.</summary>
            MS_data_file_content = 1000524,

            /// <summary>Way in which the spectrum is represented, either with regularly spaced data points or with a list of centroided peaks.</summary>
            MS_spectrum_representation = 1000525,

            /// <summary>Waters data file format found in a Waters RAW directory, generated from an MS acquisition.</summary>
            MS_Waters_raw_format = 1000526,

            /// <summary>Highest m/z value observed in the m/z array.</summary>
            MS_highest_observed_m_z = 1000527,

            /// <summary>Lowest m/z value observed in the m/z array.</summary>
            MS_lowest_observed_m_z = 1000528,

            /// <summary>Serial Number of the instrument.</summary>
            MS_instrument_serial_number = 1000529,

            /// <summary>Conversion of one file format to another.</summary>
            MS_file_format_conversion = 1000530,

            /// <summary>Software related to the recording or transformation of spectra.</summary>
            MS_software = 1000531,

            /// <summary>Thermo Finnigan software for data acquisition and analysis.</summary>
            MS_Xcalibur = 1000532,

            /// <summary>Thermo Finnigan software for data analysis of peptides and proteins.</summary>
            MS_Bioworks = 1000533,

            /// <summary>Micromass software for data acquisition and analysis.</summary>
            MS_MassLynx = 1000534,

            /// <summary>Bruker software for data analysis.</summary>
            MS_FlexAnalysis = 1000535,

            /// <summary>Applied Biosystems software for data acquisition and analysis.</summary>
            MS_Data_Explorer = 1000536,

            /// <summary>Applied Biosystems software for data acquisition and analysis.</summary>
            MS_4700_Explorer = 1000537,

            /// <summary>A software for converting Waters raw directory format to mzXML or mzML. MassWolf was originally developed at the Institute for Systems Biology.</summary>
            MS_massWolf = 1000538,

            /// <summary>Applied Biosystems MALDI-TOF data acquisition and analysis system.</summary>
            MS_Voyager_Biospectrometry_Workstation_System = 1000539,

            /// <summary>Bruker software for data acquisition.</summary>
            MS_FlexControl = 1000540,

            /// <summary>A software program for converting Thermo Finnigan RAW file format to mzXML or mzML. ReAdW was originally developed at the Institute for Systems Biology. Its whimsical interleaved spelling and capitalization is pronounced \ readraw\ .</summary>
            MS_ReAdW = 1000541,

            /// <summary>A software program for converting Applied Biosystems wiff file format to mzXML format. MzStar was originally developed at the Institute for Systems Biology. It is now obsoleted by the MzWiff program.</summary>
            MS_MzStar = 1000542,

            /// <summary>Data processing attribute used to describe the type of data processing performed on the data file.</summary>
            MS_data_processing_action = 1000543,

            /// <summary>Conversion of a file format to Proteomics Standards Initiative mzML file format.</summary>
            MS_Conversion_to_mzML = 1000544,

            /// <summary>Conversion of a file format to Institute of Systems Biology mzXML file format.</summary>
            MS_Conversion_to_mzXML = 1000545,

            /// <summary>Conversion of a file format to Proteomics Standards Initiative mzData file format.</summary>
            MS_Conversion_to_mzData = 1000546,

            /// <summary>Object Attribute.</summary>
            MS_object_attribute = 1000547,

            /// <summary>Sample properties that are associated with a value.</summary>
            MS_sample_attribute = 1000548,

            /// <summary>Selection window properties that are associated with a value.</summary>
            MS_selection_window_attribute = 1000549,

            /// <summary>OBSOLETE Time Unit.</summary>
            MS_time_unit_OBSOLETE = 1000550,

            /// <summary>SCIEX or Applied Biosystems|MDS SCIEX software for data acquisition.</summary>
            MS_Analyst = 1000551,

            /// <summary>OBSOLETE Maldi Spot Identifier.</summary>
            MS_maldi_spot_identifier_OBSOLETE = 1000552,

            /// <summary>A software program for converting Agilent MassHunter format to mzXML or mzML. Trapper was originally developed at the Institute for Systems Biology.</summary>
            MS_Trapper = 1000553,

            /// <summary>ThermoFinnigan LCQ Deca.</summary>
            MS_LCQ_Deca = 1000554,

            /// <summary>LTQ Orbitrap Discovery.</summary>
            MS_LTQ_Orbitrap_Discovery = 1000555,

            /// <summary>LTQ Orbitrap XL.</summary>
            MS_LTQ_Orbitrap_XL = 1000556,

            /// <summary>LTQ FT Ultra.</summary>
            MS_LTQ_FT_Ultra = 1000557,

            /// <summary>GC Quantum.</summary>
            MS_GC_Quantum = 1000558,

            /// <summary>Spectrum type.</summary>
            MS_spectrum_type = 1000559,

            /// <summary>The format of the file being used. This could be a instrument or vendor specific proprietary file format or a converted open file format.</summary>
            MS_mass_spectrometer_file_format = 1000560,

            /// <summary>Checksum is a form of redundancy check, a simple way to protect the integrity of data by detecting errors in data.</summary>
            MS_data_file_checksum_type = 1000561,

            /// <summary>Applied Biosystems WIFF file format.</summary>
            MS_ABI_WIFF_format = 1000562,

            /// <summary>Thermo Scientific RAW file format.</summary>
            MS_Thermo_RAW_format = 1000563,

            /// <summary>Proteomics Standards Inititative mzData file format.</summary>
            MS_PSI_mzData_format = 1000564,

            /// <summary>Micromass PKL file format.</summary>
            MS_Micromass_PKL_format = 1000565,

            /// <summary>Institute of Systems Biology mzXML file format.</summary>
            MS_ISB_mzXML_format = 1000566,

            /// <summary>Bruker/Agilent YEP file format.</summary>
            MS_Bruker_Agilent_YEP_format = 1000567,

            /// <summary>MD5 (Message-Digest algorithm 5) is a (now deprecated) cryptographic hash function with a 128-bit hash value used to check the integrity of files.</summary>
            MS_MD5 = 1000568,

            /// <summary>SHA-1 (Secure Hash Algorithm-1) is a cryptographic hash function designed by the National Security Agency (NSA). It is also used to verify file integrity. Since 2011 it has been deprecated by the NIST as a U. S. government standard.</summary>
            MS_SHA_1 = 1000569,

            /// <summary>Method used to combine the mass spectra.</summary>
            MS_spectra_combination = 1000570,

            /// <summary>Spectra Sum.</summary>
            MS_sum_of_spectra = 1000571,

            /// <summary>Compression Type.</summary>
            MS_binary_data_compression_type = 1000572,

            /// <summary>Spectra is combined by calculating the median of the spectra.</summary>
            MS_median_of_spectra = 1000573,

            /// <summary>Zlib.</summary>
            MS_zlib_compression = 1000574,

            /// <summary>Spectra is combined by calculating the mean of the spectra.</summary>
            MS_mean_of_spectra = 1000575,

            /// <summary>No Compression.</summary>
            MS_no_compression = 1000576,

            /// <summary>Data file from which an entity is sourced.</summary>
            MS_source_data_file = 1000577,

            /// <summary>LCQ Fleet.</summary>
            MS_LCQ_Fleet = 1000578,

            /// <summary>Mass spectrum created by a single-stage MS experiment or the first stage of a multi-stage experiment.</summary>
            MS_MS1_spectrum = 1000579,

            /// <summary>MSn refers to multi-stage MS2 experiments designed to record product ion spectra where n is the number of product ion stages (progeny ions). For ion traps, sequential MS/MS experiments can be undertaken where n &gt; 2 whereas for a simple triple quadrupole system n=2. Use the term ms level (MS:1000511) for specifying n.</summary>
            MS_MSn_spectrum = 1000580,

            /// <summary>Spectrum generated from MSn experiment with three or more stages of m/z separation and in which a particular multi-step reaction path is monitored.</summary>
            MS_CRM_spectrum = 1000581,

            /// <summary>Spectrum obtained with the operation of a mass spectrometer in which the abundances of one ion or several ions of specific m/z values are recorded rather than the entire mass spectrum (Selected Ion Monitoring).</summary>
            MS_SIM_spectrum = 1000582,

            /// <summary>Spectrum obtained when data are acquired from specific product ions corresponding to m/z values of selected precursor ions a recorded via two or more stages of mass spectrometry. The precursor/product ion pair is called a transition pair. Data can be obtained for a single transition pair or multiple transition pairs. Multiple time segments of different transition pairs can exist in a single file. Single precursor ions can have multiple product ions consitituting multiple transition pairs. Selected reaction monitoring can be performed as tandem mass spectrometry in time or tandem mass spectrometry in space.</summary>
            MS_SRM_spectrum = 1000583,

            /// <summary>Proteomics Standards Inititative mzML file format.</summary>
            MS_mzML_format = 1000584,

            /// <summary>Details about a person or organization to contact in case of concern or discussion about the file.</summary>
            MS_contact_attribute = 1000585,

            /// <summary>Name of the contact person or organization.</summary>
            MS_contact_name = 1000586,

            /// <summary>Postal address of the contact person or organization.</summary>
            MS_contact_address = 1000587,

            /// <summary>Uniform Resource Locator related to the contact person or organization.</summary>
            MS_contact_URL = 1000588,

            /// <summary>Email address of the contact person or organization.</summary>
            MS_contact_email = 1000589,

            /// <summary>Home institution of the contact person.</summary>
            MS_contact_affiliation = 1000590,

            /// <summary>A software program for converting Applied Biosystems wiff file format to the mzXML or mzML format. MzWiff is currently maintained at the Institute for Systems Biology. It replaces the slower mzStar program.</summary>
            MS_MzWiff = 1000591,

            /// <summary>A process of reducing spikes of intensity in order to reduce noise while preserving real peak signal. Many algorithms can be applied for this process.</summary>
            MS_smoothing = 1000592,

            /// <summary>A process of removal of varying intensities generated due to variable energy absorption before further processing can take place. Baseline reduction facilitates meaningful comparision between intensities of m/z values.</summary>
            MS_baseline_reduction = 1000593,

            /// <summary>The removal of very low intensity data points that are likely to be spurious noise rather than real signal.</summary>
            MS_low_intensity_data_point_removal = 1000594,

            /// <summary>A data array of relative time offset values from a reference time.</summary>
            MS_time_array = 1000595,

            /// <summary>An attribute of resolution when recording the detector response in absence of the analyte.</summary>
            MS_measurement_method = 1000596,

            /// <summary>The electrical potential used to impart kinetic energy to ions in a mass spectrometer.</summary>
            MS_ion_optics_type = 1000597,

            /// <summary>A process to fragment ions in a mass spectrometer by inducing fragmentation of cations (e.g. peptides or proteins) by transferring electrons from radical-anions.</summary>
            MS_electron_transfer_dissociation = 1000598,

            /// <summary>A process that involves precursor ion activation at high Q, a time delay to allow the precursor to fragment, then a rapid pulse to low Q where all fragment ions are trapped. The product ions can then be scanned out of the ion trap and detected.</summary>
            MS_pulsed_q_dissociation = 1000599,

            /// <summary>Database application and analysis platform for proteomics.</summary>
            MS_Proteios = 1000600,

            /// <summary>Waters software for data analysis.</summary>
            MS_ProteinLynx_Global_Server = 1000601,

            /// <summary>Shimadzu MALDI-TOF instrument model.</summary>
            MS_Shimadzu_MALDI_TOF_instrument_model = 1000602,

            /// <summary>Shimadzu Scientific Instruments instrument model.</summary>
            MS_Shimadzu_Scientific_Instruments_instrument_model = 1000603,

            /// <summary>Shimadzu Scientific Instruments LCMS-IT-TOF MS.</summary>
            MS_LCMS_IT_TOF = 1000604,

            /// <summary>Shimadzu Scientific Instruments LCMS-2010EV MS.</summary>
            MS_LCMS_2010EV = 1000605,

            /// <summary>Shimadzu Scientific Instruments LCMS-2010A MS.</summary>
            MS_LCMS_2010A = 1000606,

            /// <summary>Shimadzu Biotech AXIMA CFR MALDI-TOF MS.</summary>
            MS_AXIMA_CFR_MALDI_TOF = 1000607,

            /// <summary>Shimadzu Biotech AXIMA-QIT MS.</summary>
            MS_AXIMA_QIT = 1000608,

            /// <summary>Shimadzu Biotech AXIMA-CFR plus MS.</summary>
            MS_AXIMA_CFR_plus = 1000609,

            /// <summary>Shimadzu Biotech AXIMA Performance MALDI-TOF/TOF MS.</summary>
            MS_AXIMA_Performance_MALDI_TOF_TOF = 1000610,

            /// <summary>Shimadzu Biotech AXIMA Confidence MALDI-TOF (curved field reflectron) MS.</summary>
            MS_AXIMA_Confidence_MALDI_TOF = 1000611,

            /// <summary>Shimadzu Biotech AXIMA Assurance Linear MALDI-TOF MS.</summary>
            MS_AXIMA_Assurance_Linear_MALDI_TOF = 1000612,

            /// <summary>SEQUEST DTA file format.</summary>
            MS_DTA_format = 1000613,

            /// <summary>Peak list file format used by ProteinLynx Global Server.</summary>
            MS_ProteinLynx_Global_Server_mass_spectrum_XML_format = 1000614,

            /// <summary>ProteoWizard software for data processing and analysis. Primarily developed by the labs of P. Malick and D. Tabb.</summary>
            MS_ProteoWizard_software = 1000615,

            /// <summary>A user-defined scan configuration that specifies the instrumental settings in which a spectrum is acquired. An instrument may cycle through a list of preset scan configurations to acquire data. This is a more generic term for the Thermo \ scan event\ , which is defined in the Thermo Xcalibur glossary as: a mass spectrometer scan that is defined by choosing the necessary scan parameter settings. Multiple scan events can be defined for each segment of time.</summary>
            MS_preset_scan_configuration = 1000616,

            /// <summary>A data array of electromagnetic radiation wavelength values.</summary>
            MS_wavelength_array = 1000617,

            /// <summary>Highest wavelength observed in an EMR spectrum.</summary>
            MS_highest_observed_wavelength = 1000618,

            /// <summary>Lowest wavelength observed in an EMR spectrum.</summary>
            MS_lowest_observed_wavelength = 1000619,

            /// <summary>OBSOLETE Spectrum generated from a photodiode array detector (ultraviolet/visible spectrum).</summary>
            MS_PDA_spectrum_OBSOLETE = 1000620,

            /// <summary>An array detector used to record spectra in the ultraviolet and visible region of light.</summary>
            MS_photodiode_array_detector = 1000621,

            /// <summary>Surveyor PDA.</summary>
            MS_Surveyor_PDA = 1000622,

            /// <summary>Accela PDA.</summary>
            MS_Accela_PDA = 1000623,

            /// <summary>Inductive detector.</summary>
            MS_inductive_detector = 1000624,

            /// <summary>Representation of a chromatographic separation attribute measurement versus time.</summary>
            MS_chromatogram = 1000625,

            /// <summary>Type of chromatogram measurement being represented.</summary>
            MS_chromatogram_type = 1000626,

            /// <summary>Representation of an array of the measurements of a specific single ion current versus time.</summary>
            MS_selected_ion_current_chromatogram = 1000627,

            /// <summary>Representation of an array of the most intense peaks versus time.</summary>
            MS_basepeak_chromatogram = 1000628,

            /// <summary>Threshold below which some action is taken.</summary>
            MS_low_intensity_threshold = 1000629,

            /// <summary>Data processing parameter used in the data processing performed on the data file.</summary>
            MS_data_processing_parameter = 1000630,

            /// <summary>Threshold above which some action is taken.</summary>
            MS_high_intensity_threshold = 1000631,

            /// <summary>Waters oa-ToF based Q-Tof Premier.</summary>
            MS_Q_Tof_Premier = 1000632,

            /// <summary>A possible charge state of the ion in a situation where the charge of an ion is known to be one of several possible values rather than a completely unknown value or determined to be a specific charge with reasonable certainty.</summary>
            MS_possible_charge_state = 1000633,

            /// <summary>ThermoFinnigan DSQ GC-MS.</summary>
            MS_DSQ = 1000634,

            /// <summary>Thermo Scientific ITQ 700 GC-MS.</summary>
            MS_ITQ_700 = 1000635,

            /// <summary>Thermo Scientific ITQ 900 GC-MS.</summary>
            MS_ITQ_900 = 1000636,

            /// <summary>Thermo Scientific ITQ 1100 GC-MS.</summary>
            MS_ITQ_1100 = 1000637,

            /// <summary>Thermo Scientific LTQ XL MS with ETD.</summary>
            MS_LTQ_XL_ETD = 1000638,

            /// <summary>Thermo Scientific LTQ Orbitrap XL MS with ETD.</summary>
            MS_LTQ_Orbitrap_XL_ETD = 1000639,

            /// <summary>Thermo Scientific DFS HR GC-MS.</summary>
            MS_DFS = 1000640,

            /// <summary>Thermo Scientific DSQ II GC-MS.</summary>
            MS_DSQ_II = 1000641,

            /// <summary>Thermo Scientific MALDI LTQ XL MS.</summary>
            MS_MALDI_LTQ_XL = 1000642,

            /// <summary>Thermo Scientific MALDI LTQ Orbitrap MS.</summary>
            MS_MALDI_LTQ_Orbitrap = 1000643,

            /// <summary>Thermo Scientific TSQ Quantum Access MS.</summary>
            MS_TSQ_Quantum_Access = 1000644,

            /// <summary>Thermo Scientific Element XR HR-ICP-MS.</summary>
            MS_Element_XR = 1000645,

            /// <summary>Thermo Scientific Element 2 HR-ICP-MS.</summary>
            MS_Element_2 = 1000646,

            /// <summary>Thermo Scientific Element GD Glow Discharge MS.</summary>
            MS_Element_GD = 1000647,

            /// <summary>Thermo Scientific GC IsoLink Isotope Ratio MS.</summary>
            MS_GC_IsoLink = 1000648,

            /// <summary>Thermo Scientific Exactive MS.</summary>
            MS_Exactive = 1000649,

            /// <summary>Thermo Scientific software for data analysis of peptides and proteins.</summary>
            MS_Proteome_Discoverer = 1000650,

            /// <summary>SCIEX or Applied Biosystems|MDS SCIEX QTRAP 3200.</summary>
            MS_3200_QTRAP = 1000651,

            /// <summary>SCIEX or Applied Biosystems|MDS SCIEX 4800 Plus MALDI TOF-TOF Analyzer.</summary>
            MS_4800_Plus_MALDI_TOF_TOF = 1000652,

            /// <summary>SCIEX or Applied Biosystems|MDS SCIEX API 3200 MS.</summary>
            MS_API_3200 = 1000653,

            /// <summary>SCIEX or Applied Biosystems|MDS SCIEX API 5000 MS.</summary>
            MS_API_5000 = 1000654,

            /// <summary>SCIEX or Applied Biosystems|MDS SCIEX QSTAR Elite.</summary>
            MS_QSTAR_Elite = 1000655,

            /// <summary>Applied Biosystems|MDS SCIEX QSTAR Pulsar.</summary>
            MS_QSTAR_Pulsar = 1000656,

            /// <summary>Applied Biosystems|MDS SCIEX QSTAR XL.</summary>
            MS_QSTAR_XL = 1000657,

            /// <summary>Applied Biosystems|MDS SCIEX 4800 Proteomics Analyzer.</summary>
            MS_4800_Proteomics_Analyzer = 1000658,

            /// <summary>SCIEX or Applied Biosystems software for data acquisition and analysis.</summary>
            MS_4000_Series_Explorer_Software = 1000659,

            /// <summary>SCIEX or Applied Biosystems software for data acquisition and analysis.</summary>
            MS_GPS_Explorer = 1000661,

            /// <summary>SCIEX or Applied Biosystems|MDS SCIEX software metabolite identification.</summary>
            MS_LightSight_Software = 1000662,

            /// <summary>SCIEX or Applied Biosystems|MDS SCIEX software for protein ID and quant.</summary>
            MS_ProteinPilot_Software = 1000663,

            /// <summary>Applied Biosystems|MDS SCIEX software for tissue imaging.</summary>
            MS_TissueView_Software = 1000664,

            /// <summary>Applied Biosystems|MDS SCIEX software for metabolomics and biomarker profiling.</summary>
            MS_MarkerView_Software = 1000665,

            /// <summary>Applied Biosystems|MDS SCIEX software for MRM assay development.</summary>
            MS_MRMPilot_Software = 1000666,

            /// <summary>Applied Biosystems|MDS SCIEX software for bio-related data exploration.</summary>
            MS_BioAnalyst = 1000667,

            /// <summary>Applied Biosystems|MDS SCIEX software for protein identification.</summary>
            MS_Pro_ID = 1000668,

            /// <summary>Applied Biosystems|MDS SCIEX software for protein ID and quant by ICAT.</summary>
            MS_Pro_ICAT = 1000669,

            /// <summary>Applied Biosystems|MDS SCIEX software for protein ID and quant by iTRAQ.</summary>
            MS_Pro_Quant = 1000670,

            /// <summary>Applied Biosystems|MDS SCIEX software for MS-BLAST identification.</summary>
            MS_Pro_BLAST = 1000671,

            /// <summary>SCIEX Cliquid software for data analysis and quantitation.</summary>
            MS_Cliquid = 1000672,

            /// <summary>Applied Biosystems|MDS SCIEX software for MRM assay development.</summary>
            MS_MIDAS_Workflow_Designer = 1000673,

            /// <summary>Applied Biosystems|MDS SCIEX software for MRM-based quantitation.</summary>
            MS_MultiQuant = 1000674,

            /// <summary>The 6220 Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer. This time of flight mass spectrometer has a m/z range of 50-12000, mass accuracy of less than 2 ppm and resolution greater than 13,000 at m/z 2722. It has multiple ion sources and can be used with multimode ion sources.</summary>
            MS_6220_Time_of_Flight_LC_MS = 1000675,

            /// <summary>The 6510 Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer. This time of flight mass spectrometer has a m/z range of 50-12000, mass accuracy of less than 2 ppm and resolution greater than 13,000 at m/z 2722. It has multiple ion sources and can be used with multimode ion sources.</summary>
            MS_6510_Quadrupole_Time_of_Flight_LC_MS = 1000676,

            /// <summary>The 6520A Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer. This time of flight mass spectrometer has a m/z range of 50-12000, mass accuracy of less than 2 ppm and resolution greater than 26,000 at m/z 2722. It has multiple ion sources and can be used with multimode ion sources.</summary>
            MS_6520A_Quadrupole_Time_of_Flight_LC_MS = 1000677,

            /// <summary>Software for data acquisition of 6000 series instruments.</summary>
            MS_MassHunter_Data_Acquisition = 1000678,

            /// <summary>Software for open access data acquisition.</summary>
            MS_MassHunter_Easy_Access = 1000679,

            /// <summary>Software for data analysis of data from 6000 series instruments.</summary>
            MS_MassHunter_Qualitative_Analysis = 1000680,

            /// <summary>Software for quantitation of Triple Quadrupole and Quadrupole Time-of-Flight data.</summary>
            MS_MassHunter_Quantitative_Analysis = 1000681,

            /// <summary>Software for identification of metabolites.</summary>
            MS_MassHunter_Metabolite_ID = 1000682,

            /// <summary>Software for protein characterization.</summary>
            MS_MassHunter_BioConfirm = 1000683,

            /// <summary>Software for quantitation and statistical analysis of TOF and Q-TOF LC/MS data.</summary>
            MS_Genespring_MS = 1000684,

            /// <summary>Software for quantitation and statistical analysis of TOF and Q-TOF LC/MS data.</summary>
            MS_MassHunter_Mass_Profiler = 1000685,

            /// <summary>Personal Metabolite Database for MassHunter Workstation. Software for identification of human metabolites.</summary>
            MS_METLIN = 1000686,

            /// <summary>Software for protein identification and characterization of complex protein digest mixtures.</summary>
            MS_Spectrum_Mill_for_MassHunter_Workstation = 1000687,

            /// <summary>Software for data analysis of 6300 series ion trap mass spectrometers.</summary>
            MS_6300_Series_Ion_Trap_Data_Analysis_Software = 1000688,

            /// <summary>Agilent software for data acquisition and analysis.</summary>
            MS_Agilent_software = 1000689,

            /// <summary>SCIEX or Applied Biosystems software for data acquisition and analysis.</summary>
            MS_SCIEX_software = 1000690,

            /// <summary>Applied Biosystems|MDS SCIEX software for data acquisition and analysis.</summary>
            MS_Applied_Biosystems_software = 1000691,

            /// <summary>Bruker software for data acquisition and analysis.</summary>
            MS_Bruker_software = 1000692,

            /// <summary>Thermo Finnigan software for data acquisition and analysis.</summary>
            MS_Thermo_Finnigan_software = 1000693,

            /// <summary>Waters software for data acquisition and analysis.</summary>
            MS_Waters_software = 1000694,

            /// <summary>Bruker Daltonics' apex ultra: ESI, MALDI, Nanospray, APCI, APPI, Qh-FT_ICR.</summary>
            MS_apex_ultra = 1000695,

            /// <summary>Bruker Daltonics' autoflex III smartbeam: MALDI TOF.</summary>
            MS_autoflex_III_smartbeam = 1000696,

            /// <summary>Bruker Daltonics' HCT Series.</summary>
            MS_Bruker_Daltonics_HCT_Series = 1000697,

            /// <summary>Bruker Daltonics' HCTultra: ESI TOF, Nanospray, APCI, APPI.</summary>
            MS_HCTultra = 1000698,

            /// <summary>Bruker Daltonics' HCTultra PTM: ESI TOF, Nanospray, APCI, APPI, PTR.</summary>
            MS_HCTultra_PTM = 1000699,

            /// <summary>Bruker Daltonics' HCTultra ETD II: ESI Q-TOF, Nanospray, APCI, APPI, ETD.</summary>
            MS_HCTultra_ETD_II = 1000700,

            /// <summary>Bruker Daltonics' microflex LT: MALDI TOF.</summary>
            MS_microflex_LT = 1000701,

            /// <summary>Bruker Daltonics' micrOTOF: ESI TOF, APCI, APPI.</summary>
            MS_micrOTOF = 1000702,

            /// <summary>Bruker Daltonics' micrOTOF-Q: ESI Q-TOF, Nanospray, APCI, APPI.</summary>
            MS_micrOTOF_Q = 1000703,

            /// <summary>Bruker Daltonics' micrOTOF-Q II: ESI Q-TOF, Nanospray, APCI, APPI.</summary>
            MS_micrOTOF_Q_II = 1000704,

            /// <summary>Bruker Daltonics' ultraflex III TOF/TOF: MALDI TOF.</summary>
            MS_ultraflex_III_TOF_TOF = 1000705,

            /// <summary>Bruker software for data acquisition.</summary>
            MS_apexControl = 1000706,

            /// <summary>Bruker software for data analysis.</summary>
            MS_BioTools = 1000707,

            /// <summary>Bruker CLINPROT software.</summary>
            MS_CLINPROT = 1000708,

            /// <summary>Bruker CLINPROT micro software.</summary>
            MS_CLINPROT_micro = 1000709,

            /// <summary>Bruker CLINPROT robot software.</summary>
            MS_CLINPROT_robot = 1000710,

            /// <summary>Bruker ClinProTools software.</summary>
            MS_ClinProTools = 1000711,

            /// <summary>Bruker Compass software.</summary>
            MS_Compass = 1000712,

            /// <summary>Bruker Compass for HCT/esquire software.</summary>
            MS_Compass_for_HCT_esquire = 1000713,

            /// <summary>Bruker Compass for micrOTOF software.</summary>
            MS_Compass_for_micrOTOF = 1000714,

            /// <summary>Bruker compass OpenAccess software.</summary>
            MS_Compass_OpenAccess = 1000715,

            /// <summary>Bruker compass Security Pack software.</summary>
            MS_Compass_Security_Pack = 1000716,

            /// <summary>Bruker stand-alone software for data conversion.</summary>
            MS_CompassXport = 1000717,

            /// <summary>Bruker software library for data access.</summary>
            MS_CompassXtract = 1000718,

            /// <summary>Bruker software for data analysis.</summary>
            MS_DataAnalysis = 1000719,

            /// <summary>Bruker software for data acquisition.</summary>
            MS_dpControl = 1000720,

            /// <summary>Bruker software for data acquisition.</summary>
            MS_esquireControl = 1000721,

            /// <summary>Bruker software for data analysis.</summary>
            MS_flexImaging = 1000722,

            /// <summary>Bruker GENOLINK software.</summary>
            MS_GENOLINK = 1000723,

            /// <summary>Bruker GenoTools software.</summary>
            MS_GenoTools = 1000724,

            /// <summary>Bruker software for data acquisition.</summary>
            MS_HCTcontrol = 1000725,

            /// <summary>Bruker software for data acquisition.</summary>
            MS_micrOTOFcontrol = 1000726,

            /// <summary>Bruker PolyTools software.</summary>
            MS_PolyTools = 1000727,

            /// <summary>Bruker software for data analysis.</summary>
            MS_ProfileAnalysis = 1000728,

            /// <summary>Bruker PROTEINEER software.</summary>
            MS_PROTEINEER = 1000729,

            /// <summary>Bruker PROTEINEER dp software.</summary>
            MS_PROTEINEER_dp = 1000730,

            /// <summary>Bruker PROTEINEER fc software.</summary>
            MS_PROTEINEER_fc = 1000731,

            /// <summary>Bruker PROTEINEER spII software.</summary>
            MS_PROTEINEER_spII = 1000732,

            /// <summary>Bruker PROTEINEER-LC software.</summary>
            MS_PROTEINEER_LC = 1000733,

            /// <summary>Bruker ProteinScape software.</summary>
            MS_ProteinScape = 1000734,

            /// <summary>BrukerPureDisk software.</summary>
            MS_PureDisk = 1000735,

            /// <summary>Bruker software for data analysis.</summary>
            MS_QuantAnalysis = 1000736,

            /// <summary>Bruker software for data acquisition.</summary>
            MS_spControl = 1000737,

            /// <summary>Bruker TargetAnalysis software.</summary>
            MS_TargetAnalysis = 1000738,

            /// <summary>Bruker WARP-LC software.</summary>
            MS_WARP_LC = 1000739,

            /// <summary>Parameter file used to configure the acquisition of raw data on the instrument.</summary>
            MS_parameter_file = 1000740,

            /// <summary>Conversion to dta format.</summary>
            MS_Conversion_to_dta = 1000741,

            /// <summary>Thermo Finnigan SRF file format.</summary>
            MS_Bioworks_SRF_format = 1000742,

            /// <summary>Thermo Scientific TSQ Quantum Ultra AM.</summary>
            MS_TSQ_Quantum_Ultra_AM = 1000743,

            /// <summary>Mass-to-charge ratio of an selected ion.</summary>
            MS_selected_ion_m_z = 1000744,

            /// <summary>The correction of the spectrum scan times, as used e.g. in label-free proteomics.</summary>
            MS_retention_time_alignment = 1000745,

            /// <summary>The removal of very high intensity data points.</summary>
            MS_high_intensity_data_point_removal = 1000746,

            /// <summary>The time that a data processing action was finished.</summary>
            MS_completion_time = 1000747,

            /// <summary>ThermoFinnigan SSQ 7000 MS.</summary>
            MS_SSQ_7000 = 1000748,

            /// <summary>ThermoFinnigan TSQ 7000 MS.</summary>
            MS_TSQ_7000 = 1000749,

            /// <summary>ThermoFinnigan TSQ MS.</summary>
            MS_TSQ = 1000750,

            /// <summary>Thermo Scientific TSQ Quantum Ultra.</summary>
            MS_TSQ_Quantum_Ultra = 1000751,

            /// <summary>TOPP (The OpenMS proteomics pipeline) software.</summary>
            MS_TOPP_software = 1000752,

            /// <summary>Removes the baseline from profile spectra using a top-hat filter.</summary>
            MS_BaselineFilter = 1000753,

            /// <summary>Exports data from an OpenMS database to a file.</summary>
            MS_DBExporter = 1000754,

            /// <summary>Imports data to an OpenMS database.</summary>
            MS_DBImporter = 1000755,

            /// <summary>Converts between different MS file formats.</summary>
            MS_FileConverter = 1000756,

            /// <summary>Extracts or manipulates portions of data from peak, feature or consensus feature files.</summary>
            MS_FileFilter = 1000757,

            /// <summary>Merges several MS files into one file.</summary>
            MS_FileMerger = 1000758,

            /// <summary>Applies an internal calibration.</summary>
            MS_InternalCalibration = 1000759,

            /// <summary>OBSOLETE Corrects retention time distortions between maps.</summary>
            MS_MapAligner_OBSOLETE = 1000760,

            /// <summary>Normalizes peak intensities in an MS run.</summary>
            MS_MapNormalizer = 1000761,

            /// <summary>OBSOLETE Removes noise from profile spectra by using different smoothing techniques.</summary>
            MS_NoiseFilter_OBSOLETE = 1000762,

            /// <summary>OBSOLETE Finds mass spectrometric peaks in profile mass spectra.</summary>
            MS_PeakPicker_OBSOLETE = 1000763,

            /// <summary>Transforms an LC/MS map into a resampled map or a png image.</summary>
            MS_Resampler = 1000764,

            /// <summary>OBSOLETE Applies a filter to peak spectra.</summary>
            MS_SpectraFilter_OBSOLETE = 1000765,

            /// <summary>Applies time of flight calibration.</summary>
            MS_TOFCalibration = 1000766,

            /// <summary>Describes how the native spectrum identifiers are formated.</summary>
            MS_native_spectrum_identifier_format = 1000767,

            /// <summary>Native format defined by controllerType=xsd:nonNegativeInteger controllerNumber=xsd:positiveInteger scan=xsd:positiveInteger.</summary>
            MS_Thermo_nativeID_format = 1000768,

            /// <summary>Native format defined by function=xsd:positiveInteger process=xsd:nonNegativeInteger scan=xsd:nonNegativeInteger.</summary>
            MS_Waters_nativeID_format = 1000769,

            /// <summary>Native format defined by sample=xsd:nonNegativeInteger period=xsd:nonNegativeInteger cycle=xsd:nonNegativeInteger experiment=xsd:nonNegativeInteger.</summary>
            MS_WIFF_nativeID_format = 1000770,

            /// <summary>Native format defined by scan=xsd:nonNegativeInteger.</summary>
            MS_Bruker_Agilent_YEP_nativeID_format = 1000771,

            /// <summary>Native format defined by scan=xsd:nonNegativeInteger.</summary>
            MS_Bruker_BAF_nativeID_format = 1000772,

            /// <summary>Native format defined by file=xsd:IDREF.</summary>
            MS_Bruker_FID_nativeID_format = 1000773,

            /// <summary>Native format defined by index=xsd:nonNegativeInteger.</summary>
            MS_multiple_peak_list_nativeID_format = 1000774,

            /// <summary>Native format defined by file=xsd:IDREF.</summary>
            MS_single_peak_list_nativeID_format = 1000775,

            /// <summary>Native format defined by scan=xsd:nonNegativeInteger.</summary>
            MS_scan_number_only_nativeID_format = 1000776,

            /// <summary>Native format defined by spectrum=xsd:nonNegativeInteger.</summary>
            MS_spectrum_identifier_nativeID_format = 1000777,

            /// <summary>A process that infers the charge state of an MSn spectrum's precursor(s) by the application of some algorithm.</summary>
            MS_charge_state_calculation = 1000778,

            /// <summary>Infers charge state as single or ambiguously multiple by determining the fraction of intensity below the precursor m/z.</summary>
            MS_below_precursor_intensity_dominance_charge_state_calculation = 1000779,

            /// <summary>A process that recalculates existing precursor selected ions with one or more algorithmically determined precursor selected ions.</summary>
            MS_precursor_recalculation = 1000780,

            /// <summary>Recalculates one or more precursor selected ions by peak detection in the isolation windows of high accuracy MS precursor scans.</summary>
            MS_msPrefix_precursor_recalculation = 1000781,

            /// <summary>Reduces intensity spikes by applying local polynomial regression (of degree k) on a distribution (of at least k+1 equally spaced points) to determine the smoothed value for each point. It tends to preserve features of the distribution such as relative maxima, minima and width, which are usually 'flattened' by other adjacent averaging techniques.</summary>
            MS_Savitzky_Golay_smoothing = 1000782,

            /// <summary>Reduces intensity spikes by applying a modelling method known as locally weighted polynomial regression. At each point in the data set a low-degree polynomial is fit to a subset of the data, with explanatory variable values near the point whose response is being estimated. The polynomial is fit using weighted least squares, giving more weight to points near the point whose response is being estimated and less weight to points further away. The value of the regression function for the point is then obtained by evaluating the local polynomial using the explanatory variable values for that data point. The LOESS fit is complete after regression function values have been computed for each of the n data points. Many of the details of this method, such as the degree of the polynomial model and the weights, are flexible.</summary>
            MS_LOWESS_smoothing = 1000783,

            /// <summary>Reduces intensity spikes by convolving the data with a one-dimensional Gaussian function.</summary>
            MS_Gaussian_smoothing = 1000784,

            /// <summary>Reduces intensity spikes by averaging each point with two or more adjacent points. The more adjacent points that used, the stronger the smoothing effect.</summary>
            MS_moving_average_smoothing = 1000785,

            /// <summary>A data array that contains data not covered by any other term in this group. Please do not use this term, if the binary data array type might be commonly used - contact the PSI-MS working group in order to have another CV term added.</summary>
            MS_non_standard_data_array = 1000786,

            /// <summary>Threshold at or below which some action is taken.</summary>
            MS_inclusive_low_intensity_threshold = 1000787,

            /// <summary>Threshold at or above which some action is taken.</summary>
            MS_inclusive_high_intensity_threshold = 1000788,

            /// <summary>MS1 spectrum that is enriched in multiply-charged ions compared to singly-charged ions.</summary>
            MS_enhanced_multiply_charged_spectrum = 1000789,

            /// <summary>MSn spectrum in which the product ions are collected after a time delay, which allows the observation of lower energy fragmentation processes after precursor ion activation.</summary>
            MS_time_delayed_fragmentation_spectrum = 1000790,

            /// <summary>OBSOLETE Scan with enhanced resolution.</summary>
            MS_enhanced_resolution_scan_OBSOLETE = 1000791,

            /// <summary>Isolation window parameter.</summary>
            MS_isolation_window_attribute = 1000792,

            /// <summary>OBSOLETE The highest m/z being isolated in an isolation window.</summary>
            MS_isolation_window_upper_limit_OBSOLETE = 1000793,

            /// <summary>OBSOLETE The lowest m/z being isolated in an isolation window.</summary>
            MS_isolation_window_lower_limit_OBSOLETE = 1000794,

            /// <summary>Use this term if only one scan was recorded or there is no information about scans available.</summary>
            MS_no_combination = 1000795,

            /// <summary>Free-form text title describing a spectrum, usually a series of key value pairs as used in an MGF file.</summary>
            MS_spectrum_title = 1000796,

            /// <summary>A list of scan numbers and or scan ranges associated with a peak list. If possible the list of scans should be converted to native spectrum identifiers instead of using this term.</summary>
            MS_peak_list_scans = 1000797,

            /// <summary>A list of raw scans and or scan ranges used to generate a peak list. If possible the list of scans should be converted to native spectrum identifiers instead of using this term.</summary>
            MS_peak_list_raw_scans = 1000798,

            /// <summary>A software tool that has not yet been released. The value should describe the software. Please do not use this term for publicly available software - contact the PSI-MS working group in order to have another CV term added.</summary>
            MS_custom_unreleased_software_tool = 1000799,

            /// <summary>The observed mass divided by the difference between two masses that can be separated: m/dm. The procedure by which dm was obtained and the mass at which the measurement was made should be reported.</summary>
            MS_mass_resolving_power = 1000800,

            /// <summary>Spectral peak processing conducted on the acquired data to convert profile data to centroided data. The area defined by all raw data points that belong to the peak is reported.</summary>
            MS_area_peak_picking = 1000801,

            /// <summary>Spectral peak processing conducted on the acquired data to convert profile data to centroided data. The maximum intensity of all raw data points that belong to the peak is reported.</summary>
            MS_height_peak_picking = 1000802,

            /// <summary>Offset between two analyzers in a constant neutral loss or neutral gain scan. The value corresponds to the neutral loss or neutral gain value.</summary>
            MS_analyzer_scan_offset = 1000803,

            /// <summary>A plot of the relative intensity of electromagnetic radiation as a function of the wavelength.</summary>
            MS_electromagnetic_radiation_spectrum = 1000804,

            /// <summary>A plot of the relative intensity of electromagnetic radiation emitted by atoms or molecules when excited.</summary>
            MS_emission_spectrum = 1000805,

            /// <summary>A plot of the relative intensity of electromagnetic radiation absorbed by atoms or molecules when excited.</summary>
            MS_absorption_spectrum = 1000806,

            /// <summary>Unit describing the scan rate of a spectrum in Thomson per second.</summary>
            MS_Th_s = 1000807,

            /// <summary>Chromatogram properties that are associated with a value.</summary>
            MS_chromatogram_attribute = 1000808,

            /// <summary>A free-form text title describing a chromatogram.</summary>
            MS_chromatogram_title = 1000809,

            /// <summary>Representation of the current of ions versus time.</summary>
            MS_ion_current_chromatogram = 1000810,

            /// <summary>Representation of electromagnetic properties versus time.</summary>
            MS_electromagnetic_radiation_chromatogram = 1000811,

            /// <summary>Representation of light absorbed by the sample versus time.</summary>
            MS_absorption_chromatogram = 1000812,

            /// <summary>Representation of light emitted by the sample versus time.</summary>
            MS_emission_chromatogram = 1000813,

            /// <summary>The number of counted events observed per second in one or a group of elements of a detector.</summary>
            MS_counts_per_second = 1000814,

            /// <summary>Bruker BAF raw file format.</summary>
            MS_Bruker_BAF_format = 1000815,

            /// <summary>Bruker HyStar U2 file format.</summary>
            MS_Bruker_U2_format = 1000816,

            /// <summary>Bruker software for hyphenated experiments.</summary>
            MS_HyStar = 1000817,

            /// <summary>Acquity UPLC Photodiode Array Detector.</summary>
            MS_Acquity_UPLC_PDA = 1000818,

            /// <summary>Acquity UPLC Fluorescence Detector.</summary>
            MS_Acquity_UPLC_FLR = 1000819,

            /// <summary>A data array of flow rate measurements.</summary>
            MS_flow_rate_array = 1000820,

            /// <summary>A data array of pressure measurements.</summary>
            MS_pressure_array = 1000821,

            /// <summary>A data array of temperature measurements.</summary>
            MS_temperature_array = 1000822,

            /// <summary>Native format defined by declaration=xsd:nonNegativeInteger collection=xsd:nonNegativeInteger scan=xsd:nonNegativeInteger.</summary>
            MS_Bruker_U2_nativeID_format = 1000823,

            /// <summary>No nativeID format indicates that the file tagged with this term does not contain spectra that can have a nativeID format.</summary>
            MS_no_nativeID_format = 1000824,

            /// <summary>Bruker FID file format.</summary>
            MS_Bruker_FID_format = 1000825,

            /// <summary>The time of elution from all used chromatographic columns (one or more) in the chromatographic separation step, relative to the start of the chromatography.</summary>
            MS_elution_time = 1000826,

            /// <summary>The primary or reference m/z about which the isolation window is defined.</summary>
            MS_isolation_window_target_m_z = 1000827,

            /// <summary>The extent of the isolation window in m/z below the isolation window target m/z. The lower and upper offsets may be asymmetric about the target m/z.</summary>
            MS_isolation_window_lower_offset = 1000828,

            /// <summary>The extent of the isolation window in m/z above the isolation window target m/z. The lower and upper offsets may be asymmetric about the target m/z.</summary>
            MS_isolation_window_upper_offset = 1000829,

            /// <summary>Properties of the preparation steps which took place before the measurement was performed.</summary>
            MS_sample_preparation = 1000831,

            /// <summary>Attributes to describe the technique how the sample is prepared with the matrix solution.</summary>
            MS_MALDI_matrix_application = 1000832,

            /// <summary>Describes the technique how the matrix is put on the sample target.</summary>
            MS_matrix_application_type = 1000833,

            /// <summary>Describes the chemical solution used as matrix.</summary>
            MS_matrix_solution = 1000834,

            /// <summary>Concentration of the chemical solution used as matrix.</summary>
            MS_matrix_solution_concentration = 1000835,

            /// <summary>Dried droplet in MALDI matrix preparation.</summary>
            MS_dried_droplet_MALDI_matrix_preparation = 1000836,

            /// <summary>Printed MALDI matrix preparation.</summary>
            MS_printed_MALDI_matrix_preparation = 1000837,

            /// <summary>Sprayed MALDI matrix preparation.</summary>
            MS_sprayed_MALDI_matrix_preparation = 1000838,

            /// <summary>Precoated MALDI sample plate.</summary>
            MS_precoated_MALDI_sample_plate = 1000839,

            /// <summary>Device that emits light (electromagnetic radiation) through a process called stimulated emission. The term is an acronym for Light Amplification by Stimulated Emission of Radiation.</summary>
            MS_laser = 1000840,

            /// <summary>Laser properties that are associated with a value.</summary>
            MS_laser_attribute = 1000841,

            /// <summary>Type of laser used for desorption purpose.</summary>
            MS_laser_type = 1000842,

            /// <summary>OBSOLETE The distance between two peaks of the emitted laser beam.</summary>
            MS_wavelength_OBSOLETE = 1000843,

            /// <summary>Describes the diameter of the laser beam in x direction.</summary>
            MS_focus_diameter_x = 1000844,

            /// <summary>Describes the diameter of the laser beam in y direction.</summary>
            MS_focus_diameter_y = 1000845,

            /// <summary>Describes output energy of the laser system. May be attenuated by filters or other means.</summary>
            MS_pulse_energy = 1000846,

            /// <summary>Describes how long the laser beam was emitted from the laser device.</summary>
            MS_pulse_duration = 1000847,

            /// <summary>Describes the reduction of the intensity of the laser beam energy.</summary>
            MS_attenuation = 1000848,

            /// <summary>Describes the angle between the laser beam and the sample target.</summary>
            MS_impact_angle = 1000849,

            /// <summary>Laser which is powered by a gaseous medium.</summary>
            MS_gas_laser = 1000850,

            /// <summary>Solid state laser materials are commonly made by doping a crystalline solid host with ions that provide the required energy states.</summary>
            MS_solid_state_laser = 1000851,

            /// <summary>Dye lasers use an organic dye as the gain medium.</summary>
            MS_dye_laser = 1000852,

            /// <summary>Free electron laser uses a relativistic electron beam as the lasing medium which move freely through a magnetic structure, hence the term.</summary>
            MS_free_electron_laser = 1000853,

            /// <summary>Thermo Scientific LTQ XL MS.</summary>
            MS_LTQ_XL = 1000854,

            /// <summary>Thermo Scientific LTQ Velos MS.</summary>
            MS_LTQ_Velos = 1000855,

            /// <summary>Thermo Scientific LTQ Velos MS with ETD.</summary>
            MS_LTQ_Velos_ETD = 1000856,

            /// <summary>Properties of the described run.</summary>
            MS_run_attribute = 1000857,

            /// <summary>Identier string that describes the sample fraction. This identifier should contain the fraction number(s) or similar information.</summary>
            MS_fraction_identifier = 1000858,

            /// <summary>Group of two or more atoms held together by chemical bonds.</summary>
            MS_molecule = 1000859,

            /// <summary>A molecule of low molecular weight that is composed of two or more amino acid residues.</summary>
            MS_peptide = 1000860,

            /// <summary>A physical characteristic of a molecular entity.</summary>
            MS_molecular_entity_property = 1000861,

            /// <summary>The pH of a solution at which a charged molecule does not migrate in an electric field.</summary>
            MS_isoelectric_point = 1000862,

            /// <summary>The pH of a solution at which a charged molecule would not migrate in an electric field, as predicted by a software algorithm.</summary>
            MS_predicted_isoelectric_point = 1000863,

            /// <summary>A combination of symbols used to express the chemical composition of a molecule.</summary>
            MS_chemical_formula = 1000864,

            /// <summary>A chemical formula which expresses the proportions of the elements present in a substance.</summary>
            MS_empirical_formula = 1000865,

            /// <summary>A chemical compound formula expressing the number of atoms of each element present in a compound, without indicating how they are linked.</summary>
            MS_molecular_formula = 1000866,

            /// <summary>A chemical formula showing the number of atoms of each element in a molecule, their spatial arrangement, and their linkage to each other.</summary>
            MS_structural_formula = 1000867,

            /// <summary>The simplified molecular input line entry specification or SMILES is a specification for unambiguously describing the structure of a chemical compound using a short ASCII string.</summary>
            MS_SMILES_formula = 1000868,

            /// <summary>The gas pressure of the collision gas used for collisional excitation.</summary>
            MS_collision_gas_pressure = 1000869,

            /// <summary>OBSOLETE SCIEX or Applied Biosystems|MDS SCIEX QTRAP 4000.</summary>
            MS_4000_QTRAP_OBSOLETE = 1000870,

            /// <summary>Software used to predict, select, or optimize transitions or analyze the results of selected reaction monitoring runs.</summary>
            MS_SRM_software = 1000871,

            /// <summary>Software used to predict transitions for selected reaction monitoring experiments based on observed spectrum libraries developed and distributed by the Institute for Systems Biology.</summary>
            MS_MaRiMba = 1000872,

            /// <summary>Software used to predict or calculate numerical attributes of peptides.</summary>
            MS_peptide_attribute_calculation_software = 1000873,

            /// <summary>Sequence Specific Retention Calculator estimates the retention time of peptides based on their sequence.</summary>
            MS_SSRCalc = 1000874,

            /// <summary>Potential difference between the orifice and the skimmer in volts.</summary>
            MS_declustering_potential = 1000875,

            /// <summary>Potential difference between the sampling cone/orifice in volts.</summary>
            MS_cone_voltage = 1000876,

            /// <summary>Potential difference setting of the tube lens in volts.</summary>
            MS_tube_lens_voltage = 1000877,

            /// <summary>An identifier/accession number to an external reference database.</summary>
            MS_external_reference_identifier = 1000878,

            /// <summary>A unique identifier for a publication in the PubMed database (MIR:00000015).</summary>
            MS_PubMed_identifier = 1000879,

            /// <summary>The duration of intervals between scanning, during which the instrument configuration is switched.</summary>
            MS_interchannel_delay = 1000880,

            /// <summary>Constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity.</summary>
            MS_molecular_entity = 1000881,

            /// <summary>A compound composed of one or more chains of amino acids in a specific order determined by the base sequence of nucleotides in the DNA of a gene.</summary>
            MS_protein = 1000882,

            /// <summary>A short name or symbol of a protein (e.g., HSF 1 or HSF1_HUMAN).</summary>
            MS_protein_short_name = 1000883,

            /// <summary>An nonphysical characterstic attributed to a specific protein.</summary>
            MS_protein_attribute = 1000884,

            /// <summary>Identifier for a specific protein in a database.</summary>
            MS_protein_accession = 1000885,

            /// <summary>A long name describing the function of the protein.</summary>
            MS_protein_name = 1000886,

            /// <summary>Nonphysical characteristic attributed to a peptide.</summary>
            MS_peptide_attribute = 1000887,

            /// <summary>Sequence of letter symbols denoting the order of amino acids that compose the peptide, with any amino acid mass modifications that might be present having been stripped away.</summary>
            MS_stripped_peptide_sequence = 1000888,

            /// <summary>Sequence of letter symbols denoting the order of amino acid residues that compose the peptidoform including the encoding of any residue modifications that are present.</summary>
            MS_peptidoform_sequence = 1000889,

            /// <summary>A state description of how a peptide might be isotopically or isobarically labelled.</summary>
            MS_peptidoform_labeling_state = 1000890,

            /// <summary>A peptide that has been created or labelled with some heavier-than-usual isotopes.</summary>
            MS_heavy_labeled_peptidoform = 1000891,

            /// <summary>A peptide that has not been labelled with heavier-than-usual isotopes. This is often referred to as \ light\  to distinguish from \ heavy\ .</summary>
            MS_unlabeled_peptidoform = 1000892,

            /// <summary>An arbitrary string label used to mark a set of peptides that belong together in a set, whereby the members are differentiated by different isotopic labels. For example, the heavy and light forms of the same peptide will both be assigned the same peptide group label.</summary>
            MS_peptidoform_group_label = 1000893,

            /// <summary>A time interval from the start of chromatography when an analyte exits a chromatographic column.</summary>
            MS_retention_time = 1000894,

            /// <summary>A time interval from the start of chromatography when an analyte exits an unspecified local chromatographic column and instrumental setup.</summary>
            MS_local_retention_time = 1000895,

            /// <summary>A time interval from the start of chromatography when an analyte exits a standardized reference chromatographic column and instrumental setup.</summary>
            MS_normalized_retention_time = 1000896,

            /// <summary>A time interval from the start of chromatography when an analyte exits a chromatographic column as predicted by a referenced software application.</summary>
            MS_predicted_retention_time = 1000897,

            /// <summary>Something, such as a practice or a product, that is widely recognized or employed, especially because of its excellence.</summary>
            MS_standard = 1000898,

            /// <summary>A practice or product that has become a standard not because it has been approved by a standards organization but because it is widely used and recognized by the industry as being standard.</summary>
            MS_de_facto_standard = 1000899,

            /// <summary>A specification of a minimum amount of information needed to reproduce or fully interpret a scientific result.</summary>
            MS_minimum_information_standard = 1000900,

            /// <summary>A standard providing the retention times at which a set of reference compounds exit the reference chromatographic column.</summary>
            MS_retention_time_normalization_standard = 1000901,

            /// <summary>The de facto standard providing the retention times at which a set of halogenated reference peptides exit the reference chromatographic column.</summary>
            MS_H_PINS_retention_time_normalization_standard = 1000902,

            /// <summary>The ordinal of the fragment within a specified ion series. (e.g. 8 for a y8 ion).</summary>
            MS_product_ion_series_ordinal = 1000903,

            /// <summary>The difference in m/z of the predicted m/z based on the assigned product ion minus the actual observed peak m/z.</summary>
            MS_product_ion_m_z_delta = 1000904,

            /// <summary>The magnitude of a peak expressed in terms of the percentage of the magnitude of the base peak intensity multiplied by 100. The base peak is therefore 10000. This unit is common in normalized spectrum libraries.</summary>
            MS_percent_of_base_peak_times_100 = 1000905,

            /// <summary>Ordinal specifying the rank in intensity of a peak in a spectrum. Base peak is 1. The next most intense peak is 2, etc.</summary>
            MS_peak_intensity_rank = 1000906,

            /// <summary>Ordinal specifying the rank of a peak in a spectrum in terms of suitability for targeting. The most suitable peak is 1. The next most suitability peak is 2, etc. Suitability is algorithm and context dependant.</summary>
            MS_peak_targeting_suitability_rank = 1000907,

            /// <summary>A set of two m/z values corresponding to the precursor m/z and a fragment m/z that in combination can be used to identify or quantify a specific ion, although not necessarily uniquely.</summary>
            MS_transition = 1000908,

            /// <summary>The strategy used to validate that a transition is effective.</summary>
            MS_transition_validation_method = 1000909,

            /// <summary>The transition has been optimized by direct injection of the peptide into an instrument specified in a separate term, and the optimum voltages and fragmentation energies have been determined.</summary>
            MS_transition_optimized_on_specified_instrument = 1000910,

            /// <summary>The transition has been validated by obtaining an MS2 spectrum and demonstrating that the peak is detectable on the instrument specified with a separate term.</summary>
            MS_transition_validated_with_an_MS_MS_spectrum_on_specified_instrument = 1000911,

            /// <summary>The transition has been purported by obtaining an MS2 spectrum and demonstrating that the peak is detectable on the instrument specified with a separate term. However, the detecting instrument is of a different type (e.g. ion trap) than the instrument that the transition will eventually be used on (e.g. triple quad).</summary>
            MS_transition_purported_from_an_MS_MS_spectrum_on_a_different__specified_instrument = 1000912,

            /// <summary>The transition has been predicted by informatics software without any direct spectral evidence.</summary>
            MS_transition_predicted_by_informatic_analysis = 1000913,

            /// <summary>A file format that has two or more columns of tabular data where each column is separated by a TAB character.</summary>
            MS_tab_delimited_text_format = 1000914,

            /// <summary>An attribute of a window in time about which a peptide might elute from the column.</summary>
            MS_retention_time_window_attribute = 1000915,

            /// <summary>The extent of the retention time window in time units below the target retention time. The lower and upper offsets may be asymmetric about the target time.</summary>
            MS_retention_time_window_lower_offset = 1000916,

            /// <summary>The extent of the retention time window in time units above the target retention time. The lower and upper offsets may be asymmetric about the target time.</summary>
            MS_retention_time_window_upper_offset = 1000917,

            /// <summary>A list of peptides or compounds and their expected m/z coordinates that can be used to cause a mass spectrometry to obtain spectra of those molecules specifically.</summary>
            MS_target_list = 1000918,

            /// <summary>A priority setting specifying whether included or excluded targets have priority over the other.</summary>
            MS_target_inclusion_exclusion_priority = 1000919,

            /// <summary>A priority setting specifying that included targets have priority over the excluded targets if there is a conflict.</summary>
            MS_includes_supersede_excludes = 1000920,

            /// <summary>A priority setting specifying that excluded targets have priority over the included targets if there is a conflict.</summary>
            MS_excludes_supersede_includes = 1000921,

            /// <summary>Software used to predict, select, and optimize transitions as well as analyze the results of selected reaction monitoring runs developed and distributed by the MacCoss lab at the University of Washington.</summary>
            MS_Skyline = 1000922,

            /// <summary>Software used to predict, select, and optimize transitions for selected reaction monitoring experiments developed and distributed by the Institute for Systems Biology.</summary>
            MS_TIQAM = 1000923,

            /// <summary>OBSOLETE Software used to predict transitions for selected reaction monitoring experiments based on observed spectrum libraries developed and distributed by the Institute for Systems Biology.</summary>
            MS_MaRiMba_OBSOLETE = 1000924,

            /// <summary>Software suite used to predict, select, and optimize transitions as well as analyze the results of selected reaction monitoring runs developed and distributed by the Institute for Systems Biology.</summary>
            MS_ATAQS = 1000925,

            /// <summary>The integer rank given an interpretation of an observed product ion. For example, if y8 is selected as the most likely interpretation of a peak, then it is assigned a rank of 1.</summary>
            MS_product_interpretation_rank = 1000926,

            /// <summary>The length of time spent filling an ion trapping device.</summary>
            MS_ion_injection_time = 1000927,

            /// <summary>A spectrum derived from a special calibration source, rather than from the primary injected sample. A calibration spectrum is typically derived from a substance that can be used to correct systematic shift in m/z for spectra of the primary inject sample.</summary>
            MS_calibration_spectrum = 1000928,

            /// <summary>Native format defined by source=xsd:string start=xsd:nonNegativeInteger end=xsd:nonNegativeInteger.</summary>
            MS_Shimadzu_Biotech_nativeID_format = 1000929,

            /// <summary>Shimadzu Biotech format.</summary>
            MS_Shimadzu_Biotech_database_entity = 1000930,

            /// <summary>Applied Biosystems|MDS SCIEX QTRAP 5500.</summary>
            MS_QTRAP_5500 = 1000931,

            /// <summary>SCIEX TripleTOF 5600, a quadrupole - quadrupole - time-of-flight mass spectrometer.</summary>
            MS_TripleTOF_5600 = 1000932,

            /// <summary>Encoding of modifications of the protein sequence from the specified accession, written in PEFF notation.</summary>
            MS_protein_modifications = 1000933,

            /// <summary>Name of the gene from which the protein is translated.</summary>
            MS_gene_name = 1000934,

            /// <summary>The 6470A Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_6470A_Triple_Quadrupole_LC_MS = 1000935,

            /// <summary>The 6470B Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_6470B_Triple_Quadrupole_LC_MS = 1000936,

            /// <summary>The 6495C Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_6495C_Triple_Quadrupole_LC_MS = 1000937,

            /// <summary>Collection of terms from the PSI Proteome Informatics standards describing the interpretation of spectra.</summary>
            MS_spectrum_interpretation = 1001000,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_CleavesAt = 1001005,

            /// <summary>SEQUEST View Input Parameters.</summary>
            MS_SEQUEST_ViewCV = 1001006,

            /// <summary>Number of peptide results to show.</summary>
            MS_SEQUEST_OutputLines = 1001007,

            /// <summary>Number of full protein descriptions to show for top N peptides.</summary>
            MS_SEQUEST_DescriptionLines = 1001009,

            /// <summary>A de novo sequencing search (without database).</summary>
            MS_de_novo_search = 1001010,

            /// <summary>Details about the database searched.</summary>
            MS_search_database_details = 1001011,

            /// <summary>The organisation, project or laboratory from where the database is obtained (UniProt, NCBI, EBI, other).</summary>
            MS_database_source = 1001012,

            /// <summary>The name of the search database (nr, SwissProt or est_human).</summary>
            MS_database_name = 1001013,

            /// <summary>OBSOLETE: Use attribute in mzIdentML instead. Local file path of the search database from the search engine's point of view.</summary>
            MS_database_local_file_path_OBSOLETE = 1001014,

            /// <summary>URI, from where the search database was originally downloaded.</summary>
            MS_database_original_uri = 1001015,

            /// <summary>Version of the search database. In mzIdentML use the attribute instead.</summary>
            MS_database_version = 1001016,

            /// <summary>Date and time at which a product was publicly released. For mzIdentML, use the database release date XML attribute instead of this term.</summary>
            MS_release_date = 1001017,

            /// <summary>Database containing amino acid or nucleic acid sequences.</summary>
            MS_database_type = 1001018,

            /// <summary>Was there filtering used on the database.</summary>
            MS_database_filtering = 1001019,

            /// <summary>A taxonomy filter was to the database search.</summary>
            MS_DB_filter_taxonomy = 1001020,

            /// <summary>Filtering applied specifically by accession number pattern.</summary>
            MS_DB_filter_on_accession_numbers = 1001021,

            /// <summary>Filtering applied specifically by protein molecular weight, specified as either a range or above/below a threshold value.</summary>
            MS_DB_MW_filter = 1001022,

            /// <summary>Filtering applied specifically by predicted protein isoelectric focussing point (pI), specified as either a range or above/below a threshold value.</summary>
            MS_DB_PI_filter = 1001023,

            /// <summary>The translated open reading frames from a nucleotide database considered in the search (range: 1-6).</summary>
            MS_translation_frame = 1001024,

            /// <summary>The translation table used to translate the nucleotides to amino acids.</summary>
            MS_translation_table = 1001025,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_NormalizeXCorrValues = 1001026,

            /// <summary>Filtering applied specifically by amino acid sequence pattern.</summary>
            MS_DB_filter_on_sequence_pattern = 1001027,

            /// <summary>String in the header of a sequence entry for that entry to be searched.</summary>
            MS_SEQUEST_SequenceHeaderFilter = 1001028,

            /// <summary>The number of sequences (proteins / nucleotides) from the database search after filtering.</summary>
            MS_number_of_sequences_searched = 1001029,

            /// <summary>Number of peptide seqs compared to each spectrum.</summary>
            MS_number_of_peptide_seqs_compared_to_each_spectrum = 1001030,

            /// <summary>A search using a library of spectra.</summary>
            MS_spectral_library_search = 1001031,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_SequencePartialFilter = 1001032,

            /// <summary>OBSOLETE: use attribute in mzIdentML instead. Date and time of the actual search run.</summary>
            MS_date___time_search_performed_OBSOLETE = 1001035,

            /// <summary>The time taken to complete the search in seconds.</summary>
            MS_search_time_taken = 1001036,

            /// <summary>Flag indicating that fragment ions should be shown.</summary>
            MS_SEQUEST_ShowFragmentIons = 1001037,

            /// <summary>Specify depth as value of the CVParam.</summary>
            MS_SEQUEST_Consensus = 1001038,

            /// <summary>Type of the source file, the mzIdentML was created from.</summary>
            MS_intermediate_analysis_format = 1001040,

            /// <summary>SEQUEST View / Sort Input Parameters.</summary>
            MS_SEQUEST_sortCV = 1001041,

            /// <summary>Specify \ number of dtas shown\  as value of the CVParam.</summary>
            MS_SEQUEST_LimitTo = 1001042,

            /// <summary>Details of cleavage agent (enzyme).</summary>
            MS_cleavage_agent_details = 1001044,

            /// <summary>The name of the cleavage agent.</summary>
            MS_cleavage_agent_name = 1001045,

            /// <summary>Sort order of SEQUEST search results by the delta of the normalized correlation score.</summary>
            MS_SEQUEST_sort_by_dCn = 1001046,

            /// <summary>Sort order of SEQUEST search results by the difference between a theoretically calculated and the corresponding experimentally measured molecular mass M.</summary>
            MS_SEQUEST_sort_by_dM = 1001047,

            /// <summary>Sort order of SEQUEST search results given by the ions.</summary>
            MS_SEQUEST_sort_by_Ions = 1001048,

            /// <summary>Sort order of SEQUEST search results given by the mass of the protonated ion.</summary>
            MS_SEQUEST_sort_by_MH_ = 1001049,

            /// <summary>Sort order of SEQUEST search results given by the probability.</summary>
            MS_SEQUEST_sort_by_P = 1001050,

            /// <summary>OBSOLETE: use attribute independent in mzIdentML instead. Description of multiple enzyme digestion protocol, if any.</summary>
            MS_multiple_enzyme_combination_rules_OBSOLETE = 1001051,

            /// <summary>Sort order of SEQUEST search results given by the previous amino acid.</summary>
            MS_SEQUEST_sort_by_PreviousAminoAcid = 1001052,

            /// <summary>Sort order of SEQUEST search results given by the reference.</summary>
            MS_SEQUEST_sort_by_Ref = 1001053,

            /// <summary>Modification parameters for the search engine run.</summary>
            MS_modification_parameters = 1001055,

            /// <summary>The specificity rules for the modifications applied by the search engine.</summary>
            MS_modification_specificity_rule = 1001056,

            /// <summary>OBSOLETE: Tolerance on types.</summary>
            MS_tolerance_on_types_OBSOLETE = 1001057,

            /// <summary>The quality estimation was done manually.</summary>
            MS_quality_estimation_by_manual_validation = 1001058,

            /// <summary>Sort order of SEQUEST search results given by the result 'Sp' of 'Rank/Sp' in the out file (peptide).</summary>
            MS_SEQUEST_sort_by_RSp = 1001059,

            /// <summary>Method for quality estimation (manually or with decoy database).</summary>
            MS_quality_estimation_method_details = 1001060,

            /// <summary>OBSOLETE: replaced by MS:1000336 (neutral loss): Leave this to PSI-MOD.</summary>
            MS_neutral_loss_OBSOLETE = 1001061,

            /// <summary>Mascot MGF file format.</summary>
            MS_Mascot_MGF_format = 1001062,

            /// <summary>OBSOLETE: There is Phenyx:ScoringModel for Phenyx! Scoring model (more detailed granularity). TODO: add some child terms.</summary>
            MS_TODOscoring_model_OBSOLETE = 1001065,

            /// <summary>The description of the DEPRECATED ion fragment series (including charges and neutral losses) that are considered by the search engine.</summary>
            MS_ions_series_considered_in_search = 1001066,

            /// <summary>Sort order of SEQUEST search results by the Sp score.</summary>
            MS_SEQUEST_sort_by_Sp = 1001068,

            /// <summary>Sort order of SEQUEST search results given by the total ion current.</summary>
            MS_SEQUEST_sort_by_TIC = 1001069,

            /// <summary>Sort order of SEQUEST search results given by the scan number.</summary>
            MS_SEQUEST_sort_by_Scan = 1001070,

            /// <summary>Sort order of SEQUEST search results given by the sequence.</summary>
            MS_SEQUEST_sort_by_Sequence = 1001071,

            /// <summary>Sort order of SEQUEST search results given by the SEQUEST result 'Sf'.</summary>
            MS_SEQUEST_sort_by_Sf = 1001072,

            /// <summary>Database contains amino acid sequences.</summary>
            MS_database_type_amino_acid = 1001073,

            /// <summary>Database contains nucleic acid sequences.</summary>
            MS_database_type_nucleotide = 1001079,

            /// <summary>Enumeration of type of search value (i.e. from PMF, sequence tag, MS2).</summary>
            MS_search_type = 1001080,

            /// <summary>A peptide mass fingerprint search.</summary>
            MS_pmf_search = 1001081,

            /// <summary>A sequence tag search.</summary>
            MS_tag_search = 1001082,

            /// <summary>An MS2 search (with fragment ions).</summary>
            MS_ms_ms_search = 1001083,

            /// <summary>Non-redundant GenBank sequence database.</summary>
            MS_database_nr = 1001084,

            /// <summary>Protein level information.</summary>
            MS_protein_level_identification_attribute = 1001085,

            /// <summary>Sort order of SEQUEST search results by the correlation score.</summary>
            MS_SEQUEST_sort_by_XCorr = 1001086,

            /// <summary>SEQUEST View / Process Input Parameters.</summary>
            MS_SEQUEST_ProcessCV = 1001087,

            /// <summary>The protein description line from the sequence entry in the source database FASTA file.</summary>
            MS_protein_description = 1001088,

            /// <summary>The taxonomy of the resultant molecule from the search.</summary>
            MS_molecule_taxonomy = 1001089,

            /// <summary>OBSOLETE: The system used to indicate taxonomy. There should be an enumerated list of options: latin name, NCBI TaxID, common name, Swiss-Prot species ID (ex. RABIT from the full protein ID ALBU_RABIT).</summary>
            MS_taxonomy_nomenclature_OBSOLETE = 1001090,

            /// <summary>Description not provided</summary>
            MS_NoEnzyme_OBSOLETE = 1001091,

            /// <summary>Identification confidence metric for a peptide.</summary>
            MS_peptide_sequence_level_identification_statistic = 1001092,

            /// <summary>The percent coverage for the protein based upon the matched peptide sequences (can be calculated).</summary>
            MS_sequence_coverage = 1001093,

            /// <summary>Sort order of SEQUEST search results given by the charge.</summary>
            MS_SEQUEST_sort_by_z = 1001094,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_ProcessAll = 1001095,

            /// <summary>Specify \ percentage\  as value of the CVParam.</summary>
            MS_SEQUEST_TopPercentMostIntense = 1001096,

            /// <summary>This counts distinct sequences hitting the protein without regard to a minimal confidence threshold.</summary>
            MS_distinct_peptide_sequences = 1001097,

            /// <summary>This counts the number of distinct peptide sequences. Multiple charge states and multiple modification states do NOT count as multiple sequences. The definition of 'confident' must be qualified elsewhere.</summary>
            MS_confident_distinct_peptide_sequences = 1001098,

            /// <summary>The point of this entry is to define what is meant by confident for the term Confident distinct peptide sequence and/or Confident peptides. Example 1 - metric=Paragon:Confidence value=95 sense=greater than Example 2 - metric=Mascot:Eval value=0.05 sense=less than.</summary>
            MS_confident_peptide_qualification = 1001099,

            /// <summary>This counts the number of peptide sequences without regard to whether they are distinct. Multiple charges states and multiple modification states DO count as multiple peptides. The definition of 'confident' must be qualified elsewhere.</summary>
            MS_confident_peptide_sequence_number = 1001100,

            /// <summary>Protein group or subset relationships.</summary>
            MS_protein_group_or_subset_relationship = 1001101,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_Chromatogram = 1001102,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_InfoAndLog = 1001103,

            /// <summary>The name of the UniProtKB/Swiss-Prot knowledgebase.</summary>
            MS_database_UniProtKB_Swiss_Prot = 1001104,

            /// <summary>Peptide level information.</summary>
            MS_peptide_sequence_level_identification_attribute = 1001105,

            /// <summary>Specify \ number\  as value of the CVParam.</summary>
            MS_SEQUEST_TopNumber = 1001106,

            /// <summary>Source file for this mzIdentML was a data set in a database.</summary>
            MS_data_stored_in_database = 1001107,

            /// <summary>Parameter information, type of product: a ion with charge on the N-terminal side.</summary>
            MS_param__a_ion = 1001108,

            /// <summary>Specify cull string as value of the CVParam.</summary>
            MS_SEQUEST_CullTo = 1001109,

            /// <summary>SEQUEST Mode Input Parameters.</summary>
            MS_SEQUEST_modeCV = 1001110,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_Full = 1001111,

            /// <summary>Residue preceding the first amino acid in the peptide sequence as it occurs in the protein. Use 'N-term' to denote if the peptide starts at the N terminus of the protein.</summary>
            MS_n_terminal_flanking_residue = 1001112,

            /// <summary>Residue following the last amino acid in the peptide sequence as it occurs in the protein. Use 'C-term' to denote if the peptide ends at the C terminus of the protein.</summary>
            MS_c_terminal_flanking_residue = 1001113,

            /// <summary>OBSOLETE Retention time of the spectrum from the source file.</summary>
            MS_retention_time_s__OBSOLETE = 1001114,

            /// <summary>OBSOLETE: use spectrumID attribute of SpectrumIdentificationResult. Take from mzData.</summary>
            MS_scan_number_s__OBSOLETE = 1001115,

            /// <summary>Results specific for one protein as part of a protein ambiguity group (a result not valid for all the other proteins in the protein ambiguity group).</summary>
            MS_single_protein_identification_statistic = 1001116,

            /// <summary>The theoretical neutral mass of the molecule (e.g. the peptide sequence and its modifications) not including its charge carrier.</summary>
            MS_theoretical_mass = 1001117,

            /// <summary>Parameter information, type of product: b ion with charge on the N-terminal side.</summary>
            MS_param__b_ion = 1001118,

            /// <summary>Parameter information, type of product: c ion with charge on the N-terminal side.</summary>
            MS_param__c_ion = 1001119,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_FormatAndLinks = 1001120,

            /// <summary>The number of peaks that were matched as qualified by the ion series considered field. If a peak matches multiple ions then only 1 would be added the count.</summary>
            MS_number_of_matched_peaks = 1001121,

            /// <summary>The ion series that were used during the calculation of the count (e.g. a, b, c, d, v, w, x, y, z, a-H2O, a-NH3, b-H2O, b-NH3, y-H2O, y-NH3, b-H20, b+, z-, z+1, z+2, b-H3PO4, y-H3PO4, immonium, internal ya, internal yb).</summary>
            MS_ions_series_considered = 1001122,

            /// <summary>The number of peaks from the original peak list that are used to calculate the scores for a particular search engine. All ions that have the opportunity to match or be counted even if they don't.</summary>
            MS_number_of_peaks_used = 1001123,

            /// <summary>The number of peaks from the original peaks listed that were submitted to the search engine.</summary>
            MS_number_of_peaks_submitted = 1001124,

            /// <summary>Result of quality estimation: decision of a manual validation.</summary>
            MS_manual_validation = 1001125,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_Fast = 1001126,

            /// <summary>Accessions Containing Sequence - Accessions for each protein containing this peptide.</summary>
            MS_peptide_sharing_details = 1001127,

            /// <summary>SEQUEST Select Input Parameters.</summary>
            MS_SEQUEST_selectCV = 1001128,

            /// <summary>Quantification information.</summary>
            MS_quantification_information = 1001129,

            /// <summary>OBSOLETE Peptide raw area.</summary>
            MS_peptide_raw_area_OBSOLETE = 1001130,

            /// <summary>Error on peptide area.</summary>
            MS_error_on_peptide_area = 1001131,

            /// <summary>Peptide ratio.</summary>
            MS_peptide_ratio = 1001132,

            /// <summary>Error on peptide ratio.</summary>
            MS_error_on_peptide_ratio = 1001133,

            /// <summary>Protein ratio.</summary>
            MS_protein_ratio = 1001134,

            /// <summary>Error on protein ratio.</summary>
            MS_error_on_protein_ratio = 1001135,

            /// <summary>OBSOLETE P-value (protein diff from 1 randomly).</summary>
            MS_p_value__protein_diff_from_1_randomly__OBSOLETE = 1001136,

            /// <summary>Absolute quantity in terms of real concentration or molecule copy number in sample.</summary>
            MS_absolute_quantity = 1001137,

            /// <summary>Error on absolute quantity.</summary>
            MS_error_on_absolute_quantity = 1001138,

            /// <summary>Quantitation software name.</summary>
            MS_quantitation_software_name = 1001139,

            /// <summary>OBSOLETE Quantitation software version.</summary>
            MS_quantitation_software_version_OBSOLETE = 1001140,

            /// <summary>The intensity of the precursor ion.</summary>
            MS_intensity_of_precursor_ion = 1001141,

            /// <summary>International Protein Index database for Homo sapiens sequences.</summary>
            MS_database_IPI_human = 1001142,

            /// <summary>Search engine specific peptide spectrum match scores.</summary>
            MS_PSM_level_search_engine_specific_statistic = 1001143,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_SelectDefault = 1001144,

            /// <summary>SEQUEST Select Advanced Input Parameters.</summary>
            MS_SEQUEST_SelectAdvancedCV = 1001145,

            /// <summary>Ion a-NH3 parameter information, type of product: a ion with lost ammonia.</summary>
            MS_param__a_ion_NH3_DEPRECATED = 1001146,

            /// <summary>Description not provided</summary>
            MS_protein_ambiguity_group_result_details = 1001147,

            /// <summary>Ion a-H2O if a significant and fragment includes STED.</summary>
            MS_param__a_ion_H2O_DEPRECATED = 1001148,

            /// <summary>Ion b-NH3 parameter information, type of product: b ion with lost ammonia.</summary>
            MS_param__b_ion_NH3_DEPRECATED = 1001149,

            /// <summary>Ion b-H2O if b significant and fragment includes STED.</summary>
            MS_param__b_ion_H2O_DEPRECATED = 1001150,

            /// <summary>Ion y-NH3 parameter information, type of product: y ion with lost ammonia.</summary>
            MS_param__y_ion_NH3_DEPRECATED = 1001151,

            /// <summary>Description not provided</summary>
            MS_param__y_ion_H2O_DEPRECATED = 1001152,

            /// <summary>Search engine specific scores.</summary>
            MS_search_engine_specific_score = 1001153,

            /// <summary>The SEQUEST result 'Probability'.</summary>
            MS_SEQUEST_probability = 1001154,

            /// <summary>The SEQUEST result 'XCorr'.</summary>
            MS_SEQUEST_xcorr = 1001155,

            /// <summary>The SEQUEST result 'DeltaCn'.</summary>
            MS_SEQUEST_deltacn = 1001156,

            /// <summary>The SEQUEST result 'Sp' (protein).</summary>
            MS_SEQUEST_sp = 1001157,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_Uniq = 1001158,

            /// <summary>The SEQUEST result 'Expectation value'.</summary>
            MS_SEQUEST_expectation_value = 1001159,

            /// <summary>The SEQUEST result 'Sf'.</summary>
            MS_SEQUEST_sf = 1001160,

            /// <summary>The SEQUEST result 'Matched Ions'.</summary>
            MS_SEQUEST_matched_ions = 1001161,

            /// <summary>The SEQUEST result 'Total Ions'.</summary>
            MS_SEQUEST_total_ions = 1001162,

            /// <summary>The SEQUEST result 'Consensus Score'.</summary>
            MS_SEQUEST_consensus_score = 1001163,

            /// <summary>The Paragon result 'Unused ProtScore'.</summary>
            MS_Paragon_unused_protscore = 1001164,

            /// <summary>The Paragon result 'Total ProtScore'.</summary>
            MS_Paragon_total_protscore = 1001165,

            /// <summary>The Paragon result 'Score'.</summary>
            MS_Paragon_score = 1001166,

            /// <summary>The Paragon result 'Confidence'.</summary>
            MS_Paragon_confidence = 1001167,

            /// <summary>The Paragon result 'Expression Error Factor'.</summary>
            MS_Paragon_expression_error_factor = 1001168,

            /// <summary>The Paragon result 'Expression change P-value'.</summary>
            MS_Paragon_expression_change_p_value = 1001169,

            /// <summary>The Paragon result 'Contrib'.</summary>
            MS_Paragon_contrib = 1001170,

            /// <summary>The Mascot result 'Score'.</summary>
            MS_Mascot_score = 1001171,

            /// <summary>The Mascot result 'expectation value'.</summary>
            MS_Mascot_expectation_value = 1001172,

            /// <summary>The Mascot result 'Matched ions'.</summary>
            MS_Mascot_matched_ions = 1001173,

            /// <summary>The Mascot result 'Total ions'.</summary>
            MS_Mascot_total_ions = 1001174,

            /// <summary>A peptide matching multiple proteins.</summary>
            MS_peptide_shared_in_multiple_proteins = 1001175,

            /// <summary>Regular expression for Trypsin.</summary>
            MS______KR______P_ = 1001176,

            /// <summary>Number of Molecular Hypothesis Considered - This is the number of molecules (e.g. peptides for proteomics) considered for a particular search.</summary>
            MS_number_of_molecular_hypothesis_considered = 1001177,

            /// <summary>Expressed sequence tag nucleotide sequence database.</summary>
            MS_database_EST = 1001178,

            /// <summary>Regular expressions for cleavage enzymes.</summary>
            MS_Cleavage_agent_regular_expression = 1001180,

            /// <summary>The details of the actual run of the search.</summary>
            MS_search_statistics = 1001184,

            /// <summary>As parameter for search engine: apply the modification only at the N-terminus of a peptide.</summary>
            MS_modification_specificity_peptide_N_term = 1001189,

            /// <summary>As parameter for search engine: apply the modification only at the C-terminus of a peptide.</summary>
            MS_modification_specificity_peptide_C_term = 1001190,

            /// <summary>OBSOLETE Quality estimation by p-value.</summary>
            MS_p_value_OBSOLETE = 1001191,

            /// <summary>Result of quality estimation: Expect value.</summary>
            MS_Expect_value = 1001192,

            /// <summary>Result of quality estimation: confidence score.</summary>
            MS_confidence_score = 1001193,

            /// <summary>Quality estimation by decoy database.</summary>
            MS_quality_estimation_with_decoy_database = 1001194,

            /// <summary>Decoy type: Amino acids of protein sequences are used in reverse order.</summary>
            MS_decoy_DB_type_reverse = 1001195,

            /// <summary>Decoy type: Amino acids of protein sequences are randomized (keeping the original protein mass).</summary>
            MS_decoy_DB_type_randomized = 1001196,

            /// <summary>Decoy database composition: database contains original (target) and decoy entries.</summary>
            MS_DB_composition_target_decoy = 1001197,

            /// <summary>Identification confidence metric for a protein.</summary>
            MS_protein_identification_confidence_metric = 1001198,

            /// <summary>Source file for this mzIdentML was in Mascot DAT file format.</summary>
            MS_Mascot_DAT_format = 1001199,

            /// <summary>Source file for this mzIdentML was in SEQUEST out file format.</summary>
            MS_SEQUEST_out_file_format = 1001200,

            /// <summary>Maximum value of molecular weight filter.</summary>
            MS_DB_MW_filter_maximum = 1001201,

            /// <summary>Minimum value of molecular weight filter.</summary>
            MS_DB_MW_filter_minimum = 1001202,

            /// <summary>Maximum value of isoelectric point filter.</summary>
            MS_DB_PI_filter_maximum = 1001203,

            /// <summary>Minimum value of isoelectric point filter.</summary>
            MS_DB_PI_filter_minimum = 1001204,

            /// <summary>The name of the Mascot search engine.</summary>
            MS_Mascot = 1001207,

            /// <summary>The name of the SEQUEST search engine.</summary>
            MS_SEQUEST = 1001208,

            /// <summary>The name of the Phenyx search engine.</summary>
            MS_Phenyx = 1001209,

            /// <summary>The type of mass difference value to be considered by the search engine (monoisotopic or average).</summary>
            MS_mass_type_settings = 1001210,

            /// <summary>Mass type setting for parent mass was monoisotopic.</summary>
            MS_parent_mass_type_mono = 1001211,

            /// <summary>Mass type setting for parent mass was average isotopic.</summary>
            MS_parent_mass_type_average = 1001212,

            /// <summary>OBSOLETE: Scores and global result characteristics.</summary>
            MS_search_result_details_OBSOLETE = 1001213,

            /// <summary>Estimation of the global false discovery rate of proteins.</summary>
            MS_protein_level_global_FDR = 1001214,

            /// <summary>The SEQUEST result 'Sp' in out file (peptide).</summary>
            MS_SEQUEST_PeptideSp = 1001215,

            /// <summary>The SEQUEST result 'Sp' of 'Rank/Sp' in out file (peptide). Also called 'rsp'.</summary>
            MS_SEQUEST_PeptideRankSp = 1001217,

            /// <summary>The SEQUEST result '#' in out file (peptide).</summary>
            MS_SEQUEST_PeptideNumber = 1001218,

            /// <summary>The SEQUEST result 'Id#' in out file (peptide).</summary>
            MS_SEQUEST_PeptideIdnumber = 1001219,

            /// <summary>Fragmentation information, type of product: y ion.</summary>
            MS_frag__y_ion = 1001220,

            /// <summary>Fragmentation information like ion types.</summary>
            MS_product_ion_attribute = 1001221,

            /// <summary>Fragmentation information, type of product: b ion without water.</summary>
            MS_frag__b_ion___H2O = 1001222,

            /// <summary>Fragmentation information, type of product: y ion without water.</summary>
            MS_frag__y_ion___H2O = 1001223,

            /// <summary>Fragmentation information, type of product: b ion.</summary>
            MS_frag__b_ion = 1001224,

            /// <summary>The m/z of the product ion.</summary>
            MS_product_ion_m_z = 1001225,

            /// <summary>The intensity of a single product ion.</summary>
            MS_product_ion_intensity = 1001226,

            /// <summary>The product ion m/z error.</summary>
            MS_product_ion_m_z_error = 1001227,

            /// <summary>Fragmentation information, type of product: x ion.</summary>
            MS_frag__x_ion = 1001228,

            /// <summary>Fragmentation information, type of product: a ion.</summary>
            MS_frag__a_ion = 1001229,

            /// <summary>Fragmentation information, type of product: z ion.</summary>
            MS_frag__z_ion = 1001230,

            /// <summary>Fragmentation information, type of product: c ion.</summary>
            MS_frag__c_ion = 1001231,

            /// <summary>Ion b-NH3 fragmentation information, type of product: b ion without ammonia.</summary>
            MS_frag__b_ion___NH3 = 1001232,

            /// <summary>Ion y-NH3 fragmentation information, type of product: y ion without ammonia.</summary>
            MS_frag__y_ion___NH3 = 1001233,

            /// <summary>Fragmentation information, type of product: a ion without water.</summary>
            MS_frag__a_ion___H2O = 1001234,

            /// <summary>Ion a-NH3 fragmentation information, type of product: a ion without ammonia.</summary>
            MS_frag__a_ion___NH3 = 1001235,

            /// <summary>Fragmentation information, type of product: d ion.</summary>
            MS_frag__d_ion = 1001236,

            /// <summary>Fragmentation information, type of product: v ion.</summary>
            MS_frag__v_ion = 1001237,

            /// <summary>Fragmentation information, type of product: w ion.</summary>
            MS_frag__w_ion = 1001238,

            /// <summary>Fragmentation information, type of product: immonium ion.</summary>
            MS_frag__immonium_ion = 1001239,

            /// <summary>Non-identified ion.</summary>
            MS_non_identified_ion = 1001240,

            /// <summary>Co-eluting ion.</summary>
            MS_co_eluting_ion = 1001241,

            /// <summary>Source file for this mzIdentML was a SEQUEST folder with its out files.</summary>
            MS_SEQUEST_out_folder = 1001242,

            /// <summary>Source file for this mzIdentML was a SEQUEST summary page (proteins).</summary>
            MS_SEQUEST_summary = 1001243,

            /// <summary>PerSeptive peak list file format.</summary>
            MS_PerSeptive_PKS_format = 1001245,

            /// <summary>PE SCIEX peak list file format.</summary>
            MS_SCIEX_API_III_format = 1001246,

            /// <summary>Bruker data exchange XML format.</summary>
            MS_Bruker_XML_format = 1001247,

            /// <summary>Details describing the search input.</summary>
            MS_search_input_details = 1001249,

            /// <summary>Result of quality estimation: the local FDR at the current position of a sorted list.</summary>
            MS_local_FDR = 1001250,

            /// <summary>Enzyme trypsin.</summary>
            MS_Trypsin = 1001251,

            /// <summary>Database source EBI.</summary>
            MS_DB_source_EBI = 1001252,

            /// <summary>Database source NCBI.</summary>
            MS_DB_source_NCBI = 1001253,

            /// <summary>Database source UniProt.</summary>
            MS_DB_source_UniProt = 1001254,

            /// <summary>Mass type setting for fragment mass was average isotopic.</summary>
            MS_fragment_mass_type_average = 1001255,

            /// <summary>Mass type setting for fragment mass was monoisotopic.</summary>
            MS_fragment_mass_type_mono = 1001256,

            /// <summary>Parameter information, type of product: side chain loss v ion.</summary>
            MS_param__v_ion = 1001257,

            /// <summary>Parameter information, type of product: side chain loss d ion.</summary>
            MS_param__d_ion = 1001258,

            /// <summary>Parameter information, type of product: immonium ion.</summary>
            MS_param__immonium_ion = 1001259,

            /// <summary>Parameter information, type of product: side chain loss w ion.</summary>
            MS_param__w_ion = 1001260,

            /// <summary>Parameter information, type of product: x ion with charge on the C-terminal side.</summary>
            MS_param__x_ion = 1001261,

            /// <summary>Parameter information, type of product: y ion with charge on the C-terminal side.</summary>
            MS_param__y_ion = 1001262,

            /// <summary>Parameter information, type of product: z ion with charge on the C-terminal side.</summary>
            MS_param__z_ion = 1001263,

            /// <summary>Role of a Person or Organization.</summary>
            MS_role_type = 1001266,

            /// <summary>Software vendor role.</summary>
            MS_software_vendor = 1001267,

            /// <summary>Programmer role.</summary>
            MS_programmer = 1001268,

            /// <summary>Instrument vendor role.</summary>
            MS_instrument_vendor = 1001269,

            /// <summary>Lab personnel role.</summary>
            MS_lab_personnel = 1001270,

            /// <summary>Researcher role.</summary>
            MS_researcher = 1001271,

            /// <summary>Regular expression for Arg-C.</summary>
            MS_____R_____P_ = 1001272,

            /// <summary>Regular expression for Asp-N.</summary>
            MS_____BD__ = 1001273,

            /// <summary>Regular expression for Asp-N-ambic.</summary>
            MS_____DE__ = 1001274,

            /// <summary>Source data for this mzIdentML was a ProteinScape SearchEvent.</summary>
            MS_ProteinScape_SearchEvent = 1001275,

            /// <summary>Source data for this mzIdentML was a ProteinScape Gel.</summary>
            MS_ProteinScape_Gel = 1001276,

            /// <summary>Specify the regular expression for decoy accession numbers.</summary>
            MS_decoy_DB_accession_regexp = 1001283,

            /// <summary>OBSOLETE The name of the database, the search database was derived from.</summary>
            MS_decoy_DB_derived_from_OBSOLETE = 1001284,

            /// <summary>International Protein Index database for Mus musculus sequences.</summary>
            MS_database_IPI_mouse = 1001285,

            /// <summary>International Protein Index database for Rattus norvegicus sequences.</summary>
            MS_database_IPI_rat = 1001286,

            /// <summary>International Protein Index database for Danio rerio sequences.</summary>
            MS_database_IPI_zebrafish = 1001287,

            /// <summary>International Protein Index database for Gallus gallus sequences.</summary>
            MS_database_IPI_chicken = 1001288,

            /// <summary>International Protein Index database for Bos taurus sequences.</summary>
            MS_database_IPI_cow = 1001289,

            /// <summary>International Protein Index database for Arabidopsis thaliana sequences.</summary>
            MS_database_IPI_arabidopsis = 1001290,

            /// <summary>OBSOLETE Decoy database from a non-redundant GenBank sequence database.</summary>
            MS_decoy_DB_from_nr_OBSOLETE = 1001291,

            /// <summary>OBSOLETE Decoy database from a International Protein Index database for Rattus norvegicus.</summary>
            MS_decoy_DB_from_IPI_rat_OBSOLETE = 1001292,

            /// <summary>OBSOLETE Decoy database from a International Protein Index database for Mus musculus.</summary>
            MS_decoy_DB_from_IPI_mouse_OBSOLETE = 1001293,

            /// <summary>OBSOLETE Decoy database from a International Protein Index database for Arabidopsis thaliana.</summary>
            MS_decoy_DB_from_IPI_arabidopsis_OBSOLETE = 1001294,

            /// <summary>OBSOLETE Decoy database from an expressed sequence tag nucleotide sequence database.</summary>
            MS_decoy_DB_from_EST_OBSOLETE = 1001295,

            /// <summary>OBSOLETE Decoy database from a International Protein Index database for Danio rerio.</summary>
            MS_decoy_DB_from_IPI_zebrafish_OBSOLETE = 1001296,

            /// <summary>OBSOLETE Decoy database from a Swiss-Prot protein sequence database.</summary>
            MS_decoy_DB_from_UniProtKB_Swiss_Prot_OBSOLETE = 1001297,

            /// <summary>OBSOLETE Decoy database from a International Protein Index database for Gallus gallus.</summary>
            MS_decoy_DB_from_IPI_chicken_OBSOLETE = 1001298,

            /// <summary>OBSOLETE Decoy database from a International Protein Index database for Bos taurus.</summary>
            MS_decoy_DB_from_IPI_cow_OBSOLETE = 1001299,

            /// <summary>OBSOLETE Decoy database from a International Protein Index database for Homo sapiens.</summary>
            MS_decoy_DB_from_IPI_human_OBSOLETE = 1001300,

            /// <summary>The rank of the protein in a list sorted by the search engine.</summary>
            MS_protein_rank = 1001301,

            /// <summary>Search engine specific input parameters.</summary>
            MS_search_engine_specific_input_parameter = 1001302,

            /// <summary>Endoproteinase Arg-C.</summary>
            MS_Arg_C = 1001303,

            /// <summary>Endoproteinase Asp-N.</summary>
            MS_Asp_N = 1001304,

            /// <summary>Enzyme Asp-N, Ammonium Bicarbonate (AmBic).</summary>
            MS_Asp_N_ambic = 1001305,

            /// <summary>Enzyme chymotrypsin.</summary>
            MS_Chymotrypsin = 1001306,

            /// <summary>Cyanogen bromide.</summary>
            MS_CNBr = 1001307,

            /// <summary>Formic acid.</summary>
            MS_Formic_acid = 1001308,

            /// <summary>Endoproteinase Lys-C.</summary>
            MS_Lys_C = 1001309,

            /// <summary>Proteinase Lys-C/P.</summary>
            MS_Lys_C_P = 1001310,

            /// <summary>PepsinA proteinase.</summary>
            MS_PepsinA = 1001311,

            /// <summary>Cleavage agent TrypChymo.</summary>
            MS_TrypChymo = 1001312,

            /// <summary>Cleavage agent Trypsin/P.</summary>
            MS_Trypsin_P = 1001313,

            /// <summary>Cleavage agent V8-DE.</summary>
            MS_V8_DE = 1001314,

            /// <summary>Cleavage agent V8-E.</summary>
            MS_V8_E = 1001315,

            /// <summary>Significance threshold below which the p-value of a peptide match must lie to be considered statistically significant (default 0.05).</summary>
            MS_Mascot_SigThreshold = 1001316,

            /// <summary>The number of protein hits to display in the report. If 'Auto', all protein hits that have a protein score exceeding the average peptide identity threshold are reported. Otherwise an integer at least 1.</summary>
            MS_Mascot_MaxProteinHits = 1001317,

            /// <summary>Mascot protein scoring method; either 'Standard' or 'MudPIT'.</summary>
            MS_Mascot_ProteinScoringMethod = 1001318,

            /// <summary>Mascot peptide match ion score threshold. If between 0 and 1, then peptide matches whose expect value exceeds the thresholds are suppressed; if at least 1, then peptide matches whose ion score is below the threshold are suppressed.</summary>
            MS_Mascot_MinMSMSThreshold = 1001319,

            /// <summary>If true, show (sequence or spectrum) same-set proteins. Otherwise they are suppressed.</summary>
            MS_Mascot_ShowHomologousProteinsWithSamePeptides = 1001320,

            /// <summary>If true, show (sequence or spectrum) sub-set and subsumable proteins. Otherwise they are suppressed.</summary>
            MS_Mascot_ShowHomologousProteinsWithSubsetOfPeptides = 1001321,

            /// <summary>Only used in Peptide Summary and Select Summary reports. If true, a peptide match must be 'bold red' to be included in the report; bold red means the peptide is a top ranking match in a query and appears for the first time (in linear order) in the list of protein hits.</summary>
            MS_Mascot_RequireBoldRed = 1001322,

            /// <summary>If true, then the search results are against a nucleic acid database and Unigene clustering is enabled. Otherwise UniGene clustering is not in use.</summary>
            MS_Mascot_UseUnigeneClustering = 1001323,

            /// <summary>If true, then the search results are error tolerant and peptide matches from the second pass are included in search results. Otherwise no error tolerant peptide matches are included.</summary>
            MS_Mascot_IncludeErrorTolerantMatches = 1001324,

            /// <summary>If true, then the search results are against an automatically generated decoy database and the reported peptide matches and protein hits come from the decoy database. Otherwise peptide matches and protein hits come from the original database.</summary>
            MS_Mascot_ShowDecoyMatches = 1001325,

            /// <summary>OBSOLETE.</summary>
            MS_add_others_OBSOLETE = 1001326,

            /// <summary>OMSSA E-value.</summary>
            MS_OMSSA_evalue = 1001328,

            /// <summary>OMSSA p-value.</summary>
            MS_OMSSA_pvalue = 1001329,

            /// <summary>The X!Tandem expectation value.</summary>
            MS_X__Tandem_expect = 1001330,

            /// <summary>The X!Tandem hyperscore.</summary>
            MS_X__Tandem_hyperscore = 1001331,

            /// <summary>Regular expression for Chymotrypsin.</summary>
            MS______FYWL______P_ = 1001332,

            /// <summary>Regular expression for CNBr.</summary>
            MS_____M_ = 1001333,

            /// <summary>Regular expression for formic acid.</summary>
            MS______D_______D__ = 1001334,

            /// <summary>Regular expression for Lys-C.</summary>
            MS_____K_____P_ = 1001335,

            /// <summary>Regular expression for Lys-C/P.</summary>
            MS_____K_ = 1001336,

            /// <summary>Regular expression for PepsinA.</summary>
            MS______FL__ = 1001337,

            /// <summary>Regular expression for TrypChymo.</summary>
            MS______FYWLKR______P_ = 1001338,

            /// <summary>Regular expression for Trypsin/P.</summary>
            MS______KR__ = 1001339,

            /// <summary>Regular expression for V8-DE.</summary>
            MS______BDEZ______P_ = 1001340,

            /// <summary>Regular expression for V8-E.</summary>
            MS______EZ______P_ = 1001341,

            /// <summary>Details about a single database sequence.</summary>
            MS_database_sequence_details = 1001342,

            /// <summary>The sequence is a nucleic acid sequence.</summary>
            MS_NA_sequence = 1001343,

            /// <summary>The sequence is a amino acid sequence.</summary>
            MS_AA_sequence = 1001344,

            /// <summary>Children of this term specify the source of the mass table used.</summary>
            MS_mass_table_source = 1001345,

            /// <summary>The masses used in the mass table are taken from AAIndex.</summary>
            MS_AAIndex_mass_table = 1001346,

            /// <summary>The children of this term define file formats of the sequence database used.</summary>
            MS_database_file_formats = 1001347,

            /// <summary>The sequence database was stored in the FASTA format.</summary>
            MS_FASTA_format = 1001348,

            /// <summary>The sequence database was stored in the Abstract Syntax Notation 1 format.</summary>
            MS_ASN_1 = 1001349,

            /// <summary>The sequence database was stored in the NCBI formatdb (*.p*) format.</summary>
            MS_NCBI___p_ = 1001350,

            /// <summary>ClustalW ALN (multiple alignment) format.</summary>
            MS_clustal_aln = 1001351,

            /// <summary>EMBL entry format.</summary>
            MS_embl_em = 1001352,

            /// <summary>The NBRF PIR was used as format.</summary>
            MS_NBRF_PIR = 1001353,

            /// <summary>Root node for options for the mass table used.</summary>
            MS_mass_table_options = 1001354,

            /// <summary>Descriptions of peptides.</summary>
            MS_peptide_descriptions = 1001355,

            /// <summary>Descriptions of the input spectra.</summary>
            MS_spectrum_descriptions = 1001356,

            /// <summary>Description of the quality of the input spectrum.</summary>
            MS_spectrum_quality_descriptions = 1001357,

            /// <summary>This term reports the quality of the spectrum assigned by msmsEval.</summary>
            MS_msmsEval_quality = 1001358,

            /// <summary>Children of this term describe ambiguous residues.</summary>
            MS_ambiguous_residues = 1001359,

            /// <summary>List of standard residue one letter codes which are used to replace a non-standard.</summary>
            MS_alternate_single_letter_codes = 1001360,

            /// <summary>List of masses a non-standard letter code is replaced with.</summary>
            MS_alternate_mass = 1001361,

            /// <summary>The number of unmatched peaks.</summary>
            MS_number_of_unmatched_peaks = 1001362,

            /// <summary>A peptide matching only one.</summary>
            MS_peptide_unique_to_one_protein = 1001363,

            /// <summary>Estimation of the global false discovery rate for distinct peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs have been collapsed to one entry).</summary>
            MS_peptide_sequence_level_global_FDR = 1001364,

            /// <summary>Fragmentation information, type of product: internal yb ion.</summary>
            MS_frag__internal_yb_ion = 1001365,

            /// <summary>Fragmentation information, type of product: internal ya ion.</summary>
            MS_frag__internal_ya_ion = 1001366,

            /// <summary>Fragmentation information, type of product: z+1 ion.</summary>
            MS_frag__z_1_ion = 1001367,

            /// <summary>Fragmentation information, type of product: z+2 ion.</summary>
            MS_frag__z_2_ion = 1001368,

            /// <summary>Simple text file format of \ m/z</summary>
            MS_text_format = 1001369,

            /// <summary>The Mascot result 'homology threshold'.</summary>
            MS_Mascot_homology_threshold = 1001370,

            /// <summary>The Mascot result 'identity threshold'.</summary>
            MS_Mascot_identity_threshold = 1001371,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_Sequences = 1001372,

            /// <summary>SEQUEST total ion current.</summary>
            MS_SEQUEST_TIC = 1001373,

            /// <summary>Description not provided</summary>
            MS_SEQUEST_Sum = 1001374,

            /// <summary>The instrument type parameter value in Phenyx.</summary>
            MS_Phenyx_Instrument_Type = 1001375,

            /// <summary>The selected scoring model in Phenyx.</summary>
            MS_Phenyx_Scoring_Model = 1001376,

            /// <summary>The default parent charge value in Phenyx.</summary>
            MS_Phenyx_Default_Parent_Charge = 1001377,

            /// <summary>The parameter in Phenyx that specifies if the experimental charge state is to be considered as correct.</summary>
            MS_Phenyx_Trust_Parent_Charge = 1001378,

            /// <summary>The turbo mode parameter in Phenyx.</summary>
            MS_Phenyx_Turbo = 1001379,

            /// <summary>The maximal allowed fragment m/z error filter considered in the turbo mode of Phenyx.</summary>
            MS_Phenyx_Turbo_ErrorTol = 1001380,

            /// <summary>The minimal peptide sequence coverage value, expressed in percent, considered in the turbo mode of Phenyx.</summary>
            MS_Phenyx_Turbo_Coverage = 1001381,

            /// <summary>The list of ion series considered in the turbo mode of Phenyx.</summary>
            MS_Phenyx_Turbo_Series = 1001382,

            /// <summary>The minimal number of residues for a peptide to be considered for a valid identification in Phenyx.</summary>
            MS_Phenyx_MinPepLength = 1001383,

            /// <summary>The minimal peptide z-score for a peptide to be considered for a valid identification in Phenyx.</summary>
            MS_Phenyx_MinPepzscore = 1001384,

            /// <summary>The maximal peptide p-value for a peptide to be considered for a valid identification in Phenyx.</summary>
            MS_Phenyx_MaxPepPvalue = 1001385,

            /// <summary>The minimal protein score required for a protein database entry to be displayed in the list of identified proteins in Phenyx.</summary>
            MS_Phenyx_AC_Score = 1001386,

            /// <summary>The parameter in Phenyx that specifies if the conflict resolution algorithm is to be used.</summary>
            MS_Phenyx_Conflict_Resolution = 1001387,

            /// <summary>The primary sequence database identifier of a protein in Phenyx.</summary>
            MS_Phenyx_AC = 1001388,

            /// <summary>A secondary sequence database identifier of a protein in Phenyx.</summary>
            MS_Phenyx_ID = 1001389,

            /// <summary>The protein score of a protein match in Phenyx.</summary>
            MS_Phenyx_Score = 1001390,

            /// <summary>First number of phenyx result \ #Peptides\ .</summary>
            MS_Phenyx_Peptides1 = 1001391,

            /// <summary>Second number of phenyx result \ #Peptides\ .</summary>
            MS_Phenyx_Peptides2 = 1001392,

            /// <summary>The value of the automatic peptide acceptance filter in Phenyx.</summary>
            MS_Phenyx_Auto = 1001393,

            /// <summary>The value of the user-defined peptide acceptance filter in Phenyx.</summary>
            MS_Phenyx_User = 1001394,

            /// <summary>The z-score value of a peptide sequence match in Phenyx.</summary>
            MS_Phenyx_Pepzscore = 1001395,

            /// <summary>The p-value of a peptide sequence match in Phenyx.</summary>
            MS_Phenyx_PepPvalue = 1001396,

            /// <summary>The number of missed cleavages of a peptide sequence in Phenyx.</summary>
            MS_Phenyx_NumberOfMC = 1001397,

            /// <summary>The expression of the nature and position(s) of modified residue(s) on a matched peptide sequence in Phenyx.</summary>
            MS_Phenyx_Modif = 1001398,

            /// <summary>Source file for this mzIdentML was in OMSSA csv file format.</summary>
            MS_OMSSA_csv_format = 1001399,

            /// <summary>Source file for this mzIdentML was in OMSSA xml file format.</summary>
            MS_OMSSA_xml_format = 1001400,

            /// <summary>Source file for this mzIdentML was in X!Tandem xml file format.</summary>
            MS_X__Tandem_xml_format = 1001401,

            /// <summary>This subsection describes terms which can describe details of spectrum identification results.</summary>
            MS_spectrum_identification_result_details = 1001405,

            /// <summary>Parameter information, type of product: internal yb ion.</summary>
            MS_param__internal_yb_ion = 1001406,

            /// <summary>Parameter information, type of product: internal ya ion.</summary>
            MS_param__internal_ya_ion = 1001407,

            /// <summary>Parameter information, type of product: z+1 ion.</summary>
            MS_param__z_1_ion = 1001408,

            /// <summary>Parameter information, type of product: z+2 ion.</summary>
            MS_param__z_2_ion = 1001409,

            /// <summary>The translation start codons used to translate the nucleotides to amino acids.</summary>
            MS_translation_start_codons = 1001410,

            /// <summary>Specification of the search tolerance.</summary>
            MS_search_tolerance_specification = 1001411,

            /// <summary>Description not provided</summary>
            MS_search_tolerance_plus_value = 1001412,

            /// <summary>Description not provided</summary>
            MS_search_tolerance_minus_value = 1001413,

            /// <summary>OBSOLETE: replaced by MS:1000797 (peak list scans): This term can hold the scans attribute from an MGF input file.</summary>
            MS_MGF_scans_OBSOLETE = 1001414,

            /// <summary>OBSOLETE: replaced by MS:1000798 (peak list raw scans): This term can hold the raw scans attribute from an MGF input file.</summary>
            MS_MGF_raw_scans_OBSOLETE = 1001415,

            /// <summary>OBSOLETE: replaced by MS:1000796 (spectrum title): Holds the spectrum title from different input file formats, e.g. MGF TITLE.</summary>
            MS_spectrum_title_OBSOLETE = 1001416,

            /// <summary>SpectraST dot product of two spectra, measuring spectral similarity.</summary>
            MS_SpectraST_dot = 1001417,

            /// <summary>SpectraST measure of how much of the dot product is dominated by a few peaks.</summary>
            MS_SpectraST_dot_bias = 1001418,

            /// <summary>SpectraST spectrum score.</summary>
            MS_SpectraST_discriminant_score_F = 1001419,

            /// <summary>SpectraST normalised difference between dot product of top hit and runner-up.</summary>
            MS_SpectraST_delta = 1001420,

            /// <summary>The XML-based pepXML file format for encoding PSM information, created and maintained by the Trans-Proteomic Pipeline developers.</summary>
            MS_pepXML_format = 1001421,

            /// <summary>The XML-based protXML file format for encoding protein identifications, created and maintained by the Trans-Proteomic Pipeline developers.</summary>
            MS_protXML_format = 1001422,

            /// <summary>A URL that describes the translation table used to translate the nucleotides to amino acids.</summary>
            MS_translation_table_description = 1001423,

            /// <summary>Name of the used method in the ProteinExtractor algorithm.</summary>
            MS_ProteinExtractor_Methodname = 1001424,

            /// <summary>Flag indicating if a non redundant scoring should be generated.</summary>
            MS_ProteinExtractor_GenerateNonRedundant = 1001425,

            /// <summary>Flag indicating if identified proteins should be included.</summary>
            MS_ProteinExtractor_IncludeIdentified = 1001426,

            /// <summary>The maximum number of proteins to consider.</summary>
            MS_ProteinExtractor_MaxNumberOfProteins = 1001427,

            /// <summary>The maximum considered mass for a protein.</summary>
            MS_ProteinExtractor_MaxProteinMass = 1001428,

            /// <summary>The minimum number of proteins to consider.</summary>
            MS_ProteinExtractor_MinNumberOfPeptides = 1001429,

            /// <summary>Flag indicating to include Mascot scoring for calculation of the ProteinExtractor meta score.</summary>
            MS_ProteinExtractor_UseMascot = 1001430,

            /// <summary>Only peptides with scores higher than that threshold are taken into account in Mascot scoring for calculation of the ProteinExtractor meta score.</summary>
            MS_ProteinExtractor_MascotPeptideScoreThreshold = 1001431,

            /// <summary>In the final result each protein must have at least one peptide above this Mascot score threshold in ProteinExtractor meta score calculation.</summary>
            MS_ProteinExtractor_MascotUniqueScore = 1001432,

            /// <summary>Description not provided</summary>
            MS_ProteinExtractor_MascotUseIdentityScore = 1001433,

            /// <summary>Influence of Mascot search engine in the process of merging the search engine specific protein lists into the global protein list of ProteinExtractor.</summary>
            MS_ProteinExtractor_MascotWeighting = 1001434,

            /// <summary>Flag indicating to include SEQUEST scoring for calculation of the ProteinExtractor meta score.</summary>
            MS_ProteinExtractor_UseSequest = 1001435,

            /// <summary>Only peptides with scores higher than that threshold are taken into account in SEQUEST scoring for calculation of the ProteinExtractor meta score.</summary>
            MS_ProteinExtractor_SequestPeptideScoreThreshold = 1001436,

            /// <summary>In the final result each protein must have at least one peptide above this SEQUEST score threshold in ProteinExtractor meta score calculation.</summary>
            MS_ProteinExtractor_SequestUniqueScore = 1001437,

            /// <summary>Influence of SEQUEST search engine in the process of merging the search engine specific protein lists into the global protein list of ProteinExtractor.</summary>
            MS_ProteinExtractor_SequestWeighting = 1001438,

            /// <summary>Flag indicating to include ProteinSolver scoring for calculation of the ProteinExtractor meta score.</summary>
            MS_ProteinExtractor_UseProteinSolver = 1001439,

            /// <summary>Only peptides with scores higher than that threshold are taken into account in ProteinSolver scoring for calculation of the ProteinExtractor meta score.</summary>
            MS_ProteinExtractor_ProteinSolverPeptideScoreThreshold = 1001440,

            /// <summary>In the final result each protein must have at least one peptide above this ProteinSolver score threshold in ProteinExtractor meta score calculation.</summary>
            MS_ProteinExtractor_ProteinSolverUniqueScore = 1001441,

            /// <summary>Influence of ProteinSolver search engine in the process of merging the search engine specific protein lists into the global protein list of ProteinExtractor.</summary>
            MS_ProteinExtractor_ProteinSolverWeighting = 1001442,

            /// <summary>Flag indicating to include Phenyx scoring for calculation of the ProteinExtractor meta score.</summary>
            MS_ProteinExtractor_UsePhenyx = 1001443,

            /// <summary>Only peptides with scores higher than that threshold are taken into account in Phenyx scoring for calculation of the ProteinExtractor meta score.</summary>
            MS_ProteinExtractor_PhenyxPeptideScoreThreshold = 1001444,

            /// <summary>In the final result each protein must have at least one peptide above this Phenyx score threshold in ProteinExtractor meta score calculation.</summary>
            MS_ProteinExtractor_PhenyxUniqueScore = 1001445,

            /// <summary>Influence of Phenyx search engine in the process of merging the search engine specific protein lists into the global protein list of ProteinExtractor.</summary>
            MS_ProteinExtractor_PhenyxWeighting = 1001446,

            /// <summary>False-discovery rate threshold for proteins.</summary>
            MS_prot_FDR_threshold = 1001447,

            /// <summary>False-discovery rate threshold for peptides.</summary>
            MS_pep_FDR_threshold = 1001448,

            /// <summary>Threshold for OMSSA e-value for quality estimation.</summary>
            MS_OMSSA_e_value_threshold = 1001449,

            /// <summary>Details of decoy generation and database structure.</summary>
            MS_decoy_DB_details = 1001450,

            /// <summary>Name of algorithm used for decoy generation.</summary>
            MS_decoy_DB_generation_algorithm = 1001451,

            /// <summary>Decoy type: Amino acids of protein sequences are used in a random order.</summary>
            MS_decoy_DB_type_shuffle = 1001452,

            /// <summary>Decoy database composition: database contains only decoy entries.</summary>
            MS_DB_composition_only_decoy = 1001453,

            /// <summary>Decoy entries are generated during the search, not explicitly stored in a database (like Mascot Decoy).</summary>
            MS_quality_estimation_with_implicite_decoy_sequences = 1001454,

            /// <summary>Acquisition software.</summary>
            MS_acquisition_software = 1001455,

            /// <summary>Analysis software.</summary>
            MS_analysis_software = 1001456,

            /// <summary>Data processing software.</summary>
            MS_data_processing_software = 1001457,

            /// <summary>Vocabularies describing the spectrum generation information.</summary>
            MS_spectrum_generation_information = 1001458,

            /// <summary>Format of data files.</summary>
            MS_file_format = 1001459,

            /// <summary>This term should be given if the modification was unknown.</summary>
            MS_unknown_modification = 1001460,

            /// <summary>Greylag identification software.</summary>
            MS_greylag = 1001461,

            /// <summary>The sequence database was stored in the PEFF (PSI enhanced FastA file) format.</summary>
            MS_PEFF_format = 1001462,

            /// <summary>Phenyx open XML file format.</summary>
            MS_Phenyx_XML_format = 1001463,

            /// <summary>DTASelect file format.</summary>
            MS_DTASelect_format = 1001464,

            /// <summary>MS2 file format for MS2 spectral data.</summary>
            MS_MS2_format = 1001466,

            /// <summary>This term is used if a NCBI TaxID is specified, e.g. 9606 for Homo sapiens.</summary>
            MS_taxonomy__NCBI_TaxID = 1001467,

            /// <summary>This term is used if a common name is specified, e.g. human. Recommend using MS:1001467 (taxonomy: NCBI TaxID) where possible.</summary>
            MS_taxonomy__common_name = 1001468,

            /// <summary>This term is used if a scientific name is specified, e.g. Homo sapiens. Recommend using MS:1001467 (taxonomy: NCBI TaxID) where possible.</summary>
            MS_taxonomy__scientific_name = 1001469,

            /// <summary>This term is used if a swiss prot taxonomy id is specified, e.g. Human. Recommend using MS:1001467 (taxonomy: NCBI TaxID) where possible.</summary>
            MS_taxonomy__Swiss_Prot_ID = 1001470,

            /// <summary>The children of this term can be used to describe modifications.</summary>
            MS_peptide_modification_details = 1001471,

            /// <summary>Representation of an array of the measurements of a selectively monitored ion versus time.</summary>
            MS_selected_ion_monitoring_chromatogram = 1001472,

            /// <summary>Representation of an array of the measurements of a selectively monitored reaction versus time.</summary>
            MS_selected_reaction_monitoring_chromatogram = 1001473,

            /// <summary>OBSOLETE Representation of an array of the measurements of a series of monitored reactions versus time.</summary>
            MS_consecutive_reaction_monitoring_chromatogram_OBSOLETE = 1001474,

            /// <summary>Open Mass Spectrometry Search Algorithm was used to analyze the spectra.</summary>
            MS_OMSSA = 1001475,

            /// <summary>X!Tandem was used to analyze the spectra.</summary>
            MS_X__Tandem = 1001476,

            /// <summary>Open-source software for mass spectral library creation and searching, developed at the Institute for Systems Biology and the Hong Kong University of Science and Technology. Part of the Trans-Proteomic Pipeline.</summary>
            MS_SpectraST = 1001477,

            /// <summary>Mascot Parser was used to analyze the spectra.</summary>
            MS_Mascot_Parser = 1001478,

            /// <summary>Sequence of zero or more non-zero ASCII characters terminated by a single null (0) byte.</summary>
            MS_null_terminated_ASCII_string = 1001479,

            /// <summary>Native format defined by jobRun=xsd:nonNegativeInteger spotLabel=xsd:string spectrum=xsd:nonNegativeInteger.</summary>
            MS_SCIEX_TOF_TOF_nativeID_format = 1001480,

            /// <summary>Applied Biosystems/MDS Analytical Technologies TOF/TOF instrument database.</summary>
            MS_SCIEX_TOF_TOF_database = 1001481,

            /// <summary>SCIEX 5800 TOF-TOF Analyzer.</summary>
            MS_5800_TOF_TOF = 1001482,

            /// <summary>SCIEX or Applied Biosystems software for TOF/TOF data acquisition and analysis.</summary>
            MS_SCIEX_TOF_TOF_Series_Explorer_Software = 1001483,

            /// <summary>Normalization of data point intensities.</summary>
            MS_intensity_normalization = 1001484,

            /// <summary>Calibration of data point m/z positions.</summary>
            MS_m_z_calibration = 1001485,

            /// <summary>Filtering out part of the data.</summary>
            MS_data_filtering = 1001486,

            /// <summary>An algorithm for protein determination/assembly integrated into Bruker's ProteinScape.</summary>
            MS_ProteinExtractor = 1001487,

            /// <summary>Mascot Distiller.</summary>
            MS_Mascot_Distiller = 1001488,

            /// <summary>Mascot Integra.</summary>
            MS_Mascot_Integra = 1001489,

            /// <summary>Percolator.</summary>
            MS_Percolator = 1001490,

            /// <summary>Percolator:Q value.</summary>
            MS_percolator_Q_value = 1001491,

            /// <summary>Percolator:score.</summary>
            MS_percolator_score = 1001492,

            /// <summary>Posterior error probability.</summary>
            MS_percolator_PEP = 1001493,

            /// <summary>In case no threshold was used.</summary>
            MS_no_threshold = 1001494,

            /// <summary>The SearchResultId of this peptide as SearchResult in the ProteinScape database.</summary>
            MS_ProteinScape_SearchResultId = 1001495,

            /// <summary>The SearchEventId of the SearchEvent in the ProteinScape database.</summary>
            MS_ProteinScape_SearchEventId = 1001496,

            /// <summary>The Profound probability score stored by ProteinScape.</summary>
            MS_ProteinScape_ProfoundProbability = 1001497,

            /// <summary>The Profound z value.</summary>
            MS_Profound_z_value = 1001498,

            /// <summary>The Profound cluster score.</summary>
            MS_Profound_Cluster = 1001499,

            /// <summary>The Profound cluster rank.</summary>
            MS_Profound_ClusterRank = 1001500,

            /// <summary>The MSFit Mowse score.</summary>
            MS_MSFit_Mowse_score = 1001501,

            /// <summary>The Sonar score.</summary>
            MS_Sonar_Score = 1001502,

            /// <summary>The ProteinSolver exp value stored by ProteinScape.</summary>
            MS_ProteinScape_PFFSolverExp = 1001503,

            /// <summary>The ProteinSolver score stored by ProteinScape.</summary>
            MS_ProteinScape_PFFSolverScore = 1001504,

            /// <summary>The intensity coverage of the identified peaks in the spectrum calculated by ProteinScape.</summary>
            MS_ProteinScape_IntensityCoverage = 1001505,

            /// <summary>The SEQUEST meta score calculated by ProteinScape from the original SEQUEST scores.</summary>
            MS_ProteinScape_SequestMetaScore = 1001506,

            /// <summary>The score calculated by ProteinExtractor.</summary>
            MS_ProteinExtractor_Score = 1001507,

            /// <summary>Native format defined by scanId=xsd:nonNegativeInteger.</summary>
            MS_Agilent_MassHunter_nativeID_format = 1001508,

            /// <summary>A data file format found in an Agilent MassHunter directory which contains raw data acquired by an Agilent mass spectrometer.</summary>
            MS_Agilent_MassHunter_format = 1001509,

            /// <summary>TSQ Vantage.</summary>
            MS_TSQ_Vantage = 1001510,

            /// <summary>Filter types which are used to filter a sequence database.</summary>
            MS_Sequence_database_filter_types = 1001511,

            /// <summary>Sequence database filters which actually can contains values, e.g. to limit PI value of the sequences used to search.</summary>
            MS_Sequence_database_filters = 1001512,

            /// <summary>DB sequence filter pattern.</summary>
            MS_DB_sequence_filter_pattern = 1001513,

            /// <summary>DB accession filter string.</summary>
            MS_DB_accession_filter_string = 1001514,

            /// <summary>Fragmentation information, type of product: c ion without water.</summary>
            MS_frag__c_ion___H2O = 1001515,

            /// <summary>Fragmentation information, type of product: c ion without ammonia.</summary>
            MS_frag__c_ion___NH3 = 1001516,

            /// <summary>Fragmentation information, type of product: z ion without water.</summary>
            MS_frag__z_ion___H2O = 1001517,

            /// <summary>Fragmentation information, type of product: z ion without ammonia.</summary>
            MS_frag__z_ion___NH3 = 1001518,

            /// <summary>Fragmentation information, type of product: x ion without water.</summary>
            MS_frag__x_ion___H2O = 1001519,

            /// <summary>Fragmentation information, type of product: x ion without ammonia.</summary>
            MS_frag__x_ion___NH3 = 1001520,

            /// <summary>Fragmentation information, type of product: precursor ion without water.</summary>
            MS_frag__precursor_ion___H2O = 1001521,

            /// <summary>Fragmentation information, type of product: precursor ion without ammonia.</summary>
            MS_frag__precursor_ion___NH3 = 1001522,

            /// <summary>Fragmentation information, type of product: precursor ion.</summary>
            MS_frag__precursor_ion = 1001523,

            /// <summary>This term can describe a neutral loss m/z value that is lost from an ion.</summary>
            MS_fragment_neutral_loss = 1001524,

            /// <summary>This term can describe a neutral loss m/z value that is lost from an ion.</summary>
            MS_precursor_neutral_loss = 1001525,

            /// <summary>Native format defined by databasekey=xsd:long.</summary>
            MS_spectrum_from_database_integer_nativeID_format = 1001526,

            /// <summary>Spectra from Bruker/Protagen Proteinscape database.</summary>
            MS_Proteinscape_spectra = 1001527,

            /// <summary>Native format defined by query=xsd:nonNegativeInteger.</summary>
            MS_Mascot_query_number = 1001528,

            /// <summary>Child-terms contain information to map the results back to spectra.</summary>
            MS_spectra_data_details = 1001529,

            /// <summary>Native format defined by mzMLid=xsd:IDREF.</summary>
            MS_mzML_unique_identifier = 1001530,

            /// <summary>Native format defined by databasekey=xsd:long.</summary>
            MS_spectrum_from_ProteinScape_database_nativeID_format = 1001531,

            /// <summary>Native format defined by databasekey=xsd:string.</summary>
            MS_spectrum_from_database_string_nativeID_format = 1001532,

            /// <summary>Bruker Daltonics' esquire series.</summary>
            MS_Bruker_Daltonics_esquire_series = 1001533,

            /// <summary>Bruker Daltonics' flex series.</summary>
            MS_Bruker_Daltonics_flex_series = 1001534,

            /// <summary>Bruker Daltonics' BioTOF series.</summary>
            MS_Bruker_Daltonics_BioTOF_series = 1001535,

            /// <summary>Bruker Daltonics' micrOTOF series.</summary>
            MS_Bruker_Daltonics_micrOTOF_series = 1001536,

            /// <summary>Bruker Daltonics' BioTOF: ESI TOF.</summary>
            MS_BioTOF = 1001537,

            /// <summary>Bruker Daltonics' BioTOF III: ESI TOF.</summary>
            MS_BioTOF_III = 1001538,

            /// <summary>Bruker Daltonics' UltroTOF-Q: ESI Q-TOF (MALDI optional).</summary>
            MS_UltroTOF_Q = 1001539,

            /// <summary>Bruker Daltonics' micrOTOF II: ESI TOF, Nanospray, APCI, APPI.</summary>
            MS_micrOTOF_II = 1001540,

            /// <summary>Bruker Daltonics' maXis: ESI Q-TOF, Nanospray, APCI, APPI.</summary>
            MS_maXis = 1001541,

            /// <summary>Bruker Daltonics' amaZon ETD: ESI quadrupole ion trap, Nanospray, APCI, APPI, ETD, PTR.</summary>
            MS_amaZon_ETD = 1001542,

            /// <summary>Bruker Daltonics' microflex LRF: MALDI TOF.</summary>
            MS_microflex_LRF = 1001543,

            /// <summary>Bruker Daltonics' ultrafleXtreme: MALDI TOF.</summary>
            MS_ultrafleXtreme = 1001544,

            /// <summary>Bruker Daltonics' amaZon series.</summary>
            MS_Bruker_Daltonics_amaZon_series = 1001545,

            /// <summary>Bruker Daltonics' amaZon X: ESI quadrupole ion trap, APCI, APPI, ETD, PTR.</summary>
            MS_amaZon_X = 1001546,

            /// <summary>Bruker Daltonics' maXis series.</summary>
            MS_Bruker_Daltonics_maXis_series = 1001547,

            /// <summary>Bruker Daltonics' solarix: ESI quadrupole ion trap, APCI, APPI, ETD, PTR.</summary>
            MS_Bruker_Daltonics_solarix_series = 1001548,

            /// <summary>Bruker Daltonics' solariX: ESI, MALDI, Qh-FT_ICR.</summary>
            MS_solariX = 1001549,

            /// <summary>Bruker Daltonics' microflex II: MALDI TOF.</summary>
            MS_microflex_II = 1001550,

            /// <summary>Bruker Daltonics' autoflex II TOF/TOF: MALDI TOF.</summary>
            MS_autoflex_II_TOF_TOF = 1001553,

            /// <summary>Bruker Daltonics' autoflex III TOF/TOF smartbeam: MALDI TOF.</summary>
            MS_autoflex_III_TOF_TOF_smartbeam = 1001554,

            /// <summary>Bruker Daltonics' autoflex: MALDI TOF.</summary>
            MS_autoflex = 1001555,

            /// <summary>Bruker Daltonics' apex series.</summary>
            MS_Bruker_Daltonics_apex_series = 1001556,

            /// <summary>Shimadzu Corporation software.</summary>
            MS_Shimadzu_Corporation_software = 1001557,

            /// <summary>Shimadzu Biotech software for data acquisition, processing, and analysis.</summary>
            MS_MALDI_Solutions = 1001558,

            /// <summary>Native format defined by file=xsd:IDREF.</summary>
            MS_SCIEX_TOF_TOF_T2D_nativeID_format = 1001559,

            /// <summary>Applied Biosystems/MDS Analytical Technologies TOF/TOF instrument export format.</summary>
            MS_SCIEX_TOF_TOF_T2D_format = 1001560,

            /// <summary>Scaffold analysis software.</summary>
            MS_Scaffold = 1001561,

            /// <summary>Scaffold native ID format.</summary>
            MS_Scaffold_nativeID_format = 1001562,

            /// <summary>Source file for this mzIdentML was in SEQUEST SQT format.</summary>
            MS_SEQUEST_SQT_format = 1001563,

            /// <summary>Source file for this mzIdentML was in Thermo Scientific Discoverer MSF format.</summary>
            MS_Discoverer_MSF_format = 1001564,

            /// <summary>Source file for this mzIdentML was in Waters IdentityE XML format.</summary>
            MS_IdentityE_XML_format = 1001565,

            /// <summary>Source file for this mzIdentML was in Waters ProteinLynx XML format.</summary>
            MS_ProteinLynx_XML_format = 1001566,

            /// <summary>Source file for this mzIdentML was in Agilent SpectrumMill directory format.</summary>
            MS_SpectrumMill_directories = 1001567,

            /// <summary>Scaffold peptide probability score.</summary>
            MS_Scaffold_Peptide_Probability = 1001568,

            /// <summary>Waters IdentityE peptide score.</summary>
            MS_IdentityE_Score = 1001569,

            /// <summary>ProteinLynx log likelihood score.</summary>
            MS_ProteinLynx_Log_Likelihood = 1001570,

            /// <summary>Waters ProteinLynx Ladder score.</summary>
            MS_ProteinLynx_Ladder_Score = 1001571,

            /// <summary>Spectrum mill peptide score.</summary>
            MS_SpectrumMill_Score = 1001572,

            /// <summary>SpectrumMill SPI score (%).</summary>
            MS_SpectrumMill_SPI = 1001573,

            /// <summary>Flag indicating to report only the spectra assigned to identified proteins.</summary>
            MS_report_only_spectra_assigned_to_identified_proteins = 1001574,

            /// <summary>Minimum number of peptides a protein must have to be accepted.</summary>
            MS_Scaffold__Minimum_Peptide_Count = 1001575,

            /// <summary>Minimum protein probability a protein must have to be accepted.</summary>
            MS_Scaffold__Minimum_Protein_Probability = 1001576,

            /// <summary>Minimum probability a peptide must have to be accepted for protein scoring.</summary>
            MS_Scaffold__Minimum_Peptide_Probability = 1001577,

            /// <summary>Minimum number of enzymatic termini a peptide must have to be accepted.</summary>
            MS_minimum_number_of_enzymatic_termini = 1001578,

            /// <summary>Scaffold protein probability score.</summary>
            MS_Scaffold_Protein_Probability = 1001579,

            /// <summary>Discriminant score from Agilent SpectrumMill software.</summary>
            MS_SpectrumMill_Discriminant_Score = 1001580,

            /// <summary>The DC potential applied to the asymmetric waveform in FAIMS that compensates for the difference between high and low field mobility of an ion.</summary>
            MS_FAIMS_compensation_voltage = 1001581,

            /// <summary>Bioconductor package XCMS for preprocessing high-throughput, untargeted analyte profiling data.</summary>
            MS_XCMS = 1001582,

            /// <summary>MaxQuant is a quantitative proteomics software package designed for analyzing large mass spectrometric data sets. It is specifically aimed at high resolution MS data.</summary>
            MS_MaxQuant = 1001583,

            /// <summary>Search that includes data from Peptide Mass Fingerprint (PMF) and MS2 (aka Peptide Fragment Fingerprint - PFF).</summary>
            MS_combined_pmf___ms_ms_search = 1001584,

            /// <summary>Tabb Lab software for directly comparing peptides in a database to tandem mass spectra.</summary>
            MS_MyriMatch = 1001585,

            /// <summary>Tabb Lab software for generating sequence tags from tandem mass spectra.</summary>
            MS_DirecTag = 1001586,

            /// <summary>Tabb Lab software for reconciling sequence tags to a protein database.</summary>
            MS_TagRecon = 1001587,

            /// <summary>Tabb Lab software for spectral library searches on tandem mass spectra.</summary>
            MS_Pepitome = 1001588,

            /// <summary>Using the multivariate hypergeometric distribution and a peak list divided into several intensity classes, this score is the negative natural log probability that the predicted peaks matched to experimental peaks by random chance.</summary>
            MS_MyriMatch_MVH = 1001589,

            /// <summary>The negative natural log probability that predicted peaks match to experimental peaks by random chance by scoring the m/z delta of the matches in a multinomial distribution.</summary>
            MS_MyriMatch_mzFidelity = 1001590,

            /// <summary>A representative protein selected from a set of sequence same-set or spectrum same-set proteins.</summary>
            MS_anchor_protein = 1001591,

            /// <summary>A protein with significant homology to another protein, but some distinguishing peptide matches.</summary>
            MS_family_member_protein = 1001592,

            /// <summary>TO ENDETAIL: a really generic relationship OR ortholog protein.</summary>
            MS_group_member_with_undefined_relationship_OR_ortholog_protein = 1001593,

            /// <summary>A protein which is indistinguishable or equivalent to another protein, having matches to an identical set of peptide sequences.</summary>
            MS_sequence_same_set_protein = 1001594,

            /// <summary>A protein which is indistinguishable or equivalent to another protein, having matches to a set of peptide sequences that cannot be distinguished using the evidence in the mass spectra.</summary>
            MS_spectrum_same_set_protein = 1001595,

            /// <summary>A protein with a sub-set of the peptide sequence matches for another protein, and no distinguishing peptide matches.</summary>
            MS_sequence_sub_set_protein = 1001596,

            /// <summary>A protein with a sub-set of the matched spectra for another protein, where the matches cannot be distinguished using the evidence in the mass spectra, and no distinguishing peptide matches.</summary>
            MS_spectrum_sub_set_protein = 1001597,

            /// <summary>A sequence same-set or sequence sub-set protein where the matches are distributed across two or more proteins.</summary>
            MS_sequence_subsumable_protein = 1001598,

            /// <summary>A spectrum same-set or spectrum sub-set protein where the matches are distributed across two or more proteins.</summary>
            MS_spectrum_subsumable_protein = 1001599,

            /// <summary>Confidence category of inferred protein (conclusive, non conclusive, ambiguous group or indistinguishable).</summary>
            MS_protein_inference_confidence_category = 1001600,

            /// <summary>OBSOLETE Name and location of the .raw file or files.</summary>
            MS_ProteomeDiscoverer_Spectrum_Files_Raw_File_names_OBSOLETE = 1001601,

            /// <summary>OBSOLETE Path and name of the .srf (SEQUEST Result Format) file.</summary>
            MS_ProteomeDiscoverer_SRF_File_Selector_SRF_File_Path_OBSOLETE = 1001602,

            /// <summary>OBSOLETE Ionization source (electro-, nano-, thermospray, electron impact, APCI, MALDI, FAB etc).</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Ionization_Source_OBSOLETE = 1001603,

            /// <summary>OBSOLETE Fragmentation method used (CID, MPD, ECD, PQD, ETD, HCD, Any).</summary>
            MS_ProteomeDiscoverer_Activation_Type_OBSOLETE = 1001604,

            /// <summary>Lower retention-time limit.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Lower_RT_Limit = 1001605,

            /// <summary>OBSOLETE Type of mass spectrometer used (ITMS, FTMS, TOFMS, SQMS, TQMS, SectorMS).</summary>
            MS_ProteomeDiscoverer_Mass_Analyzer_OBSOLETE = 1001606,

            /// <summary>Maximum mass limit of a singly charged precursor ion.</summary>
            MS_ProteomeDiscoverer_Max_Precursor_Mass = 1001607,

            /// <summary>Minimum mass limit of a singly charged precursor ion.</summary>
            MS_ProteomeDiscoverer_Min_Precursor_Mass = 1001608,

            /// <summary>Minimum number of peaks in a tandem mass spectrum that is allowed to pass the filter and to be subjected to further processing in the workflow.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Minimum_Peak_Count = 1001609,

            /// <summary>OBSOLETE Level of the mass spectrum (MS2 ... MS10).</summary>
            MS_ProteomeDiscoverer_MS_Order_OBSOLETE = 1001610,

            /// <summary>OBSOLETE Polarity mode (positive or negative).</summary>
            MS_ProteomeDiscoverer_Polarity_Mode_OBSOLETE = 1001611,

            /// <summary>Determines which precursor mass to use for a given MSn scan. This option applies only to higher-order MSn scans (n &gt;= 3).</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Precursor_Selection = 1001612,

            /// <summary>Signal-to-Noise ratio below which peaks are removed.</summary>
            MS_ProteomeDiscoverer_SN_Threshold = 1001613,

            /// <summary>OBSOLETE Scan type for the precursor ion (full, Single Ion Monitoring (SIM), Single Reaction Monitoring (SRM)).</summary>
            MS_ProteomeDiscoverer_Scan_Type_OBSOLETE = 1001614,

            /// <summary>Used to filter out tandem mass spectra that have a total intensity current(sum of the intensities of all peaks in a spectrum) below the specified value.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Total_Intensity_Threshold = 1001615,

            /// <summary>Specifies the fragmentation method to use in the search algorithm if it is not included in the scan header.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Unrecognized_Activation_Type_Replacements = 1001616,

            /// <summary>Specifies the charge state of the precursor ions, if it is not defined in the scan header.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Unrecognized_Charge_Replacements = 1001617,

            /// <summary>Specifies the mass spectrometer to use to produce the spectra, if it is not included in the scan header.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Unrecognized_Mass_Analyzer_Replacements = 1001618,

            /// <summary>Specifies the MS scan order used to produce the product spectra, if it is not included in the scan header.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Unrecognized_MS_Order_Replacements = 1001619,

            /// <summary>Specifies the polarity of the ions monitored if it is not included in the scan header.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Unrecognized_Polarity_Replacements = 1001620,

            /// <summary>Upper retention-time limit.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Upper_RT_Limit = 1001621,

            /// <summary>Specifies the size of the mass-to-charge ratio (m/z) window in daltons used to remove precursors.</summary>
            MS_ProteomeDiscoverer_Non_Fragment_Filter_Mass_Window_Offset = 1001622,

            /// <summary>Maximum allowed mass of a neutral loss.</summary>
            MS_ProteomeDiscoverer_Non_Fragment_Filter_Maximum_Neutral_Loss_Mass = 1001623,

            /// <summary>Determines whether the charge-reduced precursor peaks found in an ETD or ECD spectrum are removed.</summary>
            MS_ProteomeDiscoverer_Non_Fragment_Filter_Remove_Charge_Reduced_Precursor = 1001624,

            /// <summary>Determines whether neutral loss peaks are removed from ETD and ECD spectra.</summary>
            MS_ProteomeDiscoverer_Non_Fragment_Filter_Remove_Neutral_Loss_Peaks = 1001625,

            /// <summary>Determines whether overtone peaks are removed from LTQ FT or LTQ FT Ultra ECD spectra.</summary>
            MS_ProteomeDiscoverer_Non_Fragment_Filter_Remove_Only_Known_Masses = 1001626,

            /// <summary>Determines whether precursor overtone peaks in the spectrum are removed from the input spectrum.</summary>
            MS_ProteomeDiscoverer_Non_Fragment_Filter_Remove_Precursor_Overtones = 1001627,

            /// <summary>Determines whether precursor artifact peaks from the MS2 input spectra are removed.</summary>
            MS_ProteomeDiscoverer_Non_Fragment_Filter_Remove_Precursor_Peak = 1001628,

            /// <summary>Determines whether the fragment spectrum for scans with the same precursor mass is grouped, regardless of mass analyzer and activation type.</summary>
            MS_ProteomeDiscoverer_Spectrum_Grouper_Allow_Mass_Analyzer_Mismatch = 1001629,

            /// <summary>Determines whether spectra from different MS order scans can be grouped together.</summary>
            MS_ProteomeDiscoverer_Spectrum_Grouper_Allow_MS_Order_Mismatch = 1001630,

            /// <summary>OBSOLETE Chromatographic window where precursors to be grouped must reside to be considered the same species.</summary>
            MS_ProteomeDiscoverer_Spectrum_Grouper_Max_RT_Difference_OBSOLETE = 1001631,

            /// <summary>Groups spectra measured within the given mass and retention-time tolerances into a single spectrum for analysis.</summary>
            MS_ProteomeDiscoverer_Spectrum_Grouper_Precursor_Mass_Criterion = 1001632,

            /// <summary>Highest charge state that is allowed for the deconvolution of multiply charged data.</summary>
            MS_ProteomeDiscoverer_Xtract_Highest_Charge = 1001633,

            /// <summary>OBSOLETE Highest mass-to-charge (mz) value for spectral peaks in the measured spectrum that are considered for Xtract.</summary>
            MS_ProteomeDiscoverer_Xtract_Highest_MZ_OBSOLETE = 1001634,

            /// <summary>Lowest charge state that is allowed for the deconvolution of multiply charged data.</summary>
            MS_ProteomeDiscoverer_Xtract_Lowest_Charge = 1001635,

            /// <summary>OBSOLETE Lowest mass-to-charge (mz) value for spectral peaks in the measured spectrum that are considered for Xtract.</summary>
            MS_ProteomeDiscoverer_Xtract_Lowest_MZ_OBSOLETE = 1001636,

            /// <summary>Determines whether the isotopic pattern, i.e. all isotopes of a mass are removed from the spectrum.</summary>
            MS_ProteomeDiscoverer_Xtract_Monoisotopic_Mass_Only = 1001637,

            /// <summary>Fraction of the more abundant peak that an overlapping multiplet must exceed in order to be processed (deconvoluted).</summary>
            MS_ProteomeDiscoverer_Xtract_Overlapping_Remainder = 1001638,

            /// <summary>Accuracy required for a pattern fit to be considered valid.</summary>
            MS_ProteomeDiscoverer_Xtract_Required_Fitting_Accuracy = 1001639,

            /// <summary>Resolution at mass 400.</summary>
            MS_ProteomeDiscoverer_Xtract_Resolution_At_400 = 1001640,

            /// <summary>Minimum charge state below which peptides are filtered out.</summary>
            MS_ProteomeDiscoverer_Lowest_Charge_State = 1001641,

            /// <summary>Maximum charge above which peptides are filtered out.</summary>
            MS_ProteomeDiscoverer_Highest_Charge_State = 1001642,

            /// <summary>Determines whether spectra with scores above the threshold score are retained rather than filtered out.</summary>
            MS_ProteomeDiscoverer_Spectrum_Score_Filter_Let_Pass_Above_Scores = 1001643,

            /// <summary>Determine dynamic post-translational modifications (PTMs).</summary>
            MS_ProteomeDiscoverer_Dynamic_Modification = 1001644,

            /// <summary>Static Modification to all occurrences of a named amino acid.</summary>
            MS_ProteomeDiscoverer_Static_Modification = 1001645,

            /// <summary>OBSOLETE Determines whether the Proteome Discoverer application searches an additional decoy database.</summary>
            MS_ProteomeDiscoverer_Mascot_Decoy_Search_OBSOLETE = 1001646,

            /// <summary>Determines whether to search error-tolerant.</summary>
            MS_ProteomeDiscoverer_Mascot_Error_tolerant_Search = 1001647,

            /// <summary>Maximum size of the .mgf (Mascot Generic Format) file in MByte.</summary>
            MS_ProteomeDiscoverer_Mascot_Max_MGF_File_Size = 1001648,

            /// <summary>URL (Uniform resource Locator) of the Mascot server.</summary>
            MS_ProteomeDiscoverer_Mascot_Mascot_Server_URL = 1001649,

            /// <summary>Number of attempts to submit the Mascot search.</summary>
            MS_ProteomeDiscoverer_Mascot_Number_of_attempts_to_submit_the_search = 1001650,

            /// <summary>Number of attempts to submit the Mascot search.</summary>
            MS_ProteomeDiscoverer_Mascot_X_Static_Modification = 1001651,

            /// <summary>OBSOLETE Name of the user submitting the Mascot search.</summary>
            MS_ProteomeDiscoverer_Mascot_User_Name_OBSOLETE = 1001652,

            /// <summary>Time interval between attempts to submit a search in seconds.</summary>
            MS_ProteomeDiscoverer_Mascot_Time_interval_between_attempts_to_submit_a_search = 1001653,

            /// <summary>OBSOLETE Specifies the enzyme reagent used for protein digestion.</summary>
            MS_ProteomeDiscoverer_Enzyme_Name_OBSOLETE = 1001654,

            /// <summary>OBSOLETE Mass tolerance used for matching fragment peaks in Da or mmu.</summary>
            MS_ProteomeDiscoverer_Fragment_Mass_Tolerance_OBSOLETE = 1001655,

            /// <summary>Type of instrument used to acquire the data in the raw file.</summary>
            MS_Mascot_Instrument = 1001656,

            /// <summary>Maximum number of missed cleavage sites to consider during the digest.</summary>
            MS_ProteomeDiscoverer_Maximum_Missed_Cleavage_Sites = 1001657,

            /// <summary>Minimum score in the IonScore column that each peptide must exceed in order to be reported.</summary>
            MS_ProteomeDiscoverer_Mascot_Peptide_CutOff_Score = 1001658,

            /// <summary>OBSOLETE Mass window for which precursor ions are considered to be the same species.</summary>
            MS_ProteomeDiscoverer_Precursor_Mass_Tolerance_OBSOLETE = 1001659,

            /// <summary>Minimum protein score in the IonScore column that each protein must exceed in order to be reported.</summary>
            MS_ProteomeDiscoverer_Mascot_Protein_CutOff_Score = 1001660,

            /// <summary>OBSOLETE Database to use in the search (configured on the Mascot server).</summary>
            MS_ProteomeDiscoverer_Protein_Database_OBSOLETE = 1001661,

            /// <summary>Specifies a factor that is used in calculating a threshold that determines whether a protein appears in the results report.</summary>
            MS_ProteomeDiscoverer_Mascot_Protein_Relevance_Factor = 1001662,

            /// <summary>Specifies the relaxed target false discovery rate (FDR, 0.0 - 1.0) for peptide hits with moderate confidence.</summary>
            MS_ProteomeDiscoverer_Target_FDR_Relaxed = 1001663,

            /// <summary>Specifies the strict target false discovery rate (FDR, 0.0 - 1.0) for peptide hits with high confidence.</summary>
            MS_ProteomeDiscoverer_Target_FDR_Strict = 1001664,

            /// <summary>OBSOLETE Limits searches to entries from a particular species or group of species.</summary>
            MS_ProteomeDiscoverer_Mascot_Taxonomy_OBSOLETE = 1001665,

            /// <summary>OBSOLETE Use average mass for the precursor.</summary>
            MS_ProteomeDiscoverer_Use_Average_Precursor_Mass_OBSOLETE = 1001666,

            /// <summary>OBSOLETE Determines whether to use MudPIT or normal scoring.</summary>
            MS_Mascot_use_MudPIT_scoring_OBSOLETE = 1001667,

            /// <summary>Minimum cross-correlation threshold that determines whether peptides in an .srf file are imported.</summary>
            MS_ProteomeDiscoverer_Absolute_XCorr_Threshold = 1001668,

            /// <summary>Determines whether to calculate a probability score for every peptide match.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Calculate_Probability_Score = 1001669,

            /// <summary>Dynamic C-terminal modification that is used during the search.</summary>
            MS_ProteomeDiscoverer_SEQUEST_CTerminal_Modification = 1001670,

            /// <summary>Percentage of the theoretical ions that must be found in order for a peptide to be scored and retained.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Fragment_Ion_Cutoff_Percentage = 1001671,

            /// <summary>Maximum number of identical modifications that a single peptide can have.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Max_Identical_Modifications_Per_Peptide = 1001672,

            /// <summary>Maximum number of different modifications that a peptide can have, e.g. because of steric hindrance.</summary>
            MS_ProteomeDiscoverer_Max_Modifications_Per_Peptide = 1001673,

            /// <summary>Maximum number of peptides that are searched and scored per spectrum.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Maximum_Peptides_Considered = 1001674,

            /// <summary>Maximum number of peptide matches reported per spectrum.</summary>
            MS_ProteomeDiscoverer_Maximum_Peptides_Output = 1001675,

            /// <summary>Maximum number of proteins that a single identified peptide can be associated with during protein assembly.</summary>
            MS_ProteomeDiscoverer_Maximum_Protein_References_Per_Peptide = 1001676,

            /// <summary>Dynamic N-terminal modification that is used during the search.</summary>
            MS_ProteomeDiscoverer_SEQUEST_NTerminal_Modification = 1001677,

            /// <summary>Static modification for the C terminal of the peptide used during the search.</summary>
            MS_ProteomeDiscoverer_Peptide_CTerminus = 1001678,

            /// <summary>Static modification for the N terminal of the peptide used during the search.</summary>
            MS_ProteomeDiscoverer_Peptide_NTerminus = 1001679,

            /// <summary>Specifies a factor to apply to the protein score.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Peptide_Relevance_Factor = 1001680,

            /// <summary>Specifies a peptide threshold that determines whether the protein that it is a part of is scored and retained in the report.</summary>
            MS_ProteomeDiscoverer_Protein_Relevance_Threshold = 1001681,

            /// <summary>OBSOLETE Determines whether the Proteome Discoverer application searches against a decoy database.</summary>
            MS_ProteomeDiscoverer_Search_Against_Decoy_Database_OBSOLETE = 1001682,

            /// <summary>Use average masses for the fragments.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Use_Average_Fragment_Masses = 1001683,

            /// <summary>Determines whether a ions with neutral loss are used for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Use_Neutral_Loss_a_Ions = 1001684,

            /// <summary>Determines whether b ions with neutral loss are used for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Use_Neutral_Loss_b_Ions = 1001685,

            /// <summary>Determines whether y ions with neutral loss are used for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Use_Neutral_Loss_y_Ions = 1001686,

            /// <summary>Determines whether z ions with neutral loss are used for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Use_Neutral_Loss_z_Ions = 1001687,

            /// <summary>Uses a ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_a_Ions = 1001688,

            /// <summary>Uses b ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_b_Ions = 1001689,

            /// <summary>Uses c ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_c_Ions = 1001690,

            /// <summary>Uses c ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_d_Ions = 1001691,

            /// <summary>Uses c ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_v_Ions = 1001692,

            /// <summary>Uses c ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_w_Ions = 1001693,

            /// <summary>Uses x ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_x_Ions = 1001694,

            /// <summary>Uses y ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_y_Ions = 1001695,

            /// <summary>Uses z ions for spectrum matching with this relative factor.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Weight_of_z_Ions = 1001696,

            /// <summary>Sets a minimum protein score that each protein must exceed in order to be reported.</summary>
            MS_ProteomeDiscoverer_ZCore_Protein_Score_Cutoff = 1001697,

            /// <summary>Specifies which peak to select if more than one peak is found inside the integration window.</summary>
            MS_ProteomeDiscoverer_Reporter_Ions_Quantizer_Integration_Method = 1001698,

            /// <summary>Specifies the mass-to-charge window that enables one to look for the reporter peaks.</summary>
            MS_ProteomeDiscoverer_Reporter_Ions_Quantizer_Integration_Window_Tolerance = 1001699,

            /// <summary>Quantitation method for isobarically labeled quantitation.</summary>
            MS_ProteomeDiscoverer_Reporter_Ions_Quantizer_Quantitation_Method = 1001700,

            /// <summary>OBSOLETE Format of the exported spectra (dta, mgf or mzData).</summary>
            MS_ProteomeDiscoverer_Spectrum_Exporter_Export_Format_OBSOLETE = 1001701,

            /// <summary>Name of the output file that contains the exported data.</summary>
            MS_ProteomeDiscoverer_Spectrum_Exporter_File_name = 1001702,

            /// <summary>Influences the modifications search.</summary>
            MS_ProteomeDiscoverer_Search_Modifications_Only_For_Identified_Proteins = 1001703,

            /// <summary>Standard high confidence XCorr parameter for charge = 1.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Std_High_Confidence_XCorr_Charge1 = 1001704,

            /// <summary>Standard high confidence XCorr parameter for charge = 2.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Std_High_Confidence_XCorr_Charge2 = 1001705,

            /// <summary>Standard high confidence XCorr parameter for charge = 3.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Std_High_Confidence_XCorr_Charge3 = 1001706,

            /// <summary>Standard high confidence XCorr parameter for charge &gt;= 4.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Std_High_Confidence_XCorr_Charge4 = 1001707,

            /// <summary>Standard medium confidence XCorr parameter for charge = 1.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Std_Medium_Confidence_XCorr_Charge1 = 1001708,

            /// <summary>Standard medium confidence XCorr parameter for charge = 2.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Std_Medium_Confidence_XCorr_Charge2 = 1001709,

            /// <summary>Standard medium confidence XCorr parameter for charge = 3.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Std_Medium_Confidence_XCorr_Charge3 = 1001710,

            /// <summary>Standard medium confidence XCorr parameter for charge &gt;= 4.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Std_Medium_Confidence_XCorr_Charge4 = 1001711,

            /// <summary>FT high confidence XCorr parameter for charge = 1.</summary>
            MS_ProteomeDiscoverer_SEQUEST_FT_High_Confidence_XCorr_Charge1 = 1001712,

            /// <summary>FT high confidence XCorr parameter for charge = 2.</summary>
            MS_ProteomeDiscoverer_SEQUEST_FT_High_Confidence_XCorr_Charge2 = 1001713,

            /// <summary>FT high confidence XCorr parameter for charge = 3.</summary>
            MS_ProteomeDiscoverer_SEQUEST_FT_High_Confidence_XCorr_Charge3 = 1001714,

            /// <summary>FT high confidence XCorr parameter for charge &gt;= 4.</summary>
            MS_ProteomeDiscoverer_SEQUEST_FT_High_Confidence_XCorr_Charge4 = 1001715,

            /// <summary>FT medium confidence XCorr parameter for charge = 1.</summary>
            MS_ProteomeDiscoverer_SEQUEST_FT_Medium_Confidence_XCorr_Charge1 = 1001716,

            /// <summary>FT medium confidence XCorr parameter for charge = 2.</summary>
            MS_ProteomeDiscoverer_SEQUEST_FT_Medium_Confidence_XCorr_Charge2 = 1001717,

            /// <summary>FT medium confidence XCorr parameter for charge = 3.</summary>
            MS_ProteomeDiscoverer_SEQUEST_FT_Medium_Confidence_XCorr_Charge3 = 1001718,

            /// <summary>FT medium confidence XCorr parameter for charge &gt;= 4.</summary>
            MS_ProteomeDiscoverer_SEQUEST_FT_Medium_Confidence_XCorr_Charge4 = 1001719,

            /// <summary>OBSOLETE ProteomeDiscoverer's 1st dynamic post-translational modification (PTM) input parameter.</summary>
            MS_ProteomeDiscoverer_1__Dynamic_Modification_OBSOLETE = 1001720,

            /// <summary>OBSOLETE ProteomeDiscoverer's 2nd dynamic post-translational modification (PTM) input parameter.</summary>
            MS_ProteomeDiscoverer_2__Dynamic_Modification_OBSOLETE = 1001721,

            /// <summary>OBSOLETE ProteomeDiscoverer's 3rd dynamic post-translational modification (PTM) input parameter.</summary>
            MS_ProteomeDiscoverer_3__Dynamic_Modification_OBSOLETE = 1001722,

            /// <summary>OBSOLETE ProteomeDiscoverer's 4th dynamic post-translational modification (PTM) input parameter.</summary>
            MS_ProteomeDiscoverer_4__Dynamic_Modification_OBSOLETE = 1001723,

            /// <summary>Static Modification for X.</summary>
            MS_ProteomeDiscoverer_Static_Modification_for_X = 1001724,

            /// <summary>Minimal initial peptide probability to contribute to analysis.</summary>
            MS_ProteomeDiscoverer_Initial_minimal_peptide_probability = 1001725,

            /// <summary>Minimum adjusted peptide probability contributing to protein probability.</summary>
            MS_ProteomeDiscoverer_Minimal_peptide_probability = 1001726,

            /// <summary>Minimum peptide weight contributing to protein probability.</summary>
            MS_ProteomeDiscoverer_Minimal_peptide_weight = 1001727,

            /// <summary>Number of spectra from 1+ precursor ions.</summary>
            MS_ProteomeDiscoverer_Number_of_input1_spectra = 1001728,

            /// <summary>Number of spectra from 2+ precursor ions.</summary>
            MS_ProteomeDiscoverer_Number_of_input2_spectra = 1001729,

            /// <summary>Number of spectra from 3+ precursor ions.</summary>
            MS_ProteomeDiscoverer_Number_of_input3_spectra = 1001730,

            /// <summary>Number of spectra from 4+ precursor ions.</summary>
            MS_ProteomeDiscoverer_Number_of_input4_spectra = 1001731,

            /// <summary>Number of spectra from 5+ precursor ions.</summary>
            MS_ProteomeDiscoverer_Number_of_input5_spectra = 1001732,

            /// <summary>Total number of predicted correct protein ids (sum of probabilities).</summary>
            MS_ProteomeDiscoverer_Number_of_predicted_correct_proteins = 1001733,

            /// <summary>OBSOLETE Sample organism (used for annotation purposes).</summary>
            MS_ProteomeDiscoverer_Organism_OBSOLETE = 1001734,

            /// <summary>OBSOLETE Full path database name.</summary>
            MS_ProteomeDiscoverer_Reference_Database_OBSOLETE = 1001735,

            /// <summary>Residues considered equivalent when comparing peptides.</summary>
            MS_ProteomeDiscoverer_Residue_substitution_list = 1001736,

            /// <summary>OBSOLETE File type (if not pepXML).</summary>
            MS_ProteomeDiscoverer_Source_file_extension_OBSOLETE = 1001737,

            /// <summary>OBSOLETE Input pepXML files.</summary>
            MS_ProteomeDiscoverer_Source_Files_OBSOLETE = 1001738,

            /// <summary>OBSOLETE Input pepXML files (old).</summary>
            MS_ProteomeDiscoverer_Source_Files_old_OBSOLETE = 1001739,

            /// <summary>Windows full path for database.</summary>
            MS_ProteomeDiscoverer_WinCyg_reference_database = 1001740,

            /// <summary>Windows pepXML file names.</summary>
            MS_ProteomeDiscoverer_WinCyg_source_files = 1001741,

            /// <summary>Finnigan LTQ Orbitrap Velos MS.</summary>
            MS_LTQ_Orbitrap_Velos = 1001742,

            /// <summary>Determines if to use A ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_A_Ions = 1001743,

            /// <summary>Determines if to use B ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_B_Ions = 1001744,

            /// <summary>Determines if to use C ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_C_Ions = 1001745,

            /// <summary>Determines if to use D ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_D_Ions = 1001746,

            /// <summary>Determines if to use V ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_V_Ions = 1001747,

            /// <summary>Determines if to use W ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_W_Ions = 1001748,

            /// <summary>Determines if to use X ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_X_Ions = 1001749,

            /// <summary>Determines if to use Y ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_Y_Ions = 1001750,

            /// <summary>Determines if to use z ions for spectrum matching.</summary>
            MS_ProteomeDiscoverer_Mascot_Weight_of_Z_Ions = 1001751,

            /// <summary>Determines if to use precursor reevaluation.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Use_New_Precursor_Reevaluation = 1001752,

            /// <summary>Signal-to-Noise ratio below which peaks are removed (in FT mode only).</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_SN_Threshold_FTonly = 1001753,

            /// <summary>Unknown Mascot parameter which ProteomeDiscoverer uses for mascot searches.</summary>
            MS_ProteomeDiscoverer_Mascot_Please_Do_not_Touch_this = 1001754,

            /// <summary>Phone number of the contact person or organization.</summary>
            MS_contact_phone_number = 1001755,

            /// <summary>Fax number for the contact person or organization.</summary>
            MS_contact_fax_number = 1001756,

            /// <summary>Toll-free phone number of the contact person or organization.</summary>
            MS_contact_toll_free_phone_number = 1001757,

            /// <summary>Significance threshold type used in Mascot reporting (either 'identity' or 'homology').</summary>
            MS_Mascot_SigThresholdType = 1001758,

            /// <summary>Strategy used by Mascot to group proteins with same peptide matches (one of 'none', 'Occam's razor' or 'family clustering').</summary>
            MS_Mascot_ProteinGrouping = 1001759,

            /// <summary>List of Percolator features that were used in processing the peptide matches. Typical Percolator features are 'retentionTime', 'dM', 'mScore', 'lgDScore', 'mrCalc', 'charge' and 'dMppm'.</summary>
            MS_Percolator_features = 1001760,

            /// <summary>Waters LC-system ACQUITY UPLC.</summary>
            MS_ACQUITY_UPLC = 1001761,

            /// <summary>Waters LC-system ACQUITY UPLC H-Class.</summary>
            MS_ACQUITY_UPLC_H_Class = 1001762,

            /// <summary>Waters LC-system ACQUITY UPLC H-Class Bio.</summary>
            MS_ACQUITY_UPLC_H_Class_Bio = 1001763,

            /// <summary>Waters LC-system ACQUITY UPLC I-Class.</summary>
            MS_ACQUITY_UPLC_I_Class = 1001764,

            /// <summary>Waters LC-system ACQUITY UPLC Systems with 2D Technology.</summary>
            MS_ACQUITY_UPLC_Systems_with_2D_Technology = 1001765,

            /// <summary>Waters LC-system nanoACQUITY UPLC.</summary>
            MS_nanoACQUITY_UPLC = 1001766,

            /// <summary>Waters LC-system nanoACQUITY UPLC System with 1D Technology.</summary>
            MS_nanoACQUITY_UPLC_System_with_1D_Technology = 1001767,

            /// <summary>Waters LC-system nanoACQUITY UPLC with HDX Technology.</summary>
            MS_nanoACQUITY_UPLC_with_HDX_Technology = 1001768,

            /// <summary>Waters LC-system TRIZAIC UPLC nanoTile.</summary>
            MS_TRIZAIC_UPLC_nanoTile = 1001769,

            /// <summary>Waters oa-ToF based GCT Premier.</summary>
            MS_GCT_Premier = 1001770,

            /// <summary>Waters oa-ToF based MALDI Synapt G2 HDMS.</summary>
            MS_MALDI_Synapt_G2_HDMS = 1001771,

            /// <summary>Waters oa-ToF based MALDI Synapt G2 MS.</summary>
            MS_MALDI_Synapt_G2_MS = 1001772,

            /// <summary>Waters oa-ToF based MALDI Synapt G2 MS.</summary>
            MS_MALDI_Synapt_G2_S_HDMS = 1001773,

            /// <summary>Waters oa-ToF based MALDI Synapt G2-S MS.</summary>
            MS_MALDI_Synapt_G2_S_MS = 1001774,

            /// <summary>Waters oa-ToF based MALDI Synapt HDMS.</summary>
            MS_MALDI_Synapt_HDMS = 1001775,

            /// <summary>Waters oa-ToF based MALDI Synapt MS.</summary>
            MS_MALDI_Synapt_MS = 1001776,

            /// <summary>Waters oa-ToF based Synapt G2 HDMS.</summary>
            MS_Synapt_G2_HDMS = 1001777,

            /// <summary>Waters oa-ToF based Synapt G2 MS.</summary>
            MS_Synapt_G2_MS = 1001778,

            /// <summary>Waters oa-ToF based Synapt G2-S HDMS.</summary>
            MS_Synapt_G2_S_HDMS = 1001779,

            /// <summary>Waters oa-ToF based Synapt G2-S MS.</summary>
            MS_Synapt_G2_S_MS = 1001780,

            /// <summary>Waters oa-ToF based Synapt HDMS.</summary>
            MS_Synapt_HDMS = 1001781,

            /// <summary>Waters oa-ToF based Synapt MS.</summary>
            MS_Synapt_MS = 1001782,

            /// <summary>Waters oa-ToF based Xevo G2 Q-Tof.</summary>
            MS_Xevo_G2_Q_Tof = 1001783,

            /// <summary>Waters oa-ToF based Xevo G2 Tof.</summary>
            MS_Xevo_G2_Tof = 1001784,

            /// <summary>Waters oa-ToF based Xevo Q-Tof.</summary>
            MS_Xevo_Q_Tof = 1001785,

            /// <summary>Waters quadrupole based 3100.</summary>
            MS_3100 = 1001786,

            /// <summary>Waters quadrupole based Acquity SQD.</summary>
            MS_Acquity_SQD = 1001787,

            /// <summary>Waters quadrupole based Acquity TQD.</summary>
            MS_Acquity_TQD = 1001788,

            /// <summary>Waters (triple) quadrupole based Quattro micro GC.</summary>
            MS_Quattro_micro_GC = 1001789,

            /// <summary>Waters quadrupole based Xevo TQ MS.</summary>
            MS_Xevo_TQ_MS = 1001790,

            /// <summary>Waters quadrupole based Xevo TQD.</summary>
            MS_Xevo_TQD = 1001791,

            /// <summary>Waters quadrupole based Xevo TQ-S.</summary>
            MS_Xevo_TQ_S = 1001792,

            /// <summary>NCBI TaxID taxonomy ID to prefer when two or more proteins match the same set of peptides or when protein entry in database represents multiple sequences.</summary>
            MS_Mascot_PreferredTaxonomy = 1001793,

            /// <summary>Waters Empower software for liquid chromatography and mass spectrometry acquisition.</summary>
            MS_Empower = 1001795,

            /// <summary>Waters UNIFY software for liquid chromatography and mass spectrometry acquisition.</summary>
            MS_UNIFY = 1001796,

            /// <summary>OBSOLETE An ion mobility mass spectrometry technique based on the superimposition of travelling voltage waves on a radially-confining RF voltage in a gas-filled, stacked-ring ion guide.</summary>
            MS_travelling_wave_ion_mobility_mass_spectrometer_OBSOLETE = 1001797,

            /// <summary>LECO software for data acquisition and analysis.</summary>
            MS_LECO_software = 1001798,

            /// <summary>Software for acquisition, processing and analysis of data for LECO instruments.</summary>
            MS_ChromaTOF_software = 1001799,

            /// <summary>LECO instrument model.</summary>
            MS_LECO_instrument_model = 1001800,

            /// <summary>LECO high resolution time-of-flight GC mass spectrometer.</summary>
            MS_Pegasus_HRT = 1001801,

            /// <summary>LECO high resolution time-of-flight LC mass spectrometer.</summary>
            MS_Citius_HRT = 1001802,

            /// <summary>LECO GC time-of-flight mass spectrometer.</summary>
            MS_Pegasus = 1001803,

            /// <summary>LECO bench-top GC time-of-flight mass spectrometer.</summary>
            MS_TruTOF = 1001804,

            /// <summary>The data type of the value reported in a QuantLayer.</summary>
            MS_quantification_datatype = 1001805,

            /// <summary>Attributes describing the details of an object relevant for reporting quantification workflows or values.</summary>
            MS_quantification_object_attribute = 1001806,

            /// <summary>Attribute describing a study variable.</summary>
            MS_study_variable_attribute = 1001807,

            /// <summary>The study variable is 'technical replicate'. The string value denotes the category of technical replicate, e.g. 'run generated from same sample'.</summary>
            MS_technical_replicate = 1001808,

            /// <summary>The study variable is 'biological replicate'. This means, the run was generated from another individual or sample.</summary>
            MS_biological_replicate = 1001809,

            /// <summary>The experimental condition is 'case' in contrast to 'control'.</summary>
            MS_experimental_condition__case_ = 1001810,

            /// <summary>The experimental condition is 'control' in contrast to 'case'.</summary>
            MS_experimental_condition__control_ = 1001811,

            /// <summary>The experimental condition is 'disease' in contrast to 'healthy'.</summary>
            MS_experimental_condition__disease_ = 1001812,

            /// <summary>The experimental condition is 'healthy' in contrast to 'disease'.</summary>
            MS_experimental_condition__healthy_ = 1001813,

            /// <summary>The experimental condition is given in the value of this term.</summary>
            MS_generic_experimental_condition = 1001814,

            /// <summary>The experimental design followed a time series design. The time point of this run is given in the value of this term.</summary>
            MS_time_series__time_point_X = 1001815,

            /// <summary>The experimental design followed a dilution series design. The concentration of this run is given in the value of this term.</summary>
            MS_dilution_series__concentration_X = 1001816,

            /// <summary>Attribute describing a raw file.</summary>
            MS_raw_file_attribute = 1001817,

            /// <summary>The raw file contains the run of one sample (e.g. spectral counting, LC-MS label-free).</summary>
            MS_one_sample_run = 1001818,

            /// <summary>The raw file contains the run of two samples (e.g. SILAC, metabolic labelling).</summary>
            MS_two_sample_run = 1001819,

            /// <summary>The raw file contains the run of three samples (e.g. 3-plex SILAC).</summary>
            MS_three_sample_run = 1001820,

            /// <summary>The raw file contains the run of four samples (e.g. 4-plex iTraq).</summary>
            MS_four_sample_run = 1001821,

            /// <summary>The raw file contains the run of eight samples (e.g. 8-plex iTraq).</summary>
            MS_eight_sample_run = 1001822,

            /// <summary>Attribute describing, how raw files build a raw file group.</summary>
            MS_raw_files_group_attribute = 1001823,

            /// <summary>Attribute describing, how raw files build a raw file group.</summary>
            MS_merge_of_runs_of_1D_gel_bands = 1001824,

            /// <summary>Attribute describing a feature list.</summary>
            MS_feature_list_attribute = 1001825,

            /// <summary>The mass trace of the features of this feature list specifies rectangles. Each mass trace has the syntax (RT_start,MZ_start,RT_end,MZ_end), i.e. opposite corners are given.</summary>
            MS_mass_trace_reporting__rectangles = 1001826,

            /// <summary>The mass trace of the features of this feature list specifies polygons. Each mass trace has the syntax (RT_1, MZ_1, RT_2, MZ_2, ... , RT_i, MZ_i, ... , RT_n, MZ_n), where the line (RT_n, MZ_n)-&gt;(RT_1, MZ_1) is implicit.</summary>
            MS_mass_trace_reporting__polygons = 1001827,

            /// <summary>Attribute describing a feature.</summary>
            MS_feature_attribute = 1001828,

            /// <summary>Identifier for an SRM transition in an external document describing additional information about the transition.</summary>
            MS_SRM_transition_ID = 1001829,

            /// <summary>Software from Nonlinear Dynamics for LC-MS label-free workflow.</summary>
            MS_Progenesis_LC_MS = 1001830,

            /// <summary>Software for SILAC workflow.</summary>
            MS_SILACAnalyzer = 1001831,

            /// <summary>Quantitation software comment or any customizations to the default setup of the software.</summary>
            MS_quantitation_software_comment_or_customizations = 1001832,

            /// <summary>The overall workflow of this quantitation report.</summary>
            MS_quantitation_analysis_summary = 1001833,

            /// <summary>LC-MS label-free workflow (RT m/z map).</summary>
            MS_LC_MS_label_free_quantitation_analysis = 1001834,

            /// <summary>SILAC workflow (heavy, light, and sometimes medium peak).</summary>
            MS_SILAC_quantitation_analysis = 1001835,

            /// <summary>Spectral counting workflow (number of identified MS2 spectra as approximation of peptide / protein quant).</summary>
            MS_spectral_counting_quantitation_analysis = 1001836,

            /// <summary>Quantification analysis using the SCIEX amine-reactive isobaric tags for relative and absolute quantification (iTRAQ) labelling workflow, wherein 2-8 reporter ions are measured in MS2 spectra near in the 114-121 m/z range.</summary>
            MS_iTRAQ_quantitation_analysis = 1001837,

            /// <summary>Selected Reaction Monitoring workflow (XIC quantitation of precursor / fragment mass pair).</summary>
            MS_SRM_quantitation_analysis = 1001838,

            /// <summary>Metabolic labeling workflow (heavy and light versions of peptides, depending on number of nitrogens).</summary>
            MS_metabolic_labeling_14N___15N_quantitation_analysis = 1001839,

            /// <summary>Maximum peak intensity of the LC-MS feature.</summary>
            MS_LC_MS_feature_intensity = 1001840,

            /// <summary>Real (intensity times area) volume of the LC-MS feature.</summary>
            MS_LC_MS_feature_volume = 1001841,

            /// <summary>The number of MS2 spectra identified for a raw peptide sequence without PTMs and charge state in spectral counting.</summary>
            MS_sequence_level_spectral_count = 1001842,

            /// <summary>Maximum intensity of MS1 feature.</summary>
            MS_MS1_feature_maximum_intensity = 1001843,

            /// <summary>Area of MS1 feature.</summary>
            MS_MS1_feature_area = 1001844,

            /// <summary>OBSOLETE Area of MS1 peak (e.g. SILAC, 15N).</summary>
            MS_peak_area_OBSOLETE = 1001845,

            /// <summary>Area of all peaks belonging to the isotopic pattern of light or heavy peak (e.g. 15N).</summary>
            MS_isotopic_pattern_area = 1001846,

            /// <summary>Intensity of MS2 reporter ion (e.g. iTraq).</summary>
            MS_reporter_ion_intensity = 1001847,

            /// <summary>Simple ratio of two values (enumerator and denominator).</summary>
            MS_simple_ratio_of_two_values = 1001848,

            /// <summary>OBSOLETE Peptide quantification value calculated as sum of MatchedFeature quantification values.</summary>
            MS_sum_of_MatchedFeature_values_OBSOLETE = 1001849,

            /// <summary>Normalized peptide value.</summary>
            MS_normalized_peptide_value = 1001850,

            /// <summary>Protein quantification value calculated as sum of peptide values.</summary>
            MS_protein_value__sum_of_peptide_values = 1001851,

            /// <summary>Normalized protein value.</summary>
            MS_normalized_protein_value = 1001852,

            /// <summary>Global datatype: Maximum of all pair-wise fold changes of group means (e.g. Progenesis).</summary>
            MS_max_fold_change = 1001853,

            /// <summary>Global datatype: p-value of ANOVA of group means (e.g. Progenesis).</summary>
            MS_ANOVA_p_value = 1001854,

            /// <summary>P-value of t-Test of two groups.</summary>
            MS_t_test_p_value = 1001855,

            /// <summary>Intensity (or area) of MS2 reporter ion (e.g. iTraq).</summary>
            MS_reporter_ion_raw_value = 1001856,

            /// <summary>Normalized value of MS2 reporter ion (e.g. iTraq).</summary>
            MS_reporter_ion_normalized_value = 1001857,

            /// <summary>Area of the extracted ion chromatogram (e.g. of a transition in SRM).</summary>
            MS_XIC_area = 1001858,

            /// <summary>Normalized area of the extracted ion chromatogram (e.g. of a transition in SRM).</summary>
            MS_normalized_XIC_area = 1001859,

            /// <summary>Protein quantification value calculated as mean of peptide ratios.</summary>
            MS_protein_value__mean_of_peptide_ratios = 1001860,

            /// <summary>Terms used to describe types of quantification data processing.</summary>
            MS_quantification_data_processing = 1001861,

            /// <summary>Normalization of protein values to the mean of the sum of all protein PSM counts (e.g. spectral counting).</summary>
            MS_normalization_to_mean_of_sum_of_all_proteins = 1001862,

            /// <summary>Normalization of protein values to approach the same distribution.</summary>
            MS_quantile_normalization__proteins = 1001863,

            /// <summary>Normalization of peptide values to approach the same distribution.</summary>
            MS_quantile_normalization__peptides = 1001864,

            /// <summary>Automatic RT alignment of Progenesis software.</summary>
            MS_Progenesis_automatic_alignment = 1001865,

            /// <summary>RT alignment of Progenesis software using automatic and manual vectors.</summary>
            MS_Progenesis_manual_alignment = 1001866,

            /// <summary>Normalization as performed by Progenesis LC-MS.</summary>
            MS_Progenesis_normalization = 1001867,

            /// <summary>Estimation of the q-value for distinct peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs, possibly with different mass modifications, mapping to the same sequence have been collapsed to one entry).</summary>
            MS_distinct_peptide_level_q_value = 1001868,

            /// <summary>Estimation of the q-value for proteins.</summary>
            MS_protein_level_q_value = 1001869,

            /// <summary>Estimation of the p-value for distinct peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs have been collapsed to one entry).</summary>
            MS_peptide_sequence_level_p_value = 1001870,

            /// <summary>Estimation of the p-value for proteins.</summary>
            MS_protein_level_p_value = 1001871,

            /// <summary>Estimation of the e-value for distinct peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs have been collapsed to one entry).</summary>
            MS_peptide_sequence_level_e_value = 1001872,

            /// <summary>Estimation of the e-value for proteins.</summary>
            MS_protein_level_e_value = 1001873,

            /// <summary>OBSOLETE A smoothing of the distribution of q-values calculated for PSMs from individual search engines, such that ordering of result quality is maintained and all FDRScore values are guaranteed to have a value &gt; 0.</summary>
            MS_FDRScore_OBSOLETE = 1001874,

            /// <summary>The regular expression describing the sequence motif for a modification.</summary>
            MS_modification_motif = 1001875,

            /// <summary>The a priori probability of a modification.</summary>
            MS_modification_probability = 1001876,

            /// <summary>Software for acquisition, processing and analysis of data for LECO instruments.</summary>
            MS_ChromaTOF_HRT_software = 1001877,

            /// <summary>Shimadzu Biotech software for data acquisition, processing, and analysis.</summary>
            MS_MALDI_Solutions_Microbial_Identification = 1001878,

            /// <summary>The potential difference between two adjacent interface voltages affecting in-source collision induced dissociation.</summary>
            MS_offset_voltage = 1001879,

            /// <summary>The dissociation of an ion as a result of collisional excitation during ion transfer from an atmospheric pressure ion source and the mass spectrometer vacuum.</summary>
            MS_in_source_collision_induced_dissociation = 1001880,

            /// <summary>mz5 file format, modelled after mzML.</summary>
            MS_mz5_format = 1001881,

            /// <summary>Attributes of the quality of a transition that affect its selection as appropriate.</summary>
            MS_transition_validation_attribute = 1001882,

            /// <summary>Variation of a set of signal measurements calculated as the standard deviation relative to the mean.</summary>
            MS_coefficient_of_variation = 1001883,

            /// <summary>Unitless number providing the ratio of the total measured intensity of a signal relative to the estimated noise level for that signal.</summary>
            MS_signal_to_noise_ratio = 1001884,

            /// <summary>Parameters string passed to a command-line interface software application, omitting the executable name.</summary>
            MS_command_line_parameters = 1001885,

            /// <summary>Software for data analysis of peptides and proteins.</summary>
            MS_SQID = 1001886,

            /// <summary>The SQID result 'Score'.</summary>
            MS_SQID_score = 1001887,

            /// <summary>The SQID result 'deltaScore'.</summary>
            MS_SQID_deltaScore = 1001888,

            /// <summary>The SQID result 'protein score'.</summary>
            MS_SQID_protein_score = 1001889,

            /// <summary>The data type normalised abundance for proteins produced by Progenesis LC-MS.</summary>
            MS_Progenesis_protein_normalised_abundance = 1001890,

            /// <summary>The data type normalised abundance for peptides produced by Progenesis LC-MS.</summary>
            MS_Progenesis_peptide_normalised_abundance = 1001891,

            /// <summary>The data type raw abundance for proteins produced by Progenesis LC-MS.</summary>
            MS_Progenesis_protein_raw_abundance = 1001892,

            /// <summary>The data type raw abundance for peptide produced by Progenesis LC-MS.</summary>
            MS_Progenesis_peptide_raw_abundance = 1001893,

            /// <summary>The data type confidence score produced by Progenesis LC-MS.</summary>
            MS_Progenesis_confidence_score = 1001894,

            /// <summary>The data type peptide count produced by Progenesis LC-MS.</summary>
            MS_Progenesis_peptide_count = 1001895,

            /// <summary>The data type feature intensity produced by Progenesis LC-MS.</summary>
            MS_Progenesis_feature_intensity = 1001896,

            /// <summary>The data type peptide counts (unique) produced by MaxQuant.</summary>
            MS_MaxQuant_peptide_counts__unique_ = 1001897,

            /// <summary>The data type peptide counts (all) produced by MaxQuant.</summary>
            MS_MaxQuant_peptide_counts__all_ = 1001898,

            /// <summary>The data type peptide counts (razor+unique) produced by MaxQuant.</summary>
            MS_MaxQuant_peptide_counts__razor_unique_ = 1001899,

            /// <summary>The data type sequence length produced by MaxQuant.</summary>
            MS_MaxQuant_sequence_length = 1001900,

            /// <summary>The data type PEP (posterior error probability) produced by MaxQuant.</summary>
            MS_MaxQuant_PEP = 1001901,

            /// <summary>The data type LFQ intensity produced by MaxQuant.</summary>
            MS_MaxQuant_LFQ_intensity = 1001902,

            /// <summary>The data type feature intensity produced by MaxQuant.</summary>
            MS_MaxQuant_feature_intensity = 1001903,

            /// <summary>The data type MS2 count produced by MaxQuant.</summary>
            MS_MaxQuant_MS_MS_count = 1001904,

            /// <summary>The emPAI value of protein abundance, produced from the emPAI algorithm.</summary>
            MS_emPAI_value = 1001905,

            /// <summary>The APEX value of protein abundance, produced from the APEX software.</summary>
            MS_APEX_value = 1001906,

            /// <summary>The full width of a retention time window for a chromatographic peak.</summary>
            MS_retention_time_window_width = 1001907,

            /// <summary>Thermo Scientific ISQ single quadrupole MS with the ExtractraBrite source.</summary>
            MS_ISQ = 1001908,

            /// <summary>Thermo Scientific second generation Velos.</summary>
            MS_Velos_Plus = 1001909,

            /// <summary>Thermo Scientific LTQ Orbitrap Elite, often just referred to as the Orbitrap Elite.</summary>
            MS_LTQ_Orbitrap_Elite = 1001910,

            /// <summary>Thermo Scientific Q Exactive.</summary>
            MS_Q_Exactive = 1001911,

            /// <summary>Thermo Scientific PinPoint SRM analysis software.</summary>
            MS_PinPoint = 1001912,

            /// <summary>Potential difference setting of the Thermo Scientific S-lens stacked-ring ion guide in volts.</summary>
            MS_S_lens_voltage = 1001913,

            /// <summary>Python module to interface mzML Data.</summary>
            MS_pymzML = 1001914,

            /// <summary>Enzyme leukocyte elastase (EC 3.4.21.37).</summary>
            MS_leukocyte_elastase = 1001915,

            /// <summary>Enzyme proline endopeptidase (EC 3.4.21.26).</summary>
            MS_proline_endopeptidase = 1001916,

            /// <summary>Enzyme glutamyl endopeptidase (EC 3.4.21.19).</summary>
            MS_glutamyl_endopeptidase = 1001917,

            /// <summary>Chemical iodobenzoate. Cleaves after W.</summary>
            MS_2_iodobenzoate = 1001918,

            /// <summary>Main identifier of a ProteomeXchange dataset.</summary>
            MS_ProteomeXchange_accession_number = 1001919,

            /// <summary>Version number of a ProteomeXchange accession number.</summary>
            MS_ProteomeXchange_accession_number_version_number = 1001921,

            /// <summary>DOI unique identifier of a publication.</summary>
            MS_Digital_Object_Identifier__DOI_ = 1001922,

            /// <summary>Free text attribute that can enrich the information about an entity.</summary>
            MS_external_reference_keyword = 1001923,

            /// <summary>Keyword present in a scientific publication.</summary>
            MS_journal_article_keyword = 1001924,

            /// <summary>Keyword assigned by the data submitter.</summary>
            MS_submitter_keyword = 1001925,

            /// <summary>Keyword assigned by a data curator.</summary>
            MS_curator_keyword = 1001926,

            /// <summary>Hash assigned by the Tranche resource to an individual file.</summary>
            MS_Tranche_file_hash = 1001927,

            /// <summary>Hash assigned by the Tranche resource to a whole project.</summary>
            MS_Tranche_project_hash = 1001928,

            /// <summary>URI that allows the access to one experiment in the PRIDE database.</summary>
            MS_PRIDE_experiment_URI = 1001929,

            /// <summary>URI that allows the access to one project in the PRIDE database.</summary>
            MS_PRIDE_project_URI = 1001930,

            /// <summary>The source interface.</summary>
            MS_source_interface = 1001931,

            /// <summary>The source interface model.</summary>
            MS_source_interface_model = 1001932,

            /// <summary>The source sprayer.</summary>
            MS_source_sprayer = 1001933,

            /// <summary>The source sprayer type.</summary>
            MS_source_sprayer_type = 1001934,

            /// <summary>The source sprayer manufacturer.</summary>
            MS_source_sprayer_manufacturer = 1001935,

            /// <summary>The source sprayer model.</summary>
            MS_source_sprayer_model = 1001936,

            /// <summary>Plate where the sample solution is spotted in a MALDI or similar instrument.</summary>
            MS_sample_plate = 1001937,

            /// <summary>The sample plate type.</summary>
            MS_sample_plate_type = 1001938,

            /// <summary>Stainless steel plate.</summary>
            MS_stainless_steel_plate = 1001939,

            /// <summary>Coated glass plate.</summary>
            MS_coated_glass_plate = 1001940,

            /// <summary>Whether the sprayer is fed or is loaded with sample once.</summary>
            MS_electrospray_supply_type = 1001941,

            /// <summary>The sprayer is loaded with sample once.</summary>
            MS_static_supply_electrospray = 1001942,

            /// <summary>The sprayer is continuously fed with sample.</summary>
            MS_fed_supply_electrospray = 1001943,

            /// <summary>Potential difference between Q2 and Q3 in a triple quadrupole instrument in volts.</summary>
            MS_Collision_cell_exit_potential = 1001944,

            /// <summary>LECO nominal mass resolution time-of-flight GCxGC mass spectrometer.</summary>
            MS_Pegasus_4D = 1001945,

            /// <summary>PEAKS Studio software for data analysis.</summary>
            MS_PEAKS_Studio = 1001946,

            /// <summary>PEAKS Online software for high throughput data analysis.</summary>
            MS_PEAKS_Online = 1001947,

            /// <summary>PEAKS Node software for high throughput data analysis.</summary>
            MS_PEAKS_Node = 1001948,

            /// <summary>Bioinformatics Solutions Inc. Software for data processing and analysis.</summary>
            MS_BSI_software = 1001949,

            /// <summary>The PEAKS peptide '-10lgP Score'.</summary>
            MS_PEAKS_peptideScore = 1001950,

            /// <summary>The PEAKS protein '-10lgP Score'.</summary>
            MS_PEAKS_proteinScore = 1001951,

            /// <summary>The ZCore probability score.</summary>
            MS_ZCore_probScore = 1001952,

            /// <summary>The source interface manufacturer.</summary>
            MS_source_interface_manufacturer = 1001953,

            /// <summary>Parameters used in the mass spectrometry acquisition.</summary>
            MS_acquisition_parameter = 1001954,

            /// <summary>No cleavage.</summary>
            MS_no_cleavage = 1001955,

            /// <summary>Unspecific cleavage.</summary>
            MS_unspecific_cleavage = 1001956,

            /// <summary>Regular expression for leukocyte elastase.</summary>
            MS______ALIV_____P_ = 1001957,

            /// <summary>Regular expression for proline endopeptidase.</summary>
            MS______HKR_P____P_ = 1001958,

            /// <summary>Regular expression for glutamyl endopeptidase.</summary>
            MS_______E_E_ = 1001959,

            /// <summary>Regular expression for 2-iodobenzoate.</summary>
            MS_____W_ = 1001960,

            /// <summary>Algorithm used to score the match between a spectrum and a peptide ion.</summary>
            MS_peptide_spectrum_match_scoring_algorithm = 1001961,

            /// <summary>C13 peaks to use in peak detection.</summary>
            MS_Mascot_C13_counts = 1001962,

            /// <summary>Weighting factor for protein list compilation by ProteinExtractor.</summary>
            MS_ProteinExtractor_Weighting = 1001963,

            /// <summary>Flag indicating a second round search with Mascot.</summary>
            MS_ProteinScape_second_round_Mascot = 1001964,

            /// <summary>Flag indicating a second round search with Phenyx.</summary>
            MS_ProteinScape_second_round_Phenyx = 1001965,

            /// <summary>The mobility of an MS2 product ion, as measured by ion mobility mass spectrometry.</summary>
            MS_product_ion_mobility = 1001966,

            /// <summary>OBSOLETE The ion drift time of an MS2 product ion.</summary>
            MS_product_ion_drift_time_OBSOLETE = 1001967,

            /// <summary>Statistic to convey the confidence of the localization of an amino acid modification on a peptide sequence at the PSM-level.</summary>
            MS_PTM_localization_PSM_level_statistic = 1001968,

            /// <summary>phosphoRS score for PTM site location at the PSM-level.</summary>
            MS_phosphoRS_score = 1001969,

            /// <summary>Probability that the respective isoform is correct.</summary>
            MS_phosphoRS_sequence_probability = 1001970,

            /// <summary>Estimate of the probability that the respective site is truly phosphorylated.</summary>
            MS_phosphoRS_site_probability = 1001971,

            /// <summary>Version of the post-translational modification scoring algorithm.</summary>
            MS_PTM_scoring_algorithm_version = 1001972,

            /// <summary>DeBunker software.</summary>
            MS_DeBunker = 1001973,

            /// <summary>Score specific to DeBunker.</summary>
            MS_DeBunker_score = 1001974,

            /// <summary>The difference between a theoretically calculated m/z and the corresponding experimentally measured m/z. It can be expressed as absolute or relative value.</summary>
            MS_delta_m_z = 1001975,

            /// <summary>The difference between a theoretically calculated molecular mass M and the corresponding experimentally measured M. It can be expressed as absolute or relative value.</summary>
            MS_delta_M = 1001976,

            /// <summary>MSQuant software.</summary>
            MS_MSQuant = 1001977,

            /// <summary>The PTM score from MSQuant software.</summary>
            MS_MSQuant_PTM_score = 1001978,

            /// <summary>The PTM score from MaxQuant software.</summary>
            MS_MaxQuant_PTM_Score = 1001979,

            /// <summary>The Phospho (STY) Probabilities from MaxQuant software.</summary>
            MS_MaxQuant_Phospho__STY__Probabilities = 1001980,

            /// <summary>The Phospho (STY) Score Diffs from MaxQuant software.</summary>
            MS_MaxQuant_Phospho__STY__Score_Diffs = 1001981,

            /// <summary>The P-site localization probability value from MaxQuant software.</summary>
            MS_MaxQuant_P_site_localization_probability = 1001982,

            /// <summary>The PTM Delta Score value from MaxQuant software (Difference between highest scoring site and second highest).</summary>
            MS_MaxQuant_PTM_Delta_Score = 1001983,

            /// <summary>Ascore software.</summary>
            MS_Ascore_software = 1001984,

            /// <summary>A-score for PTM site location at the PSM-level.</summary>
            MS_Ascore = 1001985,

            /// <summary>H-Score for peptide phosphorylation site location.</summary>
            MS_H_Score = 1001986,

            /// <summary>Vacuum-drying MALDI sample preparation crystallization method.</summary>
            MS_vacuum_drying_MALDI_sample_preparation = 1001987,

            /// <summary>Crushed-crystal MALDI sample preparation method.</summary>
            MS_crushed_crystal_MALDI_sample_preparation = 1001988,

            /// <summary>Fast-evaporation MALDI sample preparation method.</summary>
            MS_fast_evaporation_MALDI_sample_preparation = 1001989,

            /// <summary>Overlayer method combining features of the crushed-crystal method and the fast-evaporation method.</summary>
            MS_overlayer_MALDI_sample_preparation = 1001990,

            /// <summary>Sandwich MALDI sample preparation method.</summary>
            MS_sandwich_MALDI_sample_preparation = 1001991,

            /// <summary>Spin coating MALDI sample preparation method.</summary>
            MS_spin_coating_MALDI_sample_preparation = 1001992,

            /// <summary>Quick &amp; dirty (Q&amp;D) sample preparation separating matrix handling from sample handling.</summary>
            MS_quick_and_dirty_MALDI_sample_preparation = 1001993,

            /// <summary>Top-hat morphological filter based on the basic morphological operations 'erosion' and 'dilatation'.</summary>
            MS_top_hat_baseline_reduction = 1001994,

            /// <summary>Constructs the baseline by fitting multiple parabolas to the spectrum starting with the large scale structures.</summary>
            MS_convex_hull_baseline_reduction = 1001995,

            /// <summary>The spectrum that will be baseline subtracted is divided into a number of segments.</summary>
            MS_median_baseline_reduction = 1001996,

            /// <summary>The random noise is removed by using the undecimated wavelet transform.</summary>
            MS_wavelet_transformation_smoothing = 1001997,

            /// <summary>It searches for known patterns in the measured spectrum.</summary>
            MS_sophisticated_numerical_annotation_procedure = 1001998,

            /// <summary>Normalization of areas below the curves.</summary>
            MS_area_normalization = 1001999,

            /// <summary>A Bruker's proprietary technique where molecular ions are initially accelerated at lower energy, then collide with inert gas in a collision cell that is then 'lifted' to high potential. The use of inert gas is optional, as it could lift also fragments provided by LID.</summary>
            MS_LIFT = 1002000,

            /// <summary>MS1 label-based raw feature quantitation.</summary>
            MS_MS1_label_based_raw_feature_quantitation = 1002001,

            /// <summary>MS1 label-based peptide level quantitation.</summary>
            MS_MS1_label_based_peptide_level_quantitation = 1002002,

            /// <summary>MS1 label-based protein level quantitation.</summary>
            MS_MS1_label_based_protein_level_quantitation = 1002003,

            /// <summary>MS1 label-based proteingroup level quantitation.</summary>
            MS_MS1_label_based_proteingroup_level_quantitation = 1002004,

            /// <summary>A de facto standard providing the retention times at which a specific set of 10 reference peptides exit the reference chromatographic column. The kit may be obtain from Biognosys.</summary>
            MS_iRT_retention_time_normalization_standard = 1002005,

            /// <summary>The type of the transitions, e.g. target or decoy.</summary>
            MS_SRM_transition_type = 1002006,

            /// <summary>A transition used to target a specific compound that may be in the sample.</summary>
            MS_target_SRM_transition = 1002007,

            /// <summary>A transition not expected to be present in the sample and used to calculate statistical confidence of target transition detections in some workflows.</summary>
            MS_decoy_SRM_transition = 1002008,

            /// <summary>Quantitation analysis using an isobaric labelling workflow.</summary>
            MS_isobaric_label_quantitation_analysis = 1002009,

            /// <summary>Quantitation analysis using the Thermo Fisher amine-reactive tandem mass tag (TMT) labelling workflow, wherein 2-10 reporter ions are measured in MS2 spectra in the 126-131 m/z.</summary>
            MS_TMT_quantitation_analysis = 1002010,

            /// <summary>Combination of electrospray and desorption ionization method that ionizes gases, liquids and solids in open air under atmospheric pressure.</summary>
            MS_desorption_electrospray_ionization = 1002011,

            /// <summary>Relative probability that PTM site assignment is correct, derived from the Mascot score difference between matches to the same spectrum (Mascot Delta Score).</summary>
            MS_Mascot_PTM_site_assignment_confidence = 1002012,

            /// <summary>Collision energy at the start of the collision energy ramp.</summary>
            MS_collision_energy_ramp_start = 1002013,

            /// <summary>Collision energy at the end of the collision energy ramp.</summary>
            MS_collision_energy_ramp_end = 1002014,

            /// <summary>Spectral count peptide level quantitation.</summary>
            MS_spectral_count_peptide_level_quantitation = 1002015,

            /// <summary>Spectral count protein level quantitation.</summary>
            MS_spectral_count_protein_level_quantitation = 1002016,

            /// <summary>Spectral count proteingroup level quantitation.</summary>
            MS_spectral_count_proteingroup_level_quantitation = 1002017,

            /// <summary>MS1 label-based analysis.</summary>
            MS_MS1_label_based_analysis = 1002018,

            /// <summary>Label-free raw feature quantitation.</summary>
            MS_label_free_raw_feature_quantitation = 1002019,

            /// <summary>Label-free peptide level quantitation.</summary>
            MS_label_free_peptide_level_quantitation = 1002020,

            /// <summary>Label-free protein level quantitation.</summary>
            MS_label_free_protein_level_quantitation = 1002021,

            /// <summary>Label-free proteingroup level quantitation.</summary>
            MS_label_free_proteingroup_level_quantitation = 1002022,

            /// <summary>MS2 tag-based analysis.</summary>
            MS_MS2_tag_based_analysis = 1002023,

            /// <summary>MS2 tag-based feature level quantitation.</summary>
            MS_MS2_tag_based_feature_level_quantitation = 1002024,

            /// <summary>MS2 tag-based peptide level quantitation.</summary>
            MS_MS2_tag_based_peptide_level_quantitation = 1002025,

            /// <summary>MS2 tag-based protein level quantitation.</summary>
            MS_MS2_tag_based_protein_level_quantitation = 1002026,

            /// <summary>MS2 tag-based proteingroup level quantitation.</summary>
            MS_MS2_tag_based_proteingroup_level_quantitation = 1002027,

            /// <summary>Nucleic acid base modification (substitution, insertion or deletion).</summary>
            MS_nucleic_acid_base_modification = 1002028,

            /// <summary>Specification of the original nucleic acid sequence, prior to a modification. The value slot should hold the DNA or RNA sequence.</summary>
            MS_original_nucleic_acid_sequence = 1002029,

            /// <summary>Specification of the modified nucleic acid sequence. The value slot should hold the DNA or RNA sequence.</summary>
            MS_modified_nucleic_acid_sequence = 1002030,

            /// <summary>URI to retrieve transition group data for a PASSEL (PeptideAtlas SRM Experiment Library) experiment.</summary>
            MS_PASSEL_transition_group_browser_URI = 1002031,

            /// <summary>URI that allows access to a PeptideAtlas dataset.</summary>
            MS_PeptideAtlas_dataset_URI = 1002032,

            /// <summary>Role of the contact person.</summary>
            MS_contact_role = 1002033,

            /// <summary>The first of a set of authors associated with a publication or release. There may be more than one first author in cases where several authors share primary attribution.</summary>
            MS_first_author = 1002034,

            /// <summary>The last of a set of authors associated with a publication or release. There may be more than one senior author in cases where several authors share senior attribution.</summary>
            MS_senior_author = 1002035,

            /// <summary>One of a set of authors associated with a publication or release.</summary>
            MS_co_author = 1002036,

            /// <summary>A person who submits a dataset to a repository.</summary>
            MS_dataset_submitter = 1002037,

            /// <summary>A sample that has not been labelled or modified. This is often referred to as \ light\  to distinguish from \ heavy\ .</summary>
            MS_label_free_sample = 1002038,

            /// <summary>Inlet properties that are associated with a value.</summary>
            MS_inlet_attribute = 1002039,

            /// <summary>The temperature of the inlet of a mass spectrometer.</summary>
            MS_inlet_temperature = 1002040,

            /// <summary>The temperature of the source of a mass spectrometer.</summary>
            MS_source_temperature = 1002041,

            /// <summary>The duration of a complete cycle of modulation in a comprehensive two-dimensional separation system, equals the length of a second dimension chromatogram, i.e., the time between two successive injections into the second column.</summary>
            MS_modulation_time = 1002042,

            /// <summary>ProteinProspector software for data acquisition and analysis.</summary>
            MS_ProteinProspector = 1002043,

            /// <summary>The ProteinProspector result 'Score'.</summary>
            MS_ProteinProspector_score = 1002044,

            /// <summary>The ProteinProspector result 'Expectation value'.</summary>
            MS_ProteinProspector_expectation_value = 1002045,

            /// <summary>The original source path used for directory-based sources.</summary>
            MS_native_source_path = 1002046,

            /// <summary>MS-GF software used to re-score the peptide-spectrum matches.</summary>
            MS_MS_GF = 1002047,

            /// <summary>MS-GF+ software used to analyze the spectra.</summary>
            MS_MS_GF_ = 1002048,

            /// <summary>MS-GF raw score.</summary>
            MS_MS_GF_RawScore = 1002049,

            /// <summary>MS-GF de novo score.</summary>
            MS_MS_GF_DeNovoScore = 1002050,

            /// <summary>MS-GF energy score.</summary>
            MS_MS_GF_Energy = 1002051,

            /// <summary>MS-GF spectral E-value.</summary>
            MS_MS_GF_SpecEValue = 1002052,

            /// <summary>MS-GF E-value.</summary>
            MS_MS_GF_EValue = 1002053,

            /// <summary>MS-GF Q-value.</summary>
            MS_MS_GF_QValue = 1002054,

            /// <summary>MS-GF peptide-level Q-value.</summary>
            MS_MS_GF_PepQValue = 1002055,

            /// <summary>MS-GF posterior error probability.</summary>
            MS_MS_GF_PEP = 1002056,

            /// <summary>As parameter for search engine: apply the modification only at the N-terminus of a protein.</summary>
            MS_modification_specificity_protein_N_term = 1002057,

            /// <summary>As parameter for search engine: apply the modification only at the C-terminus of a protein.</summary>
            MS_modification_specificity_protein_C_term = 1002058,

            /// <summary>Microsoft Excel (can be used for spectral counting).</summary>
            MS_Microsoft_Excel = 1002059,

            /// <summary>The name of the UniProtKB/TrEMBL database.</summary>
            MS_database_UniProtKB_TrEMBL = 1002060,

            /// <summary>OBSOLETE Decoy database from a TrEMBL protein sequence database.</summary>
            MS_decoy_DB_from_UniProtKB_TrEMBL_OBSOLETE = 1002061,

            /// <summary>Metabolic labelling: natural N (mainly 14N).</summary>
            MS_metabolic_labelling__natural_N__mainly_14N_ = 1002062,

            /// <summary>Software e.g. for SILAC and 14N/15N workflow, part of the PeakQuant suite.</summary>
            MS_FindPairs = 1002063,

            /// <summary>Peptide consensus retention time.</summary>
            MS_peptide_consensus_RT = 1002064,

            /// <summary>Peptide consensus mass/charge ratio.</summary>
            MS_peptide_consensus_m_z = 1002065,

            /// <summary>Method used to calculate the ratio.</summary>
            MS_ratio_calculation_method = 1002066,

            /// <summary>Protein quantification value calculated as median of peptide ratios.</summary>
            MS_protein_value__median_of_peptide_ratios = 1002067,

            /// <summary>Metabolic labelling: heavy N (mainly 15N).</summary>
            MS_metabolic_labelling__heavy_N__mainly_15N_ = 1002068,

            /// <summary>Metabolic labelling: Description of labelling purity. Usually the purity of feeding material (e.g. 95%), or the inclusion rate derived from isotopic peak pattern shape.</summary>
            MS_metabolic_labelling_purity = 1002069,

            /// <summary>Perform a t-test (two groups). Specify in string value, whether paired / unpaired, variance equal / different, one- / two-sided version is performed.</summary>
            MS_t_test = 1002070,

            /// <summary>Perform an ANOVA-test (more than two groups). Specify in string value, which version is performed.</summary>
            MS_ANOVA_test = 1002071,

            /// <summary>P-value as result of one of the processing steps described. Specify in the description, which processing step it was.</summary>
            MS_p_value = 1002072,

            /// <summary>The mzIdentML format for peptide and protein identification data from the PSI. File extension '.mzid'.</summary>
            MS_mzIdentML_format = 1002073,

            /// <summary>File format containing quantification results.</summary>
            MS_quantification_file_format = 1002074,

            /// <summary>The mzQuantML format for quantification data from the PSI. File extension '.mzq'.</summary>
            MS_mzQuantML_format = 1002075,

            /// <summary>PAnalyzer software for getting protein evidence categories.</summary>
            MS_PAnalyzer = 1002076,

            /// <summary>Bruker Daltonics' impact: ESI Q-TOF, Nanospray, APCI, APPI, GC-APCI, CaptiveSpray.</summary>
            MS_impact = 1002077,

            /// <summary>OBSOLETE ProteomeDiscoverer's 1st static post-translational modification (PTM) input parameter.</summary>
            MS_ProteomeDiscoverer_1__Static_Modification_OBSOLETE = 1002078,

            /// <summary>OBSOLETE ProteomeDiscoverer's 2nd static post-translational modification (PTM) input parameter.</summary>
            MS_ProteomeDiscoverer_2__Static_Modification_OBSOLETE = 1002079,

            /// <summary>Precursor clipping range before.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Precursor_Clipping_Range_Before = 1002080,

            /// <summary>Precursor clipping range after.</summary>
            MS_ProteomeDiscoverer_Spectrum_Selector_Precursor_Clipping_Range_After = 1002081,

            /// <summary>The time of elution from the first chromatographic column in the chromatographic separation step, relative to the start of chromatography on the first column.</summary>
            MS_first_column_elution_time = 1002082,

            /// <summary>The time of elution from the second chromatographic column in the chromatographic separation step, relative to the start of the chromatography on the second column.</summary>
            MS_second_column_elution_time = 1002083,

            /// <summary>Multidimensional chromatography modulation description.</summary>
            MS_multidimensional_chromatography_modulation_description = 1002084,

            /// <summary>Two-dimensional gas chromatography where a single modulation time is used throughout the acquisition.</summary>
            MS_two_dimensional_gas_chromatography_with_fixed_modulation_time = 1002085,

            /// <summary>Two-dimensional gas chromatography where the acquisition is divided into steps, each with a different modulation time.</summary>
            MS_two_dimensional_gas_chromatography_with_discrete_modulation_time_steps = 1002086,

            /// <summary>Two-dimensional liquid chromatography where a single modulation time is used throughout the acquisition.</summary>
            MS_two_dimensional_liquid_chromatography_with_fixed_modulation_time = 1002087,

            /// <summary>Two-dimensional liquid chromatography where the acquisition is divided into steps, each with a different modulation time.</summary>
            MS_two_dimensional_liquid_chromatography_with_discrete_modulation_time_steps = 1002088,

            /// <summary>XCorr threshold for storing peptides that do not belong to a protein.</summary>
            MS_ProteomeDiscoverer_Peptide_Without_Protein_XCorr_Threshold = 1002089,

            /// <summary>Flag indicating that a probability score for the assessment that a reported peptide match is a random occurrence is calculated.</summary>
            MS_Calculate_Probability_Scores = 1002090,

            /// <summary>Delta Cn threshold for filtering out PSM's.</summary>
            MS_ProteomeDiscoverer_Maximum_Delta_Cn = 1002091,

            /// <summary>Algorithm (e.g. q-value or PEP) used for calculation of the validation score using Percolator.</summary>
            MS_Percolator_Validation_based_on = 1002092,

            /// <summary>Search engine input parameter.</summary>
            MS_search_engine_input_parameter = 1002093,

            /// <summary>Search engine input parameter that is shared by more than one search engine.</summary>
            MS_common_search_engine_input_parameter = 1002094,

            /// <summary>Search engine input parameters specific to Mascot.</summary>
            MS_Mascot_input_parameter = 1002095,

            /// <summary>Search engine input parameters specific to SEQUEST.</summary>
            MS_SEQUEST_input_parameter = 1002096,

            /// <summary>Search engine input parameters specific to Phenyx.</summary>
            MS_Phenyx_input_parameter = 1002097,

            /// <summary>Search engine input parameters specific to ProteinExtractor.</summary>
            MS_ProteinExtractor_input_parameter = 1002098,

            /// <summary>Search engine input parameters specific to OMSSA.</summary>
            MS_OMSSA_input_parameter = 1002099,

            /// <summary>Search engine input parameters specific to ProteinScape.</summary>
            MS_ProteinScape_input_parameter = 1002100,

            /// <summary>Search engine input parameters specific to ProteomeDiscoverer.</summary>
            MS_ProteomeDiscoverer_input_parameter = 1002101,

            /// <summary>Software input parameters.</summary>
            MS_software_input_parameter = 1002103,

            /// <summary>Software input parameter that is shared by more than one software.</summary>
            MS_common_software_input_parameter = 1002104,

            /// <summary>Software specific input parameter.</summary>
            MS_software_specific_input_parameter = 1002105,

            /// <summary>Search engine input parameters specific to Scaffold.</summary>
            MS_Scaffold_input_parameter = 1002106,

            /// <summary>Search engine input parameters specific to Percolator.</summary>
            MS_Percolator_input_parameter = 1002107,

            /// <summary>Indicates that a higher score is better.</summary>
            MS_higher_score_better = 1002108,

            /// <summary>Indicates that a lower score is better.</summary>
            MS_lower_score_better = 1002109,

            /// <summary>Attribute describing an assay.</summary>
            MS_assay_attribute = 1002110,

            /// <summary>Attribute describing an assay label.</summary>
            MS_assay_label_attribute = 1002111,

            /// <summary>Attribute describing a protein group list.</summary>
            MS_protein_group_list_attribute = 1002112,

            /// <summary>Attribute describing a protein group.</summary>
            MS_protein_group_attribute = 1002113,

            /// <summary>Attribute describing a protein list.</summary>
            MS_protein_list_attribute = 1002114,

            /// <summary>Attribute describing a peptide consensus list.</summary>
            MS_peptide_consensus_list_attribute = 1002115,

            /// <summary>Attribute describing a peptide consensus.</summary>
            MS_peptide_consensus_attribute = 1002116,

            /// <summary>Attribute describing a small molecule list.</summary>
            MS_small_molecule_list_attribute = 1002117,

            /// <summary>Attribute describing a small molecule.</summary>
            MS_small_molecule_attribute = 1002118,

            /// <summary>Attribute describing a small molecule modification.</summary>
            MS_small_molecule_modification_attribute = 1002119,

            /// <summary>The name for identifying an experiment.</summary>
            MS_experiment_name = 1002120,

            /// <summary>Dummy decribing a spectral count feature.</summary>
            MS_spectral_count_feature = 1002121,

            /// <summary>FeatureList of spectral counts.</summary>
            MS_counts_reporting = 1002122,

            /// <summary>X-Tracker generic tool for quantitative proteomics.</summary>
            MS_x_Tracker = 1002123,

            /// <summary>ProteoSuite software for the analysis of quantitative proteomics data.</summary>
            MS_ProteoSuite = 1002124,

            /// <summary>OBSOLETE FDRScore values specifically obtained for distinct combinations of single, pairs or triplets of search engines making a given PSM, used for integrating results from these distinct pools.</summary>
            MS_combined_FDRScore_OBSOLETE = 1002125,

            /// <summary>The name of the UniProtKB knowledgebase.</summary>
            MS_database_UniProtKB = 1002126,

            /// <summary>Attribute describing an identification file.</summary>
            MS_identification_file_attribute = 1002127,

            /// <summary>Attribute describing a method file format.</summary>
            MS_method_file_format = 1002128,

            /// <summary>Software for iTRAQ workflow. Extracts and normalizes iTRAQ information from an MS experiment.</summary>
            MS_ITRAQAnalyzer = 1002129,

            /// <summary>Attribute describing an identification file format.</summary>
            MS_identification_file_format = 1002130,

            /// <summary>Noise filter component of the TOPP software.</summary>
            MS_TOPP_noise_filter = 1002131,

            /// <summary>Removes noise from profile spectra by using a gaussian smoothing.</summary>
            MS_TOPP_NoiseFilterGaussian = 1002132,

            /// <summary>Removes noise from profile spectra by using a Savitzky-Golay smoothing.</summary>
            MS_TOPP_NoiseFilterSGolay = 1002133,

            /// <summary>Peak picker component of the TOPP software.</summary>
            MS_TOPP_peak_picker = 1002134,

            /// <summary>Finds mass spectrometric peaks in high-resoluted profile mass spectra.</summary>
            MS_TOPP_PeakPickerHiRes = 1002135,

            /// <summary>Finds mass spectrometric peaks with a wavelet algorithm in low-resoluted profile mass spectra.</summary>
            MS_TOPP_PeakPickerWavelet = 1002136,

            /// <summary>Spectra filter component of the TOPP software.</summary>
            MS_TOPP_spectra_filter = 1002137,

            /// <summary>Applies a Bern et al normalization to peak spectra.</summary>
            MS_TOPP_SpectraFilterBernNorm = 1002138,

            /// <summary>Applies a filter to peak spectra for marked peaks.</summary>
            MS_TOPP_SpectraFilterMarkerMower = 1002139,

            /// <summary>Retains the n largest peaks of a peak spectra.</summary>
            MS_TOPP_SpectraFilterNLargest = 1002140,

            /// <summary>Applies a TIC/maximal intensity normalization to peak spectra.</summary>
            MS_TOPP_SpectraFilterNormalizer = 1002141,

            /// <summary>Filters putative unfragmented precursor ions from tandem spectra.</summary>
            MS_TOPP_SpectraFilterParentPeakMower = 1002142,

            /// <summary>Applies a filter to peak spectra after intensity scaling according to rank.</summary>
            MS_TOPP_SpectraFilterScaler = 1002143,

            /// <summary>Applies a filter to peak spectra after intensity scaling to the square root.</summary>
            MS_TOPP_SpectraFilterSqrtMower = 1002144,

            /// <summary>Applies a filter of peaks below a given threshold to peak spectra.</summary>
            MS_TOPP_SpectraFilterThresholdMower = 1002145,

            /// <summary>Applies a filter of the largest peaks in a sliding window over a peak spectrum.</summary>
            MS_TOPP_SpectraFilterWindowMower = 1002146,

            /// <summary>Map aligner component of the TOPP software.</summary>
            MS_TOPP_map_aligner = 1002147,

            /// <summary>Corrects retention time distortions between maps based on common peptide identifications.</summary>
            MS_TOPP_MapAlignerIdentification = 1002148,

            /// <summary>Corrects retention time distortions between maps using a pose clustering approach.</summary>
            MS_TOPP_MapAlignerPoseClustering = 1002149,

            /// <summary>Corrects retention time distortions between maps by spectrum alignment.</summary>
            MS_TOPP_MapAlignerSpectrum = 1002150,

            /// <summary>Attribute describing the data type of the numerator of a ratio.</summary>
            MS_numerator_data_type_attribute = 1002151,

            /// <summary>Attribute describing the data type of the denominator of a ratio.</summary>
            MS_denominator_data_type_attribute = 1002152,

            /// <summary>The number of spectra identified for this protein in spectral counting.</summary>
            MS_protein_level_PSM_counts = 1002153,

            /// <summary>Extracts spectra of an MS run file to several files in DTA format.</summary>
            MS_TOPP_DTAExtractor = 1002154,

            /// <summary>Merges several protein/peptide identification files into one file.</summary>
            MS_TOPP_IDMerger = 1002155,

            /// <summary>Converts identification engine file formats.</summary>
            MS_TOPP_IDFileConverter = 1002156,

            /// <summary>Merges spectra from an LC/MS map, either by precursor or by RT blocks.</summary>
            MS_TOPP_SpectraMerger = 1002157,

            /// <summary>Exports various XML formats to an mzTab file.</summary>
            MS_TOPP_MzTabExporter = 1002158,

            /// <summary>Annotates mass traces in centroided LC/MS maps.</summary>
            MS_TOPP_MassTraceExtractor = 1002159,

            /// <summary>Correct the precursor entries of tandem MS scans.</summary>
            MS_TOPP_PrecursorMassCorrector = 1002160,

            /// <summary>Performs precursor mz correction on centroided high resolution data.</summary>
            MS_TOPP_HighResPrecursorMassCorrector = 1002161,

            /// <summary>Computes an additive series to quantify a peptide in a set of samples.</summary>
            MS_TOPP_AdditiveSeries = 1002162,

            /// <summary>Decharges and merges different feature charge variants of the same chemical entity.</summary>
            MS_TOPP_Decharger = 1002163,

            /// <summary>Quantifies signals at given positions in (raw or picked) LC/MS maps.</summary>
            MS_TOPP_EICExtractor = 1002164,

            /// <summary>Feature finder component of the TOPP software.</summary>
            MS_TOPP_feature_finder = 1002165,

            /// <summary>Detects two-dimensional features in centroided LC-MS data.</summary>
            MS_TOPP_FeatureFinderCentroided = 1002166,

            /// <summary>Detects two-dimensional features in uncentroided LC-MS data.</summary>
            MS_TOPP_FeatureFinderRaw = 1002167,

            /// <summary>Detects two-dimensional features in uncentroided LC-MS data with a wavelet algorithm.</summary>
            MS_TOPP_FeatureFinderIsotopeWavelet = 1002168,

            /// <summary>Detects two-dimensional features in centroided LC-MS data of metabolites.</summary>
            MS_TOPP_FeatureFinderMetabo = 1002169,

            /// <summary>Quantifies features LC-MS/MS MRM data.</summary>
            MS_TOPP_FeatureFinderMRM = 1002170,

            /// <summary>Computes protein abundances from annotated feature/consensus maps.</summary>
            MS_TOPP_ProteinQuantifier = 1002171,

            /// <summary>Normalizes maps of one consensus XML file (after linking).</summary>
            MS_TOPP_ConsensusMapNormalizer = 1002172,

            /// <summary>Applies retention time transformations to maps.</summary>
            MS_TOPP_MapRTTransformer = 1002173,

            /// <summary>Feature linker component of the TOPP software.</summary>
            MS_TOPP_feature_linker = 1002174,

            /// <summary>Groups corresponding isotope-labeled features in a feature map.</summary>
            MS_TOPP_FeatureLinkerLabeled = 1002175,

            /// <summary>Groups corresponding features from multiple maps.</summary>
            MS_TOPP_FeatureLinkerUnlabeled = 1002176,

            /// <summary>Groups corresponding features from multiple maps using a quality threshold clustering approach.</summary>
            MS_TOPP_FeatureLinkerUnlabeledQT = 1002177,

            /// <summary>Performs a peptide/protein identification with the CompNovo engine.</summary>
            MS_TOPP_CompNovo = 1002178,

            /// <summary>Performs a peptide/protein identification with the CompNovo engine in collision-induced dissociation (CID) mode.</summary>
            MS_TOPP_CompNovoCID = 1002179,

            /// <summary>Software adaptor to an external program in the TOPP software.</summary>
            MS_TOPP_software_adaptor = 1002180,

            /// <summary>Identifies MS2 spectra using the external program Inspect.</summary>
            MS_TOPP_InspectAdapter = 1002181,

            /// <summary>Identifies MS2 spectra using the external program Mascot.</summary>
            MS_TOPP_MascotAdapter = 1002182,

            /// <summary>Identifies MS2 spectra using the online version of the external program Mascot.</summary>
            MS_TOPP_MascotAdapterOnline = 1002183,

            /// <summary>Identifies MS2 spectra using the external program OMSSA.</summary>
            MS_TOPP_OMSSAAdapter = 1002184,

            /// <summary>Identifies MS2 spectra using the external program PepNovo.</summary>
            MS_TOPP_PepNovoAdapter = 1002185,

            /// <summary>Identifies MS2 spectra using the external program XTandem.</summary>
            MS_TOPP_XTandemAdapter = 1002186,

            /// <summary>Identifies peptide MS2 spectra by spectral matching with a searchable spectral library.</summary>
            MS_TOPP_SpecLibSearcher = 1002187,

            /// <summary>Computes a consensus identification from peptide identifications of several identification engines.</summary>
            MS_TOPP_ConsensusID = 1002188,

            /// <summary>Resolves ambiguous annotations of features with peptide identifications.</summary>
            MS_TOPP_IDConflictResolver = 1002189,

            /// <summary>Filters results from protein or peptide identification engines based on different criteria.</summary>
            MS_TOPP_IDFilter = 1002190,

            /// <summary>Assigns protein/peptide identifications to feature or consensus features.</summary>
            MS_TOPP_IDMapper = 1002191,

            /// <summary>Estimates posterior error probabilities using a mixture model.</summary>
            MS_TOPP_IDPosteriorErrorProbability = 1002192,

            /// <summary>Calibrate Retention times of peptide hits to standards.</summary>
            MS_TOPP_IDRTCalibration = 1002193,

            /// <summary>Refreshes the protein references for all peptide hits.</summary>
            MS_TOPP_PeptideIndexer = 1002194,

            /// <summary>A tool for precursor ion selection based on identification results.</summary>
            MS_TOPP_PrecursorIonSelector = 1002195,

            /// <summary>MRMMapper maps measured chromatograms (mzML) and the transitions used (TraML).</summary>
            MS_TOPP_MRMMapper = 1002196,

            /// <summary>OpenSwath component of the TOPP software.</summary>
            MS_TOPP_OpenSwath_component = 1002197,

            /// <summary>Picks peaks and finds features in an SRM experiment.</summary>
            MS_TOPP_OpenSwathAnalyzer = 1002198,

            /// <summary>Extract chromatograms (XIC) from a MS2 map file.</summary>
            MS_TOPP_OpenSwathChromatogramExtractor = 1002199,

            /// <summary>Generates decoys according to different models for a specific TraML.</summary>
            MS_TOPP_OpenSwathDecoyGenerator = 1002200,

            /// <summary>Converts a featureXML to a mProphet tsv (tab separated values).</summary>
            MS_TOPP_OpenSwathFeatureXMLToTSV = 1002201,

            /// <summary>Generates a transformation file for retention time space into normalized space.</summary>
            MS_TOPP_OpenSwathRTNormalizer = 1002202,

            /// <summary>Infer proteins from a list of (high-confidence) peptides.</summary>
            MS_TOPP_ProteinInference = 1002203,

            /// <summary>Estimates the false discovery rate on peptide and protein level using decoy searches.</summary>
            MS_TOPP_FalseDiscoveryRate = 1002204,

            /// <summary>Converts, filters, and processes mass spectrometry data in variety of formats.</summary>
            MS_ProteoWizard_msconvert = 1002205,

            /// <summary>Converts, filters, and processes identifications from shotgun proteomics experiments.</summary>
            MS_ProteoWizard_idconvert = 1002206,

            /// <summary>Filters and processes protein sequence databases.</summary>
            MS_ProteoWizard_chainsaw = 1002207,

            /// <summary>Filters, processes, and displays mass spectrometry data in a variety of ways.</summary>
            MS_ProteoWizard_msaccess = 1002208,

            /// <summary>An interactive GUI application to view and filter mass spectrometry data in a variety of formats.</summary>
            MS_ProteoWizard_SeeMS = 1002209,

            /// <summary>A quantitative software package designed for analysis of IPTL, TMT and iTRAQ data.</summary>
            MS_IsobariQ = 1002210,

            /// <summary>The model incorporates data calibration (normalization), a model for the dependence of the variance on the mean intensity, and a variance stabilizing data transformation.</summary>
            MS_Variance_stabilizing_normalization = 1002211,

            /// <summary>Quantification analysis using a labelling strategy where both peptide termini are labelled so that the peptides from different labelling schema are isobaric.</summary>
            MS_IPTL_quantitation_analysis = 1002212,

            /// <summary>A protein identified by at least one unique (distinct, discrete) peptide (peptides are considered different only if they can be distinguished by evidence in mass spectrum).</summary>
            MS_PAnalyzer_conclusive_protein = 1002213,

            /// <summary>A member of a group of proteins sharing all peptides that are exclusive to the group (peptides are considered different only if they can be distinguished by evidence in mass spectrum).</summary>
            MS_PAnalyzer_indistinguishable_protein = 1002214,

            /// <summary>A protein sharing all its matched peptides with either conclusive or indistinguishable proteins (peptides are considered different only if they can be distinguished by evidence in mass spectrum).</summary>
            MS_PAnalyzer_non_conclusive_protein = 1002215,

            /// <summary>A protein sharing at least one peptide not matched to either conclusive or indistinguishable proteins (peptides are considered different only if they can be distinguished by evidence in mass spectrum).</summary>
            MS_PAnalyzer_ambiguous_group_member = 1002216,

            /// <summary>A putative identified peptide issued from a decoy sequence database.</summary>
            MS_decoy_peptide = 1002217,

            /// <summary>Collision energy at the start of the collision energy ramp in percent, normalized to the mass of the ion.</summary>
            MS_percent_collision_energy_ramp_start = 1002218,

            /// <summary>Collision energy at the end of the collision energy ramp in percent, normalized to the mass of the ion.</summary>
            MS_percent_collision_energy_ramp_end = 1002219,

            /// <summary>A web-based SRM assay design tool whose transitions are generated by mining the millions of identified peptide spectra held in the EBI's PRIDE database.</summary>
            MS_MRMaid = 1002220,

            /// <summary>Score in MRMaid to indicate the expected performance of the peptide in SRM.</summary>
            MS_MRMaid_peptide_score = 1002221,

            /// <summary>Attribute associated with a SRM transition.</summary>
            MS_SRM_transition_attribute = 1002222,

            /// <summary>Probability of detecting precursor when parent protein is present.</summary>
            MS_precursor_ion_detection_probability = 1002223,

            /// <summary>Probability of detecting product ion when precursor ion is present.</summary>
            MS_product_ion_detection_probability = 1002224,

            /// <summary>Average value of product ion intensity in a collection of identified spectra.</summary>
            MS_average_product_ion_intensity = 1002225,

            /// <summary>Standard deviation of product ion intensity in a collection of identified spectra.</summary>
            MS_product_ion_intensity_standard_deviation = 1002226,

            /// <summary>The number of times the specific product ion has been observed in a series of SRM experiments.</summary>
            MS_number_of_product_ion_observations = 1002227,

            /// <summary>The number of times the specific precursor ion has been observed in a series of SRM experiments.</summary>
            MS_number_of_precursor_ion_observations = 1002228,

            /// <summary>Calculated relaxed significance when performing a decoy search for high-confidence peptides.</summary>
            MS_ProteomeDiscoverer_Mascot_Significance_Middle = 1002229,

            /// <summary>Calculated relaxed significance when performing a decoy search for medium-confidence peptides.</summary>
            MS_ProteomeDiscoverer_Mascot_Significance_High = 1002230,

            /// <summary>(</summary>
            MS_regular_expressions_for_a_GUID = 1002231,

            /// <summary>The default FDR calculator as globally unique identifier (GUID).</summary>
            MS_ProteomeDiscoverer_Default_FDR_calculator = 1002232,

            /// <summary>Flag indicating if low-resolution spectra are taken into consideration.</summary>
            MS_ProteomeDiscoverer_SEQUEST_Low_resolution_spectra_contained = 1002233,

            /// <summary>Mass-to-charge ratio of a precursor ion selected for fragmentation.</summary>
            MS_selected_precursor_m_z = 1002234,

            /// <summary>A score assigned to a single protein accession (modelled as ProteinDetectionHypothesis in mzIdentML), based on summed peptide level scores.</summary>
            MS_ProteoGrouper_PDH_score = 1002235,

            /// <summary>A score assigned to a protein group (modelled as ProteinAmbiguityGroup in mzIdentML), based on all summed peptide level scores that have been assigned to the group as unique or razor peptides.</summary>
            MS_ProteoGrouper_PAG_score = 1002236,

            /// <summary>A library of Java routines for manipulating mzIdentML files.</summary>
            MS_mzidLib = 1002237,

            /// <summary>A converter for OMSSA OMX to mzIdentML.</summary>
            MS_mzidLib_Omssa2Mzid = 1002238,

            /// <summary>A converter for Tandem XML to mzIdentML.</summary>
            MS_mzidLib_Tandem2Mzid = 1002239,

            /// <summary>A converter for CSV files (following OMSSA CSV style) to mzIdentML.</summary>
            MS_mzidLib_Csv2Mzid = 1002240,

            /// <summary>A generic and parameterizable protein inference algorithm for mzIdentML files.</summary>
            MS_mzidLib_ProteoGrouper = 1002241,

            /// <summary>A routine for keeping only identifications passing a given threshold or setting passThreshold to true or false for SpectrumIdentificationItem or ProteinDetectionHypothesis in mzIdentML files.</summary>
            MS_mzidLib_Thresholder = 1002242,

            /// <summary>A routine for adding emPAI quantitative values to an mzIdentML file.</summary>
            MS_mzidLib_Perform_emPAI_on_mzid = 1002243,

            /// <summary>A routine for calculating local FDR, q-value and FDRScore for mzIdentML files, based on a decoy search.</summary>
            MS_mzidLib_FalseDiscoveryRate = 1002244,

            /// <summary>A tool for converting mzIdentML files to CSV format.</summary>
            MS_mzidLib_Mzidentml2Csv = 1002245,

            /// <summary>A tool for combining results analysed in parallel in two or three search engines into a single mzIdentML file.</summary>
            MS_mzidLib_CombineSearchEngines = 1002246,

            /// <summary>A tool for adding additional meta data from a FASTA file to DBSequence entries (sequence and description) in mzIdentML files.</summary>
            MS_mzidLib_InsertMetaDataFromFasta = 1002247,

            /// <summary>The SEQUEST result 'SpScore'.</summary>
            MS_SEQUEST_spscore = 1002248,

            /// <summary>The SEQUEST result 'SpRank'.</summary>
            MS_SEQUEST_sprank = 1002249,

            /// <summary>The SEQUEST result 'DeltaCnStar'.</summary>
            MS_SEQUEST_deltacnstar = 1002250,

            /// <summary>Comet open-source sequence search engine developed at the University of Washington.</summary>
            MS_Comet = 1002251,

            /// <summary>The Comet result 'XCorr'.</summary>
            MS_Comet_xcorr = 1002252,

            /// <summary>The Comet result 'DeltaCn'.</summary>
            MS_Comet_deltacn = 1002253,

            /// <summary>The Comet result 'DeltaCnStar'.</summary>
            MS_Comet_deltacnstar = 1002254,

            /// <summary>The Comet result 'SpScore'.</summary>
            MS_Comet_spscore = 1002255,

            /// <summary>The Comet result 'SpRank'.</summary>
            MS_Comet_sprank = 1002256,

            /// <summary>The Comet result 'Expectation value'.</summary>
            MS_Comet_expectation_value = 1002257,

            /// <summary>The Comet result 'Matched Ions'.</summary>
            MS_Comet_matched_ions = 1002258,

            /// <summary>The Comet result 'Total Ions'.</summary>
            MS_Comet_total_ions = 1002259,

            /// <summary>False-discovery rate threshold for peptide-spectrum matches.</summary>
            MS_PSM_FDR_threshold = 1002260,

            /// <summary>Byonic search engine from Protein Metrics.</summary>
            MS_Byonic = 1002261,

            /// <summary>The Byonic score is the primary indicator of PSM correctness. The Byonic score reflects the absolute quality of the peptide-spectrum match, not the relative quality compared to other candidate peptides. Byonic scores range from 0 to about 1000, with 300 a good score, 400 a very good score, and PSMs with scores over 500 almost sure to be correct.</summary>
            MS_Byonic_Score = 1002262,

            /// <summary>The drop in Byonic score from the top-scoring peptide to the next peptide with distinct sequence. In this computation, the same peptide with different modifications is not considered distinct.</summary>
            MS_Byonic_Delta_Score = 1002263,

            /// <summary>The drop in Byonic score from the top-scoring peptide to the next peptide different in any way, including placement of modifications. DeltaMod gives an indication of whether modifications are confidently localized; DeltaMod over 10.0 means that there is high likelihood that all modification placements are correct.</summary>
            MS_Byonic_DeltaMod_Score = 1002264,

            /// <summary>Byonic posterior error probability.</summary>
            MS_Byonic_PEP = 1002265,

            /// <summary>The log p-value of the PSM. This is the log of the probability that the PSM with such a score and delta would arise by chance in a search of this size (the size of the protein database, as expanded by the modification rules). A log p-value of -3.0 should happen by chance on only one of a thousand spectra. Caveat: it is very hard to compute a p-value that works for all searches and all spectra, so read Byonic p-values with a certain amount of skepticism.</summary>
            MS_Byonic_Peptide_LogProb = 1002266,

            /// <summary>The log p-value of the protein.</summary>
            MS_Byonic_Protein_LogProb = 1002267,

            /// <summary>Best (most negative) log p-value of an individual PSM.</summary>
            MS_Byonic_Best_LogProb = 1002268,

            /// <summary>Best (largest) Byonic score of a PSM.</summary>
            MS_Byonic_Best_Score = 1002269,

            /// <summary>A technique by which molecules are separated by chemical and physical properties such as hydrophobicity or vapour pressure.</summary>
            MS_chromatography_separation = 1002270,

            /// <summary>Liquid chromatography (LC) is a separation technique in which the mobile phase is a liquid.</summary>
            MS_liquid_chromatography_separation = 1002271,

            /// <summary>Gas chromatography (GC) is a separation technique in which the mobile phase is a gas.</summary>
            MS_gas_chromatography_separation = 1002272,

            /// <summary>Detector potential difference in volts.</summary>
            MS_detector_potential = 1002273,

            /// <summary>Waters quadrupole based SQ Detector 2.</summary>
            MS_SQ_Detector_2 = 1002274,

            /// <summary>Waters oa-ToF based Xevo G2-S Tof.</summary>
            MS_Xevo_G2_S_Tof = 1002275,

            /// <summary>Waters oa-ToF based Xevo G2-S QTof.</summary>
            MS_Xevo_G2_S_QTof = 1002276,

            /// <summary>Waters AutoSpec Premier magnetic sector instrument.</summary>
            MS_AutoSpec_Premier = 1002277,

            /// <summary>LECO nominal mass resolution time-of-flight GC mass spectrometer.</summary>
            MS_Pegasus_III = 1002278,

            /// <summary>Bruker Daltonics' maXis 4G: ESI Q-TOF, Nanospray, APCI, APPI, GC-APCI, CaptiveSpray.</summary>
            MS_maXis_4G = 1002279,

            /// <summary>Bruker Daltonics' compact: ESI Q-TOF, Nanospray, APCI, APPI, GC-APCI, CaptiveSpray.</summary>
            MS_compact = 1002280,

            /// <summary>Selected Reaction Monitoring feature level quantitation.</summary>
            MS_SRM_feature_level_quantitation = 1002281,

            /// <summary>Selected Reaction Monitoring peptide level quantitation.</summary>
            MS_SRM_peptide_level_quantitation = 1002282,

            /// <summary>Selected Reaction Monitoring protein level quantitation.</summary>
            MS_SRM_protein_level_quantitation = 1002283,

            /// <summary>Selected Reaction Monitoring proteingroup level quantitation.</summary>
            MS_SRM_proteingroup_level_quantitation = 1002284,

            /// <summary>A suite of open source tools for the processing of MS2 proteomics data developed by the Seattle Proteome Center at the Institute for Systems Biology.</summary>
            MS_Trans_Proteomic_Pipeline = 1002285,

            /// <summary>A software program that is a component of the Trans-Proteomic Pipeline.</summary>
            MS_Trans_Proteomic_Pipeline_software = 1002286,

            /// <summary>A program in the TPP that calculates PSM probabilities for MS2 proteomics data searched with any of the supported sequence or spectral library search engines via the pepXML format.</summary>
            MS_PeptideProphet = 1002287,

            /// <summary>A program in the TPP that calculates distinct peptide probabilities based on several lines of corroborating evidence including search results from multiple search engines via the pepXML format.</summary>
            MS_iProphet = 1002288,

            /// <summary>A program in the TPP that calculates protein-level probabilities based on input PSM or peptide-level probabilities from PeptideProphet or iProphet. The output is written in the protXML format.</summary>
            MS_ProteinProphet = 1002289,

            /// <summary>A program in the TPP that calculates PSM-level abundances based on 2-channel isotope-labelled data such as ICAT, SILAC, etc.</summary>
            MS_XPRESS = 1002290,

            /// <summary>A program in the TPP that calculates PSM, peptide, and protein-level abundances based on N-channel isobaric label peptide data such as iTRAQ, TMT, etc.</summary>
            MS_Libra = 1002291,

            /// <summary>A program in the TPP that calculates PTM localization probabilities by re-analyzing the peaks that are available to distinguish between possible modification sites.</summary>
            MS_PTMProphet = 1002292,

            /// <summary>Bruker Daltonics' SCION series.</summary>
            MS_Bruker_Daltonics_SCION_series = 1002293,

            /// <summary>Bruker Daltonics' EVOQ series.</summary>
            MS_Bruker_Daltonics_EVOQ_series = 1002294,

            /// <summary>Bruker Daltonics' SCION SQ: GC-single quadrupole.</summary>
            MS_SCION_SQ = 1002295,

            /// <summary>Bruker Daltonics' SCION TQ: GC-triple quadrupole.</summary>
            MS_SCION_TQ = 1002296,

            /// <summary>Bruker Daltonics' EVOQ Elite: LC-triple quadrupole.</summary>
            MS_EVOQ_Elite = 1002297,

            /// <summary>Bruker Daltonics' EVOQ Qube: LC-triple quadrupole.</summary>
            MS_EVOQ_Qube = 1002298,

            /// <summary>Bruker Daltonics' micrOTOF-Q III: ESI Q-TOF, Nanospray, APCI, APPI, GC-APCI, CaptiveSpray.</summary>
            MS_micrOTOF_Q_III = 1002299,

            /// <summary>Bruker Daltonics' amaZon Speed ETD: ESI quadrupole ion trap, Nanospray, APCI, APPI, ETD, PTR, GC-APCI, CaptiveSpray.</summary>
            MS_amaZon_Speed_ETD = 1002300,

            /// <summary>Bruker Daltonics' amaZon ETD: ESI quadrupole ion trap, Nanospray, APCI, APPI, GC-APCI, CaptiveSpray.</summary>
            MS_amaZon_Speed = 1002301,

            /// <summary>Bruker Container raw file format.</summary>
            MS_Bruker_Container_format = 1002302,

            /// <summary>Native identifier (UUID).</summary>
            MS_Bruker_Container_nativeID_format = 1002303,

            /// <summary>Domain range of a numerical value.</summary>
            MS_domain_range = 1002304,

            /// <summary>Value range for probabilities.</summary>
            MS_value_between_0_and_1_inclusive = 1002305,

            /// <summary>Positive value range.</summary>
            MS_value_greater_than_zero = 1002306,

            /// <summary>Type of fragment ion based on where the backbone breaks, such as a y ion or a c ion.</summary>
            MS_fragmentation_ion_type = 1002307,

            /// <summary>A detector using a fluorescent signal after excitation with light.</summary>
            MS_fluorescence_detector = 1002308,

            /// <summary>The absolute value of the log-base10 of the Byonic posterior error probability (PEP) of the PSM.</summary>
            MS_Byonic__Peptide_AbsLogProb = 1002309,

            /// <summary>The absolute value of the log-base10 of the Byonic posterior error probability (PEP) of the protein.</summary>
            MS_Byonic__Protein_AbsLogProb = 1002310,

            /// <summary>The absolute value of the log-base10 Byonic two-dimensional posterior error probability (PEP) of the PSM. The two-dimensional PEP takes into account protein ranking information as well as PSM information.</summary>
            MS_Byonic__Peptide_AbsLogProb2D = 1002311,

            /// <summary>Compression using MS-Numpress linear prediction compression.</summary>
            MS_MS_Numpress_linear_prediction_compression = 1002312,

            /// <summary>Compression using MS-Numpress positive integer compression.</summary>
            MS_MS_Numpress_positive_integer_compression = 1002313,

            /// <summary>Compression using MS-Numpress short logged float compression.</summary>
            MS_MS_Numpress_short_logged_float_compression = 1002314,

            /// <summary>Indicates a consensus result from several search engine runs.</summary>
            MS_consensus_result = 1002315,

            /// <summary>Strict confidence probability score.</summary>
            MS_ProteomeDiscoverer_Amanda_high_confidence_threshold = 1002316,

            /// <summary>Relaxed confidence probability score.</summary>
            MS_ProteomeDiscoverer_Amanda_middle_confidence_threshold = 1002317,

            /// <summary>Flag indicating automatic estimation of the workload level.</summary>
            MS_ProteomeDiscoverer_automatic_workload = 1002318,

            /// <summary>The Amanda score of the scoring function for a PSM.</summary>
            MS_Amanda_AmandaScore = 1002319,

            /// <summary>Maximum dynamic modifications per PSM.</summary>
            MS_ProteomeDiscoverer_max_differential_modifications = 1002320,

            /// <summary>Maximum equal modifications per PSM.</summary>
            MS_ProteomeDiscoverer_max_equal_modifications = 1002321,

            /// <summary>Minimum peptide length.</summary>
            MS_ProteomeDiscoverer_min_peptide_length = 1002322,

            /// <summary>Maximum peptide length.</summary>
            MS_ProteomeDiscoverer_max_peptide_length = 1002323,

            /// <summary>Maximum number of same neutral losses.</summary>
            MS_ProteomeDiscoverer_max_number_neutral_loss = 1002324,

            /// <summary>Max number of same neutral losses of modifications.</summary>
            MS_ProteomeDiscoverer_max_number_neutral_loss_modifications = 1002325,

            /// <summary>Flag for usage of flanking ions.</summary>
            MS_ProteomeDiscoverer_use_flanking_ions = 1002326,

            /// <summary>The maximum number of possible equal modifications per PSM.</summary>
            MS_ProteomeDiscoverer_max_number_of_same_modifs = 1002327,

            /// <summary>Defines whether a simple deisotoping shall be performed.</summary>
            MS_ProteomeDiscoverer_perform_deisotoping = 1002328,

            /// <summary>Specifies the fragment ions and neutral losses that are calculated.</summary>
            MS_ProteomeDiscoverer_ion_settings = 1002329,

            /// <summary>OBSOLETE ProteomeDiscoverer's 3rd static post-translational modification (PTM) input parameter.</summary>
            MS_ProteomeDiscoverer_3__Static_Modification_OBSOLETE = 1002330,

            /// <summary>OBSOLETE ProteomeDiscoverer's 5th dynamic post-translational modification (PTM) input parameter.</summary>
            MS_ProteomeDiscoverer_5__Dynamic_Modification_OBSOLETE = 1002331,

            /// <summary>The scientist responsible for personnel, grants, and instrumentation in a functional laboratory group.</summary>
            MS_lab_head = 1002332,

            /// <summary>Computer software primarily designed to convert data represented in one format to another format, sometimes with minor data alterations in the process.</summary>
            MS_conversion_software = 1002333,

            /// <summary>Java software designed to convert one of several proteomics identification results formats into mzIdentML or PRIDE XML.</summary>
            MS_ProCon = 1002334,

            /// <summary>Java software designed to convert one of several proteomics identification results formats into PRIDE XML.</summary>
            MS_PRIDE_Converter2 = 1002335,

            /// <summary>Amanda scoring system for PSM identification.</summary>
            MS_Amanda = 1002336,

            /// <summary>Andromeda is a peptide search engine.</summary>
            MS_Andromeda = 1002337,

            /// <summary>The probability based score of the Andromeda search engine.</summary>
            MS_Andromeda_score = 1002338,

            /// <summary>Estimation of global false discovery rate of peptides with a post-translational modification.</summary>
            MS_site_global_FDR = 1002339,

            /// <summary>Tag that can be added to a ProteomeXchange dataset, to enable the grouping of datasets. One tag can be used for indicating that a given dataset is part of a bigger project, like e.g. the Human Proteome Project.</summary>
            MS_ProteomeXchange_project_tag = 1002340,

            /// <summary>A putative identified peptide found in a second-pass search of protein sequences selected from a first-pass search.</summary>
            MS_second_pass_peptide_identification = 1002341,

            /// <summary>A framework for differential analysis of mass spectrometry data.</summary>
            MS_MZmine = 1002342,

            /// <summary>OBSOLETE Stability type of the ion.</summary>
            MS_ion_stability_type_OBSOLETE = 1002343,

            /// <summary>Modular Application Toolkit for Chromatography Mass-Spectrometry is an application framework mainly for developers.</summary>
            MS_Maltcms = 1002344,

            /// <summary>Attribute of a single peptide-spectrum match.</summary>
            MS_PSM_level_attribute = 1002345,

            /// <summary>Protein group level information.</summary>
            MS_protein_group_level_identification_attribute = 1002346,

            /// <summary>Identification confidence metric for a peptide spectrum match.</summary>
            MS_PSM_level_identification_statistic = 1002347,

            /// <summary>Identification confidence metric for a protein group.</summary>
            MS_protein_group_level_identification_statistic = 1002348,

            /// <summary>Positive value range less than or equal to 1.</summary>
            MS_value_greater_than_zero_but_less_than_or_equal_to_one = 1002349,

            /// <summary>Estimation of the global false discovery rate of peptide spectrum matches.</summary>
            MS_PSM_level_global_FDR = 1002350,

            /// <summary>Estimation of the local false discovery rate of peptide spectrum matches.</summary>
            MS_PSM_level_local_FDR = 1002351,

            /// <summary>Estimation of the p-value for peptide spectrum matches.</summary>
            MS_PSM_level_p_value = 1002352,

            /// <summary>Estimation of the e-value for peptide spectrum matches.</summary>
            MS_PSM_level_e_value = 1002353,

            /// <summary>Estimation of the q-value for peptide spectrum matches.</summary>
            MS_PSM_level_q_value = 1002354,

            /// <summary>mzidLibrary FDRScore for peptide spectrum matches.</summary>
            MS_PSM_level_FDRScore = 1002355,

            /// <summary>mzidLibrary Combined FDRScore for peptide spectrum matches specifically obtained for distinct combinations of single, pairs or triplets of search engines making a given PSM, used for integrating results from these distinct pools.</summary>
            MS_PSM_level_combined_FDRScore = 1002356,

            /// <summary>Probability that the reported peptide ion is truly responsible for some or all of the components of the specified mass spectrum.</summary>
            MS_PSM_level_probability = 1002357,

            /// <summary>Search engine specific distinct peptide score.</summary>
            MS_search_engine_specific_peptide_sequence_level_identification_statistic = 1002358,

            /// <summary>Estimation of the local false discovery rate for distinct peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs have been collapsed to one entry).</summary>
            MS_peptide_sequence_level_local_FDR = 1002359,

            /// <summary>MzidLibrary FDRScore for distinct peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs have been collapsed to one entry).</summary>
            MS_distinct_peptide_level_FDRScore = 1002360,

            /// <summary>Combined FDRScore for peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs have been collapsed to one entry) specifically obtained for distinct combinations of single, pairs or triplets of search engines making a given peptide, used for integrating results from these distinct pools.</summary>
            MS_distinct_peptide_level_combined_FDRScore = 1002361,

            /// <summary>Probability that the reported distinct peptide sequence (irrespective of mass modifications) has been correctly identified via the referenced PSMs.</summary>
            MS_peptide_sequence_level_probability = 1002362,

            /// <summary>Search engine specific protein scores.</summary>
            MS_search_engine_specific_score_for_proteins = 1002363,

            /// <summary>Estimation of the local false discovery rate of proteins.</summary>
            MS_protein_level_local_FDR = 1002364,

            /// <summary>MzidLibrary FDRScore for proteins specifically obtained for distinct combinations of single, pairs or triplets of search engines making a given PSM, used for integrating results from these distinct pools.</summary>
            MS_FDRScore_for_proteins = 1002365,

            /// <summary>MzidLibrary Combined FDRScore for proteins.</summary>
            MS_combined_FDRScore_for_proteins = 1002366,

            /// <summary>Probability that a specific protein sequence has been correctly identified from the PSM and distinct peptide evidence, and based on the available protein sequences presented to the analysis software.</summary>
            MS_probability_for_proteins = 1002367,

            /// <summary>Search engine specific protein group scores.</summary>
            MS_search_engine_specific_score_for_protein_groups = 1002368,

            /// <summary>Estimation of the global false discovery rate of protein groups.</summary>
            MS_protein_group_level_global_FDR = 1002369,

            /// <summary>Estimation of the local false discovery rate of protein groups.</summary>
            MS_protein_group_level_local_FDR = 1002370,

            /// <summary>Estimation of the p-value for protein groups.</summary>
            MS_protein_group_level_p_value = 1002371,

            /// <summary>Estimation of the e-value for protein groups.</summary>
            MS_protein_group_level_e_value = 1002372,

            /// <summary>Estimation of the q-value for protein groups.</summary>
            MS_protein_group_level_q_value = 1002373,

            /// <summary>mzidLibrary FDRScore for protein groups.</summary>
            MS_protein_group_level_FDRScore = 1002374,

            /// <summary>mzidLibrary Combined FDRScore for proteins specifically obtained for distinct combinations of single, pairs or triplets of search engines making a given PSM, used for integrating results from these distinct pools.</summary>
            MS_protein_group_level_combined_FDRScore = 1002375,

            /// <summary>Probability that at least one of the members of a group of protein sequences has been correctly identified from the PSM and distinct peptide evidence, and based on the available protein sequences presented to the analysis software.</summary>
            MS_protein_group_level_probability = 1002376,

            /// <summary>Specifies the threshold value for relaxed scoring.</summary>
            MS_ProteomeDiscoverer_Relaxed_Score_Threshold = 1002377,

            /// <summary>Specifies the threshold value for strict scoring.</summary>
            MS_ProteomeDiscoverer_Strict_Score_Threshold = 1002378,

            /// <summary>Cut off score for storing peptides that do not belong to a protein.</summary>
            MS_ProteomeDiscoverer_Peptide_Without_Protein_Cut_Off_Score = 1002379,

            /// <summary>Estimation of the false localization rate for modification site assignment.</summary>
            MS_false_localization_rate = 1002380,

            /// <summary>Software for automated LC-MALDI analysis and reporting.</summary>
            MS_MALDI_Solutions_LC_MALDI = 1002381,

            /// <summary>Shimadzu MALDI-7090: MALDI-TOF-TOF.</summary>
            MS_Shimadzu_MALDI_7090 = 1002382,

            /// <summary>SCiLS software for data acquisition and analysis.</summary>
            MS_SCiLS_software = 1002383,

            /// <summary>SCiLS Lab software.</summary>
            MS_SCiLS_Lab = 1002384,

            /// <summary>SCiLS Lab file format.</summary>
            MS_SCiLS_Lab_format = 1002385,

            /// <summary>Preprocessing software.</summary>
            MS_preprocessing_software = 1002386,

            /// <summary>PIA - Protein Inference Algorithms, a toolbox for protein inference and identification analysis.</summary>
            MS_PIA = 1002387,

            /// <summary>A compilation of search engine results in the PIA XML format.</summary>
            MS_PIA_XML_format = 1002388,

            /// <summary>A parameter set for a single PIA analysis.</summary>
            MS_PIA_workflow_parameter = 1002389,

            /// <summary>Indicates whether the FDR score was calculated for the input file.</summary>
            MS_PIA_FDRScore_calculated = 1002390,

            /// <summary>Indicates whether the combined FDR score was calculated for the PIA compilation.</summary>
            MS_PIA_Combined_FDRScore_calculated = 1002391,

            /// <summary>Indicates whether PSM sets were created.</summary>
            MS_PIA_PSM_sets_created = 1002392,

            /// <summary>The number of top identifications per spectrum used for the FDR calculation, 0 means all.</summary>
            MS_PIA_used_top_identifications_for_FDR = 1002393,

            /// <summary>The score given to a protein by any protein inference.</summary>
            MS_PIA_protein_score = 1002394,

            /// <summary>The used algorithm for the protein inference using PIA.</summary>
            MS_PIA_protein_inference = 1002395,

            /// <summary>A filter used by PIA for the protein inference.</summary>
            MS_PIA_protein_inference_filter = 1002396,

            /// <summary>The used scoring method for the protein inference using PIA.</summary>
            MS_PIA_protein_inference_scoring = 1002397,

            /// <summary>The used base score for the protein inference using PIA.</summary>
            MS_PIA_protein_inference_used_score = 1002398,

            /// <summary>The method to determine the PSMs used for scoring by the protein inference.</summary>
            MS_PIA_protein_inference_used_PSMs = 1002399,

            /// <summary>A filter used for the report generation.</summary>
            MS_PIA_filter = 1002400,

            /// <summary>At least one protein within each group should be annotated as a leading protein to indicate it has the strongest evidence, or approximately equal evidence as other group members.</summary>
            MS_leading_protein = 1002401,

            /// <summary>Zero to many proteins within each group should be annotated as non-leading to indicate that other proteins have stronger evidence.</summary>
            MS_non_leading_protein = 1002402,

            /// <summary>An arbitrary and optional flag applied to exactly one protein per group to indicate it can serve as the representative of the group, amongst leading proteins, in effect serving as a tiebreaker for approaches that require exactly one group representative.</summary>
            MS_group_representative = 1002403,

            /// <summary>The number of proteins that have been identified, which must match the number of groups that pass the threshold in the file.</summary>
            MS_count_of_identified_proteins = 1002404,

            /// <summary>Details describing a protein cluster.</summary>
            MS_protein_group_level_result_list_attribute = 1002405,

            /// <summary>The number of protein clusters that have been identified, which must match the number of clusters that pass the threshold in the file.</summary>
            MS_count_of_identified_clusters = 1002406,

            /// <summary>An identifier applied to protein groups to indicate that they are linked by shared peptides.</summary>
            MS_cluster_identifier = 1002407,

            /// <summary>The number of protein clusters that have been identified, which must match the number of clusters that pass the threshold in the file.</summary>
            MS_number_of_distinct_protein_sequences = 1002408,

            /// <summary>Assigned to a non-leading protein that has some independent evidence to support its presence relative to the leading protein(s), e.g. the protein may have a unique peptide but not sufficient to be promoted as a leading protein of another group.</summary>
            MS_marginally_distinguished_protein = 1002409,

            /// <summary>Anubis software for selected reaction monitoring data.</summary>
            MS_Anubis = 1002410,

            /// <summary>The TraML format for transitions in SRM from the PSI. File extension '.TraML'.</summary>
            MS_TraML_format = 1002411,

            /// <summary>Summed area of all the extracted ion chromatogram for the peptide (e.g. of all the transitions in SRM).</summary>
            MS_total_XIC_area = 1002412,

            /// <summary>The background area for the quantified transition.</summary>
            MS_product_background = 1002413,

            /// <summary>Postprocessing software.</summary>
            MS_postprocessing_software = 1002414,

            /// <summary>A Boolean attribute to determine whether the protein group has passed the threshold indicated in the file.</summary>
            MS_protein_group_passes_threshold = 1002415,

            /// <summary>Thermo Scientific Orbitrap Fusion.</summary>
            MS_Orbitrap_Fusion = 1002416,

            /// <summary>Thermo Scientific Orbitrap Fusion with ETD.</summary>
            MS_Orbitrap_Fusion_ETD = 1002417,

            /// <summary>Thermo Scientific TSQ Quantiva MS.</summary>
            MS_TSQ_Quantiva = 1002418,

            /// <summary>Thermo Scientific TSQ Endura MS.</summary>
            MS_TSQ_Endura = 1002419,

            /// <summary>URI that allows access to a PASSEL experiment.</summary>
            MS_PASSEL_experiment_URI = 1002420,

            /// <summary>Search engine input parameters specific to Paragon.</summary>
            MS_Paragon_input_parameter = 1002421,

            /// <summary>The Paragon method setting indicating the type of sample at the high level, generally meaning the type of quantitation labelling or lack thereof. 'Identification' is indicated for samples without any labels for quantitation.</summary>
            MS_Paragon__sample_type = 1002422,

            /// <summary>The Paragon method setting indicating the actual cysteine alkylation agent; 'None' is indicated if there was no cysteine alkylation.</summary>
            MS_Paragon__cysteine_alkylation = 1002423,

            /// <summary>The Paragon method setting (translating to a large number of lower level settings) indicating the instrument used or a category of instrument.</summary>
            MS_Paragon__instrument_setting = 1002424,

            /// <summary>The Paragon method setting that controls the two major modes of search effort of the Paragon algorithm: the Rapid mode uses a conventional database search, while the Thorough mode uses a hybrid search, starting with the same approach as the Rapid mode but then follows it with a separate tag-based approach enabling a more extensive search.</summary>
            MS_Paragon__search_effort = 1002425,

            /// <summary>A Paragon method setting that allows the inclusion of large sets of features such as biological modification or substitutions.</summary>
            MS_Paragon__ID_focus = 1002426,

            /// <summary>The Paragon method setting that controls whether FDR analysis is conducted.</summary>
            MS_Paragon__FDR_analysis = 1002427,

            /// <summary>The Paragon method setting that controls whether quantitation analysis is conducted.</summary>
            MS_Paragon__quantitation = 1002428,

            /// <summary>The Paragon method setting that controls whether the 'Background Correction' analysis is conducted; this processing estimates a correction to the attenuation in extremity ratios that can occur in isobaric quantatitation workflows on complex samples.</summary>
            MS_Paragon__background_correction = 1002429,

            /// <summary>The Paragon method setting that controls whether 'Bias Correction' is invoked in quantitation analysis; this correction is a normalization to set the central tendency of protein ratios to unity.</summary>
            MS_Paragon__bias_correction = 1002430,

            /// <summary>The Paragon method setting that controls which label channel is used as the denominator in calculating relative expression ratios.</summary>
            MS_Paragon__channel_to_use_as_denominator_in_ratios = 1002431,

            /// <summary>Search engine specific metadata that are not user-controlled settings.</summary>
            MS_search_engine_specific_input_metadata = 1002432,

            /// <summary>This metric detects if any changes have been made to the originally installed key control files for the software; if no changes have been made, then the software version and settings are sufficient to enable exact reproduction; if changes have been made, then the modified ParameterTranslation- and ProteinPilot DataDictionary-XML files much also be provided in order to exactly reproduce a result.</summary>
            MS_Paragon__modified_data_dictionary_or_parameter_translation = 1002433,

            /// <summary>Number of spectra in a search.</summary>
            MS_number_of_spectra_searched = 1002434,

            /// <summary>The time that a data processing action was started.</summary>
            MS_data_processing_start_time = 1002435,

            /// <summary>The Paragon method setting indicating the actual digestion agent - unlike other search tools, this setting does not include options that control partial specificity like 'semitrypsin'; if trypsin is used, trypsin is set, and partially conforming peptides are found in the Thorough mode of search; 'None' should be indicated only if there was really no digestion done.</summary>
            MS_Paragon__digestion = 1002436,

            /// <summary>The number of decoy sequences, if the concatenated target-decoy approach is used.</summary>
            MS_number_of_decoy_sequences = 1002437,

            /// <summary>Information about the list of PSMs (SpectrumIdentificationList).</summary>
            MS_spectrum_identification_list_result_details = 1002438,

            /// <summary>A flag on a list of PSMs (SpectrumIdentificationList) to indicate that this is the final set of identifications to be interpreted by consuming software. Amongst the set of SpectrumIdentificationList(s) that are flagged with the term, each spectrum must not be referenced from more than one SpectrumIdentificationResult.</summary>
            MS_final_PSM_list = 1002439,

            /// <summary>A flag on a list of PSMs (SpectrumIdentificationList) to indicate that this is not the final set of identifications to be interpreted by consuming software. This term should be used when results are provided from multiple search engines for the results from each search engine before they are combined to give consensus identifications. Amongst the set of SpectrumIdentificationList(s) that are flagged with the term, each spectrum may be referenced from more than one SpectrumIdentificationResult.</summary>
            MS_intermediate_PSM_list = 1002440,

            /// <summary>AIA Analytical Data Interchange file format for mass spectrometry data.</summary>
            MS_Andi_MS_format = 1002441,

            /// <summary>The format of the chromatography file being used. This could be a instrument or vendor specific proprietary file format or a converted open file format.</summary>
            MS_chromatograph_file_format = 1002442,

            /// <summary>AIA Analytical Data Interchange file format for chromatography data.</summary>
            MS_Andi_CHROM_format = 1002443,

            /// <summary>The 6420 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_6420_Triple_Quadrupole_LC_MS = 1002444,

            /// <summary>The 6460 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer. It is similar to the 6420 but adds Agilent Jet Stream (AJS) technology to increase sensitivity.</summary>
            MS_6460_Triple_Quadrupole_LC_MS = 1002445,

            /// <summary>The 6490 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer. It is similar to the 6420 but adds the Agilent iFunnel technology to increase sensitivity.</summary>
            MS_6490_Triple_Quadrupole_LC_MS = 1002446,

            /// <summary>The Paragon method setting indicating a list of one or more 'special factors', which generally capture secondary effects (relative to other settings) as a set of probabilities of modification features that override the assumed levels. For example the 'gel-based ID' special factor causes an increase probability of oxidation on several resides because of the air exposure impact on a gel, in addition to other effects.</summary>
            MS_Paragon_special_factor = 1002447,

            /// <summary>The PEAKS inChorus peptide score.</summary>
            MS_PEAKS_inChorusPeptideScore = 1002448,

            /// <summary>The PEAKS inChorus protein score.</summary>
            MS_PEAKS_inChorusProteinScore = 1002449,

            /// <summary>Ion b-H3PO4: b ion with lost phosphoric acid.</summary>
            MS_param__b_ion_H3PO4_DEPRECATED = 1002450,

            /// <summary>Ion y-H3PO4: y ion with lost phosphoric acid.</summary>
            MS_param__y_ion_H3PO4_DEPRECATED = 1002451,

            /// <summary>The Maltcms Graphical User Interface.</summary>
            MS_Maui = 1002452,

            /// <summary>No fixed modifications are included as a parameter for the search, and therefore they are not reported.</summary>
            MS_No_fixed_modifications_searched = 1002453,

            /// <summary>No variable modifications are included as a parameter for the search, and therefore they are not reported.</summary>
            MS_No_variable_modifications_searched = 1002454,

            /// <summary>OBSOLETE Neutral loss of water.</summary>
            MS_H2O_neutral_loss_OBSOLETE = 1002455,

            /// <summary>OBSOLETE Neutral loss of ammonia.</summary>
            MS_NH3_neutral_loss_OBSOLETE = 1002456,

            /// <summary>OBSOLETE Neutral loss of phosphoric acid.</summary>
            MS_H3PO4_neutral_loss_OBSOLETE = 1002457,

            /// <summary>PeptideShaker is a software for the interpretation of proteomics identification results.</summary>
            MS_PeptideShaker = 1002458,

            /// <summary>MS Amanda csv output format.</summary>
            MS_MS_Amanda_csv_format = 1002459,

            /// <summary>Estimation of the global false negative rate of protein groups.</summary>
            MS_protein_group_level_global_FNR = 1002460,

            /// <summary>Estimation of the global confidence of protein groups.</summary>
            MS_protein_group_level_confidence = 1002461,

            /// <summary>Estimation of the global false negative rate for distinct peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs have been collapsed to one entry).</summary>
            MS_peptide_sequence_level_global_FNR = 1002462,

            /// <summary>Estimation of the global confidence for distinct peptides once redundant identifications of the same peptide have been removed (id est multiple PSMs have been collapsed to one entry).</summary>
            MS_peptide_sequence_level_global_confidence = 1002463,

            /// <summary>Estimation of the global false negative rate of peptide spectrum matches.</summary>
            MS_PSM_level_global_FNR = 1002464,

            /// <summary>Estimation of the global confidence of peptide spectrum matches.</summary>
            MS_PSM_level_global_confidence = 1002465,

            /// <summary>The probability based PeptideShaker PSM score.</summary>
            MS_PeptideShaker_PSM_score = 1002466,

            /// <summary>The probability based PeptideShaker PSM confidence.</summary>
            MS_PeptideShaker_PSM_confidence = 1002467,

            /// <summary>The probability based PeptideShaker peptide score.</summary>
            MS_PeptideShaker_peptide_score = 1002468,

            /// <summary>The probability based PeptideShaker peptide confidence.</summary>
            MS_PeptideShaker_peptide_confidence = 1002469,

            /// <summary>The probability based PeptideShaker protein group score.</summary>
            MS_PeptideShaker_protein_group_score = 1002470,

            /// <summary>The probability based PeptideShaker protein group confidence.</summary>
            MS_PeptideShaker_protein_group_confidence = 1002471,

            /// <summary>A collision-induced dissociation process that occurs in a trap-type collision cell.</summary>
            MS_trap_type_collision_induced_dissociation = 1002472,

            /// <summary>The description of the ion fragment series (including charges and neutral losses) that are considered by the search engine.</summary>
            MS_ion_series_considered_in_search = 1002473,

            /// <summary>The sum of peptide-level scores for peptides mapped only to non-canonical gene models within the group.</summary>
            MS_ProteoAnnotator_non_canonical_gene_model_score = 1002474,

            /// <summary>The count of the number of peptide sequences mapped to non-canonical gene models only within the group.</summary>
            MS_ProteoAnnotator_count_alternative_peptides = 1002475,

            /// <summary>Drift time of an ion or spectrum of ions as measured in an ion mobility mass spectrometer. This time might refer to the central value of a bin into which all ions within a narrow range of drift time have been aggregated.</summary>
            MS_ion_mobility_drift_time = 1002476,

            /// <summary>Array of population mean ion mobility values from a drift time device, reported in seconds (or milliseconds), corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_mean_ion_mobility_drift_time_array = 1002477,

            /// <summary>Array of mean charge values where the mean charge is calculated as a weighted mean of the charges of individual peaks that are aggregated into a processed spectrum.</summary>
            MS_mean_charge_array = 1002478,

            /// <summary>Regular expression.</summary>
            MS_regular_expression = 1002479,

            /// <summary>(10</summary>
            MS_regular_expression_for_a_digital_object_identifier__DOI_ = 1002480,

            /// <summary>A collision-induced dissociation process wherein the projectile ion has the translational energy higher than approximately 1000 eV.</summary>
            MS_higher_energy_beam_type_collision_induced_dissociation = 1002481,

            /// <summary>Estimated statistical threshold.</summary>
            MS_statistical_threshold = 1002482,

            /// <summary>Estimated statistical threshold at PSM-level.</summary>
            MS_PSM_level_statistical_threshold = 1002483,

            /// <summary>Estimated statistical threshold at peptide-level.</summary>
            MS_peptide_level_statistical_threshold = 1002484,

            /// <summary>Estimated statistical threshold at protein-level.</summary>
            MS_protein_level_statistical_threshold = 1002485,

            /// <summary>Estimated statistical threshold at protein group-level.</summary>
            MS_protein_group_level_statistical_threshold = 1002486,

            /// <summary>Dataset identifier issued by the MassIVE repository. A dataset can refer to either a single sample as part of a study, or all samples that are part of the study corresponding to a publication.</summary>
            MS_MassIVE_dataset_identifier = 1002487,

            /// <summary>URI that allows the access to one dataset in the MassIVE repository. A dataset can refer to either a single sample as part of a study, or all samples that are part of the study corresponding to a publication.</summary>
            MS_MassIVE_dataset_URI = 1002488,

            /// <summary>Details describing a special processing.</summary>
            MS_special_processing = 1002489,

            /// <summary>Peptide-level scoring performed.</summary>
            MS_peptide_level_scoring = 1002490,

            /// <summary>Modification localization scoring performed.</summary>
            MS_modification_localization_scoring = 1002491,

            /// <summary>Consensus multiple search engine approach performed.</summary>
            MS_consensus_scoring = 1002492,

            /// <summary>Sample pre-fractionation performed.</summary>
            MS_sample_pre_fractionation = 1002493,

            /// <summary>Cross-linking search performed.</summary>
            MS_cross_linking_search = 1002494,

            /// <summary>No special processing performed.</summary>
            MS_no_special_processing = 1002495,

            /// <summary>Group PSMs by distinct peptide sequence ignoring modifications.</summary>
            MS_group_PSMs_by_sequence = 1002496,

            /// <summary>Group PSMs by distinct peptide sequence with taking modifications into account.</summary>
            MS_group_PSMs_by_sequence_with_modifications = 1002497,

            /// <summary>Group PSMs by distinct peptide sequence with taking modifications and charge into account.</summary>
            MS_group_PSMs_by_sequence_with_modifications_and_charge = 1002498,

            /// <summary>OBSOLETE Peptide level score.</summary>
            MS_peptide_level_score_OBSOLETE = 1002499,

            /// <summary>A Boolean attribute to determine whether the peptide has passed the threshold indicated in the file.</summary>
            MS_peptide_passes_threshold = 1002500,

            /// <summary>Indicating that no PSM threshold was used.</summary>
            MS_no_PSM_threshold = 1002501,

            /// <summary>Indicating that no peptide-level threshold was used.</summary>
            MS_no_peptide_level_threshold = 1002502,

            /// <summary>Flags a PSM that it is used for peptide-level scoring.</summary>
            MS_PSM_is_used_for_peptide_level_scoring = 1002503,

            /// <summary>The order of modifications to be referenced elsewhere in the document.</summary>
            MS_modification_index = 1002504,

            /// <summary>(</summary>
            MS_regular_expression_for_modification_localization_scoring = 1002505,

            /// <summary>Modification position score.</summary>
            MS_modification_position_score = 1002506,

            /// <summary>Mod position score: false localization rate.</summary>
            MS_modification_rescoring_false_localization_rate = 1002507,

            /// <summary>Cross-linking attribute.</summary>
            MS_cross_linking_attribute = 1002508,

            /// <summary>The Cross-linking donor, assigned according to the following rules: the export software SHOULD use the following rules to choose the cross-link donor as the: longer peptide, then higher peptide neutral mass, then alphabetical order.</summary>
            MS_cross_link_donor = 1002509,

            /// <summary>Cross-linking acceptor, assigned according to the following rules: the export software SHOULD use the following rules to choose the cross-link donor as the: longer peptide, then higher peptide neutral mass, then alphabetical order.</summary>
            MS_cross_link_acceptor = 1002510,

            /// <summary>Cross-linked spectrum identification item.</summary>
            MS_cross_link_spectrum_identification_item = 1002511,

            /// <summary>Cross-linking scoring value.</summary>
            MS_cross_linking_score = 1002512,

            /// <summary>The absolute abundance of protein in a cell.</summary>
            MS_molecules_per_cell = 1002513,

            /// <summary>Absolute quantitation analysis.</summary>
            MS_absolute_quantitation_analysis = 1002514,

            /// <summary>States whether an internal peptide reference is used or not in absolute quantitation analysis.</summary>
            MS_internal_peptide_reference_used = 1002515,

            /// <summary>States whether an internal protein reference is used or not in absolute quantitation analysis.</summary>
            MS_internal_protein_reference_used = 1002516,

            /// <summary>The absolute abundance of the spiked in reference peptide or protein used for absolute quantitation analysis.</summary>
            MS_internal_reference_abundance = 1002517,

            /// <summary>The data type normalised abundance for protein groups produced by Progenesis LC-MS.</summary>
            MS_Progenesis_protein_group_normalised_abundance = 1002518,

            /// <summary>The data type raw abundance for protein groups produced by Progenesis LC-MS.</summary>
            MS_Progenesis_protein_group_raw_abundance = 1002519,

            /// <summary>Peptide group identifier for peptide-level stats.</summary>
            MS_peptide_group_ID = 1002520,

            /// <summary>A technique in which mass spectra are acquired in a spatially resolved manner. This is typically achieved by scanning a laser or primary ion beam over a sample and acquiring a mass spectrum at each position.</summary>
            MS_mass_spectrometry_imaging = 1002521,

            /// <summary>Determine 1st static terminal post-translational modifications (PTMs).</summary>
            MS_ProteomeDiscoverer_1__Static_Terminal_Modification = 1002522,

            /// <summary>Thermo Scientific Q Exactive.</summary>
            MS_Q_Exactive_HF = 1002523,

            /// <summary>Thermo Scientific PepFinder BioPharma analysis software.</summary>
            MS_PepFinder = 1002524,

            /// <summary>Thermo Scientific TSQ 8000 Evo MS.</summary>
            MS_TSQ_8000_Evo = 1002525,

            /// <summary>Thermo Scientific Exactive Plus MS.</summary>
            MS_Exactive_Plus = 1002526,

            /// <summary>Instrument specific scan properties that are associated with a value.</summary>
            MS_instrument_specific_scan_attribute = 1002527,

            /// <summary>Synchronous prefilter selection.</summary>
            MS_synchronous_prefilter_selection = 1002528,

            /// <summary>A data array of resolution values.</summary>
            MS_resolution_array = 1002529,

            /// <summary>A data array of signal baseline values (the signal in the absence of analytes).</summary>
            MS_baseline_array = 1002530,

            /// <summary>SQLite-based file format created at Pacific Northwest National Lab. It stores an intermediate analysis of ion-mobility mass spectrometry data.</summary>
            MS_UIMF_format = 1002531,

            /// <summary>Native format defined by frame=xsd:nonNegativeInteger scan=xsd:nonNegativeInteger frameType=xsd:nonNegativeInteger.</summary>
            MS_UIMF_nativeID_format = 1002532,

            /// <summary>SCIEX TripleTOF 6600, a quadrupole - quadrupole - time-of-flight mass spectrometer.</summary>
            MS_TripleTOF_6600 = 1002533,

            /// <summary>The ProLuCID result 'XCorr'.</summary>
            MS_ProLuCID_xcorr = 1002534,

            /// <summary>The ProLuCID result 'DeltaCn'.</summary>
            MS_ProLuCID_deltacn = 1002535,

            /// <summary>D-Score for PTM site location at the PSM-level.</summary>
            MS_D_Score = 1002536,

            /// <summary>MD-Score for PTM site location at the PSM-level.</summary>
            MS_MD_Score = 1002537,

            /// <summary>Localization confidence metric for a post translational modification (PTM).</summary>
            MS_PTM_localization_confidence_metric = 1002538,

            /// <summary>PeptideShaker quality criteria for the confidence of PTM localizations.</summary>
            MS_PeptideShaker_PTM_confidence_type = 1002539,

            /// <summary>PeptideShaker quality criteria for the confidence of PSM's.</summary>
            MS_PeptideShaker_PSM_confidence_type = 1002540,

            /// <summary>PeptideShaker quality criteria for the confidence of peptide identifications.</summary>
            MS_PeptideShaker_peptide_confidence_type = 1002541,

            /// <summary>PeptideShaker quality criteria for the confidence of protein identifications.</summary>
            MS_PeptideShaker_protein_confidence_type = 1002542,

            /// <summary>Target/Decoy based FDR estimation for cross-linking peptide-identifications.</summary>
            MS_xiFDR = 1002543,

            /// <summary>Search engine for cross-linked peptides.</summary>
            MS_xi = 1002544,

            /// <summary>The xi result 'Score'.</summary>
            MS_xi_score = 1002545,

            /// <summary>A software package to convert Skyline report to mzQuantML.</summary>
            MS_Skyline_mzQuantML_converter = 1002546,

            /// <summary>A normalized spectral abundance factor (NSAF).</summary>
            MS_normalized_spectral_abundance_factor = 1002547,

            /// <summary>A distributed normalized spectral abundance factor (dNSAF).</summary>
            MS_distributed_normalized_spectral_abundance_factor = 1002548,

            /// <summary>Statistic to convey the confidence of the localization of an amino acid modification on a peptide sequence.</summary>
            MS_PTM_localization_distinct_peptide_level_statistic = 1002549,

            /// <summary>phosphoRS score for PTM site location at the peptide-level.</summary>
            MS_peptide_phosphoRS_score = 1002550,

            /// <summary>A-score for PTM site location at the peptide-level.</summary>
            MS_peptide_Ascore = 1002551,

            /// <summary>H-Score for peptide phosphorylation site location at the peptide-level.</summary>
            MS_peptide_H_Score = 1002552,

            /// <summary>D-Score for PTM site location at the peptide-level.</summary>
            MS_peptide_D_Score = 1002553,

            /// <summary>MD-Score for PTM site location at the peptide-level.</summary>
            MS_peptide_MD_Score = 1002554,

            /// <summary>Threshold for PTM site location score.</summary>
            MS_PTM_localization_score_threshold = 1002555,

            /// <summary>Threshold for Ascore PTM site location score.</summary>
            MS_Ascore_threshold = 1002556,

            /// <summary>Threshold for D-score PTM site location score.</summary>
            MS_D_Score_threshold = 1002557,

            /// <summary>Threshold for MD-score PTM site location score.</summary>
            MS_MD_Score_threshold = 1002558,

            /// <summary>Threshold for H-score PTM site location score.</summary>
            MS_H_Score_threshold = 1002559,

            /// <summary>Threshold for DeBunker PTM site location score.</summary>
            MS_DeBunker_score_threshold = 1002560,

            /// <summary>Threshold for Mascot PTM site assignment confidence.</summary>
            MS_Mascot_PTM_site_assignment_confidence_threshold = 1002561,

            /// <summary>Threshold for MSQuant:PTM-score.</summary>
            MS_MSQuant_PTM_score_threshold = 1002562,

            /// <summary>Threshold for MaxQuant:PTM Score.</summary>
            MS_MaxQuant_PTM_Score_threshold = 1002563,

            /// <summary>Threshold for MaxQuant:P-site localization probability.</summary>
            MS_MaxQuant_P_site_localization_probability_threshold = 1002564,

            /// <summary>Threshold for MaxQuant:PTM Delta Score.</summary>
            MS_MaxQuant_PTM_Delta_Score_threshold = 1002565,

            /// <summary>Threshold for MaxQuant:Phospho (STY) Probabilities.</summary>
            MS_MaxQuant_Phospho__STY__Probabilities_threshold = 1002566,

            /// <summary>Threshold for phosphoRS score.</summary>
            MS_phosphoRS_score_threshold = 1002567,

            /// <summary>Threshold for phosphoRS site probability.</summary>
            MS_phosphoRS_site_probability_threshold = 1002568,

            /// <summary>Number of spectra processed at once in a ProteomeDiscoverer search.</summary>
            MS_ProteomeDiscoverer_Number_of_Spectra_Processed_At_Once = 1002569,

            /// <summary>A protein for which the matched peptide sequences are the same, or a subset of, the matched peptide sequences for two or more other proteins combined. These other proteins need not all be in the same group.</summary>
            MS_sequence_multiply_subsumable_protein = 1002570,

            /// <summary>A protein for which the matched spectra are the same, or a subset of, the matched spectra for two or more other proteins combined. These other proteins need not all be in the same group.</summary>
            MS_spectrum_multiply_subsumable_protein = 1002571,

            /// <summary>Estimated statistical threshold used for protein detection.</summary>
            MS_protein_detection_statistical_threshold = 1002572,

            /// <summary>Estimated statistical threshold used for spectrum identification.</summary>
            MS_spectrum_identification_statistical_threshold = 1002573,

            /// <summary>A program in the TPP that calculates PSM, peptide, and protein-level abundances based on 2-channel isotope-labelled data such as ICAT, SILAC, etc.</summary>
            MS_ASAPRatio = 1002574,

            /// <summary>Tide open-source sequence search program developed at the University of Washington.</summary>
            MS_Tide = 1002575,

            /// <summary>Andromeda result file output format.</summary>
            MS_Andromeda_result_format = 1002576,

            /// <summary>SCIEX 2000 QTRAP.</summary>
            MS_2000_QTRAP = 1002577,

            /// <summary>SCIEX 2500 QTRAP.</summary>
            MS_2500_QTRAP = 1002578,

            /// <summary>SCIEX 3500 QTRAP.</summary>
            MS_3500_QTRAP = 1002579,

            /// <summary>SCIEX QTRAP 4500.</summary>
            MS_QTRAP_4500 = 1002580,

            /// <summary>SCIEX QTRAP 6500.</summary>
            MS_QTRAP_6500 = 1002581,

            /// <summary>SCIEX QTRAP 6500+.</summary>
            MS_QTRAP_6500_ = 1002582,

            /// <summary>SCIEX TripleTOF 4600 time-of-flight mass spectrometer.</summary>
            MS_TripleTOF_4600 = 1002583,

            /// <summary>SCIEX TripleTOF 5600+ time-of-flight mass spectrometer.</summary>
            MS_TripleTOF_5600_ = 1002584,

            /// <summary>Applied Biosystems/MDS SCIEX API 100 MS.</summary>
            MS_API_100 = 1002585,

            /// <summary>Applied Biosystems/MDS SCIEX API 100LC MS.</summary>
            MS_API_100LC = 1002586,

            /// <summary>Applied Biosystems/MDS SCIEX API 165 MS.</summary>
            MS_API_165 = 1002587,

            /// <summary>Applied Biosystems/MDS SCIEX API 300 MS.</summary>
            MS_API_300 = 1002588,

            /// <summary>Applied Biosystems/MDS SCIEX API 350 MS.</summary>
            MS_API_350 = 1002589,

            /// <summary>Applied Biosystems/MDS SCIEX API 365 MS.</summary>
            MS_API_365 = 1002590,

            /// <summary>SCIEX Triple Quad 3500.</summary>
            MS_Triple_Quad_3500 = 1002591,

            /// <summary>SCIEX Triple Quad 4500.</summary>
            MS_Triple_Quad_4500 = 1002592,

            /// <summary>SCIEX Triple Quad 5500.</summary>
            MS_Triple_Quad_5500 = 1002593,

            /// <summary>SCIEX Triple Quad 6500.</summary>
            MS_Triple_Quad_6500 = 1002594,

            /// <summary>SCIEX Triple Quad 6500+.</summary>
            MS_Triple_Quad_6500_ = 1002595,

            /// <summary>The SEQUEST-like sequence search engine ProLuCID, developed in the Yates Lab at the Scripps Research Institute.</summary>
            MS_ProLuCID = 1002596,

            /// <summary>MS1 file format for MS1 spectral data.</summary>
            MS_MS1_format = 1002597,

            /// <summary>Analysis software designed to reassemble the SEQUEST peptide identifications and to highlight the most significant matches.</summary>
            MS_DTASelect = 1002598,

            /// <summary>Spectral Hash key, an unique identifier for spectra.</summary>
            MS_splash_key = 1002599,

            /// <summary>Internal data and submission format of the PRIDE database.</summary>
            MS_PRIDE_XML = 1002600,

            /// <summary>Tabular result format for proteomics and metabolomics experiments.</summary>
            MS_mzTab = 1002601,

            /// <summary>Reagent used in labeled quantification methods.</summary>
            MS_sample_label = 1002602,

            /// <summary>Isotope coded affinity tag reagent.</summary>
            MS_ICAT_reagent = 1002603,

            /// <summary>The name of the sample labelled with the heavy ICAT label.</summary>
            MS_ICAT_heavy_reagent = 1002604,

            /// <summary>The name of the sample labelled with the light ICAT label.</summary>
            MS_ICAT_light_reagent = 1002605,

            /// <summary>Isotope coded protein labeling reagent.</summary>
            MS_ICPL_reagent = 1002606,

            /// <summary>The name of the sample labelled with the ICPL reagent 0.</summary>
            MS_ICPL_reagent_0 = 1002607,

            /// <summary>The name of the sample labelled with the ICPL reagent 4.</summary>
            MS_ICPL_reagent_4 = 1002608,

            /// <summary>The name of the sample labelled with the ICPL reagent 6.</summary>
            MS_ICPL_reagent_6 = 1002609,

            /// <summary>The name of the sample labelled with the ICPL reagent 10.</summary>
            MS_ICPL_reagent_10 = 1002610,

            /// <summary>Stable isotope labeling with amino acids in cell culture reagent.</summary>
            MS_SILAC_reagent = 1002611,

            /// <summary>The name of the sample labelled with the heavy SILAC label.</summary>
            MS_SILAC_heavy_reagent = 1002612,

            /// <summary>The name of the sample labelled with the medium SILAC label.</summary>
            MS_SILAC_medium_reagent = 1002613,

            /// <summary>The name of the sample labelled with the light SILAC label.</summary>
            MS_SILAC_light_reagent = 1002614,

            /// <summary>Tandem mass tag reagent used in TMT, glycoTMT, iodoTMT, aminoxyTMT or hydrazideTMT isobaric labeling.</summary>
            MS_TMT_reagent = 1002615,

            /// <summary>The name of the sample labelled with the TMT reagent 126.</summary>
            MS_TMT_reagent_126 = 1002616,

            /// <summary>The name of the sample labelled with the TMT reagent 127.</summary>
            MS_TMT_reagent_127 = 1002617,

            /// <summary>The name of the sample labelled with the TMT reagent 128.</summary>
            MS_TMT_reagent_128 = 1002618,

            /// <summary>The name of the sample labelled with the TMT reagent 129.</summary>
            MS_TMT_reagent_129 = 1002619,

            /// <summary>The name of the sample labelled with the TMT reagent 130.</summary>
            MS_TMT_reagent_130 = 1002620,

            /// <summary>The name of the sample labelled with the TMT reagent 131.</summary>
            MS_TMT_reagent_131 = 1002621,

            /// <summary>Isobaric tag for relative and absolute quantitation (iTRAQ or iTRAQH) reagent.</summary>
            MS_iTRAQ_reagent = 1002622,

            /// <summary>The name of the sample labelled with the iTRAQ reagent 113.</summary>
            MS_iTRAQ_reagent_113 = 1002623,

            /// <summary>The name of the sample labelled with the iTRAQ reagent 114.</summary>
            MS_iTRAQ_reagent_114 = 1002624,

            /// <summary>The name of the sample labelled with the iTRAQ reagent 115.</summary>
            MS_iTRAQ_reagent_115 = 1002625,

            /// <summary>The name of the sample labelled with the iTRAQ reagent 116.</summary>
            MS_iTRAQ_reagent_116 = 1002626,

            /// <summary>The name of the sample labelled with the iTRAQ reagent 117.</summary>
            MS_iTRAQ_reagent_117 = 1002627,

            /// <summary>The name of the sample labelled with the iTRAQ reagent 118.</summary>
            MS_iTRAQ_reagent_118 = 1002628,

            /// <summary>The name of the sample labelled with the iTRAQ reagent 119.</summary>
            MS_iTRAQ_reagent_119 = 1002629,

            /// <summary>The name of the sample labelled with the iTRAQ reagent 121.</summary>
            MS_iTRAQ_reagent_121 = 1002630,

            /// <summary>Dissociation process combining electron-transfer dissociation and higher-energy collision dissociation. It combines ETD (reaction time) followed by HCD (activation energy).</summary>
            MS_electron_transfer_higher_energy_collision_dissociation = 1002631,

            /// <summary>Dataset identifier issued by the jPOST repository. A dataset can refer to either a single sample as part of a study, or all samples that are part of the study corresponding to a publication.</summary>
            MS_jPOST_dataset_identifier = 1002632,

            /// <summary>URI that allows the access to one dataset in the jPOST repository. A dataset can refer to either a single sample as part of a study, or all samples that are part of the study corresponding to a publication.</summary>
            MS_jPOST_dataset_URI = 1002633,

            /// <summary>Thermo Scientific Q Exactive Plus.</summary>
            MS_Q_Exactive_Plus = 1002634,

            /// <summary>Proteogenomics search performed.</summary>
            MS_proteogenomics_search = 1002635,

            /// <summary>Proteogenomics attribute.</summary>
            MS_proteogenomics_attribute = 1002636,

            /// <summary>The name or number of the chromosome to which a given peptide has been mapped.</summary>
            MS_chromosome_name = 1002637,

            /// <summary>The strand (+ or -) to which the peptide has been mapped.</summary>
            MS_chromosome_strand = 1002638,

            /// <summary>OBSOLETE The overall start position on the chromosome to which a peptide has been mapped i.e. the position of the first base of the first codon, using a zero-based counting system.</summary>
            MS_peptide_start_on_chromosome_OBSOLETE = 1002639,

            /// <summary>The overall end position on the chromosome to which a peptide has been mapped i.e. the position of the third base of the last codon, using a zero-based counting system.</summary>
            MS_peptide_end_on_chromosome = 1002640,

            /// <summary>The number of exons to which the peptide has been mapped.</summary>
            MS_peptide_exon_count = 1002641,

            /// <summary>A comma separated list of the number of DNA bases within each exon to which a peptide has been mapped. Assuming standard operation of a search engine, the peptide exon sizes should sum to exactly three times the peptide length.</summary>
            MS_peptide_exon_nucleotide_sizes = 1002642,

            /// <summary>A comma separated list of start positions within exons to which the peptide has been mapped, relative to peptide-chromosome start, assuming a zero-based counting system. The first value MUST match the value in peptide start on chromosome.</summary>
            MS_peptide_start_positions_on_chromosome = 1002643,

            /// <summary>The reference genome and versioning string as used for mapping. All coordinates are within this frame of reference.</summary>
            MS_genome_reference_version = 1002644,

            /// <summary>Mass Spectrometry Development Kit (MSDK) is a Java library of algorithms for processing of mass spectrometry data.</summary>
            MS_MSDK = 1002645,

            /// <summary>Describes how the native spectrum identifiers that have been combined prior to searching or interpretation are formated.</summary>
            MS_native_spectrum_identifier_format__combined_spectra = 1002646,

            /// <summary>Thermo comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_Thermo_nativeID_format__combined_spectra = 1002647,

            /// <summary>Waters comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_Waters_nativeID_format__combined_spectra = 1002648,

            /// <summary>WIFF comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_WIFF_nativeID_format__combined_spectra = 1002649,

            /// <summary>Bruker/Agilent comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_Bruker_Agilent_YEP_nativeID_format__combined_spectra = 1002650,

            /// <summary>Bruker BAF comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_Bruker_BAF_nativeID_format__combined_spectra = 1002651,

            /// <summary>Bruker FID comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_Bruker_FID_nativeID_format__combined_spectra = 1002652,

            /// <summary>Comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_multiple_peak_list_nativeID_format__combined_spectra = 1002653,

            /// <summary>Comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_single_peak_list_nativeID_format__combined_spectra = 1002654,

            /// <summary>Comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_scan_number_only_nativeID_format__combined_spectra = 1002655,

            /// <summary>Comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_spectrum_identifier_nativeID_format__combined_spectra = 1002656,

            /// <summary>Comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_mzML_unique_identifier__combined_spectra = 1002657,

            /// <summary>Identification parameter for the search engine run.</summary>
            MS_identification_parameter = 1002658,

            /// <summary>Text-based format used by UniProtKB for sequence entries.</summary>
            MS_UniProtKB_text_sequence_format = 1002659,

            /// <summary>XML-based format used by UniProtKB for sequence entries.</summary>
            MS_UniProtKB_XML_sequence_format = 1002660,

            /// <summary>Morpheus search engine.</summary>
            MS_Morpheus = 1002661,

            /// <summary>Morpheus score for PSMs.</summary>
            MS_Morpheus_Morpheus_score = 1002662,

            /// <summary>Summed Morpheus score for protein groups.</summary>
            MS_Morpheus_summed_Morpheus_score = 1002663,

            /// <summary>Parent term for interaction scores derived from cross-linking.</summary>
            MS_interaction_score_derived_from_cross_linking = 1002664,

            /// <summary>(</summary>
            MS_regular_expression_for_interaction_scores_derived_from_cross_linking = 1002665,

            /// <summary>Bruker Daltonics' impact II.</summary>
            MS_impact_II = 1002666,

            /// <summary>Bruker Daltonics' impact HD.</summary>
            MS_impact_HD = 1002667,

            /// <summary>Standard reporter ion for iTRAQ 4Plex. The value slot holds the integer mass of the iTRAQ 4Plex reporter ion, e.g. 114.</summary>
            MS_frag__iTRAQ_4plex_reporter_ion = 1002668,

            /// <summary>Standard reporter ion for iTRAQ 8Plex. The value slot holds the integer mass of the iTRAQ 8Plex reporter ion, e.g. 113.</summary>
            MS_frag__iTRAQ_8plex_reporter_ion = 1002669,

            /// <summary>Standard reporter ion for TMT. The value slot holds the integer mass of the TMT reporter ion and can be suffixed with either N or C, indicating whether the mass difference is encoded at a Nitrogen or Carbon atom, e.g. 127N.</summary>
            MS_frag__TMT_reporter_ion = 1002670,

            /// <summary>Standard reporter ion for TMT with ETD fragmentation. The value slot holds the integer mass of the TMT ETD reporter ion and can be suffixed with either N or C, indicating whether the mass difference is encoded at a Nitrogen or Carbon atom, e.g. 127C.</summary>
            MS_frag__TMT_ETD_reporter_ion = 1002671,

            /// <summary>No statistical threshold for accepting or rejecting that a modification position.</summary>
            MS_no_modification_threshold = 1002672,

            /// <summary>Cross-Linking MS search engine.</summary>
            MS_OpenXQuest = 1002673,

            /// <summary>SCIEX X500R QTOF, a quadrupole - quadrupole - time-of-flight mass spectrometer.</summary>
            MS_X500R_QTOF = 1002674,

            /// <summary>This subsection describes terms which can describe details of cross-linking results.</summary>
            MS_cross_linking_result_details = 1002675,

            /// <summary>Estimation of the global false discovery rate of proteins-pairs in cross-linking experiments.</summary>
            MS_protein_pair_level_global_FDR = 1002676,

            /// <summary>Estimation of the global false discovery rate of residue-pairs in cross-linking experiments.</summary>
            MS_residue_pair_level_global_FDR = 1002677,

            /// <summary>A supplemental collision-induced dissociation process that occurs in a beam-type collision cell in addition to another primary type of dissociation.</summary>
            MS_supplemental_beam_type_collision_induced_dissociation = 1002678,

            /// <summary>The dissociation of an ion after supplemental collisional excitation.</summary>
            MS_supplemental_collision_induced_dissociation = 1002679,

            /// <summary>Energy for an ion experiencing supplemental collision with a stationary gas particle resulting in dissociation of the ion.</summary>
            MS_supplemental_collision_energy = 1002680,

            /// <summary>OpenXQuest's combined score for a cross-link spectrum match.</summary>
            MS_OpenXQuest_combined_score = 1002681,

            /// <summary>OpenXQuest's cross-correlation of cross-linked ions subscore.</summary>
            MS_OpenXQuest_xcorr_xlink = 1002682,

            /// <summary>OpenXQuest's cross-correlation of unlinked ions subscore.</summary>
            MS_OpenXQuest_xcorr_common = 1002683,

            /// <summary>OpenXQuest's match-odds subscore.</summary>
            MS_OpenXQuest_match_odds = 1002684,

            /// <summary>OpenXQuest's sum of matched peak intensity subscore.</summary>
            MS_OpenXQuest_intsum = 1002685,

            /// <summary>OpenXQuest's weighted percent of total ion current subscore.</summary>
            MS_OpenXQuest_wTIC = 1002686,

            /// <summary>Attribute of an item in the result of mass spectrometry proteomics data analysis.</summary>
            MS_analysis_attribute = 1002687,

            /// <summary>Statistic derived from a post-translational modification localization analysis.</summary>
            MS_PTM_localization_attribute = 1002688,

            /// <summary>Statistic for a single item derived from a post-translational modification localization analysis.</summary>
            MS_PTM_localization_single_result_statistic = 1002689,

            /// <summary>Statistic for all items derived from a post-translational modification localization analysis.</summary>
            MS_PTM_localization_result_list_statistic = 1002690,

            /// <summary>Global false localization rate for all localizations in a dataset.</summary>
            MS_global_FLR = 1002691,

            /// <summary>Local false localization rate for the bottom item in list of localizations sorted from most to least confident.</summary>
            MS_local_FLR_at_threshold = 1002692,

            /// <summary>Attribute of an identification item in the result of mass spectrometry proteomics data analysis.</summary>
            MS_identification_attribute = 1002693,

            /// <summary>Attribute of a single identification item (as opposed to a list) in the result of mass spectrometry proteomics data analysis.</summary>
            MS_single_identification_result_attribute = 1002694,

            /// <summary>Fragment ion corresponding to an isobaric label artifact.</summary>
            MS_frag__isobaric_label_ion = 1002695,

            /// <summary>Fragment ion is an isotopic peak other than that monoisotopic peak. This is used in conjuction with another ion type, such as frag: y ion.</summary>
            MS_secondary_isotope_peak = 1002697,

            /// <summary>An attribute of the protein cluster concept as used in mzIdentML.</summary>
            MS_protein_cluster_identification_attribute = 1002698,

            /// <summary>General property of an entire result list.</summary>
            MS_result_list_attribute = 1002699,

            /// <summary>General property of the list of all PSMs.</summary>
            MS_PSM_level_result_list_attribute = 1002700,

            /// <summary>Statistic pertaining to the full list of all PSMs.</summary>
            MS_PSM_level_result_list_statistic = 1002701,

            /// <summary>General property of all peptide sequences in the list.</summary>
            MS_peptide_sequence_level_result_list_attribute = 1002702,

            /// <summary>Statistic pertaining to all peptide sequences in the list.</summary>
            MS_peptide_sequence_level_result_list_statistic = 1002703,

            /// <summary>Attribute of an entire protein list.</summary>
            MS_protein_level_result_list_attribute = 1002704,

            /// <summary>A statistical metric of an entire protein list.</summary>
            MS_protein_level_result_list_statistic = 1002705,

            /// <summary>Attrbiute of an entire list of protein groups.</summary>
            MS_protein_group_level_result_list_statistic = 1002706,

            /// <summary>Regular expression for LysargiNase.</summary>
            MS_____KR__ = 1002707,

            /// <summary>Metalloproteinase found in Methanosarcina acetivorans that cleaves on the N-terminal side of lysine and arginine residues.</summary>
            MS_LysargiNase = 1002708,

            /// <summary>A data type representing more than a single value.</summary>
            MS_compound_data_type = 1002709,

            /// <summary>A data type defining a list of values of a single type.</summary>
            MS_list_of_type = 1002710,

            /// <summary>A list of xsd:string.</summary>
            MS_list_of_strings = 1002711,

            /// <summary>A list of xsd:integer.</summary>
            MS_list_of_integers = 1002712,

            /// <summary>A list of xsd:float.</summary>
            MS_list_of_floats = 1002713,

            /// <summary>LECO bench-top GC time-of-flight mass spectrometer.</summary>
            MS_Pegasus_BT = 1002719,

            /// <summary>PNNL top-down/bottom-up analysis software for identifying peptides and proteoforms in fragmentation mass spectra.</summary>
            MS_MSPathFinder = 1002720,

            /// <summary>MSPathFinder spectral E-value.</summary>
            MS_MSPathFinder_SpecEValue = 1002721,

            /// <summary>MSPathFinder E-value.</summary>
            MS_MSPathFinder_EValue = 1002722,

            /// <summary>MSPathFinder Q-value.</summary>
            MS_MSPathFinder_QValue = 1002723,

            /// <summary>MSPathFinder peptide-level Q-value.</summary>
            MS_MSPathFinder_PepQValue = 1002724,

            /// <summary>MSPathFinder raw score.</summary>
            MS_MSPathFinder_RawScore = 1002725,

            /// <summary>Waters Corporation SYNAPT G2-Si orthogonal acceleration time-of-flight mass spectrometer.</summary>
            MS_SYNAPT_G2_Si = 1002726,

            /// <summary>Waters Corporation MALDI SYNAPT G2-Si orthogonal acceleration time-of-flight mass spectrometer.</summary>
            MS_MALDI_SYNAPT_G2_Si = 1002727,

            /// <summary>Waters Corporation Vion IMS QTof orthogonal acceleration time-of-flight mass spectrometer.</summary>
            MS_Vion_IMS_QTof = 1002728,

            /// <summary>Waters Corporation Xevo G2 XS Tof orthogonal acceleration time-of-flight mass spectrometer.</summary>
            MS_Xevo_G2_XS_Tof = 1002729,

            /// <summary>Waters Corporation Xevo TQ-XS triple quadrupole mass spectrometer.</summary>
            MS_Xevo_TQ_XS = 1002730,

            /// <summary>Waters Corporation Xevo TQ-S micro triple quadrupole mass spectrometer.</summary>
            MS_Xevo_TQ_S_micro = 1002731,

            /// <summary>Thermo Scientific Orbitrap Fusion Lumos mass spectrometer with Tribrid architecture consisting of quadrupole mass filter, linear ion trap and Orbitrap mass analyzers.</summary>
            MS_Orbitrap_Fusion_Lumos = 1002732,

            /// <summary>The number of MS2 spectra identified for a peptide sequence specified by the amino acid one-letter codes plus optional PTMs in spectral counting.</summary>
            MS_peptide_level_spectral_count = 1002733,

            /// <summary>The number of MS2 spectra identified for a molecular ion defined by the peptide sequence represented by the amino acid one-letter codes, plus optional PTMs plus optional charged aducts plus the charge state, in spectral counting.</summary>
            MS_peptide_ion_level_spectral_count = 1002734,

            /// <summary>The data type of the value reported in a QuantLayer for a feature.</summary>
            MS_feature_level_quantification_datatype = 1002735,

            /// <summary>The data type of the value reported in a QuantLayer for a PSM.</summary>
            MS_PSM_level_quantification_datatype = 1002736,

            /// <summary>The data type of the value reported in a QuantLayer for a peptide.</summary>
            MS_peptide_level_quantification_datatype = 1002737,

            /// <summary>The data type of the value reported in a QuantLayer for a protein.</summary>
            MS_protein_level_quantification_datatype = 1002738,

            /// <summary>The data type of the value reported in a QuantLayer for a protein group.</summary>
            MS_protein_group_level_quantification_datatype = 1002739,

            /// <summary>Within the context of a proteogenomics approach, a peptide sequence that has not been mapped to a genomic location.</summary>
            MS_unmapped_peptide = 1002740,

            /// <summary>Within the context of a proteogenomics approach, a protein sequence that has not been mapped to a genomic location.</summary>
            MS_unmapped_protein = 1002741,

            /// <summary>A data array of noise values.</summary>
            MS_noise_array = 1002742,

            /// <summary>A data array of parallel, independent m/z values for a sampling of noise across a spectrum (typically much smaller than MS:1000514, the m/z array).</summary>
            MS_sampled_noise_m_z_array = 1002743,

            /// <summary>A data array of intensity values for the amplitude of noise variation superposed on the baseline (MS:1002745) across a spectrum (for use with MS:1002743, sampled noise m/z array).</summary>
            MS_sampled_noise_intensity_array = 1002744,

            /// <summary>A data array of baseline intensity values (the intensity in the absence of analytes) for a sampling of noise across a spectrum (for use with MS:1002743, sampled noise m/z array).</summary>
            MS_sampled_noise_baseline_array = 1002745,

            /// <summary>Compression using MS-Numpress linear prediction compression and zlib.</summary>
            MS_MS_Numpress_linear_prediction_compression_followed_by_zlib_compression = 1002746,

            /// <summary>Compression using MS-Numpress positive integer compression and zlib.</summary>
            MS_MS_Numpress_positive_integer_compression_followed_by_zlib_compression = 1002747,

            /// <summary>Compression using MS-Numpress short logged float compression and zlib.</summary>
            MS_MS_Numpress_short_logged_float_compression_followed_by_zlib_compression = 1002748,

            /// <summary>Means that Mascot has integrated the search results of database and spectral library search into a single data set.</summary>
            MS_Mascot_IntegratedSpectralLibrarySearch = 1002749,

            /// <summary>Search tool of the NIST (National Institute of Standards and Technology) for spectral library searches.</summary>
            MS_NIST_MSPepSearch = 1002750,

            /// <summary>MSP text format defined by the NIST.</summary>
            MS_NIST_MSP_format = 1002751,

            /// <summary>Database containing spectra.</summary>
            MS_database_type_spectral_library = 1002752,

            /// <summary>Value range for scores.</summary>
            MS_value_between_0_and_1000_inclusive = 1002753,

            /// <summary>MSPepSearch score (0 for entirely dissimilar and 1000 for identical observed spectrum and library spectrum.</summary>
            MS_MSPepSearch_score = 1002754,

            /// <summary>A combined MS2 (with fragment ions) and spectral library search.</summary>
            MS_combined_ms_ms___spectral_library_search = 1002755,

            /// <summary>Quantitation analysis using the Thermo Fisher sulfhydryl-reactive iodo tandem mass tag (iodoTMT) labelling workflow.</summary>
            MS_iodoTMT_quantitation_analysis = 1002756,

            /// <summary>Quantitation analysis using the Thermo Fisher carbonyl-reactive glyco-tandem mass tag (glyco-TMT) labelling workflow.</summary>
            MS_glyco_TMT_quantitation_analysis = 1002757,

            /// <summary>Quantitation analysis using the Thermo Fisher carbonyl-reactive aminoxy tandem mass tag (aminoxyTMT) labelling workflow.</summary>
            MS_aminoxyTMT_quantitation_analysis = 1002758,

            /// <summary>Quantitation analysis using the Thermo Fisher carbonyl-reactive hydrazide tandem mass tag (hydrazide-TMT) labelling workflow.</summary>
            MS_hydrazideTMT_quantitation_analysis = 1002759,

            /// <summary>Quantification analysis using the carbonyl-reactive isobaric tags for relative and absolute quantification hydrazide (iTRAQH) labelling workflow.</summary>
            MS_iTRAQH_quantitation_analysis = 1002760,

            /// <summary>Quantification analysis using the amine-reactive deuterium isobaric amine reactive tag (DiART) labelling workflow.</summary>
            MS_DiART_quantitation_analysis = 1002761,

            /// <summary>Quantification analysis using the amine-reactive dimethyl leucine (DiLeu) tag labelling workflow.</summary>
            MS_DiLeu_quantitation_analysis = 1002762,

            /// <summary>The name of the sample labelled with the TMT reagent 127N.</summary>
            MS_TMT_reagent_127N = 1002763,

            /// <summary>The name of the sample labelled with the TMT reagent 127C.</summary>
            MS_TMT_reagent_127C = 1002764,

            /// <summary>The name of the sample labelled with the TMT reagent 128N.</summary>
            MS_TMT_reagent_128N = 1002765,

            /// <summary>The name of the sample labelled with the TMT reagent 128C.</summary>
            MS_TMT_reagent_128C = 1002766,

            /// <summary>The name of the sample labelled with the TMT reagent 129N.</summary>
            MS_TMT_reagent_129N = 1002767,

            /// <summary>The name of the sample labelled with the TMT reagent 129C.</summary>
            MS_TMT_reagent_129C = 1002768,

            /// <summary>The name of the sample labelled with the TMT reagent 130N.</summary>
            MS_TMT_reagent_130N = 1002769,

            /// <summary>The name of the sample labelled with the TMT reagent 130C.</summary>
            MS_TMT_reagent_130C = 1002770,

            /// <summary>Deuterium isobaric amine reactive tag labeling reagent.</summary>
            MS_DiART_reagent = 1002771,

            /// <summary>The name of the sample labelled with the DiART reagent 114.</summary>
            MS_DiART_reagent_114 = 1002772,

            /// <summary>The name of the sample labelled with the DiART reagent 115.</summary>
            MS_DiART_reagent_115 = 1002773,

            /// <summary>The name of the sample labelled with the DiART reagent 116.</summary>
            MS_DiART_reagent_116 = 1002774,

            /// <summary>The name of the sample labelled with the DiART reagent 117.</summary>
            MS_DiART_reagent_117 = 1002775,

            /// <summary>The name of the sample labelled with the DiART reagent 118.</summary>
            MS_DiART_reagent_118 = 1002776,

            /// <summary>The name of the sample labelled with the DiART reagent 119.</summary>
            MS_DiART_reagent_119 = 1002777,

            /// <summary>Dimethyl leucine labeling reagent.</summary>
            MS_DiLeu_reagent = 1002778,

            /// <summary>The name of the sample labelled with the DiLeu reagent 115.</summary>
            MS_DiLeu_reagent_115 = 1002779,

            /// <summary>The name of the sample labelled with the DiLeu reagent 116.</summary>
            MS_DiLeu_reagent_116 = 1002780,

            /// <summary>The name of the sample labelled with the DiLeu reagent 117.</summary>
            MS_DiLeu_reagent_117 = 1002781,

            /// <summary>The name of the sample labelled with the DiLeu reagent 118.</summary>
            MS_DiLeu_reagent_118 = 1002782,

            /// <summary>The 6550 Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6550_iFunnel_Q_TOF_LC_MS = 1002783,

            /// <summary>The 6550A Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6550A_iFunnel_Q_TOF_LC_MS = 1002784,

            /// <summary>The 6520B Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6520B_Q_TOF_LC_MS = 1002785,

            /// <summary>The 6530A Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6530A_Q_TOF_LC_MS = 1002786,

            /// <summary>The 6530B Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6530B_Q_TOF_LC_MS = 1002787,

            /// <summary>The 6538 Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6538_Q_TOF_LC_MS = 1002788,

            /// <summary>The 6540 Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6540_Q_TOF_LC_MS = 1002789,

            /// <summary>The 6542 Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6542_Q_TOF_LC_MS = 1002790,

            /// <summary>The 6545 Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6545_Q_TOF_LC_MS = 1002791,

            /// <summary>The 6560 Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6560_Q_TOF_LC_MS = 1002792,

            /// <summary>The 6570 Quadrupole Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6570_Q_TOF_LC_MS = 1002793,

            /// <summary>The 6120B Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a single quadrupole mass spectrometer from the 6100 Series of Agilent mass spectrometers.</summary>
            MS_6120B_Quadrupole_LC_MS = 1002794,

            /// <summary>The 6150 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a single quadrupole mass spectrometer from the 6100 Series of Agilent mass spectrometers.</summary>
            MS_6150_Quadrupole_LC_MS = 1002795,

            /// <summary>The 6224 Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6224_Time_of_Flight_LC_MS = 1002796,

            /// <summary>The 6230A Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6230A_Time_of_Flight_LC_MS = 1002797,

            /// <summary>The 6230B Time-of-Flight LC/MS is a Agilent liquid chromatography instrument combined with a Agilent time of flight mass spectrometer.</summary>
            MS_6230B_Time_of_Flight_LC_MS = 1002798,

            /// <summary>The 6430 Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_6430_Triple_Quadrupole_LC_MS = 1002799,

            /// <summary>The 6495A Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_6495A_Triple_Quadrupole_LC_MS = 1002800,

            /// <summary>The 6495B Quadrupole LC/MS system is a Agilent liquid chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_6495B_Triple_Quadrupole_LC_MS = 1002801,

            /// <summary>The 7000A Quadrupole GC/MS system is a Agilent gas chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_7000A_Triple_Quadrupole_GC_MS = 1002802,

            /// <summary>The 7000B Quadrupole GC/MS system is a Agilent gas chromatography instrument combined with a Agilent triple quadrupole mass spectrometer.</summary>
            MS_7000B_Triple_Quadrupole_GC_MS = 1002803,

            /// <summary>The 7800 Quadrupole ICP-MS system is a Agilent inductively couple plasma instrument combined with a Agilent quadrupole mass spectrometer.</summary>
            MS_7800_Quadrupole_ICP_MS = 1002804,

            /// <summary>The 8800 Quadrupole ICP-MS system is a Agilent inductively couple plasma instrument combined with a Agilent quadrupole mass spectrometer.</summary>
            MS_8800_Triple_Quadrupole_ICP_MS = 1002805,

            /// <summary>Molecular entity having a net positive or negative electric charge.</summary>
            MS_ion = 1002806,

            /// <summary>Adduct ion with positive ionization.</summary>
            MS_positive_mode_adduct_ion = 1002807,

            /// <summary>Adduct ion with negative ionization.</summary>
            MS_negative_mode_adduct_ion = 1002808,

            /// <summary>Nonphysical characteristic attributed to an adduct ion.</summary>
            MS_adduct_ion_attribute = 1002809,

            /// <summary>Theoretical m/z of the X component in the adduct (addition product) M+X or M-X. This term was formerly called 'adduct ion mass', but it is not really a mass. It corresponds to the column mislabelled as 'mass' at https://fiehnlab.ucdavis.edu/staff/kind/Metabolomics/MS-Adduct-Calculator.</summary>
            MS_adduct_ion_X_m_z = 1002810,

            /// <summary>Isotope of the matrix molecule M of an adduct formation.</summary>
            MS_adduct_ion_isotope = 1002811,

            /// <summary>(\</summary>
            MS_Regular_expression_for_adduct_ion_formula = 1002812,

            /// <summary>Adduct formation formula of the form M+X or M-X, as constrained by the provided regular expression.</summary>
            MS_adduct_ion_formula = 1002813,

            /// <summary>An electrical mobility unit that equals the speed</summary>
            MS_volt_second_per_square_centimeter = 1002814,

            /// <summary>Ion mobility measurement for an ion or spectrum of ions as measured in an ion mobility mass spectrometer. This might refer to the central value of a bin into which all ions within a narrow range of mobilities have been aggregated.</summary>
            MS_inverse_reduced_ion_mobility = 1002815,

            /// <summary>Array of population mean ion mobility values (K or K0) based on ion separation in gaseous phase due to different ion mobilities under an electric field based on ion size, m/z and shape, corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_mean_ion_mobility_array = 1002816,

            /// <summary>Bruker TDF raw file format.</summary>
            MS_Bruker_TDF_format = 1002817,

            /// <summary>Native format defined by frame=xsd:nonNegativeInteger scan=xsd:nonNegativeInteger.</summary>
            MS_Bruker_TDF_nativeID_format = 1002818,

            /// <summary>Bruker TDF comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_Bruker_TDF_nativeID_format__combined_spectra = 1002819,

            /// <summary>Adduct formed by protonation of a matrix molecule M, i.e. the addition of a matrix molecule M plus a proton.</summary>
            MS_M_H_ion = 1002820,

            /// <summary>Adduct formed by deprotonation of a matrix molecule M, i.e. the removal of a proton from a matrix molecule M.</summary>
            MS_M_H_ion1 = 1002821,

            /// <summary>File format developed by the OpenMS team.</summary>
            MS_OpenMS_file_format = 1002822,

            /// <summary>OpenMS intermediate identification format.</summary>
            MS_idXML = 1002823,

            /// <summary>OpenMS feature file format.</summary>
            MS_featureXML = 1002824,

            /// <summary>OpenMS consensus map format.</summary>
            MS_consensusXML = 1002825,

            /// <summary>MetaMorpheus search engine.</summary>
            MS_MetaMorpheus = 1002826,

            /// <summary>MetaMorpheus score for PSMs.</summary>
            MS_MetaMorpheus_score = 1002827,

            /// <summary>MetaMorpheus score for protein groups.</summary>
            MS_MetaMorpheus_protein_score = 1002828,

            /// <summary>Feature intensity produced by XCMS findPeaks() from integrated peak intensity.</summary>
            MS_XCMS_into = 1002829,

            /// <summary>Feature intensity produced by XCMS findPeaks() from baseline corrected integrated peak intensity.</summary>
            MS_XCMS_intf = 1002830,

            /// <summary>Feature intensity produced by XCMS findPeaks() from maximum peak intensity.</summary>
            MS_XCMS_maxo = 1002831,

            /// <summary>Feature intensity produced by XCMS findPeaks() from feature area that is not normalized by the scan rate.</summary>
            MS_XCMS_area = 1002832,

            /// <summary>Polarities of the scans of a run are alternating, i.e. both positive and negative mode scans are acquired.</summary>
            MS_alternating_polarity_mode = 1002833,

            /// <summary>The Delta Score reported by Proteome Discoverer version 2.</summary>
            MS_ProteomeDiscoverer_Delta_Score = 1002834,

            /// <summary>Thermo Fisher Scientific LTQ Orbitrap Classic.</summary>
            MS_LTQ_Orbitrap_Classic = 1002835,

            /// <summary>Dataset identifier issued by the iProX repository. A dataset can refer to either a single sample as part of a study, or all samples that are part of the study corresponding to a publication.</summary>
            MS_iProX_dataset_identifier = 1002836,

            /// <summary>URI that allows the access to one dataset in the iProX repository. A dataset can refer to either a single sample as part of a study, or all samples that are part of the study corresponding to a publication.</summary>
            MS_iProX_dataset_URI = 1002837,

            /// <summary>mzMLb file format, mzML encapsulated within HDF5.</summary>
            MS_mzMLb_format = 1002838,

            /// <summary>Conversion of a file format to Proteomics Standards Initiative mzMLb file format.</summary>
            MS_Conversion_to_mzMLb = 1002839,

            /// <summary>Data belonging to an external reference.</summary>
            MS_external_reference_data = 1002840,

            /// <summary>The HDF5 dataset location containing the binary data, relative to the dataset containing the mzML. Also indicates that there is no data in the &lt;binary&gt; section of the BinaryDataArray.</summary>
            MS_external_HDF5_dataset = 1002841,

            /// <summary>The position in the external data where the array begins.</summary>
            MS_external_offset = 1002842,

            /// <summary>Describes how many fields an array contains.</summary>
            MS_external_array_length = 1002843,

            /// <summary>Root node for terms relating to the description of an Experiment in relation to the PRIDE-XML element ExperimentCollection/Experiment/additional/cvParam.</summary>
            MS_Experiment_additional_parameter = 1002844,

            /// <summary>URI of one external file associated to the PRIDE experiment (maybe through a PX submission).</summary>
            MS_Associated_file_URI = 1002845,

            /// <summary>URI of one raw data file associated to the PRIDE experiment (maybe through a PX submission).</summary>
            MS_Associated_raw_file_URI = 1002846,

            /// <summary>URI associated to one PX submission in ProteomeCentral.</summary>
            MS_ProteomeCentral_dataset_URI = 1002847,

            /// <summary>URI of one file labeled as 'Result', associated to one PX submission.</summary>
            MS_Result_file_URI = 1002848,

            /// <summary>URI of one search engine output file associated to one PX submission.</summary>
            MS_Search_engine_output_file_URI = 1002849,

            /// <summary>URI of one of one search engine output file associated to one PX submission.</summary>
            MS_Peak_list_file_URI = 1002850,

            /// <summary>URI of one file labeled as 'Other', associated to one PX submission.</summary>
            MS_Other_type_file_URI = 1002851,

            /// <summary>FTP location of one entire PX data set.</summary>
            MS_Dataset_FTP_location = 1002852,

            /// <summary>A dataset which does not have an associated published manuscript.</summary>
            MS_Dataset_with_no_associated_published_manuscript = 1002853,

            /// <summary>Dataset has been peer-reviewed somehow.</summary>
            MS_Peer_reviewed_dataset = 1002854,

            /// <summary>Dataset that has not been peer-reviewed by any means.</summary>
            MS_Non_peer_reviewed_dataset = 1002855,

            /// <summary>Dataset for which the identifications and/or spectra/traces are in formats that can be parsed by the hosting data repository such that internal references between identifications and spectra/traces are preserved and browsable at the repository. This is usually called a complete submission.</summary>
            MS_Supported_dataset_by_repository = 1002856,

            /// <summary>Dataset for which the identifications and/or spectra/traces are in formats that cannot be parsed by the hosting data repository and thus internal references between identifications and spectra/traces are not browsable at the repository. This is usually called a partial submission.</summary>
            MS_Unsupported_dataset_by_repository = 1002857,

            /// <summary>A dataset which has an associated manuscript pending for publication.</summary>
            MS_Dataset_with_its_publication_pending = 1002858,

            /// <summary>Additional URI of one raw data file associated to the PRIDE experiment (maybe through a PX submission). The URI is provided via an additional resource to PRIDE.</summary>
            MS_Additional_associated_raw_file_URI = 1002859,

            /// <summary>URI of one gel image file associated to one PX submission.</summary>
            MS_Gel_image_file_URI = 1002860,

            /// <summary>All the raw files included in the original dataset (or group of original datasets) have been reanalysed.</summary>
            MS_Reprocessed_complete_dataset = 1002861,

            /// <summary>A subset of the raw files included in the original dataset (or group of original datasets) has been reanalysed.</summary>
            MS_Reprocessed_subset_dataset = 1002862,

            /// <summary>One dataset is a reanalysis of previously published data.</summary>
            MS_Data_derived_from_previous_dataset = 1002863,

            /// <summary>No post-translational-modifications are been included in the identified peptides of one dataset.</summary>
            MS_No_PTMs_are_included_in_the_dataset = 1002864,

            /// <summary>A dataset has one associated manuscript, which has been accepted but no PubMedID is available yet.</summary>
            MS_Accepted_manuscript = 1002865,

            /// <summary>Literature reference associated with one dataset (including the authors, title, year and journal details). The value field can be used for the PubMedID, or to specify if one manuscript is just submitted or accepted, but it does not have a PubMedID yet.</summary>
            MS_Reference = 1002866,

            /// <summary>This means that the experimental information available has been improved, for instance precursor charges were added.</summary>
            MS_Experimental_information_has_been_refined_since_this_experiment_was_originally_made_publicly_available = 1002867,

            /// <summary>One dataset is not a reanalysis of previously published data.</summary>
            MS_Original_data = 1002868,

            /// <summary>Bioconductor package mzR for reading and writing mass spectrometry data files.</summary>
            MS_mzR = 1002869,

            /// <summary>Bioconductor package MSnbase provides infrastructure for manipulation, processing and visualization of mass spectrometry and proteomics data, ranging from raw to quantitative and annotated data.</summary>
            MS_MSnbase = 1002870,

            /// <summary>Bioconductor package CAMERA for annotation of peak lists generated by xcms, rule based annotation of isotopes and adducts, isotope validation, EIC correlation based tagging of unknown adducts and fragments.</summary>
            MS_CAMERA = 1002871,

            /// <summary>Dataset identifier issued by the Panorama Public repository. A dataset can refer to either a single sample as part of a study, or all samples that are part of the study corresponding to a publication.</summary>
            MS_Panorama_Public_dataset_identifier = 1002872,

            /// <summary>URI that allows the access to one dataset in the Panorama Public repository. A dataset can refer to either a single sample as part of a study, or all samples that are part of the study corresponding to a publication.</summary>
            MS_Panorama_Public_dataset_URI = 1002873,

            /// <summary>Thermo Scientific TSQ Altis Triple Quadrupole MS.</summary>
            MS_TSQ_Altis = 1002874,

            /// <summary>Thermo Scientific TSQ Quantis Triple Quadrupole MS.</summary>
            MS_TSQ_Quantis = 1002875,

            /// <summary>Thermo Scientific TSQ 9000 Triple Quadrupole MS.</summary>
            MS_TSQ_9000 = 1002876,

            /// <summary>Thermo Scientific Q Exactive HF-X Hybrid Quadrupole Orbitrap MS.</summary>
            MS_Q_Exactive_HF_X = 1002877,

            /// <summary>Software for the analysis of small molecules.</summary>
            MS_small_molecule_analysis_software = 1002878,

            /// <summary>Metabolomics analysis software for LC-MS data from Nonlinear Dynamics.</summary>
            MS_Progenesis_QI = 1002879,

            /// <summary>Metabolomics analysis software from Thermo Fisher Scientific.</summary>
            MS_Compound_Discoverer = 1002880,

            /// <summary>Metabolite identification tool MyCompoundID.</summary>
            MS_MyCompoundID = 1002881,

            /// <summary>Function used to calculate the study variable quantification value.</summary>
            MS_study_variable_average_function = 1002882,

            /// <summary>Median function.</summary>
            MS_median = 1002883,

            /// <summary>Function used to calculate the study variable quantification variation value.</summary>
            MS_study_variable_variation_function = 1002884,

            /// <summary>Standard error function.</summary>
            MS_standard_error = 1002885,

            /// <summary>The value reported in a small molecule quantification.</summary>
            MS_small_molecule_quantification_datatype = 1002886,

            /// <summary>The normalised abundance produced by Progenesis QI LC-MS.</summary>
            MS_Progenesis_QI_normalised_abundance = 1002887,

            /// <summary>The confidence score produced by a small molecule analysis software.</summary>
            MS_small_molecule_confidence_measure = 1002888,

            /// <summary>The confidence score produced by Progenesis QI.</summary>
            MS_Progenesis_MetaScope_score = 1002889,

            /// <summary>The fragmentation confidence score.</summary>
            MS_fragmentation_score = 1002890,

            /// <summary>The isotopic fit confidence score.</summary>
            MS_isotopic_fit_score = 1002891,

            /// <summary>An attribute describing ion mobility searches.</summary>
            MS_ion_mobility_attribute = 1002892,

            /// <summary>Abstract array of ion mobility data values. A more specific child term concept should be specified in data files to make precise the nature of the data being provided.</summary>
            MS_ion_mobility_array = 1002893,

            /// <summary>Unique chemical structure identifier for chemical compounds.</summary>
            MS_InChIKey = 1002894,

            /// <summary>Compound identification information.</summary>
            MS_small_molecule_identification_attribute = 1002895,

            /// <summary>Confidence level for annotation of identified compounds as defined by the Metabolomics Standards Initiative (MSI). The value slot can have the values 'Level 0' until 'Level 4'.</summary>
            MS_compound_identification_confidence_level = 1002896,

            /// <summary>OBSOLETE Identifies a peak when no de-isotoping has been performed. The value slot reports the isotopomer peak, e.g. '2H', '13C', '15N', '18O', '31P'.</summary>
            MS_isotopomer_peak_OBSOLETE = 1002897,

            /// <summary>Native format defined by scan=xsd:nonNegativeInteger.</summary>
            MS_Shimadzu_Biotech_QTOF_nativeID_format = 1002898,

            /// <summary>msalign file format.</summary>
            MS_msalign_format = 1002899,

            /// <summary>TopFD feature file format.</summary>
            MS_feature_format = 1002900,

            /// <summary>TopPIC: a software tool for top-down mass spectrometry-based proteoform identification and characterization.</summary>
            MS_TopPIC = 1002901,

            /// <summary>Top-down mass spectral feature detection.</summary>
            MS_TopFD = 1002902,

            /// <summary>A mass graph-based approach for the identification of modified proteoforms using top-down tandem mass spectra.</summary>
            MS_TopMG = 1002903,

            /// <summary>Proteoform level information.</summary>
            MS_proteoform_level_identification_attribute = 1002904,

            /// <summary>Identification confidence metric for a proteoform.</summary>
            MS_proteoform_level_identification_statistic = 1002905,

            /// <summary>Search engine specific proteoform scores.</summary>
            MS_search_engine_specific_score_for_proteoforms = 1002906,

            /// <summary>Estimation of the global false discovery rate of proteoforms.</summary>
            MS_proteoform_level_global_FDR = 1002907,

            /// <summary>Estimation of the local false discovery rate of proteoforms.</summary>
            MS_proteoform_level_local_FDR = 1002908,

            /// <summary>Estimated statistical threshold at proteoform-level.</summary>
            MS_proteoform_level_statistical_threshold = 1002909,

            /// <summary>Threshold for the global false discovery rate of proteoforms.</summary>
            MS_proteoform_level_global_FDR_threshold = 1002910,

            /// <summary>Threshold for the local false discovery rate of proteoforms.</summary>
            MS_proteoform_level_local_FDR_threshold = 1002911,

            /// <summary>Search engine input parameters specific to TopPIC.</summary>
            MS_TopPIC_input_parameter = 1002912,

            /// <summary>Fixed modifications for TopPIC searching.</summary>
            MS_TopPIC_fixed_modification = 1002913,

            /// <summary>N-terminal forms of proteins allowed in TopPIC searching.</summary>
            MS_TopPIC_N_term_form = 1002914,

            /// <summary>Error tolerance for precursor and fragment masses in PPM.</summary>
            MS_TopPIC_error_tolerance = 1002915,

            /// <summary>Maximum value of the mass shift (in Dalton) of an unexpected modification.</summary>
            MS_TopPIC_max_shift = 1002916,

            /// <summary>Minimum value of the mass shift (in Dalton) of an unexpected modification.</summary>
            MS_TopPIC_min_shift = 1002917,

            /// <summary>Maximum number of unexpected modifications in a proteoform spectrum match.</summary>
            MS_TopPIC_shift_num = 1002918,

            /// <summary>Spectrum-level cutoff type for filtering identified proteoform spectrum matches.</summary>
            MS_TopPIC_spectral_cutoff_type = 1002919,

            /// <summary>Spectrum-level cutoff value for filtering identified proteoform spectrum matches.</summary>
            MS_TopPIC_spectral_cutoff_value = 1002920,

            /// <summary>Proteoform-level cutoff type for filtering identified proteoform spectrum matches.</summary>
            MS_TopPIC_proteoform_level_cutoff_type = 1002921,

            /// <summary>Proteoform-level cutoff value for filtering identified proteoform spectrum matches.</summary>
            MS_TopPIC_proteoform_level_cutoff_value = 1002922,

            /// <summary>P-value and E-value estimation using generating function.</summary>
            MS_TopPIC_generating_function = 1002923,

            /// <summary>Number of combined spectra.</summary>
            MS_TopPIC_combined_spectrum_number = 1002924,

            /// <summary>The text file containing the information of common PTMs.</summary>
            MS_TopPIC_mod_file = 1002925,

            /// <summary>Number of threads used in TopPIC.</summary>
            MS_TopPIC_thread_number = 1002926,

            /// <summary>Proteoform identification using TopFD feature file.</summary>
            MS_TopPIC_use_TopFD_feature = 1002927,

            /// <summary>TopPIC spectrum-level E-value.</summary>
            MS_TopPIC_spectral_E_value = 1002928,

            /// <summary>TopPIC spectrum-level FDR.</summary>
            MS_TopPIC_spectral_FDR = 1002929,

            /// <summary>TopPIC proteoform-level FDR.</summary>
            MS_TopPIC_proteoform_level_FDR = 1002930,

            /// <summary>TopPIC spectrum-level p-value.</summary>
            MS_TopPIC_spectral_p_value = 1002931,

            /// <summary>Modification identification score.</summary>
            MS_TopPIC_MIScore = 1002932,

            /// <summary>TopPIC:MIScore threshold.</summary>
            MS_TopPIC_MIScore_threshold = 1002933,

            /// <summary>Search engine input parameters specific to TopMG.</summary>
            MS_TopMG_input_parameter = 1002934,

            /// <summary>Fixed modifications for TopMG searching.</summary>
            MS_TopMG_fixed_modification = 1002935,

            /// <summary>N-terminal forms of proteins allowed in TopMG searching.</summary>
            MS_TopMG_N_term_form = 1002936,

            /// <summary>Error tolerance for precursor and fragment masses in PPM.</summary>
            MS_TopMG_error_tolerance = 1002937,

            /// <summary>Maximum value of the mass shift (in Dalton).</summary>
            MS_TopMG_max_shift = 1002938,

            /// <summary>Spectrum-level cutoff type for filtering identified proteoform spectrum matches.</summary>
            MS_TopMG_spectral_cutoff_type = 1002939,

            /// <summary>Spectrum-level cutoff value for filtering identified proteoform spectrum matches.</summary>
            MS_TopMG_spectral_cutoff_value = 1002940,

            /// <summary>Proteoform-level cutoff type for filtering identified proteoform spectrum matches.</summary>
            MS_TopMG_proteoform_level_cutoff_type = 1002941,

            /// <summary>Proteoform-level cutoff value for filtering identified proteoform spectrum matches.</summary>
            MS_TopMG_proteoform_level_cutoff_value = 1002942,

            /// <summary>The text file containing the information of common PTMs.</summary>
            MS_TopMG_mod_file = 1002943,

            /// <summary>Number of threads used in TopMG.</summary>
            MS_TopMG_thread_number = 1002944,

            /// <summary>Proteoform identification using TopFD feature file.</summary>
            MS_TopMG_use_TopFD_feature = 1002945,

            /// <summary>Gap size in constructing proteoform graph.</summary>
            MS_TopMG_proteoform_graph_gap_size = 1002946,

            /// <summary>Maximum number of variable PTMs.</summary>
            MS_TopMG_variable_PTM_number = 1002947,

            /// <summary>Maximum number of variable PTMs in a proteoform graph gap.</summary>
            MS_TopMG_variable_PTM_number_in_proteoform_graph_gap = 1002948,

            /// <summary>Protein filtering using ASF-DIAGONAL method.</summary>
            MS_TopMG_use_ASF_DIAGONAL = 1002949,

            /// <summary>TopMG spectrum-level E-value.</summary>
            MS_TopMG_spectral_E_value = 1002950,

            /// <summary>TopMG spectrum-level FDR.</summary>
            MS_TopMG_spectral_FDR = 1002951,

            /// <summary>TopMG proteoform-level FDR.</summary>
            MS_TopMG_proteoform_level_FDR = 1002952,

            /// <summary>TopMG spectrum-level p-value.</summary>
            MS_TopMG_spectral_p_value = 1002953,

            /// <summary>Structural molecular descriptor for the effective interaction area between the ion and neutral gas measured in ion mobility mass spectrometry.</summary>
            MS_collisional_cross_sectional_area = 1002954,

            /// <summary>Refined High Resolution mass spectrometry confidence level for annotation of identified compounds as proposed by Schymanski et al. The value slot can have the values 'Level 1', 'Level 2', 'Level 2a', 'Level 2b', 'Level 3', 'Level 4', and 'Level 5'.</summary>
            MS_hr_ms_compound_identification_confidence_level = 1002955,

            /// <summary>A mass spectrometry peak that represents one or more isotopic ions. The value slot contains a description of the represented isotope set, e.g. 'M+1 peak'.</summary>
            MS_isotopic_ion_MS_peak = 1002956,

            /// <summary>The described isotopomer mass spectrometric signal. The value slot contains a description of the represented isotopomer, e.g. '13C peak', '15N peak', '2H peak', '18O peak' or '31P peak'.</summary>
            MS_isotopomer_MS_peak = 1002957,

            /// <summary>The described isotopologue mass spectrometric signal. The value slot contains a description of the represented isotopologue, e.g. '13C1 peak' or '15N1 peak'.</summary>
            MS_isotopologue_MS_peak = 1002958,

            /// <summary>One of several species (or molecular entities) that have the same atomic composition (molecular formula) but different line formulae or different stereochemical formulae.</summary>
            MS_isomer = 1002959,

            /// <summary>An isomer that differs from another only in the spatial distribution of the constitutive isotopic atoms.</summary>
            MS_isotopomer = 1002960,

            /// <summary>A molecular entity that differs only in isotopic composition (number of isotopic substitutions).</summary>
            MS_isotopologue = 1002961,

            /// <summary>The arithmetic mean.</summary>
            MS_mean = 1002962,

            /// <summary>The coefficient of variation.</summary>
            MS_variation_coefficient = 1002963,

            /// <summary>Lipidomics analysis software.</summary>
            MS_lipidomics_analysis_software = 1002964,

            /// <summary>Lipid Data Analyzer software for lipid quantification.</summary>
            MS_Lipid_Data_Analyzer = 1002965,

            /// <summary>The Lipid Data Analyzer native chrom format.</summary>
            MS_chrom_format = 1002966,

            /// <summary>Software for identification of phospholipids by high-throughput processing of LC-MS and shotgun lipidomics datasets.</summary>
            MS_LipidHunter = 1002967,

            /// <summary>Software for consensual cross-platform lipidomics.</summary>
            MS_LipidXplorer = 1002968,

            /// <summary>An automated workflow for rule-based lipid identification using untargeted high-resolution tandem mass spectrometry data.</summary>
            MS_LipidMatch = 1002969,

            /// <summary>Open-source software for automated phospholipid tandem mass spectrometry identification.</summary>
            MS_Greazy = 1002970,

            /// <summary>LC-MS-based lipidomics and automated identification of lipids using the LipidBlast in-silico MS/MS library.</summary>
            MS_LipidBlast = 1002971,

            /// <summary>A computational lipid identification solution for untargeted lipidomics on data-independent acquisition tandem mass spectrometry platforms.</summary>
            MS_Lipid_Pro = 1002972,

            /// <summary>A computational workflow for the discovery of lipids for the identification of eicosanoid-phosphoinositides in platelets.</summary>
            MS_LipidFinder = 1002973,

            /// <summary>An integrated software package for high-confidence lipid identification.</summary>
            MS_LipiDex = 1002974,

            /// <summary>An-open source software for identifying lipids in LC-MS/MS-based lipidomics data.</summary>
            MS_LIQUID = 1002975,

            /// <summary>Analysis of lipid experiments, a calculator for m/z values of intact lipid molecules (MS1).</summary>
            MS_ALEX = 1002976,

            /// <summary>Analysis of lipid experiments 123, a calculator with m/z values of intact lipid molecules (MS1) and their fragment ions at the MS2 and MS3 level.</summary>
            MS_ALEX123 = 1002977,

            /// <summary>Software tool for the quantitative analysis of mass spectrometric lipidome data.</summary>
            MS_LIMSA = 1002978,

            /// <summary>Adduct-Based lipidomics software for the discovery and identification of oxidative stress biomarkers.</summary>
            MS_LOBSTAHS = 1002979,

            /// <summary>Lipid qualitative/quantitative analysis software for identification and quantitation of complex lipid molecular species.</summary>
            MS_LipidQA = 1002980,

            /// <summary>The Proline software suite for mass spectrometry based proteomics.</summary>
            MS_Proline = 1002981,

            /// <summary>PepNovo tool for de novo peptide sequencing.</summary>
            MS_PepNovo = 1002982,

            /// <summary>pNovo tool for de novo peptide sequencing and identification using HCD spectra.</summary>
            MS_pNovo = 1002983,

            /// <summary>Novor real-time peptide de novo sequencing software tool.</summary>
            MS_Novor = 1002984,

            /// <summary>Digestion of proteins separated by gel electrophoresis for mass spectrometric characterization of proteins and proteomes.</summary>
            MS_in_gel_digestion = 1002985,

            /// <summary>Digestion of proteins in solution for mass spectrometric characterization of proteins and proteomes.</summary>
            MS_in_solution_digestion = 1002986,

            /// <summary>IdentiPy.</summary>
            MS_IdentiPy = 1002987,

            /// <summary>The IdentiPy result 'RHNS'.</summary>
            MS_IdentiPy_RHNS = 1002988,

            /// <summary>The IdentiPy result 'hyperscore'.</summary>
            MS_IdentiPy_hyperscore = 1002989,

            /// <summary>ms_deisotope, a library for deisotoping and charge state deconvolution of mass spectra.</summary>
            MS_ms_deisotope = 1002990,

            /// <summary>python-psims, a library for generating mzML and mzIdentML.</summary>
            MS_python_psims = 1002991,

            /// <summary>Posterior error probability of the best identified peptide of the Andromeda search engine.</summary>
            MS_Andromeda_PEP = 1002995,

            /// <summary>Peak list file format of the Andromeda search engine.</summary>
            MS_Andromeda_apl_file_format = 1002996,

            /// <summary>Index number of a reanalysis within a ProteomeXchange reprocessed dataset identifier container (RPXD).</summary>
            MS_ProteomeXchange_dataset_identifier_reanalysis_number = 1002997,

            /// <summary>Shimadzu Scientific Instruments LCMS-9030 Q-TOF MS.</summary>
            MS_LCMS_9030 = 1002998,

            /// <summary>Shimadzu Scientific Instruments LCMS-8060 MS.</summary>
            MS_LCMS_8060 = 1002999,

            /// <summary>Shimadzu Scientific Instruments LCMS-8050 MS.</summary>
            MS_LCMS_8050 = 1003000,

            /// <summary>Shimadzu Scientific Instruments LCMS-8045 MS.</summary>
            MS_LCMS_8045 = 1003001,

            /// <summary>Shimadzu Scientific Instruments LCMS-8040 MS.</summary>
            MS_LCMS_8040 = 1003002,

            /// <summary>Shimadzu Scientific Instruments LCMS-2020.</summary>
            MS_LCMS_2020 = 1003003,

            /// <summary>Bruker Daltonics' maXis II.</summary>
            MS_maXis_II = 1003004,

            /// <summary>Bruker Daltonics' timsTOF Pro.</summary>
            MS_timsTOF_Pro = 1003005,

            /// <summary>Array of population mean ion mobility values based on ion separation in gaseous phase due to different ion mobilities under an electric field based on ion size, m/z and shape, normalized for the local conditions and reported in volt-second per square centimeter, corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_mean_inverse_reduced_ion_mobility_array = 1003006,

            /// <summary>Array of raw ion mobility values (K or K0) based on ion separation in gaseous phase due to different ion mobilities under an electric field based on ion size, m/z and shape, corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_raw_ion_mobility_array = 1003007,

            /// <summary>Array of raw ion mobility values based on ion separation in gaseous phase due to different ion mobilities under an electric field based on ion size, m/z and shape, normalized for the local conditions and reported in volt-second per square centimeter, corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_raw_inverse_reduced_ion_mobility_array = 1003008,

            /// <summary>Shimadzu Biotech LCD file format.</summary>
            MS_Shimadzu_Biotech_LCD_format = 1003009,

            /// <summary>Software for lipidome-specific prediction and identification of oxidized phospholipids from LC-MS datasets.</summary>
            MS_LPPtiger = 1003010,

            /// <summary>Sequence-tag-based search engine pFind.</summary>
            MS_pFind = 1003011,

            /// <summary>Kernel mass spectral dot product scoring function.</summary>
            MS_KSDP_score = 1003012,

            /// <summary>i3-tms search engine and data-analysis software.</summary>
            MS_i3tms = 1003013,

            /// <summary>A database search-based peptide identification tool.</summary>
            MS_MSFragger = 1003014,

            /// <summary>Peptide that is shared between protein groups and assigned to the protein group with the largest number of identified peptides.</summary>
            MS_razor_peptide = 1003015,

            /// <summary>Fraction of peptide evidence attributable to a protein or a set of indistinguishable proteins.</summary>
            MS_ProteinProphet_peptide_weight = 1003016,

            /// <summary>Fraction of peptide evidence attributable to a group of proteins.</summary>
            MS_ProteinProphet_peptide_group_weight = 1003017,

            /// <summary>General proteomics processing toolkit for shotgun proteomics.</summary>
            MS_Philosopher = 1003018,

            /// <summary>Representation of chromatographic pressure versus time.</summary>
            MS_pressure_chromatogram = 1003019,

            /// <summary>Representation of the chromatographic flow rate versus time.</summary>
            MS_flow_rate_chromatogram = 1003020,

            /// <summary>Post-translational modification which is assumed to be present at each instance of a residue type.</summary>
            MS_Fixed_modification = 1003021,

            /// <summary>Post-translational modification which may or may not be present at a residue type.</summary>
            MS_Variable_modification = 1003022,

            /// <summary>Cross-Linking MS search engine.</summary>
            MS_OpenPepXL = 1003023,

            /// <summary>The OpenPepXL score for a cross-link spectrum match.</summary>
            MS_OpenPepXL_score = 1003024,

            /// <summary>A named element that is an attribute in a proteomics standards file.</summary>
            MS_named_element = 1003025,

            /// <summary>A named element that is an attribute in a mzIdentML file.</summary>
            MS_named_element_in_mzIdentML = 1003026,

            /// <summary>A named element that is an attribute in a mzML file.</summary>
            MS_named_element_in_mzML = 1003027,

            /// <summary>Thermo Scientific Orbitrap Exploris 480 Quadrupole Orbitrap MS.</summary>
            MS_Orbitrap_Exploris_480 = 1003028,

            /// <summary>Thermo Scientific Orbitrap Eclipse mass spectrometer with Tribrid architecture consisting of quadrupole mass filter, linear ion trap and Orbitrap mass analyzers.</summary>
            MS_Orbitrap_Eclipse = 1003029,

            /// <summary>Minimum number of significant unique sequences required in a protein hit. The setting is only relevant if the protein grouping strategy is 'family clustering'.</summary>
            MS_Mascot_MinNumSigUniqueSeqs = 1003030,

            /// <summary>Main identifier of a CPTAC dataset.</summary>
            MS_CPTAC_accession_number = 1003031,

            /// <summary>The confidence code to describe the confidence of annotated compounds as defined by the MS-DIAL program.</summary>
            MS_compound_identification_confidence_code_in_MS_DIAL = 1003032,

            /// <summary>Non-inherent characteristic attributed to a molecular entity.</summary>
            MS_molecular_entity_attribute = 1003033,

            /// <summary>Smallest constituent unit of ordinary matter that constitutes a chemical element.</summary>
            MS_atom = 1003034,

            /// <summary>Low molecular weight (&lt; 900 daltons) organic compound that may regulate a biological process.</summary>
            MS_small_molecule = 1003035,

            /// <summary>Small molecule that is the intermediate end product of metabolism.</summary>
            MS_metabolite = 1003036,

            /// <summary>Nucleotide containing ribose as its pentose component.</summary>
            MS_ribonucleotide = 1003037,

            /// <summary>Monomer, or single unit, of DNA, or deoxyribonucleic acid.</summary>
            MS_deoxyribonucleotide = 1003038,

            /// <summary>Organic molecule that contains amine (-NH2) and carboxyl (-COOH) functional groups, along with a side chain (R group) that is specific to each amino acid.</summary>
            MS_amino_acid = 1003039,

            /// <summary>Simplest form of sugar and the most basic units of carbohydrate that cannot be further hydrolyzed to a simpler molecule.</summary>
            MS_monosaccharide = 1003040,

            /// <summary>Molecule composed of a chain of nucleotides.</summary>
            MS_nucleic_acid = 1003041,

            /// <summary>Polymeric carbohydrate molecules composed of long chains of monosaccharide units bound together by glycosidic linkages.</summary>
            MS_polysaccharide = 1003042,

            /// <summary>Number of amino acid residues in a peptide, commonly referred to as the peptide length.</summary>
            MS_number_of_residues = 1003043,

            /// <summary>Number of amino acid residue bonds that should have been cleaved by the cleavage agent used, but were not.</summary>
            MS_number_of_missed_cleavages = 1003044,

            /// <summary>Process of mapping a peptide sequence to a protein sequence.</summary>
            MS_peptide_to_protein_mapping = 1003045,

            /// <summary>Nonphysical characteristic attributed to the result of peptide-to-protein mapping.</summary>
            MS_peptide_to_protein_mapping_attribute = 1003046,

            /// <summary>Offset in number of residues from the n terminus of the protein at which the peptide begins. Use 1 when the first residue of the peptide sequence is the first residue of the protein sequence.</summary>
            MS_protein_sequence_offset = 1003047,

            /// <summary>Total number of termini that match standard rules for the cleavage agent, 2 when both termini match cleavage agent rules, 1 when only one terminus does, and 0 if neither terminus matches cleavage agent rules.</summary>
            MS_number_of_enzymatic_termini = 1003048,

            /// <summary>Peptide that contains zero or more mass modifications on the termini or side chains of its amino acid residues, and may be differentiated from other peptidoforms with the same peptide sequence but different mass modification configurations.</summary>
            MS_peptidoform = 1003049,

            /// <summary>Non-inherent characteristic attributed to a peptidoform.</summary>
            MS_peptidoform_attribute = 1003050,

            /// <summary>Peptidoform that has formed an adduct with an ion, thereby rendering it potentially detectable with a mass spectrometer. Commonly called a 'precursor' or 'precursor ion' or 'parent ion'.</summary>
            MS_peptidoform_ion = 1003051,

            /// <summary>Inherent or measurable characteristic of a peptidoform ion.</summary>
            MS_peptidoform_ion_property = 1003052,

            /// <summary>Mass-to-charge ratio of a peptidoform ion composed of the most common isotope of each atom computed from the putative knowledge of its molecular constituents.</summary>
            MS_theoretical_monoisotopic_m_z = 1003053,

            /// <summary>Mass-to-charge ratio of a peptidoform ion computed from the putative knowledge of its molecular constituents, averaged over the distribution of naturally occurring isotopes.</summary>
            MS_theoretical_average_m_z = 1003054,

            /// <summary>Product of a direct addition of two or more distinct molecules, resulting in a single reaction product containing all atoms of all components. The resultant is considered a distinct molecular species.</summary>
            MS_adduct = 1003055,

            /// <summary>Physical measurable characteristic of an adduct ion.</summary>
            MS_adduct_ion_property = 1003056,

            /// <summary>Ordinal number of the scan indicating its order of acquisition within a mass spectrometry acquisition run.</summary>
            MS_scan_number = 1003057,

            /// <summary>Inherent or measurable characteristic of a spectrum.</summary>
            MS_spectrum_property = 1003058,

            /// <summary>Number of peaks or features in a spectrum. For a peak-picked spectrum, this will correspond to the number of data points. For a non-peak-picked spectrum, this corresponds to the number of features discernable in the spectrum, which will be fewer than the number of data points.</summary>
            MS_number_of_peaks = 1003059,

            /// <summary>Number of data points in a spectrum. For a peak-picked spectrum, this will correspond to the number of peaks. For a non-peak-picked spectrum, this corresponds to the number of values in the data array, which are not all peaks.</summary>
            MS_number_of_data_points = 1003060,

            /// <summary>Label attached to a spectrum uniquely naming it within a collection of spectra, often in a spectral library. It is often a string combination of peptide sequence, charge, mass modifications, collision energy, but will obviously be different for small molecules or unidentified spectra. It must be unique within a collection.</summary>
            MS_library_spectrum_name = 1003061,

            /// <summary>Integer index value that indicates the spectrum's ordered position within a spectral library. By custom, index counters should begin with 0.</summary>
            MS_library_spectrum_index = 1003062,

            /// <summary>PSI universal spectrum identifier (USI) multipart key that uniquely identifies a spectrum available in a ProteomeXchange datasets or spectral library.</summary>
            MS_universal_spectrum_identifier = 1003063,

            /// <summary>Non-inherent characteristic attributed to spectrum aggregation.</summary>
            MS_spectrum_aggregation_attribute = 1003064,

            /// <summary>Categorization of a spectrum based on its type of aggregation (e.g., individual spectrum, consensus spectrum, best replicate spectrum, etc.).</summary>
            MS_spectrum_aggregation_type = 1003065,

            /// <summary>Spectrum that is not the result of some aggregation process.</summary>
            MS_singleton_spectrum = 1003066,

            /// <summary>Spectrum that is the result of merging several replicate spectra to form a spectrum that is more representative of its class and ideally less noisy that any of its source replicates.</summary>
            MS_consensus_spectrum = 1003067,

            /// <summary>Spectrum that is considered the most representative from a pool of replicate spectra.</summary>
            MS_best_replicate_spectrum = 1003068,

            /// <summary>Number of replicate spectra available for use during the aggregation process.</summary>
            MS_number_of_replicate_spectra_available = 1003069,

            /// <summary>Number of replicate spectra used during the aggregation process. This is generally applicable when there are many replicates available, but some are discarded as being low S/N, blended, or otherwise unsuitable, and the remaining set is then used for merging via a consensus algorithm.</summary>
            MS_number_of_replicate_spectra_used = 1003070,

            /// <summary>Non-inherent characteristic attributed to spectrum aggregation.</summary>
            MS_spectrum_origin_attribute = 1003071,

            /// <summary>Categorization of a spectrum based on its origin (e.g., observed spectrum, predicted spectrum, demultiplexed spectrum, etc.).</summary>
            MS_spectrum_origin_type = 1003072,

            /// <summary>Spectrum that originates from an analysis attempt of a single analyte species on an instrument.</summary>
            MS_observed_spectrum = 1003073,

            /// <summary>Spectrum that originates from a compututational algorithm that attempts to predict spectra.</summary>
            MS_predicted_spectrum = 1003074,

            /// <summary>Spectrum that originates from an attempted extraction of a single ion spieces from a multiplexed spectrum that contains multiple ion species.</summary>
            MS_demultiplexed_spectrum = 1003075,

            /// <summary>Spectrum provided in the form of plain numerical values without any information pertaining to the interpretation of features.</summary>
            MS_uninterpreted_spectrum = 1003076,

            /// <summary>Spectrum provided in a form where specific features of the spectrum are interpreted to provide putative explanations for some feature.</summary>
            MS_interpreted_spectrum = 1003077,

            /// <summary>Non-inherent characteristic attributed to an interpreted spectrum.</summary>
            MS_interpreted_spectrum_attribute = 1003078,

            /// <summary>Fraction of intensity summed from all unassigned peaks divided by the intensity summed from all peaks in the spectrum.</summary>
            MS_total_unassigned_intensity_fraction = 1003079,

            /// <summary>Fraction of intensity summed from unassigned peaks among the top 20 divided by the intensity summed from all top 20 peaks in the spectrum.</summary>
            MS_top_20_peak_unassigned_intensity_fraction = 1003080,

            /// <summary>Monoisotopic mass delta in Daltons of an amino acid residue modification whose atomic composition or molecular identity has not been determined. This term should not be used for modifications of known molecular identity such as those available in Unimod, RESID or PSI-MOD. This term MUST NOT be used inside the &lt;Modification&gt; element in mzIdentML.</summary>
            MS_unidentified_modification_monoisotopic_mass_delta = 1003081,

            /// <summary>Data processing software for untargeted metabolomics and lipidomics that supports multiple instruments and MS vendors.</summary>
            MS_MS_DIAL = 1003082,

            /// <summary>Data file that contains original data as generated by an instrument, although not necessarily in the original data format (i.e. an original raw file converted to a different format is still a raw data file).</summary>
            MS_raw_data_file = 1003083,

            /// <summary>File that contains data that has been substantially processed or transformed from what was originally acquired by an instrument.</summary>
            MS_processed_data_file = 1003084,

            /// <summary>Intensity of the precursor ion in the previous MSn-1 scan (prior in time to the referencing MSn scan). For an MS2 scan, this means the MS1 precursor intensity. It is unspecified on whether this is an apex (across m/z) intensity, integrated (across m/z) intensity, a centroided peak intensity of unknown origin, or even summed across several isotopes.</summary>
            MS_previous_MSn_1_scan_precursor_intensity = 1003085,

            /// <summary>Intensity of the precursor ion current as measured by its apex point over time and m/z. It is unspecified whether this is the intensity of the selected isotope or the most intense isotope.</summary>
            MS_precursor_apex_intensity = 1003086,

            /// <summary>Dataset for which the identifications and/or spectra/traces are in formats that can be parsed by the hosting data repository such that internal references between identifications and spectra/traces are preserved and browsable at the repository. However, some metadata is not properly described due to lack of CV terms or some auxiliary data, such as data used to create a spectral library or a sequence search database crucial to the analysis, is not available.</summary>
            MS_supported_by_repository_but_incomplete_data_and_or_metadata = 1003087,

            /// <summary>Data array compression using mantissa bit truncation followed by zlib compression.</summary>
            MS_truncation_and_zlib_compression = 1003088,

            /// <summary>Data array compression using mantissa bit truncation, delta prediction and zlib compression.</summary>
            MS_truncation__delta_prediction_and_zlib_compression = 1003089,

            /// <summary>Data array compression using mantissa bit truncation, linear prediction and zlib compression.</summary>
            MS_truncation__linear_prediction_and_zlib_compression = 1003090,

            /// <summary>Settable parameter for a binary data compression event.</summary>
            MS_binary_data_compression_parameter = 1003091,

            /// <summary>Number of extraneous mantissa bits truncated to improve subsequent compression.</summary>
            MS_number_of_mantissa_bits_truncated = 1003092,

            /// <summary>Metalloendopeptidase found in the mushroom Grifola frondosa that cleaves proteins on the amino side of lysine residues.</summary>
            MS_Lys_N = 1003093,

            /// <summary>Thermo Scientific Orbitrap Exploris 240 Quadrupole Orbitrap MS.</summary>
            MS_Orbitrap_Exploris_240 = 1003094,

            /// <summary>Thermo Scientific Orbitrap Exploris 120 Quadrupole Orbitrap MS.</summary>
            MS_Orbitrap_Exploris_120 = 1003095,

            /// <summary>Thermo Scientific LTQ Orbitrap Velos Pro, often just referred to as the Orbitrap Velos Pro.</summary>
            MS_LTQ_Orbitrap_Velos_Pro = 1003096,

            /// <summary>The probability based MaxQuant protein group score.</summary>
            MS_MaxQuant_protein_group_level_score = 1003097,

            /// <summary>Peptide probability from Andromeda.</summary>
            MS_Andromeda_peptide_PEP = 1003098,

            /// <summary>Peptide probability from MaxQuant-DIA algorithm.</summary>
            MS_MaxQuant_DIA_peptide_PEP = 1003099,

            /// <summary>PSM evidence score from MaxQuant-DIA algorithm.</summary>
            MS_MaxQuant_DIA_score = 1003100,

            /// <summary>PSM evidence PEP probability from MaxQuant-DIA algorithm.</summary>
            MS_MaxQuant_DIA_PEP = 1003101,

            /// <summary>Term for a comment field withing the NIST msp file format</summary>
            MS_NIST_msp_comment = 1003102,

            /// <summary>Annotation format used for annotating individual spectrum ion peaks.</summary>
            MS_ion_annotation_format = 1003103,

            /// <summary>Annotation format designed primarily for peptides, with allowances for generic chemical formulas and other miscellaneous named ions.</summary>
            MS_peptide_ion_annotation_format = 1003104,

            /// <summary>Annotation format designed specifically for cross-linked peptide ion peaks.</summary>
            MS_cross_linked_peptide_ion_annotation_format = 1003105,

            /// <summary>Annotation format designed specifically for glycan ion peaks.</summary>
            MS_glycan_ion_annotation_format = 1003106,

            /// <summary>Annotation format designed specifically for lipid ion peaks.</summary>
            MS_lipid_ion_annotation_format = 1003107,

            /// <summary>PatternLab for Proteomics is an integrated computational environment for analyzing shotgun proteomic data.</summary>
            MS_PatternLab = 1003108,

            /// <summary>Identifying cross-linked peptides in complex protein mixtures</summary>
            MS_SIM_XL = 1003109,

            /// <summary>SIM-XL identification search engine score</summary>
            MS_SIM_XL_score = 1003110,

            /// <summary>Quantification of cross-linked peptides in complex protein mixtures</summary>
            MS_QUIN_XL = 1003111,

            /// <summary>Thermo Scientific Orbitrap ID-X mass spectrometer with Tribrid architecture consisting of quadrupole mass filter, linear ion trap and Orbitrap mass analyzers.</summary>
            MS_Orbitrap_ID_X = 1003112,

            /// <summary>The OpenMS ConsesusID tool posterior error probability</summary>
            MS_OpenMS_ConsensusID_PEP = 1003113,

            /// <summary>The score of the best PSM selected by the underlying identification tool</summary>
            MS_OpenMS_Best_PSM_Score = 1003114,

            /// <summary>The OpenMS Target-decoy q-values at PSM level</summary>
            MS_OpenMS_Target_decoy_PSM_q_value = 1003115,

            /// <summary>The OpenMS Target-decoy q-values at peptide sequence level</summary>
            MS_OpenMS_Target_decoy_peptide_q_value = 1003116,

            /// <summary>The OpenMS Target-decoy q-values at protein level</summary>
            MS_OpenMS_Target_decoy_protein_q_value = 1003117,

            /// <summary>A Method for Efficient High-Confidence Protein Inference. The tool is part of the OpenMS framework</summary>
            MS_EPIFANY = 1003118,

            /// <summary>Protein Posterior probability calculated by EPIFANY protein inference algorithm</summary>
            MS_EPIFANY_Protein_posterior_probability = 1003119,

            /// <summary>The data type LFQ intensity produced by OpenMS.</summary>
            MS_OpenMS_LFQ_intensity = 1003120,

            /// <summary>The data type LFQ spectral count produced by OpenMS.</summary>
            MS_OpenMS_LFQ_spectral_count = 1003121,

            /// <summary>Bruker Daltonics' rapiflex: MALDI TOF/TOF.</summary>
            MS_rapifleX = 1003122,

            /// <summary>Bruker Daltonics timsTOF series</summary>
            MS_Bruker_Daltonics_timsTOF_series = 1003123,

            /// <summary>Bruker Daltonics' timsTOF fleX</summary>
            MS_timsTOF_fleX = 1003124,

            /// <summary>ProSight spectrum-level Q-value.</summary>
            MS_ProSight_spectral_Q_value = 1003125,

            /// <summary>ProSight spectrum-level P-score.</summary>
            MS_ProSight_spectral_P_score = 1003126,

            /// <summary>ProSight spectrum-level E-value.</summary>
            MS_ProSight_spectral_E_value = 1003127,

            /// <summary>ProSight spectrum-level C-score.</summary>
            MS_ProSight_spectral_C_score = 1003128,

            /// <summary>Estimation of the Q-value for proteoforms.</summary>
            MS_proteoform_level_Q_value = 1003129,

            /// <summary>ProSight proteoform-level Q-value.</summary>
            MS_ProSight_proteoform_Q_value = 1003130,

            /// <summary>Isoform level information.</summary>
            MS_isoform_level_identification_attribute = 1003131,

            /// <summary>Identification confidence metric for a isoform.</summary>
            MS_isoform_level_identification_statistic = 1003132,

            /// <summary>Estimation of the Q-value for isoforms.</summary>
            MS_isoform_level_Q_value = 1003133,

            /// <summary>ProSight isoform-level Q-value.</summary>
            MS_ProSight_isoform_Q_value = 1003134,

            /// <summary>ProSight protein-level Q-value.</summary>
            MS_ProSight_protein_Q_value = 1003135,

            /// <summary>Search engine input parameters specific to ProSight.</summary>
            MS_ProSight_input_parameter = 1003136,

            /// <summary>Search engine input parameters specific to TDPortal.</summary>
            MS_TDPortal_input_parameter = 1003137,

            /// <summary>If true, runs delta m mode in ProSight.</summary>
            MS_ProSight_Run_delta_m_mode = 1003138,

            /// <summary>If true, runs Subsequence Search mode in ProSight.</summary>
            MS_ProSight_Run_Subsequence_Search_mode = 1003139,

            /// <summary>If true, runs Annotated Proteoform Search mode in ProSight.</summary>
            MS_ProSight_Run_Annotated_Proteoform_Search_mode = 1003140,

            /// <summary>ProSight: Database search engine for top-down proteomics.</summary>
            MS_ProSight = 1003141,

            /// <summary>TDPortal: Database search engine for top-down proteomics.</summary>
            MS_TDPortal = 1003142,

            /// <summary>A data array of mass values.</summary>
            MS_mass_array = 1003143,

            /// <summary>SCIEX Triple Quad 7500.</summary>
            MS_Triple_Quad_7500 = 1003144,

            /// <summary>Cross-platform software to convert Thermo RAW files to a number of open formats.</summary>
            MS_ThermoRawFileParser = 1003145,

            /// <summary>Python module that helps handling various proteomics data analysis tasks.</summary>
            MS_pyteomics = 1003146,

            /// <summary>Probability that one mass modification has been correctly localized to a specific residue as computed by PTMProphet.</summary>
            MS_PTMProphet_probability = 1003147,

            /// <summary>PSM-specific average of the m best site probabilities over all potential sites where m is the number of modifications of a specific type, as computed by PTMProphet.</summary>
            MS_PTMProphet_mean_best_probability = 1003148,

            /// <summary>PTMProphet-computed PSM-specific normalized (0.0  1.0) measure of information content across all modifications of a specific type.</summary>
            MS_PTMProphet_normalized_information_content = 1003149,

            /// <summary>PTMProphet-computed PSM-specific measure of information content per modification type ranging from 0 to m, where m is the number of modifications of a specific type.</summary>
            MS_PTMProphet_information_content = 1003150,

            /// <summary>SHA-256 (member of Secure Hash Algorithm-2 family) is a cryptographic hash function designed by the National Security Agency (NSA) and published by the NIST as a U. S. government standard. It is also used to verify file integrity.</summary>
            MS_SHA_256 = 1003151,

            /// <summary>Shimadzu Scientific Instruments GCMS-QP2010SE.</summary>
            MS_GCMS_QP2010SE = 1003152,

            /// <summary>Array of raw ion mobility values from a drift time device, reported in seconds (or milliseconds), corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_raw_ion_mobility_drift_time_array = 1003153,

            /// <summary>Array of ion mobility values (K or K0) based on ion separation in gaseous phase due to different ion mobilities under an electric field based on ion size, m/z and shape, as an average property of an analyte post peak-detection, weighted charge state reduction, and/or adduct aggregation, corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_deconvoluted_ion_mobility_array = 1003154,

            /// <summary>Array of ion mobility values based on ion separation in gaseous phase due to different ion mobilities under an electric field based on ion size, m/z and shape, normalized for the local conditions and reported in volt-second per square centimeter, as an average property of an analyte post peak-detection, weighted charge state reduction, and/or adduct aggregation, corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_deconvoluted_inverse_reduced_ion_mobility_array = 1003155,

            /// <summary>Array of mean ion mobility values from a drift time device, reported in seconds (or milliseconds), as an average property of an analyte post peak-detection, weighted charge state reduction, and/or adduct aggregation, corresponding to a spectrum of individual peaks encoded with an m/z array.</summary>
            MS_deconvoluted_ion_mobility_drift_time_array = 1003156,

            /// <summary>Array of m/z values representing the lower bound m/z of the quadrupole position at each point in the spectrum.</summary>
            MS_scanning_quadrupole_position_lower_bound_m_z_array = 1003157,

            /// <summary>Array of m/z values representing the upper bound m/z of the quadrupole position at each point in the spectrum.</summary>
            MS_scanning_quadrupole_position_upper_bound_m_z_array = 1003158,

            /// <summary>Indicates an acquisition mode in which the isolation window is a full range, rather than a subset of the full range.</summary>
            MS_isolation_window_full_range = 1003159,

            /// <summary>Proteomics Standards Initiative mzQC format for quality control data.</summary>
            MS_mzQC_format = 1003160,

            /// <summary>Grouping term for quality control data formats.</summary>
            MS_quality_control_data_format = 1003161,

            /// <summary>Proteomics (PTX) - QualityControl (QC) software for QC report generation and visualization.</summary>
            MS_PTX_QC = 1003162,

            /// <summary>The set of analyte identifiers that compose an interpretation of a spectrum.</summary>
            MS_analyte_mixture_members = 1003163,

            /// <summary>QuaMeter IDFree software for QC metric calculation.</summary>
            MS_QuaMeter_IDFree = 1003164,

            /// <summary>iMonDB software to extract, store, and manage mass spectrometry instrument parameters from raw data files.</summary>
            MS_iMonDB = 1003165,

            /// <summary>Fraction of intensity summed from all peaks that can be attributed to expected fragments of the analyte, divided by the intensity summed from all peaks in the spectrum</summary>
            MS_assigned_intensity_fraction = 1003166,

            /// <summary>The fraction of total intensities in the isolation window in the previous round of MS (i.e. the MSn-1 scan) that can be assigned to this identified analyte</summary>
            MS_MSn_1_isolation_window_precursor_purity = 1003167,

            /// <summary>A free-text string providing additional information of the library spectrum not encoded otherwise, usually for human use and not parsed by software tools.</summary>
            MS_library_spectrum_comment = 1003168,

            /// <summary>Sequence string describing the amino acids and mass modifications of a peptidoform using the PSI ProForma notation</summary>
            MS_proforma_peptidoform_sequence = 1003169,

            /// <summary>A collection of spectra organized by their originating analyte, compiled deliberately for use in MS data analysis in future experiments</summary>
            MS_spectral_library = 1003170,

            /// <summary>An attribute of a spectral library</summary>
            MS_spectral_library_attribute = 1003171,

            /// <summary>An entry in a spectral library representing a spectrum</summary>
            MS_library_spectrum = 1003172,

            /// <summary>An attribute that takes on a numeric value</summary>
            MS_numeric_attribute = 1003173,

            /// <summary>The maximum value for this attribute</summary>
            MS_attribute_maximum = 1003174,

            /// <summary>The minimum value for this attribute</summary>
            MS_attribute_minimum = 1003175,

            /// <summary>The arithmetic mean value for this attribute</summary>
            MS_attribute_mean = 1003176,

            /// <summary>The standard deviation (exact value of population, or estimate from sample) of this attribute</summary>
            MS_attribute_standard_deviation = 1003177,

            /// <summary>The coefficient of variation of this attribute, i.e. standard deviation divided by the mean</summary>
            MS_attribute_coefficient_of_variation = 1003178,

            /// <summary>The most appropriate summary value of the attribute, usually but not necessarily the mean</summary>
            MS_attribute_summary_value = 1003179,

            /// <summary>The median of this attribute</summary>
            MS_attribute_median = 1003180,

            /// <summary>Combination of two or more dissociation methods that are known by a special term.</summary>
            MS_combined_dissociation_method = 1003181,

            /// <summary>Dissociation process combining electron-transfer dissociation (ETD) and collision-induced dissociation (CID).</summary>
            MS_electron_transfer_collision_induced_dissociation = 1003182,

            /// <summary>Waters oa-ToF based Synapt XS.</summary>
            MS_Synapt_XS = 1003183,

            /// <summary>Waters oa-ToF based SELECT SERIES Cyclic IMS.</summary>
            MS_SELECT_SERIES_Cyclic_IMS = 1003184,

            /// <summary>Waters oa-ToF based SELECT SERIES MRT.</summary>
            MS_SELECT_SERIES_MRT = 1003185,

            /// <summary>Version number of the</summary>
            MS_library_format_version = 1003186,

            /// <summary>Short identifier for the library for easy reference, preferably but not necessarily globally unique</summary>
            MS_library_identifier = 1003187,

            /// <summary>A short name identifying the library to potential users. The same name may refer to multiple versions of the same continually updated library.</summary>
            MS_library_name = 1003188,

            /// <summary>Extended free-text description of the library</summary>
            MS_library_description = 1003189,

            /// <summary>Version number of the library, usually refering to a certain release of a continually updated library</summary>
            MS_library_version = 1003190,

            /// <summary>URI or URL that uniquely identifies the library</summary>
            MS_library_URI = 1003191,

            /// <summary>A spectrum deliberately introduced into a spectral library that necessarily produces incorrect identifications when matched, for the purpose of error control in spectral library searching</summary>
            MS_decoy_spectrum = 1003192,

            /// <summary>A decoy spectrum generated from a real spectrum, by shuffling the amino acid sequence of the identification of the real spectrum, followed by re-positioning annotated peaks to match the shuffled sequence.</summary>
            MS_shuffle_and_reposition_decoy_spectrum = 1003193,

            /// <summary>A decoy spectrum generated from a real spectrum, by changing the precursor m/z value of the real spectrum.</summary>
            MS_precursor_shift_decoy_spectrum = 1003194,

            /// <summary>A decoy spectrum that is either a real spectrum of an unnatural peptidoform (e.g. a synthetic peptide that cannot be found in nature), or an artificial spectrum predicted for such unnatural peptidoform</summary>
            MS_unnatural_peptidoform_decoy_spectrum = 1003195,

            /// <summary>A decoy spectrum that is a real spectrum of a naturally occuring peptidoform of an unrelated species that should not be found in the sample</summary>
            MS_unrelated_species_decoy_spectrum = 1003196,

            /// <summary>URI of the license controlling use of the library (e.g. https://creativecommons.org/publicdomain/zero/1.0/)</summary>
            MS_license_URI = 1003197,

            /// <summary>Notice of statutorily prescribed form that informs users of the underlying claim to copyright ownership in a published work</summary>
            MS_copyright_notice = 1003198,

            /// <summary>Extended free-text description of the difference from the previous version</summary>
            MS_change_log = 1003199,

            /// <summary>Version number of the software package used for library creation</summary>
            MS_software_version = 1003200,

            /// <summary>Abstract term containing several different types of provenance information</summary>
            MS_library_provenance_attribute = 1003201,

            /// <summary>A suite of software tools for creating and searching MS/MS peptide spectrum libraries, developed at the University of Washington</summary>
            MS_BiblioSpec = 1003202,

            /// <summary>Spectrum data file from which (at least) a subset of spectra were extracted from. Should use USI notation mzspec:PXDxxxx:msRunName if possible, or a URI if USI notation is not possible.</summary>
            MS_constituent_spectrum_file = 1003203,

            /// <summary>Identification file where (at least) a subset of identifications were extracted from. Should use a URI if possible</summary>
            MS_constituent_identification_file = 1003204,

            /// <summary>Source library URI which(at least) a subset of spectra were extracted from.</summary>
            MS_constituent_library_file = 1003205,

            /// <summary>String of logging information generated when the library was constructed from its constituent files. Multiple lines should be separated with escaped \n</summary>
            MS_library_creation_log = 1003206,

            /// <summary>Library creation software</summary>
            MS_library_creation_software = 1003207,

            /// <summary>The measured or inferred m/z (as reported by the mass spectrometer acquisition software or post-processing software) of the monoisotopic peak of the precursor ion based on the MSn-1 spectrum.</summary>
            MS_experimental_precursor_monoisotopic_m_z = 1003208,

            /// <summary>The measured monoisotopic m/z (as reported by the mass spectrometer acquisition software or post-processing software) minus the theoretical monoisotopic m/z of the analyte assigned to the spectrum.</summary>
            MS_monoisotopic_m_z_deviation = 1003209,

            /// <summary>The measured average m/z (as reported by the mass spectrometer acquisition software or post-processing software) minus the theoretical average m/z of the analyte assigned to the spectrum.</summary>
            MS_average_m_z_deviation = 1003210,

            /// <summary>A set of spectrum-related attributes that is shared by a subset of spectra within the same spectral library</summary>
            MS_library_spectrum_attribute_set = 1003211,

            /// <summary>A name to refer to a library attribute set</summary>
            MS_library_attribute_set_name = 1003212,

            /// <summary>Specific aspect of a mass spectrometer method by which mass ranges are selected and possibly dissociated.</summary>
            MS_mass_spectrometry_acquisition_method = 1003213,

            /// <summary>Specific aspect of a mass spectrometer method by which mass ranges are selected and possibly dissociated.</summary>
            MS_mass_spectrometry_acquisition_method_aspect = 1003214,

            /// <summary>Mass spectrometer data acquisition method wherein mass selection for fragmentation is configured according to a pre-determined program, rather than based on any detected precursor ions.</summary>
            MS_data_independent_acquisition = 1003215,

            /// <summary>Mass spectrometer data acquisition method wherein all precursor ions of which the instrument is capable are fragmented at once..</summary>
            MS_dissociation_of_full_mass_range = 1003216,

            /// <summary>Mass spectrometer data acquisition method wherein ????.</summary>
            MS_dissociation_of_scanning_quadrupole_across_a_specified_mass_range = 1003217,

            /// <summary>Mass spectrometer data acquisition method wherein a series of limited mass range fragmentation selection windows are preconfigured.</summary>
            MS_dissociation_of_sequential_mass_ranges = 1003218,

            /// <summary>Mass spectrometer data acquisition method wherein precursor ions are separated by their ion mobility properties prior to measurement.</summary>
            MS_ion_mobility_separation = 1003219,

            /// <summary>Data processing action of merging of the measurements of potentially multiple adducts into a single representation that is independent of the small ion that adds charge to a larger molecule.</summary>
            MS_adduct_deconvolution = 1003220,

            /// <summary>Mass spectrometer data acquisition method wherein MSn spectra are triggered based on the m/z of precursor ions detected in the same run.</summary>
            MS_data_dependent_acquisition = 1003221,

            /// <summary>Data processing action of merging multiple ion peaks acquired at different ion mobility steps into a single mass spectrum representing a single analyte.</summary>
            MS_ion_mobility_deconvolution = 1003222,

            /// <summary>Data independent mass spectrometer acquisition method wherein a preconfigured sequence of mass ranges are fragmented. Examples of such an approach include SWATH-MS, FT-ARM, HRM, and PAcIFIC.</summary>
            MS_data_independent_acquisition_from_dissociation_of_sequential_mass_ranges = 1003224,

            /// <summary>Data independent mass spectrometer acquisition method wherein a preconfigured sequence of mass ranges are fragmented after being separated by ion mobility. An example of such an approach is Bruker diaPASEF.</summary>
            MS_data_independent_acquisition_from_dissociation_of_sequential_mass_ranges_after_ion_mobility_separation = 1003225,

            /// <summary>Data independent mass spectrometer acquisition method wherein the full mass range is fragmented after being separated by ion mobility. Examples of such an approach include HDMS^E and IMS-AIF.</summary>
            MS_data_independent_acquisition_from_dissociation_of_full_mass_range_after_ion_mobility_separation = 1003226,

            /// <summary>Data independent mass spectrometer acquisition method wherein the full mass range is fragmented. Examples of such an approach include MS^E, AIF, and bbCID.</summary>
            MS_data_independent_acquisition_from_dissociation_of_full_mass_range = 1003227,

            /// <summary>Data independent mass spectrometer acquisition method wherein ???. An example of such an approach is Waters SONAR.</summary>
            MS_data_independent_acquisition_from_dissociation_of_scanning_quadrupole_across_mass_range = 1003228,

            /// <summary>Bruker Daltonics' timsTOF.</summary>
            MS_timsTOF = 1003229,

            /// <summary>Bruker Daltonics' timsTOF Pro 2.</summary>
            MS_timsTOF_Pro_2 = 1003230,

            /// <summary>Bruker Daltonics' timsTOF SCP.</summary>
            MS_timsTOF_SCP = 1003231,

            /// <summary>comprehensive infrastructure to organize, curate and share a multi- instrument spectral library for metabolomics data annotation developed and distributed by the French National infrastructure in metabolomics and fluxomics (MetaboHUB).</summary>
            MS_PeakForest = 1003232,

            /// <summary>An attribute that describes information about an entry in a spectral library</summary>
            MS_library_spectrum_attribute = 1003234,

            /// <summary>The ionization of analytes from a piece of paper by applying a solvent and voltage.</summary>
            MS_paper_spray_ionization = 1003235,

            /// <summary>An ordinal number uniquely identifying a spectrum in a library. Library spectrum keys should start at 1. Library spectrum keys SHOULD not change if entries are re-ordered or removed from a library.</summary>
            MS_library_spectrum_key = 1003237,

            /// <summary>A set of analyte-related attributes that is shared by a subset of spectra within the same spectral library</summary>
            MS_library_analyte_attribute_set = 1003238,

            /// <summary>A set of interpretation-related attributes that is shared by a subset of spectra within the same spectral library</summary>
            MS_library_interpretation_attribute_set = 1003239,

            /// <summary>A mathematical transformation applied to peak intensities, for example, as a way to modify the weight put on each peak when computing spectral match scores</summary>
            MS_peak_intensity_transform = 1003240,

            /// <summary>A mathematical transformation applied to peak intensities, in which peak intensities are replaced by their square roots</summary>
            MS_square_root_transform = 1003241,

            /// <summary>A mathematical transformation applied to peak intensities, in which peak intensities are replaced by their ranks</summary>
            MS_rank_transform = 1003242,

            /// <summary>The theoretical mass of the adduct ion (e.g. for a singly-charged protonated peptide ion, this value would be the neutral peptide molecules mass plus the mass of a proton)</summary>
            MS_adduct_ion_mass = 1003243,

            /// <summary>Accession number (e.g. in PeptideAtlas) of the peptide sequence</summary>
            MS_peptide_accession_number = 1003244,

            /// <summary>Thermo Scientific Q Exactive UHMR (Ultra High Mass Range) Hybrid Quadrupole Orbitrap MS.</summary>
            MS_Q_Exactive_UHMR = 1003245,

            /// <summary>Multiphoton ionization where the reactant ion dissociates as a result of the absorption of multiple UV photons.</summary>
            MS_ultraviolet_photodissociation = 1003246,

            /// <summary>A process to fragment ions in a mass spectrometer by inducing fragmentation of anions (e.g. peptides or proteins) by transferring electrons to a radical-cation.</summary>
            MS_negative_electron_transfer_dissociation = 1003247,

            /// <summary>Process to transfer a proton from a hydronium ion (H3O+) to neutral analyte, leading to a protonated analyte, which typically does not lead to fragmentation.</summary>
            MS_proton_transfer_reaction = 1003248,

            /// <summary>Process to transfer one or more protons from a multiply charged cation (peptide or protein ion) to a proton acceptor anion or neutral basic compound, thereby reducing the charge of the original analyte.</summary>
            MS_proton_transfer_charge_reduction = 1003249,

            /// <summary>The number of peptidoforms that pass the threshold to be considered identified with sufficient confidence.</summary>
            MS_count_of_identified_peptidoforms = 1003250,

            /// <summary>The number of spectra that pass the threshold to be considered identified with sufficient confidence.</summary>
            MS_count_of_identified_spectra = 1003251,

            /// <summary>Waters Corporation Xevo G2-XS QTof quadrupole time-of-flight mass spectrometer.</summary>
            MS_Xevo_G2_XS_QTof = 1003252,

            /// <summary>A universal software for data-independent acquisition (DIA) proteomics data processing</summary>
            MS_DIA_NN = 1003253,

            /// <summary>An attribute of a peak in a mass spectrum other than its m/z, intensity, and annotation.</summary>
            MS_peak_attribute = 1003254,

            /// <summary>For a de-charged spectrum, the original charge state of the ion observed in a mass spectrum determined by charge deconvolution.</summary>
            MS_pre_decharged_charge_state = 1003255,

            /// <summary>Non-inherent characteristic attributed to a peptidoform</summary>
            MS_peptidoform_ion_attribute = 1003256,

            /// <summary>A link from one spectrum to another spectrum of interest</summary>
            MS_library_spectrum_cross_reference = 1003257,

            /// <summary>A cross reference to another spectrum that has some relationship with this one, but is not necessarily spectrally similar.</summary>
            MS_related_spectrum = 1003258,

            /// <summary>A list of cross references to a related spectrum in the same library, in the form of the library spectrum key.</summary>
            MS_related_spectrum_keys = 1003259,

            /// <summary>A cross reference to a related spectrum in the form of a PSI Universal Spectrum Identifier</summary>
            MS_related_spectrum_USI = 1003260,

            /// <summary>A free-text string describing the related spectrum and/or its relationship to this spectrum</summary>
            MS_related_spectrum_description = 1003261,

            /// <summary>A cross reference to another spectrum that has high spectral similarity with this one, usually assumed to originate from the same analyte ion.</summary>
            MS_similar_spectrum = 1003262,

            /// <summary>A list of cross references to a similar spectrum in the same library, in the form of the library spectrum key.</summary>
            MS_similar_spectrum_keys = 1003263,

            /// <summary>A cross reference to a similar spectrum in the form of a PSI Universal Spectrum Identifier</summary>
            MS_similar_spectrum_USI = 1003264,

            /// <summary>A group of spectra that are spectrally similar to each other</summary>
            MS_spectrum_cluster = 1003265,

            /// <summary>A non-inherent characteristic of a spectrum cluster</summary>
            MS_spectrum_cluster_attribute = 1003266,

            /// <summary>An ordinal number uniquely identifying a spectrum cluster. It should start with 1.</summary>
            MS_spectrum_cluster_key = 1003267,

            /// <summary>A list of integers corresponding to the library spectrum keys of the members of this cluster. These members must be in the same library.</summary>
            MS_spectrum_cluster_member_spectrum_keys = 1003268,

            /// <summary>A member of this cluster external to the library, specified using a PSI Universal Spectrum Identifier.</summary>
            MS_spectrum_cluster_member_USI = 1003269,

            /// <summary>A string describing the peptidoform ion using the PSI ProForma notation, which should include the charge state, and optionally the adduct type.</summary>
            MS_proforma_peptidoform_ion_notation = 1003270,

            /// <summary>The molecular identity(-ies) of the ion(s) producing this peak, inferred manually or computationally based on its m/z and the molecular interpretation of the spectrum.</summary>
            MS_peak_annotation = 1003271,

            /// <summary>A string representing the peak annotation, in a defined format specified by the attribute 'ion annotation format'.</summary>
            MS_peak_annotation_string = 1003272,

            /// <summary>A confidence value of assigning a peak annotation to a peak, as defined by the attribute 'peak annotation confidence metric'.</summary>
            MS_peak_annotation_confidence = 1003273,

            /// <summary>A confidence metric of assigning a peak annotation to a peak. By default, this should range from 0 (no confidence) to 1 (certain), and if there are multiple annotations of the same peak, the sum of their confidence levels should be no more than 1.</summary>
            MS_peak_annotation_confidence_metric = 1003274,

            /// <summary>A user-provided name for a user-defined value describing a trait not covered by an existing controlled vocabulary term. This term should be used sparingly, preferring existing terms that describe the specific concept. Should be used with MS:1003276 to provide the attribute's value</summary>
            MS_other_attribute_name = 1003275,

            /// <summary>A user-provided value for a user-defined name describing a trait not covered by an existing controlled vocabulary term. This term should be used sparingly, preferring existing terms that describe the specific concept. Should be used with MS:1003275 to provide the attribute's name</summary>
            MS_other_attribute_value = 1003276,

            /// <summary>Value range for signed normalized score values.</summary>
            MS_value_between__1_and_1_inclusive = 1003277,

            /// <summary>A measure of the statistical variability of the m/z value of this peak, usually estimated from replicate spectra of the same analyte.</summary>
            MS_m_z_variability_of_peak = 1003278,

            /// <summary>The frequency at which this peak is observed among replicate spectra of the same analyte.</summary>
            MS_observation_frequency_of_peak = 1003279,

            /// <summary>A measure of the statistical variability of the intensity of this peak, usually estimated from replicate spectra of the same analyte.</summary>
            MS_intensity_variability_of_peak = 1003280,

            /// <summary>Casanovo is a deep learning-based de novo spectrum identification tool. Official website https://github.com/Noble-Lab/casanovo/.</summary>
            MS_Casanovo = 1003281,

            /// <summary>Bruker TSF raw file format.</summary>
            MS_Bruker_TSF_format = 1003282,

            /// <summary>Native format defined by frame=xsd:nonNegativeInteger.</summary>
            MS_Bruker_TSF_nativeID_format = 1003283,

            /// <summary>Bruker TSF comma separated list of spectra that have been combined prior to searching or interpretation.</summary>
            MS_Bruker_TSF_nativeID_format__combined_spectra = 1003284,

            /// <summary>The standard deviation of the m/z values of this peak among the replicate spectra of the same analyte.</summary>
            MS_standard_deviation_of_m_z_values_of_peak_among_replicates = 1003285,

            /// <summary>The coefficient of variation (standard deviation divided by mean) of the intensities of this peak among the replicate spectra of the same analyte.</summary>
            MS_coefficient_of_variation_of_intensity_of_peak_among_replicates = 1003286,

            /// <summary>The standard deviation of the intensities of this peak among the replicate spectra of the same analyte.</summary>
            MS_standard_deviation_of_intensity_of_peak_among_replicates = 1003287,

            /// <summary>The number of unassigned peaks in the spectrum.</summary>
            MS_number_of_unassigned_peaks = 1003288,

            /// <summary>The intensity of the highest unassigned peak in the spectrum.</summary>
            MS_intensity_of_highest_unassigned_peak = 1003289,

            /// <summary>The number of unassigned peaks among the most intense 20 peaks in the spectrum.</summary>
            MS_number_of_unassigned_peaks_among_top_20_peaks = 1003290,

            /// <summary>Luciphor phosphosite localization score</summary>
            MS_Luciphor_deltaScore = 1003291,

            /// <summary>Thermo Scientific TSQ Altis Plus Triple Quadrupole MS.</summary>
            MS_TSQ_Altis_Plus = 1003292,

            /// <summary>SCIEX ZenoTOF 7600.</summary>
            MS_ZenoTOF_7600 = 1003293,

            /// <summary>A process to fragment ions in a high intensity electron beam which results in a dissociation of various analytes ranging from singly charged small molecules to multiply protonated proteins.</summary>
            MS_electron_activated_dissociation = 1003294,

            /// <summary>Summary statistics of an attribute among all replicates that are aggregated to generate this spectrum.</summary>
            MS_summary_statistics_of_replicates = 1003295,

            /// <summary>The number of replicate spectra used during the aggregation process that originate from a specified source (e.g. a file, a dataset, a sample type, etc.). This term must appear in a group alongside a source term, which is one of: 'ProteomeXchange accession number', 'constituent spectrum file', or 'sample name'</summary>
            MS_number_of_replicates_spectra_used_from_source = 1003296,

            /// <summary>A cross reference to another spectrum that is a replicate spectrum of the same analyte and contributes to the generation of this aggregated spectrum.</summary>
            MS_contributing_replicate_spectrum = 1003297,

            /// <summary>A list of cross references to contributing replicate spectra in the same library, in the form of library spectrum keys.</summary>
            MS_contributing_replicate_spectrum_keys = 1003298,

            /// <summary>A list of cross references to contributing replicate spectra, in the form of PSI Universal Spectrum Identifiers.</summary>
            MS_contributing_replicate_spectrum_USI = 1003299,

            /// <summary>Result of a comparison of a spectrum to another spectrum, usually to assess the plausibility that the two spectra originate from the same analyte.</summary>
            MS_spectrum_match = 1003300,

            /// <summary>Result of a comparison of an observed fragment ion spectrum to the theoretically predicted fragmentation pattern of a peptide sequence, to assess the plausibility that the observed spectrum originates from the putative peptide sequence.</summary>
            MS_peptide_spectrum_match = 1003301,

            /// <summary>Result of a comparison of an observed fragment ion spectrum to another observed fragment ion spectrum, to assess the plausibility that two spectra originate from the same analyte, e.g., in spectral library searching and spectrum clustering.</summary>
            MS_spectrum_spectrum_match = 1003302,

            /// <summary>A measure of how similar two spectra are, based on the features of the spectra (the locations and intensities of peaks) alone.</summary>
            MS_spectral_similarity = 1003303,

            /// <summary>The dot (inner) product of two vectorized spectra divided by the product of their vector norms. It ranges from 0 (orthogonal vectors) to 1 (collinear vectors). Also known as cosine similarity.</summary>
            MS_spectral_dot_product = 1003304,

            /// <summary>The Euclidean distance between a pair of points representing two vectorized spectra in high-dimensional space.</summary>
            MS_spectral_Euclidean_distance = 1003305,

            /// <summary>The number of peaks with closely matching m/z values that are common to two spectra.</summary>
            MS_shared_peak_count = 1003306,

            /// <summary>The angle subtended by two vectorized spectra in high-dimensional space. It is equal to the 1 - (2 * inverse cosine of the spectral dot product) / pi.</summary>
            MS_normalized_spectral_angle = 1003307,

            /// <summary>The Pearson's rho statistic between a pair of points representing two vectorized spectra.</summary>
            MS_spectral_Pearson_correlation = 1003308,

            /// <summary>The Goslin implementations parse common lipid name dialects and normalize them to the recent lipid shorthand nomenclature based on grammars on succinct lipid nomenclature.</summary>
            MS_Goslin = 1003309,

            /// <summary>Identification confidence levels based on the updated lipid shorthand nomenclature. The value slot can have the values 'Category', 'Class', 'Species', 'Phosphate-position', 'Molecular species', 'sn-position', 'DBE position', 'Structure defined', 'Full structure', 'Complete structure'.</summary>
            MS_Lipid_shorthand_identification_confidence_level = 1003310,

            /// <summary>The lipid empirical score is a point-based lipid class-specific scoring system based on MS and other evidence sources as defined by the Lipidomics Standards Initiative (LSI). The score has a numeric value between 0 and 100. Score values are lipid class-specific and are therefore not immediately comparable between lipid classes.</summary>
            MS_Lipid_empirical_score = 1003311,

            /// <summary>Lipid shorthand identification confidence level 'Category'.</summary>
            MS_Lipid_shorthand_identification_confidence___Category = 1003312,

            /// <summary>Lipid shorthand identification confidence level 'Class'.</summary>
            MS_Lipid_shorthand_identification_confidence___Class = 1003313,

            /// <summary>Lipid shorthand identification confidence level 'Species'.</summary>
            MS_Lipid_shorthand_identification_confidence___Species = 1003314,

            /// <summary>Lipid shorthand identification confidence level 'Phosphate-position'.</summary>
            MS_Lipid_shorthand_identification_confidence___Phosphate_position = 1003315,

            /// <summary>Lipid shorthand identification confidence level 'Molecular species'.</summary>
            MS_Lipid_shorthand_identification_confidence___Molecular_species = 1003316,

            /// <summary>Lipid shorthand identification confidence level 'sn-position'.</summary>
            MS_Lipid_shorthand_identification_confidence___sn_position = 1003317,

            /// <summary>Lipid shorthand identification confidence level 'DBE pos'.</summary>
            MS_Lipid_shorthand_identification_confidence___DBE_pos = 1003318,

            /// <summary>Lipid shorthand identification confidence level 'Structure defined'.</summary>
            MS_Lipid_shorthand_identification_confidence___Structure_defined = 1003319,

            /// <summary>The number of spectra in a spectrum cluster.</summary>
            MS_spectrum_cluster_size = 1003320,

            /// <summary>Summary statistics of an attribute among all spectra of a spectrum cluster.</summary>
            MS_summary_statistics_of_clustered_spectra = 1003321,

            /// <summary>Cross reference to the spectrum that is considered the most representative among spectra in a cluster, either as a library spectrum key if the best representative is in the same library, or a universal spectrum identifier if it is not.</summary>
            MS_spectrum_cluster_best_representative = 1003322,

            /// <summary>Cross reference to a consensus spectrum that is constructed from aggregating spectra in a cluster, either as a library spectrum key if the consensus spectrum is in the same library, or a universal spectrum identifier if it is not.</summary>
            MS_spectrum_cluster_consensus_spectrum = 1003323,

            /// <summary>Spectral similarity measured by the spectral dot product between a replicate to its corresponding aggregated (e.g. consensus) spectrum.</summary>
            MS_spectral_dot_product_to_aggregated_spectrum = 1003324,

            /// <summary>Lipid shorthand identification confidence level 'Full structure'.</summary>
            MS_Lipid_shorthand_identification_confidence___Full_structure = 1003325,

            /// <summary>Lipid shorthand identification confidence level 'Complete structure'.</summary>
            MS_Lipid_shorthand_identification_confidence___Complete_structure = 1003326,

            /// <summary>PSI Quality Control controlled vocabulary term.</summary>
            MS_PSI_MS_CV_Quality_Control_Vocabulary = 4000000,

            /// <summary>Parent term for QC metrics, each metric MUST have this as an ancestor in its is_a relations.</summary>
            MS_QC_metric = 4000001,

            /// <summary>The QC metric type describes what type the corresponding metric is. Possible types are single value, n-tuple, table, or matrix.</summary>
            MS_QC_metric_value_type = 4000002,

            /// <summary>Metrics consisting of a single value. The value must have a unit (e.g. UO:0000221 ! dalton or UO:0000187 ! percent), and optionally a type (e.g. STATO:0000574 ! median or MS:1002354 ! PSM-level q-value).</summary>
            MS_single_value = 4000003,

            /// <summary>Metrics consisting of multiple values, with the number of values implicitly specified by length of the JSON array (e.g. length 4 for quartiles). All values must be given a unit (e.g. UO:0000221 ! dalton or UO:0000187 ! percent), and optionally a type (e.g. STATO:0000574 ! median or MS:1002354 ! PSM-level q-value). All values in the tuple must have the same unit and type (if applicable).</summary>
            MS_n_tuple = 4000004,

            /// <summary>Metrics consisting of a table or data frame. The values of the table may have different types in each column (in contrast to a matrix). Each column must have a unit (identical for all entries of this column) and may have a value type (e.g. STATO:0000574 ! median or MS:1002354 ! PSM-level q-value).</summary>
            MS_table = 4000005,

            /// <summary>A matrix is a rectangular array of values of the same type (in contrast to a table). All values must be given a unit (e.g. UO:0000221 ! dalton or UO:0000187 ! percent), and optionally a type (e.g. STATO:0000574 ! median or MS:1002354 ! PSM-level q-value). All values in the matrix must have the same unit and type (if applicable).</summary>
            MS_matrix = 4000006,

            /// <summary>Categorization of the QC metric.</summary>
            MS_QC_metric_category = 4000007,

            /// <summary>QC metric based on identification results.</summary>
            MS_ID_based_metric = 4000008,

            /// <summary>QC metric not based on identification results.</summary>
            MS_ID_free_metric = 4000009,

            /// <summary>QC metric based on quantification results.</summary>
            MS_quantification_based_metric = 4000010,

            /// <summary>QC metric calculated from a single run (e.g. one .raw file).</summary>
            MS_single_run_based_metric = 4000012,

            /// <summary>QC metric calculated from multiple runs (e.g. multiple .raw files).</summary>
            MS_multiple_runs_based_metric = 4000013,

            /// <summary>QC metric calculated from a single spectrum.</summary>
            MS_single_spectrum_based_metric = 4000014,

            /// <summary>QC metric calculated from multiple spectra.</summary>
            MS_multiple_spectra_based_metric = 4000015,

            /// <summary>QC metric related to retention time.</summary>
            MS_retention_time_metric = 4000016,

            /// <summary>QC metric related to a chromatogram.</summary>
            MS_chromatogram_metric = 4000017,

            /// <summary>QC metric related to an extracted ion chromatogram.</summary>
            MS_XIC_metric = 4000018,

            /// <summary>QC metric related to the mass spectrometry acquisition.</summary>
            MS_MS_metric = 4000019,

            /// <summary>QC metric related to events in the ion source.</summary>
            MS_ion_source_metric = 4000020,

            /// <summary>QC metric based on MS1 events.</summary>
            MS_MS1_metric = 4000021,

            /// <summary>QC metric based on MS2 events.</summary>
            MS_MS2_metric = 4000022,

            /// <summary>QC metric related to the sample preparation.</summary>
            MS_sample_preparation_metric = 4000023,

            /// <summary>QC metric related to measurements of the ambient environment, such as the laboratory.</summary>
            MS_environment_metric = 4000024,

            /// <summary>The number of XIC that account for the top half of all XIC-FWHM divided by the number of all XIC.</summary>
            MS_XIC50_fraction = 4000050,

            /// <summary>The first to n-th quantile of peak widths for XICs. A metric's value triplet represents the related QuaMeter metrics of of XIC-FWHM-Q1,Q2,Q3.</summary>
            MS_XIC_FWHM_quantiles = 4000051,

            /// <summary>The log ratio of successive XIC height quartiles. The metric's value triplet represents the log ratios of XIC-height-Q2 to XIC-height-Q1, XIC-height-Q3 to XIC-height-Q2, XIC-height max to XIC-height-Q3.</summary>
            MS_XIC_Height_quartile_ratios = 4000052,

            /// <summary>The retention time duration of the chromatography in seconds.</summary>
            MS_chromatography_duration = 4000053,

            /// <summary>The interval when the respective quarter of the TIC accumulates divided by retention time duration.</summary>
            MS_TIC_quarters_RT_fraction = 4000054,

            /// <summary>The interval used for acquisition of the first, second, third, and fourth quarter of all MS1 events divided by retention time duration.</summary>
            MS_MS1_quarter_RT_fraction = 4000055,

            /// <summary>The interval used for acquisition of the first, second, third, and fourth quarter of all MS2 events divided by retention time duration.</summary>
            MS_MS2_quarter_RT_fraction = 4000056,

            /// <summary>The log ratios of successive TIC-change quartiles. The TIC changes are the list of MS1 total ion current (TIC) value changes from one to the next scan, produced when each MS1 TIC is subtracted from the preceding MS1 TIC. The metric's value triplet represents the log ratio of the TIC-change Q2 to Q1, Q3 to Q2, TIC-change-max to Q3</summary>
            MS_MS1_TIC_change_quartile_ratios = 4000057,

            /// <summary>The log ratios of successive TIC quartiles. The metric's value triplet represents the log ratios of TIC-Q2 to TIC-Q1, TIC-Q3 to TIC-Q2, TIC-max to TIC-Q3.</summary>
            MS_MS1_TIC_quartile_ratios = 4000058,

            /// <summary>The number of MS1 events in the run.</summary>
            MS_number_of_MS1_spectra = 4000059,

            /// <summary>The number of MS2 events in the run.</summary>
            MS_number_of_MS2_spectra = 4000060,

            /// <summary>The first to n-th quantile of MS1 peak density (scan peak counts). A value triplet represents the original QuaMeter metrics, the quartiles of MS1 density. The number of values in the tuple implies the quantile mode.</summary>
            MS_MS1_density_quantiles = 4000061,

            /// <summary>The first to n-th quantile of MS2 peak density (scan peak counts). A value triplet represents the original QuaMeter metrics, the quartiles of MS2 density. The number of values in the tuple implies the quantile mode.</summary>
            MS_MS2_density_quantiles = 4000062,

            /// <summary>The fraction of MS/MS precursors of the corresponding charge. The fractions</summary>
            MS_MS2_known_precursor_charges_fractions = 4000063,

            /// <summary>The fractions of inferred charge state of MS/MS precursors. The fractions</summary>
            MS_MS2_unknown_and_likely_precursor_charges_fractions = 4000064,

            /// <summary>Fastest frequency for MS level 1 collection</summary>
            MS_fastest_frequency_for_MS_level_1_collection = 4000065,

            /// <summary>Fastest frequency for MS level 2 collection</summary>
            MS_fastest_frequency_for_MS_level_2_collection = 4000066,

            /// <summary>The duration of the mass spectrometry acquisition (as measured by the time between the last scan and first scan) in seconds.</summary>
            MS_MS_run_duration = 4000067,

            /// <summary>The minimal proportion of peaks needed to account for at least 50% of the total ion current in each individual spectrum considered, recorded in a mandatory fraction column. Either USI or native spectrum identifier columns must be present as well.</summary>
            MS_spectra_half_TIC = 4000068,

            /// <summary>Upper and lower limit of m/z precursor values at which MSn spectra are recorded.</summary>
            MS_m_z_acquisition_range = 4000069,

            /// <summary>Upper and lower limit of retention time at which spectra are recorded.</summary>
            MS_retention_time_acquisition_range = 4000070,

            /// <summary>The number of chromatograms recorded for the run.</summary>
            MS_number_of_chromatograms = 4000071,

            /// <summary>Observed mass accuracy in ppm, calculated by 1E6 x (observed m/z - theoretical m/z)/theoretical m/z of a selected and identified ion in a mass spectrum.</summary>
            MS_observed_mass_accuracy = 4000072,

            /// <summary>A QC metric based on a QC sample of known content.</summary>
            MS_QC_sample_metric = 4000073,

            /// <summary>A QC metric based on a QC sample of known and high complexity content.</summary>
            MS_high_complexity_QC_sample_metric = 4000074,

            /// <summary>A QC metric based on a QC sample of known and low complexity content.</summary>
            MS_low_complexity_QC_sample_metric = 4000075,

            /// <summary>A QC metric based on the results of a QC2 sample measurement. A QC2 sample is made from Pierce HeLa protein digest standard, see Pichler et al. Chiva et al. for details on QC sample design.</summary>
            MS_QC2_sample_metric = 4000076,

            /// <summary>A QC metric based on the results of a QC1 sample measurement. A QC1 sample is made from trypsin-digested BSA MS Standard (CAM modified), see Pichler et al. Chiva et al. for details on QC sample design).</summary>
            MS_QC1_sample_metric = 4000077,

            /// <summary>Observed mass accuracy for the peptides of a QC2 sample measurement. The table should contain the peptides as described in the QC2 sample metric term, missing are interpreted as not detected.</summary>
            MS_QC2_sample_mass_accuracies = 4000078,

            /// <summary>Observed intensities for the peptides of a QC2 sample measurement within 5 ppm and +/- 240 s RT tolerance. Different metrics of observed intensities are possible, at least one must be present. The table should contain the peptides as defined in the parent QC2 sample metric term, missing are interpreted as not detected.</summary>
            MS_QC2_sample_intensities = 4000079,

            /// <summary>QC terms associated but not directly metrics themselves.</summary>
            MS_QC_non_metric_term = 4000080,

            /// <summary>Data from the first principal component of a PCA.</summary>
            MS_first_principal_component = 4000081,

            /// <summary>Data from the second principal component of a PCA.</summary>
            MS_second_principal_component = 4000082,

            /// <summary>Data from the third principal component of a PCA.</summary>
            MS_third_principal_component = 4000083,

            /// <summary>Data from the fourth principal component of a PCA.</summary>
            MS_fourth_principal_component = 4000084,

            /// <summary>Data from the fifth principal component of a PCA.</summary>
            MS_fifth_principal_component = 4000085,

            /// <summary>Used to refer to data elements of input sections in mzQC, either inputFile names or metadata labels.</summary>
            MS_mzQC_input_reference = 4000086,

            /// <summary>Used to supply alternative labels for plotting figures.</summary>
            MS_mzQC_plot_label = 4000087,

            /// <summary>Used to supply batch label information with any string value.</summary>
            MS_batch_label = 4000088,

            /// <summary>Used to supply injection sequence information with consecutive whole numbers.</summary>
            MS_injection_sequence_label = 4000089,

            /// <summary>A table with the PCA results of MaxQuant's protein group raw intensities.</summary>
            MS_principal_component_analysis_of_MaxQuant_s_protein_group_raw_intensities = 4000090,

            /// <summary>A table with the PCA results of MaxQuant's protein group lfq intensities.</summary>
            MS_principal_component_analysis_of_MaxQuant_s_protein_group_lfq_intensities = 4000091,

            /// <summary>A table with the PCA results of identified MS1 feature areas.</summary>
            MS_identified_MS1_feature_area_principal_component_analysis_result = 4000092,

            /// <summary>A table with the PCA results of unidentified but multiple-run-matched MS1 feature areas.</summary>
            MS_unidentified_MS1_feature_area_principal_component_analysis_result = 4000093,

            /// <summary>A table with the PCA results of identified MS1 feature areas after batch-correction.</summary>
            MS_batch_corrected_identified_MS1_feature_area_principal_component_analysis_result = 4000094,

            /// <summary>The slowest acquisition speed with which precursor MS scans were collected. Scan acquisition frequency can be used to gauge the suitability of used instrument settings for the sample content used.</summary>
            MS_slowest_frequency_for_MS_level_1_collection = 4000095,

            /// <summary>The slowest acquisition speed with which product MS scans were collected. Scan acquisition frequency can be used to gauge the suitability of used instrument settings for the sample content used.</summary>
            MS_slowest_frequency_for_MS_level_2_collection = 4000096,

            /// <summary>The number of times where MS1 TIC increased more than 10-fold between adjacent MS1 scans. An unusual high count of signal jumps or falls can indicate ESI stability issues.</summary>
            MS_MS1_signal_jump__10x__count = 4000097,

            /// <summary>The number of times where MS1 TIC decreased more than 10-fold between adjacent MS1 scans. An unusual high count of signal jumps or falls can indicate ESI stability issues.</summary>
            MS_MS1_signal_fall__10x__count = 4000098,

            /// <summary>Number of MS1 scans where the scans' peaks intensity sums to 0 (i.e. no peaks or only 0-intensity peaks).</summary>
            MS_number_of_empty_MS1_scans = 4000099,

            /// <summary>Number of MS2 scans where the scans' peaks intensity sums to 0 (i.e. no peaks or only 0-intensity peaks).</summary>
            MS_number_of_empty_MS2_scans = 4000100,

            /// <summary>Number of MS3 scans where the scans' peaks intensity sums to 0 (i.e. no peaks or only 0-intensity peaks).</summary>
            MS_number_of_empty_MS3_scans = 4000101,

            /// <summary>The number of datapoints detected for quantification purposes within the run. These datapoints may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the meta data or analysis methods section of the recording file for the respective run.</summary>
            MS_number_of_detected_quantification_data_points = 4000102,

            /// <summary>The number of identified datapoints for quantification purposes withing the run after user defined acceptance criteria are applied. These datapoints may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the meta data or analysis methods section of the recording file for the respective run. In case of multiple acceptance criteria (FDR) available in proteomics, PSM-level FDR should be used for better comparability.</summary>
            MS_number_of_identified_quantification_data_points = 4000103,

            /// <summary>A practiced and regimented skill or series of actions.</summary>
            NCIT_Technique = 100316847,

            /// <summary>A method of analyzing or representing statistical data; a procedure for calculating a statistic.</summary>
            NCIT_Statistical_Technique = 100319044,

            /// <summary>An organizational header for concepts representing mostly abstract entities.</summary>
            NCIT_Conceptual_Entity = 100320181,

            /// <summary>A distinguishing quality or prominent aspect of a person, object, action, process, or substance.</summary>
            NCIT_Property_or_Attribute = 100320189,

            /// <summary>Terms used to indicate units of time or other terms associated with time.</summary>
            NCIT_Temporal_Qualifier = 100321514,

            /// <summary>The period of time during which something continues.</summary>
            NCIT_Duration = 100325330,

            /// <summary>A thing done.</summary>
            NCIT_Action = 100325404,

            /// <summary>The distinguishing qualities or prominent aspect of a person, object, action, process, or substance.</summary>
            NCIT_Characteristic = 100325447,

            /// <summary>A term that helps define and render a concept unique.</summary>
            NCIT_Qualifier = 100341009,

            /// <summary>An active process; excludes processes and mechanisms which fulfill biological functions.</summary>
            NCIT_Activity = 100343431,

            /// <summary>The amount of something per unit size.</summary>
            NCIT_Density = 100345781,

            /// <summary>One of the axes representing the projection of varience resulting from principal component analysis.</summary>
            NCIT_Principal_Component = 100360694,

            /// <summary>Span a region or interval of distance, space or time.</summary>
            NCIT_Cover = 100368811,

            /// <summary>An observation in a data set that is numerically distant from the rest of the data.</summary>
            NCIT_Outlier = 100379083,

            /// <summary>PSI Extended FASTA Format controlled vocabulary term.</summary>
            PEFF_PEFF_CV_term = 200000001,

            /// <summary>CV term that may appear in a PEFF file header section.</summary>
            PEFF_PEFF_file_header_section_term = 200000002,

            /// <summary>CV term that may appear in a description line of a PEFF file individual sequence entry.</summary>
            PEFF_PEFF_file_sequence_entry_term = 200000003,

            /// <summary>PEFF keyword for the sequence database name.</summary>
            PEFF_DbName = 200000008,

            /// <summary>PEFF keyword for the sequence database prefix.</summary>
            PEFF_Prefix = 200000009,

            /// <summary>PEFF keyword for the sequence database short description.</summary>
            PEFF_DbDescription = 200000010,

            /// <summary>PEFF keyword for the specifying whether the sequence database is a decoy database.</summary>
            PEFF_Decoy = 200000011,

            /// <summary>PEFF keyword for the source of the database file.</summary>
            PEFF_DbSource = 200000012,

            /// <summary>PEFF keyword for the database version (release date) according to database provider.</summary>
            PEFF_DbVersion = 200000013,

            /// <summary>OBSOLETE PEFF keyword for the database date (release or file date of the source) according to database provider.</summary>
            PEFF_DbDate_OBSOLETE = 200000014,

            /// <summary>PEFF keyword for the sumber of sequence entries in the database.</summary>
            PEFF_NumberOfEntries = 200000015,

            /// <summary>PEFF keyword for the description of the conversion from original format to this current one.</summary>
            PEFF_Conversion = 200000016,

            /// <summary>PEFF keyword for the molecular type of the sequences.</summary>
            PEFF_SequenceType = 200000017,

            /// <summary>PEFF keyword for database specific keywords not included in the current controlled vocabulary.</summary>
            PEFF_SpecificKey = 200000018,

            /// <summary>PEFF keyword for the specific values for a custom key.</summary>
            PEFF_SpecificValue = 200000019,

            /// <summary>PEFF keyword for the short description of the PEFF file.</summary>
            PEFF_DatabaseDescription = 200000020,

            /// <summary>PEFF keyword for a general comment.</summary>
            PEFF_GeneralComment = 200000021,

            /// <summary>PEFF keyword that when set to 'true' indicates that the database contains complete proteoforms.</summary>
            PEFF_ProteoformDb = 200000022,

            /// <summary>PEFF keyword for the short tag (abbreviation) and longer definition used to annotate a sequence annotation (such as variant or modification) in the OptionalTag location.</summary>
            PEFF_OptionalTagDef = 200000023,

            /// <summary>PEFF keyword that when set to 'true' indicates that entries in the database have identifiers for each annotation.</summary>
            PEFF_HasAnnotationIdentifiers = 200000024,

            /// <summary>OBSOLETE Sequence database unique identifier.</summary>
            PEFF_DbUniqueId_OBSOLETE = 200001001,

            /// <summary>PEFF keyword for the protein full name.</summary>
            PEFF_PName = 200001002,

            /// <summary>PEFF keyword for the NCBI taxonomy identifier.</summary>
            PEFF_NcbiTaxId = 200001003,

            /// <summary>PEFF keyword for the taxonomy name (latin or common name).</summary>
            PEFF_TaxName = 200001004,

            /// <summary>PEFF keyword for the gene name.</summary>
            PEFF_GName = 200001005,

            /// <summary>PEFF keyword for the sequence length.</summary>
            PEFF_Length = 200001006,

            /// <summary>PEFF keyword for the sequence version.</summary>
            PEFF_SV = 200001007,

            /// <summary>PEFF keyword for the entry version.</summary>
            PEFF_EV = 200001008,

            /// <summary>PEFF keyword for the Protein Evidence; A UniProtKB code 1-5.</summary>
            PEFF_PE = 200001009,

            /// <summary>PEFF keyword for information on how the full length original protein sequence can be processed into shorter components such as signal peptides and chains.</summary>
            PEFF_Processed = 200001010,

            /// <summary>OBSOLETE Sequence variation (substitution, insertion, deletion).</summary>
            PEFF_Variant_OBSOLETE = 200001011,

            /// <summary>PEFF keyword for the modified residue with PSI-MOD identifier.</summary>
            PEFF_ModResPsi = 200001012,

            /// <summary>PEFF keyword for the modified residue without aPSI-MOD or UniMod identifier.</summary>
            PEFF_ModRes = 200001013,

            /// <summary>PEFF keyword for the Alternative Accession Code.</summary>
            PEFF_AltAC = 200001014,

            /// <summary>PEFF keyword for the sequence status. Complete or Fragment.</summary>
            PEFF_SeqStatus = 200001015,

            /// <summary>PEFF keyword for the entry associated comment.</summary>
            PEFF_CC = 200001016,

            /// <summary>PEFF keyword for the entry associated keyword(s).</summary>
            PEFF_KW = 200001017,

            /// <summary>PEFF keyword for the Gene Ontology code.</summary>
            PEFF_GO = 200001018,

            /// <summary>PEFF keyword for the cross-reference to an external resource.</summary>
            PEFF_XRef = 200001019,

            /// <summary>Portion of a newly synthesized protein that contributes to a final structure after other components such as signal peptides are removed.</summary>
            PEFF_mature_protein = 200001020,

            /// <summary>Short peptide present at the N-terminus of a newly synthesized protein that is cleaved off and is not part of the final mature protein.</summary>
            PEFF_signal_peptide = 200001021,

            /// <summary>Short peptide present at the N-terminus of a newly synthesized protein that helps the protein through the membrane of its destination organelle.</summary>
            PEFF_transit_peptide = 200001022,

            /// <summary>PEFF keyword for the sequence conflict; a UniProtKB term.</summary>
            PEFF_Conflict = 200001023,

            /// <summary>PEFF keyword for the Sequence checksum in crc64.</summary>
            PEFF_Crc64 = 200001024,

            /// <summary>PEFF keyword for the sequence range of a domain.</summary>
            PEFF_Domain = 200001025,

            /// <summary>PEFF keyword for the UniProtKB specific Protein identifier ID; a UniProtKB term.</summary>
            PEFF_ID = 200001026,

            /// <summary>PEFF keyword for the modified residue with UniMod identifier.</summary>
            PEFF_ModResUnimod = 200001027,

            /// <summary>PEFF keyword for the simple sequence variation of a single amino acid change. A change to a stop codon is permitted with a * symbol. More complex variations must be encoded with the VariantComplex term.</summary>
            PEFF_VariantSimple = 200001028,

            /// <summary>PEFF keyword for a sequence variation that is more complex than a single amino acid change or change to a stop codon.</summary>
            PEFF_VariantComplex = 200001029,

            /// <summary>PEFF keyword for the proteoforms of this protein, constructed as a set of annotation identifiers.</summary>
            PEFF_Proteoform = 200001030,

            /// <summary>PEFF keyword for the disulfide bonds in this protein, constructed as a sets of annotation identifiers of two half-cystine modifications.</summary>
            PEFF_DisulfideBond = 200001031,

            /// <summary>PEFF keyword describing the type of processing event being described.</summary>
            PEFF_PEFF_molecule_processing_keyword = 200001032,

            /// <summary>PEFF keyword for the individual protein entry comment. It is discouraged to put parsable information here. This is only for free-text commentary.</summary>
            PEFF_Comment = 200001033,

            /// <summary>Short peptide that is cleaved off a newly synthesized protein and generally immediately degraded in the process of protein maturation, and is not a signal peptide or transit peptide.</summary>
            PEFF_propeptide = 200001034,

            /// <summary>N-terminal methionine residue of a protein that can be co-translationally cleaved.</summary>
            PEFF_initiator_methionine = 200001035,

            /// <summary>(</summary>
            PEFF_regular_expression_for_a_value_in_a_key_value_pair_of_a_PEFF_description_line_describing_one_sequence_position_followed_by_one_PEFF_term_name_and_one_optional_comment = 201002001,

            /// <summary>(AA|NA|DNA|cDNA|RNA|RNAi|tRNA|rRNA|siRNA|mDNA|mRNA|snoRNA|ncRNA|EST|</summary>
            PEFF_regular_expression_for_PEFF_molecular_sequence_type = 201002002,

            /// <summary>(Complete|Fragment|</summary>
            PEFF_regular_expression_for_PEFF_sequence_status = 201002003,

            /// <summary>The root node of the unimod modifications ontology.</summary>
            UNIMOD_unimod_root_node = 300000000,

            /// <summary>Acetylation.</summary>
            UNIMOD_Acetyl = 300000001,

            /// <summary>Amidation.</summary>
            UNIMOD_Amidated = 300000002,

            /// <summary>Biotinylation.</summary>
            UNIMOD_Biotin = 300000003,

            /// <summary>Iodoacetamide derivative.</summary>
            UNIMOD_Carbamidomethyl = 300000004,

            /// <summary>Carbamylation.</summary>
            UNIMOD_Carbamyl = 300000005,

            /// <summary>Iodoacetic acid derivative.</summary>
            UNIMOD_Carboxymethyl = 300000006,

            /// <summary>Deamidation.</summary>
            UNIMOD_Deamidated = 300000007,

            /// <summary>Gygi ICAT(TM) d0.</summary>
            UNIMOD_ICAT_G = 300000008,

            /// <summary>Gygi ICAT(TM) d8.</summary>
            UNIMOD_ICAT_G_2H_8_ = 300000009,

            /// <summary>Homoserine.</summary>
            UNIMOD_Met__Hse = 300000010,

            /// <summary>Homoserine lactone.</summary>
            UNIMOD_Met__Hsl = 300000011,

            /// <summary>Applied Biosystems original ICAT(TM) d8.</summary>
            UNIMOD_ICAT_D_2H_8_ = 300000012,

            /// <summary>Applied Biosystems original ICAT(TM) d0.</summary>
            UNIMOD_ICAT_D = 300000013,

            /// <summary>N-isopropylcarboxamidomethyl.</summary>
            UNIMOD_NIPCAM = 300000017,

            /// <summary>Biotinyl-iodoacetamidyl-3,6-dioxaoctanediamine.</summary>
            UNIMOD_PEO_Iodoacetyl_LC_Biotin = 300000020,

            /// <summary>Phosphorylation.</summary>
            UNIMOD_Phospho = 300000021,

            /// <summary>Dehydration.</summary>
            UNIMOD_Dehydrated = 300000023,

            /// <summary>Acrylamide adduct.</summary>
            UNIMOD_Propionamide = 300000024,

            /// <summary>Pyridylacetyl.</summary>
            UNIMOD_Pyridylacetyl = 300000025,

            /// <summary>S-carbamoylmethylcysteine cyclization (N-terminus).</summary>
            UNIMOD_Pyro_carbamidomethyl = 300000026,

            /// <summary>Pyro-glu from E.</summary>
            UNIMOD_Glu__pyro_Glu = 300000027,

            /// <summary>Pyro-glu from Q.</summary>
            UNIMOD_Gln__pyro_Glu = 300000028,

            /// <summary>N-Succinimidyl-2-morpholine acetate.</summary>
            UNIMOD_SMA = 300000029,

            /// <summary>Sodium adduct.</summary>
            UNIMOD_Cation_Na = 300000030,

            /// <summary>S-pyridylethylation.</summary>
            UNIMOD_Pyridylethyl = 300000031,

            /// <summary>Methylation.</summary>
            UNIMOD_Methyl = 300000034,

            /// <summary>Oxidation or Hydroxylation.</summary>
            UNIMOD_Oxidation = 300000035,

            /// <summary>Di-Methylation.</summary>
            UNIMOD_Dimethyl = 300000036,

            /// <summary>Tri-Methylation.</summary>
            UNIMOD_Trimethyl = 300000037,

            /// <summary>Beta-methylthiolation.</summary>
            UNIMOD_Methylthio = 300000039,

            /// <summary>O-Sulfonation.</summary>
            UNIMOD_Sulfo = 300000040,

            /// <summary>Hexose.</summary>
            UNIMOD_Hex = 300000041,

            /// <summary>Lipoyl.</summary>
            UNIMOD_Lipoyl = 300000042,

            /// <summary>N-Acetylhexosamine.</summary>
            UNIMOD_HexNAc = 300000043,

            /// <summary>Farnesylation.</summary>
            UNIMOD_Farnesyl = 300000044,

            /// <summary>Myristoylation.</summary>
            UNIMOD_Myristoyl = 300000045,

            /// <summary>Pyridoxal phosphate.</summary>
            UNIMOD_PyridoxalPhosphate = 300000046,

            /// <summary>Palmitoylation.</summary>
            UNIMOD_Palmitoyl = 300000047,

            /// <summary>Geranyl-geranyl.</summary>
            UNIMOD_GeranylGeranyl = 300000048,

            /// <summary>Phosphopantetheine.</summary>
            UNIMOD_Phosphopantetheine = 300000049,

            /// <summary>Flavin adenine dinucleotide.</summary>
            UNIMOD_FAD = 300000050,

            /// <summary>N-acyl diglyceride cysteine.</summary>
            UNIMOD_Tripalmitate = 300000051,

            /// <summary>Guanidination.</summary>
            UNIMOD_Guanidinyl = 300000052,

            /// <summary>4-hydroxynonenal (HNE).</summary>
            UNIMOD_HNE = 300000053,

            /// <summary>Hexuronic acid.</summary>
            UNIMOD_Glucuronyl = 300000054,

            /// <summary>Glutathione disulfide.</summary>
            UNIMOD_Glutathione = 300000055,

            /// <summary>Acetate labeling reagent (N-term &amp; K) (heavy form, +3amu).</summary>
            UNIMOD_Acetyl_2H_3_ = 300000056,

            /// <summary>Propionate labeling reagent light form (N-term &amp; K).</summary>
            UNIMOD_Propionyl = 300000058,

            /// <summary>Propionate labeling reagent heavy form (+3amu), N-term &amp; K.</summary>
            UNIMOD_Propionyl_13C_3_ = 300000059,

            /// <summary>Quaternary amine labeling reagent light form (N-term &amp; K).</summary>
            UNIMOD_GIST_Quat = 300000060,

            /// <summary>Quaternary amine labeling reagent heavy (+3amu) form, N-term &amp; K.</summary>
            UNIMOD_GIST_Quat_2H_3_ = 300000061,

            /// <summary>Quaternary amine labeling reagent heavy form (+6amu), N-term &amp; K.</summary>
            UNIMOD_GIST_Quat_2H_6_ = 300000062,

            /// <summary>Quaternary amine labeling reagent heavy form (+9amu), N-term &amp; K.</summary>
            UNIMOD_GIST_Quat_2H_9_ = 300000063,

            /// <summary>Succinic anhydride labeling reagent light form (N-term &amp; K).</summary>
            UNIMOD_Succinyl = 300000064,

            /// <summary>Succinic anhydride labeling reagent, heavy form (+4amu, 4H2), N-term &amp; K.</summary>
            UNIMOD_Succinyl_2H_4_ = 300000065,

            /// <summary>Succinic anhydride labeling reagent, heavy form (+4amu, 4C13), N-term &amp; K.</summary>
            UNIMOD_Succinyl_13C_4_ = 300000066,

            /// <summary>Iminobiotinylation.</summary>
            UNIMOD_Iminobiotin = 300000089,

            /// <summary>ESP-Tag light d0.</summary>
            UNIMOD_ESP = 300000090,

            /// <summary>ESP-Tag heavy d10.</summary>
            UNIMOD_ESP_2H_10_ = 300000091,

            /// <summary>NHS-LC-Biotin.</summary>
            UNIMOD_NHS_LC_Biotin = 300000092,

            /// <summary>EDT-maleimide-PEO-biotin.</summary>
            UNIMOD_EDT_maleimide_PEO_biotin = 300000093,

            /// <summary>IMID d0.</summary>
            UNIMOD_IMID = 300000094,

            /// <summary>IMID d4.</summary>
            UNIMOD_IMID_2H_4_ = 300000095,

            /// <summary>Acrylamide d3.</summary>
            UNIMOD_Propionamide_2H_3_ = 300000097,

            /// <summary>Applied Biosystems cleavable ICAT(TM) light.</summary>
            UNIMOD_ICAT_C = 300000105,

            /// <summary>Applied Biosystems cleavable ICAT(TM) heavy.</summary>
            UNIMOD_ICAT_C_13C_9_ = 300000106,

            /// <summary>Addition of N-formyl met.</summary>
            UNIMOD_FormylMet = 300000107,

            /// <summary>N-ethylmaleimide on cysteines.</summary>
            UNIMOD_Nethylmaleimide = 300000108,

            /// <summary>Oxidized lysine biotinylated with biotin-LC-hydrazide, reduced.</summary>
            UNIMOD_OxLysBiotinRed = 300000112,

            /// <summary>Oxidized lysine biotinylated with biotin-LC-hydrazide.</summary>
            UNIMOD_OxLysBiotin = 300000113,

            /// <summary>Oxidized proline biotinylated with biotin-LC-hydrazide, reduced.</summary>
            UNIMOD_OxProBiotinRed = 300000114,

            /// <summary>Oxidized Proline biotinylated with biotin-LC-hydrazide.</summary>
            UNIMOD_OxProBiotin = 300000115,

            /// <summary>Oxidized arginine biotinylated with biotin-LC-hydrazide.</summary>
            UNIMOD_OxArgBiotin = 300000116,

            /// <summary>Oxidized arginine biotinylated with biotin-LC-hydrazide, reduced.</summary>
            UNIMOD_OxArgBiotinRed = 300000117,

            /// <summary>EDT-iodo-PEO-biotin.</summary>
            UNIMOD_EDT_iodoacetyl_PEO_biotin = 300000118,

            /// <summary>Thio Ether Formation - BTP Adduct.</summary>
            UNIMOD_IBTP = 300000119,

            /// <summary>Ubiquitinylation residue.</summary>
            UNIMOD_GG = 300000121,

            /// <summary>Formylation.</summary>
            UNIMOD_Formyl = 300000122,

            /// <summary>N-iodoacetyl, p-chlorobenzyl-12C6-glucamine.</summary>
            UNIMOD_ICAT_H = 300000123,

            /// <summary>N-iodoacetyl, p-chlorobenzyl-13C6-glucamine.</summary>
            UNIMOD_ICAT_H_13C_6_ = 300000124,

            /// <summary>Cleaved and reduced DSP/DTSSP crosslinker.</summary>
            UNIMOD_Xlink_DTSSP_88_ = 300000126,

            /// <summary>Fluorination.</summary>
            UNIMOD_Fluoro = 300000127,

            /// <summary>5-Iodoacetamidofluorescein (Molecular Probe, Eugene, OR).</summary>
            UNIMOD_Fluorescein = 300000128,

            /// <summary>Iodination.</summary>
            UNIMOD_Iodo = 300000129,

            /// <summary>Di-Iodination.</summary>
            UNIMOD_Diiodo = 300000130,

            /// <summary>Tri-Iodination.</summary>
            UNIMOD_Triiodo = 300000131,

            /// <summary>(cis-delta 5)-tetradecaenoyl.</summary>
            UNIMOD_Myristoleyl = 300000134,

            /// <summary>(cis,cis-delta 5, delta 8)-tetradecadienoyl.</summary>
            UNIMOD_Myristoyl_Delta_H__4_ = 300000135,

            /// <summary>Labeling reagent light form (N-term &amp; K).</summary>
            UNIMOD_Benzoyl = 300000136,

            /// <summary>M5/Man5.</summary>
            UNIMOD_Hex_5_HexNAc_2_ = 300000137,

            /// <summary>5-dimethylaminonaphthalene-1-sulfonyl.</summary>
            UNIMOD_Dansyl = 300000139,

            /// <summary>ISD a-series (C-Term).</summary>
            UNIMOD_a_type_ion = 300000140,

            /// <summary>Amidination of lysines or N-terminal amines with methyl acetimidate.</summary>
            UNIMOD_Amidine = 300000141,

            /// <summary>HexNAc1dHex1.</summary>
            UNIMOD_HexNAc_1_dHex_1_ = 300000142,

            /// <summary>HexNAc2.</summary>
            UNIMOD_HexNAc_2_ = 300000143,

            /// <summary>Hex3.</summary>
            UNIMOD_Hex_3_ = 300000144,

            /// <summary>HexNAc1dHex2.</summary>
            UNIMOD_HexNAc_1_dHex_2_ = 300000145,

            /// <summary>Hex1HexNAc1dHex1.</summary>
            UNIMOD_Hex_1_HexNAc_1_dHex_1_ = 300000146,

            /// <summary>HexNAc2dHex1.</summary>
            UNIMOD_HexNAc_2_dHex_1_ = 300000147,

            /// <summary>Hex1HexNAc2.</summary>
            UNIMOD_Hex_1_HexNAc_2_ = 300000148,

            /// <summary>Hex1HexNAc1NeuAc1.</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuAc_1_ = 300000149,

            /// <summary>HexNAc2dHex2.</summary>
            UNIMOD_HexNAc_2_dHex_2_ = 300000150,

            /// <summary>Hex1HexNAc2Pent1.</summary>
            UNIMOD_Hex_1_HexNAc_2_Pent_1_ = 300000151,

            /// <summary>Hex1HexNAc2dHex1.</summary>
            UNIMOD_Hex_1_HexNAc_2_dHex_1_ = 300000152,

            /// <summary>Hex2HexNAc2.</summary>
            UNIMOD_Hex_2_HexNAc_2_ = 300000153,

            /// <summary>Hex3HexNAc1Pent1.</summary>
            UNIMOD_Hex_3_HexNAc_1_Pent_1_ = 300000154,

            /// <summary>Hex1HexNAc2dHex1Pent1.</summary>
            UNIMOD_Hex_1_HexNAc_2_dHex_1_Pent_1_ = 300000155,

            /// <summary>Hex1HexNAc2dHex2.</summary>
            UNIMOD_Hex_1_HexNAc_2_dHex_2_ = 300000156,

            /// <summary>Hex2HexNAc2Pent1.</summary>
            UNIMOD_Hex_2_HexNAc_2_Pent_1_ = 300000157,

            /// <summary>Hex2HexNAc2dHex1.</summary>
            UNIMOD_Hex_2_HexNAc_2_dHex_1_ = 300000158,

            /// <summary>M3/Man3.</summary>
            UNIMOD_Hex_3_HexNAc_2_ = 300000159,

            /// <summary>Hex HexNAc NeuAc(2) ---OR--- Hex HexNAc(3) HexA.</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuAc_2_ = 300000160,

            /// <summary>Hex(3) HexNAc(2) Phos.</summary>
            UNIMOD_Hex_3_HexNAc_2_Phos_1_ = 300000161,

            /// <summary>Selenium replaces sulfur.</summary>
            UNIMOD_Delta_S__1_Se_1_ = 300000162,

            /// <summary>Glycosylated asparagine 18O labeling.</summary>
            UNIMOD_Delta_H__1_N__1_18O_1_ = 300000170,

            /// <summary>Shimadzu NBS-13C.</summary>
            UNIMOD_NBS_13C_6_ = 300000171,

            /// <summary>Shimadzu NBS-12C.</summary>
            UNIMOD_NBS = 300000172,

            /// <summary>Michael addition of BHT quinone methide to Cysteine and Lysine.</summary>
            UNIMOD_BHT = 300000176,

            /// <summary>Phosphorylation to amine thiol.</summary>
            UNIMOD_DAET = 300000178,

            /// <summary>13C(9) Silac label.</summary>
            UNIMOD_Label_13C_9_ = 300000184,

            /// <summary>C13 label (Phosphotyrosine).</summary>
            UNIMOD_Label_13C_9__Phospho = 300000185,

            /// <summary>Hydroxyphenylglyoxal arginine.</summary>
            UNIMOD_HPG = 300000186,

            /// <summary>Bis(hydroxphenylglyoxal) arginine.</summary>
            UNIMOD_2HPG = 300000187,

            /// <summary>13C(6) Silac label.</summary>
            UNIMOD_Label_13C_6_ = 300000188,

            /// <summary>O18 label at both C-terminal oxygens.</summary>
            UNIMOD_Label_18O_2_ = 300000193,

            /// <summary>6-aminoquinolyl-N-hydroxysuccinimidyl carbamate.</summary>
            UNIMOD_AccQTag = 300000194,

            /// <summary>APTA-d0.</summary>
            UNIMOD_QAT = 300000195,

            /// <summary>APTA d3.</summary>
            UNIMOD_QAT_2H_3_ = 300000196,

            /// <summary>EAPTA d0.</summary>
            UNIMOD_EQAT = 300000197,

            /// <summary>EAPTA d5.</summary>
            UNIMOD_EQAT_2H_5_ = 300000198,

            /// <summary>DiMethyl-CHD2.</summary>
            UNIMOD_Dimethyl_2H_4_ = 300000199,

            /// <summary>EDT.</summary>
            UNIMOD_Ethanedithiol = 300000200,

            /// <summary>Acrolein addition +94.</summary>
            UNIMOD_Delta_H_6_C_6_O_1_ = 300000205,

            /// <summary>Acrolein addition +56.</summary>
            UNIMOD_Delta_H_4_C_3_O_1_ = 300000206,

            /// <summary>Acrolein addition +38.</summary>
            UNIMOD_Delta_H_2_C_3_ = 300000207,

            /// <summary>Acrolein addition +76.</summary>
            UNIMOD_Delta_H_4_C_6_ = 300000208,

            /// <summary>Acrolein addition +112.</summary>
            UNIMOD_Delta_H_8_C_6_O_2_ = 300000209,

            /// <summary>N-ethyl iodoacetamide-d0.</summary>
            UNIMOD_NEIAA = 300000211,

            /// <summary>N-ethyl iodoacetamide-d5.</summary>
            UNIMOD_NEIAA_2H_5_ = 300000212,

            /// <summary>ADP Ribose addition.</summary>
            UNIMOD_ADP_Ribosyl = 300000213,

            /// <summary>Representative mass and accurate mass for 116 &amp; 117.</summary>
            UNIMOD_iTRAQ4plex = 300000214,

            /// <summary>Light IDBEST tag for quantitation.</summary>
            UNIMOD_IGBP = 300000243,

            /// <summary>Crotonaldehyde.</summary>
            UNIMOD_Crotonaldehyde = 300000253,

            /// <summary>Acetaldehyde +26.</summary>
            UNIMOD_Delta_H_2_C_2_ = 300000254,

            /// <summary>Acetaldehyde +28.</summary>
            UNIMOD_Delta_H_4_C_2_ = 300000255,

            /// <summary>Propionaldehyde +40.</summary>
            UNIMOD_Delta_H_4_C_3_ = 300000256,

            /// <summary>O18 Labeling.</summary>
            UNIMOD_Label_18O_1_ = 300000258,

            /// <summary>13C(6) 15N(2) Silac label.</summary>
            UNIMOD_Label_13C_6_15N_2_ = 300000259,

            /// <summary>Thiophosphorylation.</summary>
            UNIMOD_Thiophospho = 300000260,

            /// <summary>4-sulfophenyl isothiocyanate.</summary>
            UNIMOD_SPITC = 300000261,

            /// <summary>Trideuteration.</summary>
            UNIMOD_Label_2H_3_ = 300000262,

            /// <summary>Phosphorylation to pyridyl thiol.</summary>
            UNIMOD_PET = 300000264,

            /// <summary>13C(6) 15N(4) Silac label.</summary>
            UNIMOD_Label_13C_6_15N_4_ = 300000267,

            /// <summary>13C(5) 15N(1) Silac label.</summary>
            UNIMOD_Label_13C_5_15N_1_ = 300000268,

            /// <summary>13C(9) 15N(1) Silac label.</summary>
            UNIMOD_Label_13C_9_15N_1_ = 300000269,

            /// <summary>Nucleophilic addtion to cytopiloyne.</summary>
            UNIMOD_Cytopiloyne = 300000270,

            /// <summary>Nucleophilic addition to cytopiloyne+H2O.</summary>
            UNIMOD_Cytopiloyne_water = 300000271,

            /// <summary>Sulfonation of N-terminus.</summary>
            UNIMOD_CAF = 300000272,

            /// <summary>Nitrosylation.</summary>
            UNIMOD_Nitrosyl = 300000275,

            /// <summary>Aminoethylbenzenesulfonylation.</summary>
            UNIMOD_AEBS = 300000276,

            /// <summary>Ethanolation.</summary>
            UNIMOD_Ethanolyl = 300000278,

            /// <summary>Ethylation.</summary>
            UNIMOD_Ethyl = 300000280,

            /// <summary>Cysteine modified Coenzyme A.</summary>
            UNIMOD_CoenzymeA = 300000281,

            /// <summary>Deuterium Methylation of Lysine.</summary>
            UNIMOD_Methyl_2H_2_ = 300000284,

            /// <summary>Light Sulfanilic Acid (SA) C12.</summary>
            UNIMOD_SulfanilicAcid = 300000285,

            /// <summary>Heavy Sulfanilic Acid (SA) C13.</summary>
            UNIMOD_SulfanilicAcid_13C_6_ = 300000286,

            /// <summary>Tryptophan oxidation to oxolactone.</summary>
            UNIMOD_Trp__Oxolactone = 300000288,

            /// <summary>Biotin polyethyleneoxide amine.</summary>
            UNIMOD_Biotin_PEO_Amine = 300000289,

            /// <summary>Pierce EZ-Link Biotin-HPDP.</summary>
            UNIMOD_Biotin_HPDP = 300000290,

            /// <summary>Mercury Mercaptan.</summary>
            UNIMOD_Delta_Hg_1_ = 300000291,

            /// <summary>(Iodo)-uracil MP.</summary>
            UNIMOD_IodoU_AMP = 300000292,

            /// <summary>3-(carbamidomethylthio)propanoyl.</summary>
            UNIMOD_CAMthiopropanoyl = 300000293,

            /// <summary>Biotinoyl-iodoacetyl-ethylenediamine.</summary>
            UNIMOD_IED_Biotin = 300000294,

            /// <summary>Fucose.</summary>
            UNIMOD_dHex = 300000295,

            /// <summary>Deuterated methyl ester.</summary>
            UNIMOD_Methyl_2H_3_ = 300000298,

            /// <summary>Carboxylation.</summary>
            UNIMOD_Carboxy = 300000299,

            /// <summary>Monobromobimane derivative.</summary>
            UNIMOD_Bromobimane = 300000301,

            /// <summary>Menadione quinone derivative.</summary>
            UNIMOD_Menadione = 300000302,

            /// <summary>Cysteine mercaptoethanol.</summary>
            UNIMOD_DeStreak = 300000303,

            /// <summary>FA2/G0F.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_4_ = 300000305,

            /// <summary>FA2G1/G1F.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_4_ = 300000307,

            /// <summary>FA2G2/G2F.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_4_ = 300000308,

            /// <summary>A2/G0.</summary>
            UNIMOD_Hex_3_HexNAc_4_ = 300000309,

            /// <summary>A2G1/G1.</summary>
            UNIMOD_Hex_4_HexNAc_4_ = 300000310,

            /// <summary>A2G2/G2.</summary>
            UNIMOD_Hex_5_HexNAc_4_ = 300000311,

            /// <summary>Cysteinylation.</summary>
            UNIMOD_Cysteinyl = 300000312,

            /// <summary>Loss of Lysine.</summary>
            UNIMOD_Lys_loss = 300000313,

            /// <summary>Nmethylmaleimide.</summary>
            UNIMOD_Nmethylmaleimide = 300000314,

            /// <summary>2,5-dimethypyrrole.</summary>
            UNIMOD_DimethylpyrroleAdduct = 300000316,

            /// <summary>MDA adduct +62.</summary>
            UNIMOD_Delta_H_2_C_5_ = 300000318,

            /// <summary>MDA adduct +54.</summary>
            UNIMOD_Delta_H_2_C_3_O_1_ = 300000319,

            /// <summary>Nethylmaleimidehydrolysis.</summary>
            UNIMOD_Nethylmaleimide_water = 300000320,

            /// <summary>Bis-((N-iodoacetyl)piperazinyl)sulfonerhodamine.</summary>
            UNIMOD_Xlink_B10621 = 300000323,

            /// <summary>Cleaved and reduced DTBP crosslinker.</summary>
            UNIMOD_Xlink_DTBP_87_ = 300000324,

            /// <summary>10-ethoxyphosphinyl-N-(biotinamidopentyl)decanamide.</summary>
            UNIMOD_FP_Biotin = 300000325,

            /// <summary>S-Ethylcystine from Serine.</summary>
            UNIMOD_Delta_H_4_C_2_O__1_S_1_ = 300000327,

            /// <summary>Monomethylation.</summary>
            UNIMOD_Methyl_2H_3_13C_1_ = 300000329,

            /// <summary>Dimethylation.</summary>
            UNIMOD_Dimethyl_2H_6_13C_2_ = 300000330,

            /// <summary>Thiophosphate labeled with biotin-HPDP.</summary>
            UNIMOD_Thiophos_S_S_biotin = 300000332,

            /// <summary>6-N-biotinylaminohexyl isopropyl phosphate.</summary>
            UNIMOD_Can_FP_biotin = 300000333,

            /// <summary>Reduced 4-Hydroxynonenal.</summary>
            UNIMOD_HNE_Delta_H_2_ = 300000335,

            /// <summary>Michael addition with methylamine.</summary>
            UNIMOD_Methylamine = 300000337,

            /// <summary>Bromination.</summary>
            UNIMOD_Bromo = 300000340,

            /// <summary>Tyrosine oxidation to 2-aminotyrosine.</summary>
            UNIMOD_Amino = 300000342,

            /// <summary>Oxidized Arginine biotinylated with biotin hydrazide.</summary>
            UNIMOD_Argbiotinhydrazide = 300000343,

            /// <summary>Arginine oxidation to glutamic semialdehyde.</summary>
            UNIMOD_Arg__GluSA = 300000344,

            /// <summary>Cysteine oxidation to cysteic acid.</summary>
            UNIMOD_Trioxidation = 300000345,

            /// <summary>His-&gt;Asn substitution.</summary>
            UNIMOD_His__Asn = 300000348,

            /// <summary>His-&gt;Asp substitution.</summary>
            UNIMOD_His__Asp = 300000349,

            /// <summary>Tryptophan oxidation to hydroxykynurenin.</summary>
            UNIMOD_Trp__Hydroxykynurenin = 300000350,

            /// <summary>Tryptophan oxidation to kynurenin.</summary>
            UNIMOD_Trp__Kynurenin = 300000351,

            /// <summary>Lysine oxidation to aminoadipic semialdehyde.</summary>
            UNIMOD_Lys__Allysine = 300000352,

            /// <summary>Oxidized Lysine biotinylated with biotin hydrazide.</summary>
            UNIMOD_Lysbiotinhydrazide = 300000353,

            /// <summary>Oxidation to nitro.</summary>
            UNIMOD_Nitro = 300000354,

            /// <summary>Oxidized proline biotinylated with biotin hydrazide.</summary>
            UNIMOD_probiotinhydrazide = 300000357,

            /// <summary>Proline oxidation to pyroglutamic acid.</summary>
            UNIMOD_Pro__pyro_Glu = 300000359,

            /// <summary>Proline oxidation to pyrrolidinone.</summary>
            UNIMOD_Pro__Pyrrolidinone = 300000360,

            /// <summary>Oxidized Threonine biotinylated with biotin hydrazide.</summary>
            UNIMOD_Thrbiotinhydrazide = 300000361,

            /// <summary>O-Diisopropylphosphorylation.</summary>
            UNIMOD_Diisopropylphosphate = 300000362,

            /// <summary>O-Isopropylphosphorylation.</summary>
            UNIMOD_Isopropylphospho = 300000363,

            /// <summary>Bruker Daltonics SERVA-ICPL(TM) quantification chemistry, heavy form.</summary>
            UNIMOD_ICPL_13C_6_ = 300000364,

            /// <summary>Bruker Daltonics SERVA-ICPL(TM) quantification chemistry, light form.</summary>
            UNIMOD_ICPL = 300000365,

            /// <summary>Deamidation in presence of O18.</summary>
            UNIMOD_Deamidated_18O_1_ = 300000366,

            /// <summary>Dehydroalanine (from Cysteine).</summary>
            UNIMOD_Cys__Dha = 300000368,

            /// <summary>Pyrrolidone from Proline.</summary>
            UNIMOD_Pro__Pyrrolidone = 300000369,

            /// <summary>Michael addition of hydroxymethylvinyl ketone to cysteine.</summary>
            UNIMOD_HMVK = 300000371,

            /// <summary>Ornithine from Arginine.</summary>
            UNIMOD_Arg__Orn = 300000372,

            /// <summary>Half of a disulfide bridge.</summary>
            UNIMOD_Dehydro = 300000374,

            /// <summary>Diphthamide.</summary>
            UNIMOD_Diphthamide = 300000375,

            /// <summary>Hydroxyfarnesyl.</summary>
            UNIMOD_Hydroxyfarnesyl = 300000376,

            /// <summary>Diacylglycerol.</summary>
            UNIMOD_Diacylglycerol = 300000377,

            /// <summary>Carboxyethyl.</summary>
            UNIMOD_Carboxyethyl = 300000378,

            /// <summary>Hypusine.</summary>
            UNIMOD_Hypusine = 300000379,

            /// <summary>Retinal.</summary>
            UNIMOD_Retinylidene = 300000380,

            /// <summary>Alpha-amino adipic acid.</summary>
            UNIMOD_Lys__AminoadipicAcid = 300000381,

            /// <summary>Pyruvic acid from N-term cys.</summary>
            UNIMOD_Cys__PyruvicAcid = 300000382,

            /// <summary>Loss of ammonia.</summary>
            UNIMOD_Ammonia_loss = 300000385,

            /// <summary>Phycocyanobilin.</summary>
            UNIMOD_Phycocyanobilin = 300000387,

            /// <summary>Phycoerythrobilin.</summary>
            UNIMOD_Phycoerythrobilin = 300000388,

            /// <summary>Phytochromobilin.</summary>
            UNIMOD_Phytochromobilin = 300000389,

            /// <summary>Heme.</summary>
            UNIMOD_Heme = 300000390,

            /// <summary>Molybdopterin.</summary>
            UNIMOD_Molybdopterin = 300000391,

            /// <summary>Quinone.</summary>
            UNIMOD_Quinone = 300000392,

            /// <summary>Glucosylgalactosyl hydroxylysine.</summary>
            UNIMOD_Glucosylgalactosyl = 300000393,

            /// <summary>Glycosylphosphatidylinositol.</summary>
            UNIMOD_GPIanchor = 300000394,

            /// <summary>Phosphoribosyl dephospho-coenzyme A.</summary>
            UNIMOD_PhosphoribosyldephosphoCoA = 300000395,

            /// <summary>Glycerylphosphorylethanolamine.</summary>
            UNIMOD_GlycerylPE = 300000396,

            /// <summary>Triiodo.</summary>
            UNIMOD_Triiodothyronine = 300000397,

            /// <summary>Tetraiodo.</summary>
            UNIMOD_Thyroxine = 300000398,

            /// <summary>Dehydroalanine (from Tyrosine).</summary>
            UNIMOD_Tyr__Dha = 300000400,

            /// <summary>2-amino-3-oxo-butanoic_acid.</summary>
            UNIMOD_Didehydro = 300000401,

            /// <summary>Oxoalanine.</summary>
            UNIMOD_Cys__Oxoalanine = 300000402,

            /// <summary>Lactic acid from N-term Ser.</summary>
            UNIMOD_Ser__LacticAcid = 300000403,

            /// <summary>AMP.</summary>
            UNIMOD_Phosphoadenosine = 300000405,

            /// <summary>Hydroxycinnamyl.</summary>
            UNIMOD_Hydroxycinnamyl = 300000407,

            /// <summary>Glycosyl-L-hydroxyproline.</summary>
            UNIMOD_Glycosyl = 300000408,

            /// <summary>Flavin mononucleotide.</summary>
            UNIMOD_FMNH = 300000409,

            /// <summary>S-diphytanylglycerol diether.</summary>
            UNIMOD_Archaeol = 300000410,

            /// <summary>Phenyl isocyanate.</summary>
            UNIMOD_Phenylisocyanate = 300000411,

            /// <summary>D5-phenyl isocyanate.</summary>
            UNIMOD_Phenylisocyanate_2H_5_ = 300000412,

            /// <summary>Phospho-guanosine.</summary>
            UNIMOD_Phosphoguanosine = 300000413,

            /// <summary>Hydroxymethyl.</summary>
            UNIMOD_Hydroxymethyl = 300000414,

            /// <summary>L-selenocysteinyl molybdenum bis(molybdopterin guanine dinucleotide).</summary>
            UNIMOD_MolybdopterinGD_Delta_S__1_Se_1_ = 300000415,

            /// <summary>Dipyrrolylmethanemethyl.</summary>
            UNIMOD_Dipyrrolylmethanemethyl = 300000416,

            /// <summary>Uridine phosphodiester.</summary>
            UNIMOD_PhosphoUridine = 300000417,

            /// <summary>Glycerophospho.</summary>
            UNIMOD_Glycerophospho = 300000419,

            /// <summary>Thiocarboxylic acid.</summary>
            UNIMOD_Carboxy__Thiocarboxy = 300000420,

            /// <summary>Persulfide.</summary>
            UNIMOD_Sulfide = 300000421,

            /// <summary>N-pyruvic acid 2-iminyl.</summary>
            UNIMOD_PyruvicAcidIminyl = 300000422,

            /// <summary>Selenyl.</summary>
            UNIMOD_Delta_Se_1_ = 300000423,

            /// <summary>Molybdenum bis(molybdopterin guanine dinucleotide).</summary>
            UNIMOD_MolybdopterinGD = 300000424,

            /// <summary>Dihydroxy.</summary>
            UNIMOD_Dioxidation = 300000425,

            /// <summary>Octanoyl.</summary>
            UNIMOD_Octanoyl = 300000426,

            /// <summary>N-acetylglucosamine-1-phosphoryl.</summary>
            UNIMOD_PhosphoHexNAc = 300000428,

            /// <summary>Phosphoglycosyl-D-mannose-1-phosphoryl.</summary>
            UNIMOD_PhosphoHex = 300000429,

            /// <summary>Palmitoleyl.</summary>
            UNIMOD_Palmitoleyl = 300000431,

            /// <summary>Cholesterol ester.</summary>
            UNIMOD_Cholesterol = 300000432,

            /// <summary>3,4-didehydroretinylidene.</summary>
            UNIMOD_Didehydroretinylidene = 300000433,

            /// <summary>Cis-14-hydroxy-10,13-dioxo-7-heptadecenoic ester.</summary>
            UNIMOD_CHDH = 300000434,

            /// <summary>4-methyl-delta-1-pyrroline-5-carboxyl.</summary>
            UNIMOD_Methylpyrroline = 300000435,

            /// <summary>Hydroxyheme.</summary>
            UNIMOD_Hydroxyheme = 300000436,

            /// <summary>(3-aminopropyl)(L-aspartyl-1-amino)phosphoryl-5-adenosine.</summary>
            UNIMOD_MicrocinC7 = 300000437,

            /// <summary>Cyano.</summary>
            UNIMOD_Cyano = 300000438,

            /// <summary>Hydrogenase diiron subcluster.</summary>
            UNIMOD_Diironsubcluster = 300000439,

            /// <summary>Amidino.</summary>
            UNIMOD_Amidino = 300000440,

            /// <summary>O3-(riboflavin phosphoryl).</summary>
            UNIMOD_FMN = 300000442,

            /// <summary>S-(4a-FMN).</summary>
            UNIMOD_FMNC = 300000443,

            /// <summary>Copper sulfido molybdopterin cytosine dinuncleotide.</summary>
            UNIMOD_CuSMo = 300000444,

            /// <summary>5-hydroxy-N6,N6,N6-trimethyl.</summary>
            UNIMOD_Hydroxytrimethyl = 300000445,

            /// <summary>Reduction.</summary>
            UNIMOD_Deoxy = 300000447,

            /// <summary>Microcin E492 siderophore ester from serine.</summary>
            UNIMOD_Microcin = 300000448,

            /// <summary>Lipid.</summary>
            UNIMOD_Decanoyl = 300000449,

            /// <summary>Monoglutamyl.</summary>
            UNIMOD_Glu = 300000450,

            /// <summary>Diglutamyl.</summary>
            UNIMOD_GluGlu = 300000451,

            /// <summary>Triglutamyl.</summary>
            UNIMOD_GluGluGlu = 300000452,

            /// <summary>Tetraglutamyl.</summary>
            UNIMOD_GluGluGluGlu = 300000453,

            /// <summary>Hexosamine.</summary>
            UNIMOD_HexN = 300000454,

            /// <summary>Free monolink of DMP crosslinker.</summary>
            UNIMOD_Xlink_DMP_154_ = 300000455,

            /// <summary>Naphthalene-2,3-dicarboxaldehyde.</summary>
            UNIMOD_NDA = 300000457,

            /// <summary>4-sulfophenyl isothiocyanate (Heavy C13).</summary>
            UNIMOD_SPITC_13C_6_ = 300000464,

            /// <summary>Aminoethylcysteine.</summary>
            UNIMOD_AEC_MAEC = 300000472,

            /// <summary>4-trimethyllammoniumbutyryl-.</summary>
            UNIMOD_TMAB = 300000476,

            /// <summary>D9-4-trimethyllammoniumbutyryl-.</summary>
            UNIMOD_TMAB_2H_9_ = 300000477,

            /// <summary>Fluorescein-5-thiosemicarbazide.</summary>
            UNIMOD_FTC = 300000478,

            /// <summary>4,4,5,5-D4 Lysine.</summary>
            UNIMOD_Label_2H_4_ = 300000481,

            /// <summary>Dehydropyrrolizidine alkaloid (dehydroretronecine) on cysteines.</summary>
            UNIMOD_DHP = 300000488,

            /// <summary>Heptose.</summary>
            UNIMOD_Hep = 300000490,

            /// <summary>Bisphenol A diglycidyl ether derivative.</summary>
            UNIMOD_BADGE = 300000493,

            /// <summary>Cy3 CyDye DIGE Fluor saturation dye.</summary>
            UNIMOD_CyDye_Cy3 = 300000494,

            /// <summary>Cy5 CyDye DIGE Fluor saturation dye.</summary>
            UNIMOD_CyDye_Cy5 = 300000495,

            /// <summary>Michael addition of t-butyl hydroxylated BHT (BHTOH) to C, H or K.</summary>
            UNIMOD_BHTOH = 300000498,

            /// <summary>Heavy IDBEST tag for quantitation.</summary>
            UNIMOD_IGBP_13C_2_ = 300000499,

            /// <summary>Nmethylmaleimidehydrolysis.</summary>
            UNIMOD_Nmethylmaleimide_water = 300000500,

            /// <summary>3-methyl-2-pyridyl isocyanate.</summary>
            UNIMOD_PyMIC = 300000501,

            /// <summary>Levuglandinyl - lysine lactam adduct.</summary>
            UNIMOD_LG_lactam_K = 300000503,

            /// <summary>Levuglandinyl - lysine hydroxylactam adduct.</summary>
            UNIMOD_LG_Hlactam_K = 300000504,

            /// <summary>Levuglandinyl - arginine lactam adduct.</summary>
            UNIMOD_LG_lactam_R = 300000505,

            /// <summary>Levuglandinyl - arginine hydroxylactam adduct.</summary>
            UNIMOD_LG_Hlactam_R = 300000506,

            /// <summary>DiMethyl-C13HD2.</summary>
            UNIMOD_Dimethyl_2H_4_13C_2_ = 300000510,

            /// <summary>Lactosylation.</summary>
            UNIMOD_Hex_2_ = 300000512,

            /// <summary></summary>
            UNIMOD_C8_QAT = 300000513,

            /// <summary>Propyl-1,2-dideoxy-2\'-methyl-alpha-D-glucopyranoso-</summary>
            UNIMOD_PropylNAGthiazoline = 300000514,

            /// <summary>Fluorescein-5-maleimide.</summary>
            UNIMOD_FNEM = 300000515,

            /// <summary>Diethylation, analogous to Dimethylation.</summary>
            UNIMOD_Diethyl = 300000518,

            /// <summary>4,4\'-dianilino-1,1\'-binaphthyl-5,5\'-disulfonic acid.</summary>
            UNIMOD_BisANS = 300000519,

            /// <summary>Piperidination.</summary>
            UNIMOD_Piperidine = 300000520,

            /// <summary>Maleimide-Biotin.</summary>
            UNIMOD_Maleimide_PEO2_Biotin = 300000522,

            /// <summary>Biot_LC_LC.</summary>
            UNIMOD_Sulfo_NHS_LC_LC_Biotin = 300000523,

            /// <summary>CLIP_TRAQ_2.</summary>
            UNIMOD_CLIP_TRAQ_2 = 300000525,

            /// <summary>Prompt loss of side chain from oxidised Met.</summary>
            UNIMOD_Dethiomethyl = 300000526,

            /// <summary>Deamidation followed by a methylation.</summary>
            UNIMOD_Methyl_Deamidated = 300000528,

            /// <summary>Dimethylation of proline residue.</summary>
            UNIMOD_Delta_H_5_C_2_ = 300000529,

            /// <summary>Replacement of proton by potassium.</summary>
            UNIMOD_Cation_K = 300000530,

            /// <summary>Replacement of proton by copper.</summary>
            UNIMOD_Cation_Cu_I_ = 300000531,

            /// <summary>Accurate mass for 114.</summary>
            UNIMOD_iTRAQ4plex114 = 300000532,

            /// <summary>Accurate mass for 115.</summary>
            UNIMOD_iTRAQ4plex115 = 300000533,

            /// <summary>Dibromo.</summary>
            UNIMOD_Dibromo = 300000534,

            /// <summary>Ubiquitination.</summary>
            UNIMOD_LRGG = 300000535,

            /// <summary>CLIP_TRAQ_3.</summary>
            UNIMOD_CLIP_TRAQ_3 = 300000536,

            /// <summary>CLIP_TRAQ_4.</summary>
            UNIMOD_CLIP_TRAQ_4 = 300000537,

            /// <summary>Was 15dB-biotin.</summary>
            UNIMOD_Biotin_Cayman_10141 = 300000538,

            /// <summary>Was PGA1-biotin.</summary>
            UNIMOD_Biotin_Cayman_10013 = 300000539,

            /// <summary>Ala-&gt;Ser substitution.</summary>
            UNIMOD_Ala__Ser = 300000540,

            /// <summary>Ala-&gt;Thr substitution.</summary>
            UNIMOD_Ala__Thr = 300000541,

            /// <summary>Ala-&gt;Asp substitution.</summary>
            UNIMOD_Ala__Asp = 300000542,

            /// <summary>Ala-&gt;Pro substitution.</summary>
            UNIMOD_Ala__Pro = 300000543,

            /// <summary>Ala-&gt;Gly substitution.</summary>
            UNIMOD_Ala__Gly = 300000544,

            /// <summary>Ala-&gt;Glu substitution.</summary>
            UNIMOD_Ala__Glu = 300000545,

            /// <summary>Ala-&gt;Val substitution.</summary>
            UNIMOD_Ala__Val = 300000546,

            /// <summary>Cys-&gt;Phe substitution.</summary>
            UNIMOD_Cys__Phe = 300000547,

            /// <summary>Cys-&gt;Ser substitution.</summary>
            UNIMOD_Cys__Ser = 300000548,

            /// <summary>Cys-&gt;Trp substitution.</summary>
            UNIMOD_Cys__Trp = 300000549,

            /// <summary>Cys-&gt;Tyr substitution.</summary>
            UNIMOD_Cys__Tyr = 300000550,

            /// <summary>Cys-&gt;Arg substitution.</summary>
            UNIMOD_Cys__Arg = 300000551,

            /// <summary>Cys-&gt;Gly substitution.</summary>
            UNIMOD_Cys__Gly = 300000552,

            /// <summary>Asp-&gt;Ala substitution.</summary>
            UNIMOD_Asp__Ala = 300000553,

            /// <summary>Asp-&gt;His substitution.</summary>
            UNIMOD_Asp__His = 300000554,

            /// <summary>Asp-&gt;Asn substitution.</summary>
            UNIMOD_Asp__Asn = 300000555,

            /// <summary>Asp-&gt;Gly substitution.</summary>
            UNIMOD_Asp__Gly = 300000556,

            /// <summary>Asp-&gt;Tyr substitution.</summary>
            UNIMOD_Asp__Tyr = 300000557,

            /// <summary>Asp-&gt;Glu substitution.</summary>
            UNIMOD_Asp__Glu = 300000558,

            /// <summary>Asp-&gt;Val substitution.</summary>
            UNIMOD_Asp__Val = 300000559,

            /// <summary>Glu-&gt;Ala substitution.</summary>
            UNIMOD_Glu__Ala = 300000560,

            /// <summary>Glu-&gt;Gln substitution.</summary>
            UNIMOD_Glu__Gln = 300000561,

            /// <summary>Glu-&gt;Asp substitution.</summary>
            UNIMOD_Glu__Asp = 300000562,

            /// <summary>Glu-&gt;Lys substitution.</summary>
            UNIMOD_Glu__Lys = 300000563,

            /// <summary>Glu-&gt;Gly substitution.</summary>
            UNIMOD_Glu__Gly = 300000564,

            /// <summary>Glu-&gt;Val substitution.</summary>
            UNIMOD_Glu__Val = 300000565,

            /// <summary>Phe-&gt;Ser substitution.</summary>
            UNIMOD_Phe__Ser = 300000566,

            /// <summary>Phe-&gt;Cys substitution.</summary>
            UNIMOD_Phe__Cys = 300000567,

            /// <summary>Phe-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Phe__Xle = 300000568,

            /// <summary>Phe-&gt;Tyr substitution.</summary>
            UNIMOD_Phe__Tyr = 300000569,

            /// <summary>Phe-&gt;Val substitution.</summary>
            UNIMOD_Phe__Val = 300000570,

            /// <summary>Gly-&gt;Ala substitution.</summary>
            UNIMOD_Gly__Ala = 300000571,

            /// <summary>Gly-&gt;Ser substitution.</summary>
            UNIMOD_Gly__Ser = 300000572,

            /// <summary>Gly-&gt;Trp substitution.</summary>
            UNIMOD_Gly__Trp = 300000573,

            /// <summary>Gly-&gt;Glu substitution.</summary>
            UNIMOD_Gly__Glu = 300000574,

            /// <summary>Gly-&gt;Val substitution.</summary>
            UNIMOD_Gly__Val = 300000575,

            /// <summary>Gly-&gt;Asp substitution.</summary>
            UNIMOD_Gly__Asp = 300000576,

            /// <summary>Gly-&gt;Cys substitution.</summary>
            UNIMOD_Gly__Cys = 300000577,

            /// <summary>Gly-&gt;Arg substitution.</summary>
            UNIMOD_Gly__Arg = 300000578,

            /// <summary>His-&gt;Pro substitution.</summary>
            UNIMOD_His__Pro = 300000580,

            /// <summary>His-&gt;Tyr substitution.</summary>
            UNIMOD_His__Tyr = 300000581,

            /// <summary>His-&gt;Gln substitution.</summary>
            UNIMOD_His__Gln = 300000582,

            /// <summary>His-&gt;Arg substitution.</summary>
            UNIMOD_His__Arg = 300000584,

            /// <summary>His-&gt;Leu/Ile substitution.</summary>
            UNIMOD_His__Xle = 300000585,

            /// <summary>Leu/Ile-&gt;Thr substitution.</summary>
            UNIMOD_Xle__Thr = 300000588,

            /// <summary>Leu/Ile-&gt;Asn substitution.</summary>
            UNIMOD_Xle__Asn = 300000589,

            /// <summary>Leu/Ile-&gt;Lys substitution.</summary>
            UNIMOD_Xle__Lys = 300000590,

            /// <summary>Lys-&gt;Thr substitution.</summary>
            UNIMOD_Lys__Thr = 300000594,

            /// <summary>Lys-&gt;Asn substitution.</summary>
            UNIMOD_Lys__Asn = 300000595,

            /// <summary>Lys-&gt;Glu substitution.</summary>
            UNIMOD_Lys__Glu = 300000596,

            /// <summary>Lys-&gt;Gln substitution.</summary>
            UNIMOD_Lys__Gln = 300000597,

            /// <summary>Lys-&gt;Met substitution.</summary>
            UNIMOD_Lys__Met = 300000598,

            /// <summary>Lys-&gt;Arg substitution.</summary>
            UNIMOD_Lys__Arg = 300000599,

            /// <summary>Lys-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Lys__Xle = 300000600,

            /// <summary>Leu/Ile-&gt;Ser substitution.</summary>
            UNIMOD_Xle__Ser = 300000601,

            /// <summary>Leu/Ile-&gt;Phe substitution.</summary>
            UNIMOD_Xle__Phe = 300000602,

            /// <summary>Leu/Ile-&gt;Trp substitution.</summary>
            UNIMOD_Xle__Trp = 300000603,

            /// <summary>Leu/Ile-&gt;Pro substitution.</summary>
            UNIMOD_Xle__Pro = 300000604,

            /// <summary>Leu/Ile-&gt;Val substitution.</summary>
            UNIMOD_Xle__Val = 300000605,

            /// <summary>Leu/Ile-&gt;His substitution.</summary>
            UNIMOD_Xle__His = 300000606,

            /// <summary>Leu/Ile-&gt;Gln substitution.</summary>
            UNIMOD_Xle__Gln = 300000607,

            /// <summary>Leu/Ile-&gt;Met substitution.</summary>
            UNIMOD_Xle__Met = 300000608,

            /// <summary>Leu/Ile-&gt;Arg substitution.</summary>
            UNIMOD_Xle__Arg = 300000609,

            /// <summary>Met-&gt;Thr substitution.</summary>
            UNIMOD_Met__Thr = 300000610,

            /// <summary>Met-&gt;Arg substitution.</summary>
            UNIMOD_Met__Arg = 300000611,

            /// <summary>Met-&gt;Lys substitution.</summary>
            UNIMOD_Met__Lys = 300000613,

            /// <summary>Met-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Met__Xle = 300000614,

            /// <summary>Met-&gt;Val substitution.</summary>
            UNIMOD_Met__Val = 300000615,

            /// <summary>Asn-&gt;Ser substitution.</summary>
            UNIMOD_Asn__Ser = 300000616,

            /// <summary>Asn-&gt;Thr substitution.</summary>
            UNIMOD_Asn__Thr = 300000617,

            /// <summary>Asn-&gt;Lys substitution.</summary>
            UNIMOD_Asn__Lys = 300000618,

            /// <summary>Asn-&gt;Tyr substitution.</summary>
            UNIMOD_Asn__Tyr = 300000619,

            /// <summary>Asn-&gt;His substitution.</summary>
            UNIMOD_Asn__His = 300000620,

            /// <summary>Asn-&gt;Asp substitution.</summary>
            UNIMOD_Asn__Asp = 300000621,

            /// <summary>Asn-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Asn__Xle = 300000622,

            /// <summary>Pro-&gt;Ser substitution.</summary>
            UNIMOD_Pro__Ser = 300000623,

            /// <summary>Pro-&gt;Ala substitution.</summary>
            UNIMOD_Pro__Ala = 300000624,

            /// <summary>Pro-&gt;His substitution.</summary>
            UNIMOD_Pro__His = 300000625,

            /// <summary>Pro-&gt;Gln substitution.</summary>
            UNIMOD_Pro__Gln = 300000626,

            /// <summary>Pro-&gt;Thr substitution.</summary>
            UNIMOD_Pro__Thr = 300000627,

            /// <summary>Pro-&gt;Arg substitution.</summary>
            UNIMOD_Pro__Arg = 300000628,

            /// <summary>Pro-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Pro__Xle = 300000629,

            /// <summary>Gln-&gt;Pro substitution.</summary>
            UNIMOD_Gln__Pro = 300000630,

            /// <summary>Gln-&gt;Lys substitution.</summary>
            UNIMOD_Gln__Lys = 300000631,

            /// <summary>Gln-&gt;Glu substitution.</summary>
            UNIMOD_Gln__Glu = 300000632,

            /// <summary>Gln-&gt;His substitution.</summary>
            UNIMOD_Gln__His = 300000633,

            /// <summary>Gln-&gt;Arg substitution.</summary>
            UNIMOD_Gln__Arg = 300000634,

            /// <summary>Gln-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Gln__Xle = 300000635,

            /// <summary>Arg-&gt;Ser substitution.</summary>
            UNIMOD_Arg__Ser = 300000636,

            /// <summary>Arg-&gt;Trp substitution.</summary>
            UNIMOD_Arg__Trp = 300000637,

            /// <summary>Arg-&gt;Thr substitution.</summary>
            UNIMOD_Arg__Thr = 300000638,

            /// <summary>Arg-&gt;Pro substitution.</summary>
            UNIMOD_Arg__Pro = 300000639,

            /// <summary>Arg-&gt;Lys substitution.</summary>
            UNIMOD_Arg__Lys = 300000640,

            /// <summary>Arg-&gt;His substitution.</summary>
            UNIMOD_Arg__His = 300000641,

            /// <summary>Arg-&gt;Gln substitution.</summary>
            UNIMOD_Arg__Gln = 300000642,

            /// <summary>Arg-&gt;Met substitution.</summary>
            UNIMOD_Arg__Met = 300000643,

            /// <summary>Arg-&gt;Cys substitution.</summary>
            UNIMOD_Arg__Cys = 300000644,

            /// <summary>Arg-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Arg__Xle = 300000645,

            /// <summary>Arg-&gt;Gly substitution.</summary>
            UNIMOD_Arg__Gly = 300000646,

            /// <summary>Ser-&gt;Phe substitution.</summary>
            UNIMOD_Ser__Phe = 300000647,

            /// <summary>Ser-&gt;Ala substitution.</summary>
            UNIMOD_Ser__Ala = 300000648,

            /// <summary>Ser-&gt;Trp substitution.</summary>
            UNIMOD_Ser__Trp = 300000649,

            /// <summary>Ser-&gt;Thr substitution.</summary>
            UNIMOD_Ser__Thr = 300000650,

            /// <summary>Ser-&gt;Asn substitution.</summary>
            UNIMOD_Ser__Asn = 300000651,

            /// <summary>Ser-&gt;Pro substitution.</summary>
            UNIMOD_Ser__Pro = 300000652,

            /// <summary>Ser-&gt;Tyr substitution.</summary>
            UNIMOD_Ser__Tyr = 300000653,

            /// <summary>Ser-&gt;Cys substitution.</summary>
            UNIMOD_Ser__Cys = 300000654,

            /// <summary>Ser-&gt;Arg substitution.</summary>
            UNIMOD_Ser__Arg = 300000655,

            /// <summary>Ser-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Ser__Xle = 300000656,

            /// <summary>Ser-&gt;Gly substitution.</summary>
            UNIMOD_Ser__Gly = 300000657,

            /// <summary>Thr-&gt;Ser substitution.</summary>
            UNIMOD_Thr__Ser = 300000658,

            /// <summary>Thr-&gt;Ala substitution.</summary>
            UNIMOD_Thr__Ala = 300000659,

            /// <summary>Thr-&gt;Asn substitution.</summary>
            UNIMOD_Thr__Asn = 300000660,

            /// <summary>Thr-&gt;Lys substitution.</summary>
            UNIMOD_Thr__Lys = 300000661,

            /// <summary>Thr-&gt;Pro substitution.</summary>
            UNIMOD_Thr__Pro = 300000662,

            /// <summary>Thr-&gt;Met substitution.</summary>
            UNIMOD_Thr__Met = 300000663,

            /// <summary>Thr-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Thr__Xle = 300000664,

            /// <summary>Thr-&gt;Arg substitution.</summary>
            UNIMOD_Thr__Arg = 300000665,

            /// <summary>Val-&gt;Phe substitution.</summary>
            UNIMOD_Val__Phe = 300000666,

            /// <summary>Val-&gt;Ala substitution.</summary>
            UNIMOD_Val__Ala = 300000667,

            /// <summary>Val-&gt;Glu substitution.</summary>
            UNIMOD_Val__Glu = 300000668,

            /// <summary>Val-&gt;Met substitution.</summary>
            UNIMOD_Val__Met = 300000669,

            /// <summary>Val-&gt;Asp substitution.</summary>
            UNIMOD_Val__Asp = 300000670,

            /// <summary>Val-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Val__Xle = 300000671,

            /// <summary>Val-&gt;Gly substitution.</summary>
            UNIMOD_Val__Gly = 300000672,

            /// <summary>Trp-&gt;Ser substitution.</summary>
            UNIMOD_Trp__Ser = 300000673,

            /// <summary>Trp-&gt;Cys substitution.</summary>
            UNIMOD_Trp__Cys = 300000674,

            /// <summary>Trp-&gt;Arg substitution.</summary>
            UNIMOD_Trp__Arg = 300000675,

            /// <summary>Trp-&gt;Gly substitution.</summary>
            UNIMOD_Trp__Gly = 300000676,

            /// <summary>Trp-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Trp__Xle = 300000677,

            /// <summary>Tyr-&gt;Phe substitution.</summary>
            UNIMOD_Tyr__Phe = 300000678,

            /// <summary>Tyr-&gt;Ser substitution.</summary>
            UNIMOD_Tyr__Ser = 300000679,

            /// <summary>Tyr-&gt;Asn substitution.</summary>
            UNIMOD_Tyr__Asn = 300000680,

            /// <summary>Tyr-&gt;His substitution.</summary>
            UNIMOD_Tyr__His = 300000681,

            /// <summary>Tyr-&gt;Asp substitution.</summary>
            UNIMOD_Tyr__Asp = 300000682,

            /// <summary>Tyr-&gt;Cys substitution.</summary>
            UNIMOD_Tyr__Cys = 300000683,

            /// <summary>Mass Defect Tag on lysine e-amino.</summary>
            UNIMOD_BDMAPP = 300000684,

            /// <summary>Nitroalkylation by Nitro Linoleic Acid.</summary>
            UNIMOD_NA_LNO2 = 300000685,

            /// <summary>Nitroalkylation by Nitro Oleic Acid.</summary>
            UNIMOD_NA_OA_NO2 = 300000686,

            /// <summary>Bruker Daltonics SERVA-ICPL(TM) quantification chemistry, medium form.</summary>
            UNIMOD_ICPL_2H_4_ = 300000687,

            /// <summary>13C(6) 15N(1) Silac label.</summary>
            UNIMOD_Label_13C_6_15N_1_ = 300000695,

            /// <summary>13C(6) 15N(2) (D)9 SILAC label.</summary>
            UNIMOD_Label_2H_9_13C_6_15N_2_ = 300000696,

            /// <summary>Nicotinic Acid.</summary>
            UNIMOD_NIC = 300000697,

            /// <summary>Deuterated Nicotinic Acid.</summary>
            UNIMOD_dNIC = 300000698,

            /// <summary>Dehydrated 4-hydroxynonenal.</summary>
            UNIMOD_HNE_Delta_H_2_O = 300000720,

            /// <summary>4-Oxononenal (ONE).</summary>
            UNIMOD_4_ONE = 300000721,

            /// <summary>O-Dimethylphosphorylation.</summary>
            UNIMOD_O_Dimethylphosphate = 300000723,

            /// <summary>O-Methylphosphorylation.</summary>
            UNIMOD_O_Methylphosphate = 300000724,

            /// <summary>O-Diethylphosphorylation.</summary>
            UNIMOD_Diethylphosphate = 300000725,

            /// <summary>O-Ethylphosphorylation.</summary>
            UNIMOD_Ethylphosphate = 300000726,

            /// <summary>O-pinacolylmethylphosphonylation.</summary>
            UNIMOD_O_pinacolylmethylphosphonate = 300000727,

            /// <summary>Methylphosphonylation.</summary>
            UNIMOD_Methylphosphonate = 300000728,

            /// <summary>O-Isopropylmethylphosphonylation.</summary>
            UNIMOD_O_Isopropylmethylphosphonate = 300000729,

            /// <summary>Representative mass and accurate mass for 113, 114, 116 &amp; 117.</summary>
            UNIMOD_iTRAQ8plex = 300000730,

            /// <summary>Accurate mass for 115, 118, 119 &amp; 121.</summary>
            UNIMOD_iTRAQ8plex_13C_6_15N_2_ = 300000731,

            /// <summary>Carboxyl modification with ethanolamine.</summary>
            UNIMOD_Ethanolamine = 300000734,

            /// <summary>Beta elimination of modified S or T followed by Michael addition of DTT.</summary>
            UNIMOD_BEMAD_ST = 300000735,

            /// <summary>Beta elimination of alkylated Cys followed by Michael addition of DTT.</summary>
            UNIMOD_BEMAD_C = 300000736,

            /// <summary>Sixplex Tandem Mass Tag.</summary>
            UNIMOD_TMT6plex = 300000737,

            /// <summary>Duplex Tandem Mass Tag.</summary>
            UNIMOD_TMT2plex = 300000738,

            /// <summary>Native Tandem Mass Tag.</summary>
            UNIMOD_TMT = 300000739,

            /// <summary>ExacTag Thiol label mass for 2-4-7-10 plex.</summary>
            UNIMOD_ExacTagThiol = 300000740,

            /// <summary>ExacTag Amine label mass for 2-4-7-10 plex.</summary>
            UNIMOD_ExacTagAmine = 300000741,

            /// <summary>Dehydrated 4-Oxononenal Michael adduct.</summary>
            UNIMOD_4_ONE_Delta_H__2_O__1_ = 300000743,

            /// <summary>Nitroso Sulfamethoxazole Sulphenamide thiol adduct.</summary>
            UNIMOD_NO_SMX_SEMD = 300000744,

            /// <summary>Nitroso Sulfamethoxazole Sulfinamide thiol adduct.</summary>
            UNIMOD_NO_SMX_SIMD = 300000746,

            /// <summary>Malonylation.</summary>
            UNIMOD_Malonyl = 300000747,

            /// <summary>Derivatization by N-term modification using 3-Sulfobenzoic succinimidyl ester.</summary>
            UNIMOD_3sulfo = 300000748,

            /// <summary>Trifluoroleucine replacement of leucine.</summary>
            UNIMOD_trifluoro = 300000750,

            /// <summary>Tri nitro benzene.</summary>
            UNIMOD_TNBS = 300000751,

            /// <summary>Isotope Distribution Encoded Tag.</summary>
            UNIMOD_IDEnT = 300000762,

            /// <summary>Beta elimination of modified S or T followed by Michael addition of labelled DTT.</summary>
            UNIMOD_BEMAD_ST_2H_6_ = 300000763,

            /// <summary>Beta elimination of alkylated Cys followed by Michael addition of labelled DTT.</summary>
            UNIMOD_BEMAD_C_2H_6_ = 300000764,

            /// <summary>Removal of initiator methionine from protein N-terminus.</summary>
            UNIMOD_Met_loss = 300000765,

            /// <summary>Removal of initiator methionine from protein N-terminus, then acetylation of the new N-terminus.</summary>
            UNIMOD_Met_loss_Acetyl = 300000766,

            /// <summary>Menadione hydroquinone derivative.</summary>
            UNIMOD_Menadione_HQ = 300000767,

            /// <summary>Mono-methylated lysine labelled with Acetyl_heavy.</summary>
            UNIMOD_Methyl_Acetyl_2H_3_ = 300000768,

            /// <summary>Lapachenole photochemically added to cysteine.</summary>
            UNIMOD_lapachenole = 300000771,

            /// <summary>13C(5) Silac label.</summary>
            UNIMOD_Label_13C_5_ = 300000772,

            /// <summary>Maleimide.</summary>
            UNIMOD_maleimide = 300000773,

            /// <summary>Alkylation by biotinylated form of phenacyl bromide.</summary>
            UNIMOD_Biotin_phenacyl = 300000774,

            /// <summary>Iodoacetic acid derivative w/ 13C label.</summary>
            UNIMOD_Carboxymethyl_13C_2_ = 300000775,

            /// <summary>D5 N-ethylmaleimide on cysteines.</summary>
            UNIMOD_NEM_2H_5_ = 300000776,

            /// <summary>Deuterium cysteamine modification to S or T.</summary>
            UNIMOD_AEC_MAEC_2H_4_ = 300000792,

            /// <summary>Hex1HexNAc1.</summary>
            UNIMOD_Hex_1_HexNAc_1_ = 300000793,

            /// <summary>13C6 labeled ubiquitinylation residue.</summary>
            UNIMOD_Label_13C_6__GG = 300000799,

            /// <summary>Was PentylamineBiotin.</summary>
            UNIMOD_Biotin_Thermo_21345 = 300000800,

            /// <summary>Labeling transglutaminase substrate on glutamine side chain.</summary>
            UNIMOD_Pentylamine = 300000801,

            /// <summary>Was Biotin-PEO4-hydrazide.</summary>
            UNIMOD_Biotin_Thermo_21360 = 300000811,

            /// <summary>Fluorescent dye that labels cysteines.</summary>
            UNIMOD_Cy3b_maleimide = 300000821,

            /// <summary>Enzymatic glycine removal leaving an amidated C-terminus.</summary>
            UNIMOD_Gly_loss_Amide = 300000822,

            /// <summary>Intact or monolink BMOE crosslinker.</summary>
            UNIMOD_Xlink_BMOE = 300000824,

            /// <summary>Intact DFDNB crosslinker.</summary>
            UNIMOD_Xlink_DFDNB = 300000825,

            /// <summary>Tris(2,4,6-trimethoxyphenyl)phosphonium acetic acid N-hydroxysuccinimide ester derivative.</summary>
            UNIMOD_TMPP_Ac = 300000827,

            /// <summary>Dihydroxy methylglyoxal adduct.</summary>
            UNIMOD_Dihydroxyimidazolidine = 300000830,

            /// <summary>Acetyl 4,4,5,5-D4 Lysine.</summary>
            UNIMOD_Label_2H_4__Acetyl = 300000834,

            /// <summary>Acetyl 13C(6) Silac label.</summary>
            UNIMOD_Label_13C_6__Acetyl = 300000835,

            /// <summary>Acetyl_13C(6) 15N(2) Silac label.</summary>
            UNIMOD_Label_13C_6_15N_2__Acetyl = 300000836,

            /// <summary>Arginine replacement by Nitropyrimidyl ornithine.</summary>
            UNIMOD_Arg__Npo = 300000837,

            /// <summary>Sumo mutant Smt3-WT tail following trypsin digestion.</summary>
            UNIMOD_EQIGG = 300000846,

            /// <summary>Adduct of phenylglyoxal with Arg.</summary>
            UNIMOD_Arg2PG = 300000848,

            /// <summary>S-guanylation.</summary>
            UNIMOD_cGMP = 300000849,

            /// <summary>S-guanylation-2.</summary>
            UNIMOD_cGMP_RMP_loss = 300000851,

            /// <summary>Ubiquitination 2H4 lysine.</summary>
            UNIMOD_Label_2H_4__GG = 300000853,

            /// <summary>Methylglyoxal-derived hydroimidazolone.</summary>
            UNIMOD_MG_H1 = 300000859,

            /// <summary>Glyoxal-derived hydroimiadazolone.</summary>
            UNIMOD_G_H1 = 300000860,

            /// <summary>NHS ester linked Green Fluorescent Bodipy Dye.</summary>
            UNIMOD_ZGB = 300000861,

            /// <summary>SILAC.</summary>
            UNIMOD_Label_13C_1_2H_3_ = 300000862,

            /// <summary>13C(6) 15N(2) Lysine glygly.</summary>
            UNIMOD_Label_13C_6_15N_2__GG = 300000864,

            /// <summary>Bruker Daltonics SERVA-ICPL(TM) quantification chemistry, +10 Da form.</summary>
            UNIMOD_ICPL_13C_6_2H_4_ = 300000866,

            /// <summary>SUMOylation by SUMO-1.</summary>
            UNIMOD_QEQTGG = 300000876,

            /// <summary>SUMOylation by SUMO-2/3.</summary>
            UNIMOD_QQQTGG = 300000877,

            /// <summary>Was ChromoBiotin.</summary>
            UNIMOD_Biotin_Thermo_21325 = 300000884,

            /// <summary>Oxidised methionine 13C(1)2H(3) SILAC label.</summary>
            UNIMOD_Label_13C_1_2H_3__Oxidation = 300000885,

            /// <summary>2-ammonio-6-</summary>
            UNIMOD_HydroxymethylOP = 300000886,

            /// <summary>Covalent linkage of maleimidyl coumarin probe (Molecular Probes D-10253).</summary>
            UNIMOD_MDCC = 300000887,

            /// <summary>MTRAQ light.</summary>
            UNIMOD_mTRAQ = 300000888,

            /// <summary>MTRAQ medium.</summary>
            UNIMOD_mTRAQ_13C_3_15N_1_ = 300000889,

            /// <summary>Thiol-reactive dye for fluorescence labelling of proteins.</summary>
            UNIMOD_DyLight_maleimide = 300000890,

            /// <summary>Methyl-PEO12-Maleimide.</summary>
            UNIMOD_Methyl_PEO12_Maleimide = 300000891,

            /// <summary>Carbamidomethylated DTT modification of cysteine.</summary>
            UNIMOD_CarbamidomethylDTT = 300000893,

            /// <summary>Carboxymethylated DTT modification of cysteine.</summary>
            UNIMOD_CarboxymethylDTT = 300000894,

            /// <summary>Biotin polyethyleneoxide (n=3) alkyne.</summary>
            UNIMOD_Biotin_PEG_PRA = 300000895,

            /// <summary>Methionine replacement by azido homoalanine.</summary>
            UNIMOD_Met__Aha = 300000896,

            /// <summary>SILAC 15N(4).</summary>
            UNIMOD_Label_15N_4_ = 300000897,

            /// <summary>Pyrophosphorylation of Ser/Thr.</summary>
            UNIMOD_pyrophospho = 300000898,

            /// <summary>Methionine replacement by homopropargylglycine.</summary>
            UNIMOD_Met__Hpg = 300000899,

            /// <summary>2,3,4,6-tetra-O-Acetyl-1-allyl-alpha-D-galactopyranoside modification of cysteine.</summary>
            UNIMOD_4AcAllylGal = 300000901,

            /// <summary>Reaction with dimethylarsinous (AsIII) acid.</summary>
            UNIMOD_DimethylArsino = 300000902,

            /// <summary>Lys-&gt;Cys substitution and carbamidomethylation.</summary>
            UNIMOD_Lys__CamCys = 300000903,

            /// <summary>Phe-&gt;Cys substitution and carbamidomethylation.</summary>
            UNIMOD_Phe__CamCys = 300000904,

            /// <summary>Leu-&gt;Met substitution and sulfoxidation.</summary>
            UNIMOD_Leu__MetOx = 300000905,

            /// <summary>Lys-&gt;Met substitution and sulfoxidation.</summary>
            UNIMOD_Lys__MetOx = 300000906,

            /// <summary>Gluconoylation.</summary>
            UNIMOD_Galactosyl = 300000907,

            /// <summary>Monolink of SMCC terminated with 3-(dimethylamino)-1-propylamine.</summary>
            UNIMOD_Xlink_SMCC_321_ = 300000908,

            /// <summary>2,4-diacetamido-2,4,6-trideoxyglucopyranose.</summary>
            UNIMOD_Bacillosamine = 300000910,

            /// <summary>Cys modification by (1-oxyl-2,2,5,5-tetramethyl-3-pyrroline-3-methyl)methanesulfonate (MTSL).</summary>
            UNIMOD_MTSL = 300000911,

            /// <summary>4-hydroxy-2-nonenal and biotinamidohexanoic acid hydrazide, reduced.</summary>
            UNIMOD_HNE_BAHAH = 300000912,

            /// <summary>Methylmalonylation on Serine.</summary>
            UNIMOD_Methylmalonylation = 300000914,

            /// <summary>13C(4) 15N(2) Lysine glygly.</summary>
            UNIMOD_Label_13C_4_15N_2__GG = 300000923,

            /// <summary>Ethyl amino.</summary>
            UNIMOD_ethylamino = 300000926,

            /// <summary>2-OH-ethyl thio-Ser.</summary>
            UNIMOD_MercaptoEthanol = 300000928,

            /// <summary>Deamidation followed by esterification with ethanol.</summary>
            UNIMOD_Ethyl_Deamidated = 300000931,

            /// <summary>SUMOylation by SUMO-2/3 (formic acid cleavage).</summary>
            UNIMOD_VFQQQTGG = 300000932,

            /// <summary>SUMOylation by SUMO-1 (formic acid cleavage).</summary>
            UNIMOD_VIEVYQEQTGG = 300000933,

            /// <summary>Photocleavable Biotin + GalNAz on O-GlcNAc.</summary>
            UNIMOD_AMTzHexNAc2 = 300000934,

            /// <summary>High molecular absorption maleimide label for proteins.</summary>
            UNIMOD_Atto495Maleimide = 300000935,

            /// <summary>Chlorination of tyrosine residues.</summary>
            UNIMOD_Chlorination = 300000936,

            /// <summary>Dichlorination.</summary>
            UNIMOD_dichlorination = 300000937,

            /// <summary>Cysteine modifier.</summary>
            UNIMOD_AROD = 300000938,

            /// <summary>Carbamidomethylated Cys that undergoes beta-elimination and Michael addition of methylamine.</summary>
            UNIMOD_Cys__methylaminoAla = 300000939,

            /// <summary>Carbamidomethylated Cys that undergoes beta-elimination and Michael addition of ethylamine.</summary>
            UNIMOD_Cys__ethylaminoAla = 300000940,

            /// <summary>2,4-Dinitrobenzenesulfenyl.</summary>
            UNIMOD_DNPS = 300000941,

            /// <summary>High molecular absorption label for proteins.</summary>
            UNIMOD_SulfoGMBS = 300000942,

            /// <summary>Modified GMBS X linker.</summary>
            UNIMOD_DimethylamineGMBS = 300000943,

            /// <summary>SILAC label.</summary>
            UNIMOD_Label_15N_2_2H_9_ = 300000944,

            /// <summary>Levuglandinyl-lysine anhydrolactam adduct.</summary>
            UNIMOD_LG_anhydrolactam = 300000946,

            /// <summary>Levuglandinyl-lysine pyrrole adduct.</summary>
            UNIMOD_LG_pyrrole = 300000947,

            /// <summary>Levuglandinyl-lysine anhyropyrrole adduct.</summary>
            UNIMOD_LG_anhyropyrrole = 300000948,

            /// <summary>Condensation product of 3-deoxyglucosone.</summary>
            UNIMOD_3_deoxyglucosone = 300000949,

            /// <summary>Replacement of proton by lithium.</summary>
            UNIMOD_Cation_Li = 300000950,

            /// <summary>Replacement of 2 protons by calcium.</summary>
            UNIMOD_Cation_Ca_II_ = 300000951,

            /// <summary>Replacement of 2 protons by iron.</summary>
            UNIMOD_Cation_Fe_II_ = 300000952,

            /// <summary>Replacement of 2 protons by nickel.</summary>
            UNIMOD_Cation_Ni_II_ = 300000953,

            /// <summary>Replacement of 2 protons by zinc.</summary>
            UNIMOD_Cation_Zn_II_ = 300000954,

            /// <summary>Replacement of proton by silver.</summary>
            UNIMOD_Cation_Ag = 300000955,

            /// <summary>Replacement of 2 protons by magnesium.</summary>
            UNIMOD_Cation_Mg_II_ = 300000956,

            /// <summary>S-(2-succinyl) cysteine.</summary>
            UNIMOD_2_succinyl = 300000957,

            /// <summary>Propargylamine.</summary>
            UNIMOD_Propargylamine = 300000958,

            /// <summary>Phospho-propargylamine.</summary>
            UNIMOD_Phosphopropargyl = 300000959,

            /// <summary>SUMOylation by SUMO-1 after tryptic cleavage.</summary>
            UNIMOD_SUMO2135 = 300000960,

            /// <summary>SUMOylation by SUMO-2/3 after tryptic cleavage.</summary>
            UNIMOD_SUMO3549 = 300000961,

            /// <summary>Membrane protein extraction.</summary>
            UNIMOD_thioacylPA = 300000967,

            /// <summary>Maleimide-3-saccharide.</summary>
            UNIMOD_maleimide3 = 300000971,

            /// <summary>Maleimide-5-saccharide.</summary>
            UNIMOD_maleimide5 = 300000972,

            /// <summary>Puromycin.</summary>
            UNIMOD_Puromycin = 300000973,

            /// <summary>2,3-dihydro-2,2-dimethyl-7-benzofuranol N-methyl carbamate.</summary>
            UNIMOD_Carbofuran = 300000977,

            /// <summary>Benzyl isothiocyanate.</summary>
            UNIMOD_BITC = 300000978,

            /// <summary>Phenethyl isothiocyanate.</summary>
            UNIMOD_PEITC = 300000979,

            /// <summary>Condensation product of glucosone.</summary>
            UNIMOD_glucosone = 300000981,

            /// <summary>Native cysteine-reactive Tandem Mass Tag.</summary>
            UNIMOD_cysTMT = 300000984,

            /// <summary>Cysteine-reactive Sixplex Tandem Mass Tag.</summary>
            UNIMOD_cysTMT6plex = 300000985,

            /// <summary>Dimethyl 13C(6) Silac label.</summary>
            UNIMOD_Label_13C_6__Dimethyl = 300000986,

            /// <summary>Dimethyl 13C(6)15N(2) Silac label.</summary>
            UNIMOD_Label_13C_6_15N_2__Dimethyl = 300000987,

            /// <summary>Replacement of proton with ammonium ion.</summary>
            UNIMOD_Ammonium = 300000989,

            /// <summary>ISD (z+2)-series.</summary>
            UNIMOD_ISD_z_2_ion = 300000991,

            /// <summary>Was Biotin-maleimide.</summary>
            UNIMOD_Biotin_Sigma_B1267 = 300000993,

            /// <summary>15N(1).</summary>
            UNIMOD_Label_15N_1_ = 300000994,

            /// <summary>15N(2).</summary>
            UNIMOD_Label_15N_2_ = 300000995,

            /// <summary>15N(3).</summary>
            UNIMOD_Label_15N_3_ = 300000996,

            /// <summary>Aminotyrosine with sulfation.</summary>
            UNIMOD_sulfo_amino = 300000997,

            /// <summary>Azidohomoalanine (AHA) bound to propargylglycine-NH2 (alkyne).</summary>
            UNIMOD_AHA_Alkyne = 300001000,

            /// <summary>Azidohomoalanine (AHA) bound to DDDDK-propargylglycine-NH2 (alkyne).</summary>
            UNIMOD_AHA_Alkyne_KDDDD = 300001001,

            /// <summary>(-)-epigallocatechin-3-gallate.</summary>
            UNIMOD_EGCG1 = 300001002,

            /// <summary>(-)-dehydroepigallocatechin.</summary>
            UNIMOD_EGCG2 = 300001003,

            /// <summary>Monomethylated Arg13C(6) 15N(4).</summary>
            UNIMOD_Label_13C_6_15N_4__Methyl = 300001004,

            /// <summary>Dimethylated Arg13C(6) 15N(4).</summary>
            UNIMOD_Label_13C_6_15N_4__Dimethyl = 300001005,

            /// <summary>2H(3) 13C(1) monomethylated Arg13C(6) 15N(4).</summary>
            UNIMOD_Label_13C_6_15N_4__Methyl_2H_3_13C_1_ = 300001006,

            /// <summary>2H(6) 13C(2) Dimethylated Arg13C(6) 15N(4).</summary>
            UNIMOD_Label_13C_6_15N_4__Dimethyl_2H_6_13C_2_ = 300001007,

            /// <summary>Sec Iodoacetamide derivative.</summary>
            UNIMOD_Cys__CamSec = 300001008,

            /// <summary>Formaldehyde adduct.</summary>
            UNIMOD_Thiazolidine = 300001009,

            /// <summary>Addition of DEDGFLYMVYASQETFG.</summary>
            UNIMOD_DEDGFLYMVYASQETFG = 300001010,

            /// <summary>Nalpha-(3-maleimidylpropionyl)biocytin.</summary>
            UNIMOD_Biotin_Invitrogen_M1602 = 300001012,

            /// <summary>Glycidamide adduct.</summary>
            UNIMOD_glycidamide = 300001014,

            /// <summary>C-terminal homoserine lactone and two aminohexanoic acids.</summary>
            UNIMOD_Ahx2_Hsl = 300001015,

            /// <summary>DMPO spin-trap nitrone adduct.</summary>
            UNIMOD_DMPO = 300001017,

            /// <summary>Isotope-Coded Dimedone light form.</summary>
            UNIMOD_ICDID = 300001018,

            /// <summary>Isotope-Coded Dimedone heavy form.</summary>
            UNIMOD_ICDID_2H_6_ = 300001019,

            /// <summary>Water-quenched monolink of DSS/BS3 crosslinker.</summary>
            UNIMOD_Xlink_DSS_156_ = 300001020,

            /// <summary>Water quenched monolink of EGS cross-linker.</summary>
            UNIMOD_Xlink_EGS_244_ = 300001021,

            /// <summary>Water quenched monolink of DST crosslinker.</summary>
            UNIMOD_Xlink_DST_132_ = 300001022,

            /// <summary>Water quenched monolink of DSP/DTSSP crosslinker.</summary>
            UNIMOD_Xlink_DTSSP_192_ = 300001023,

            /// <summary>Water quenched monolink of SMCC.</summary>
            UNIMOD_Xlink_SMCC_237_ = 300001024,

            /// <summary>Water quenched monolink of DMP crosslinker.</summary>
            UNIMOD_Xlink_DMP_140_ = 300001027,

            /// <summary>Cleavage product of EGS protein crosslinks by hydroylamine treatment.</summary>
            UNIMOD_Xlink_EGS_115_ = 300001028,

            /// <summary>Desthiobiotin modification of lysine.</summary>
            UNIMOD_Biotin_Thermo_88310 = 300001031,

            /// <summary>Tyrosine caged with 2-nitrobenzyl (ONB).</summary>
            UNIMOD_2_nitrobenzyl = 300001032,

            /// <summary>N-ethylmaleimide on selenocysteines.</summary>
            UNIMOD_Cys__SecNEM = 300001033,

            /// <summary>D5 N-ethylmaleimide on selenocysteines.</summary>
            UNIMOD_Cys__SecNEM_2H_5_ = 300001034,

            /// <summary>Thiadiazolydation of Cys.</summary>
            UNIMOD_Thiadiazole = 300001035,

            /// <summary>Modification of cystein by withaferin.</summary>
            UNIMOD_Withaferin = 300001036,

            /// <summary>Desthiobiotin fluorophosphonate.</summary>
            UNIMOD_Biotin_Thermo_88317 = 300001037,

            /// <summary>TAMRA fluorophosphonate modification of serine.</summary>
            UNIMOD_TAMRA_FP = 300001038,

            /// <summary>Maleimide-Biotin + Water.</summary>
            UNIMOD_Biotin_Thermo_21901_H2O = 300001039,

            /// <summary>Deoxyhypusine.</summary>
            UNIMOD_Deoxyhypusine = 300001041,

            /// <summary>Acetyldeoxyhypusine.</summary>
            UNIMOD_Acetyldeoxyhypusine = 300001042,

            /// <summary>Acetylhypusine.</summary>
            UNIMOD_Acetylhypusine = 300001043,

            /// <summary>Ala-&gt;Cys substitution.</summary>
            UNIMOD_Ala__Cys = 300001044,

            /// <summary>Ala-&gt;Phe substitution.</summary>
            UNIMOD_Ala__Phe = 300001045,

            /// <summary>Ala-&gt;His substitution.</summary>
            UNIMOD_Ala__His = 300001046,

            /// <summary>Ala-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Ala__Xle = 300001047,

            /// <summary>Ala-&gt;Lys substitution.</summary>
            UNIMOD_Ala__Lys = 300001048,

            /// <summary>Ala-&gt;Met substitution.</summary>
            UNIMOD_Ala__Met = 300001049,

            /// <summary>Ala-&gt;Asn substitution.</summary>
            UNIMOD_Ala__Asn = 300001050,

            /// <summary>Ala-&gt;Gln substitution.</summary>
            UNIMOD_Ala__Gln = 300001051,

            /// <summary>Ala-&gt;Arg substitution.</summary>
            UNIMOD_Ala__Arg = 300001052,

            /// <summary>Ala-&gt;Trp substitution.</summary>
            UNIMOD_Ala__Trp = 300001053,

            /// <summary>Ala-&gt;Tyr substitution.</summary>
            UNIMOD_Ala__Tyr = 300001054,

            /// <summary>Cys-&gt;Ala substitution.</summary>
            UNIMOD_Cys__Ala = 300001055,

            /// <summary>Cys-&gt;Asp substitution.</summary>
            UNIMOD_Cys__Asp = 300001056,

            /// <summary>Cys-&gt;Glu substitution.</summary>
            UNIMOD_Cys__Glu = 300001057,

            /// <summary>Cys-&gt;His substitution.</summary>
            UNIMOD_Cys__His = 300001058,

            /// <summary>Cys-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Cys__Xle = 300001059,

            /// <summary>Cys-&gt;Lys substitution.</summary>
            UNIMOD_Cys__Lys = 300001060,

            /// <summary>Cys-&gt;Met substitution.</summary>
            UNIMOD_Cys__Met = 300001061,

            /// <summary>Cys-&gt;Asn substitution.</summary>
            UNIMOD_Cys__Asn = 300001062,

            /// <summary>Cys-&gt;Pro substitution.</summary>
            UNIMOD_Cys__Pro = 300001063,

            /// <summary>Cys-&gt;Gln substitution.</summary>
            UNIMOD_Cys__Gln = 300001064,

            /// <summary>Cys-&gt;Thr substitution.</summary>
            UNIMOD_Cys__Thr = 300001065,

            /// <summary>Cys-&gt;Val substitution.</summary>
            UNIMOD_Cys__Val = 300001066,

            /// <summary>Asp-&gt;Cys substitution.</summary>
            UNIMOD_Asp__Cys = 300001067,

            /// <summary>Asp-&gt;Phe substitution.</summary>
            UNIMOD_Asp__Phe = 300001068,

            /// <summary>Asp-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Asp__Xle = 300001069,

            /// <summary>Asp-&gt;Lys substitution.</summary>
            UNIMOD_Asp__Lys = 300001070,

            /// <summary>Asp-&gt;Met substitution.</summary>
            UNIMOD_Asp__Met = 300001071,

            /// <summary>Asp-&gt;Pro substitution.</summary>
            UNIMOD_Asp__Pro = 300001072,

            /// <summary>Asp-&gt;Gln substitution.</summary>
            UNIMOD_Asp__Gln = 300001073,

            /// <summary>Asp-&gt;Arg substitution.</summary>
            UNIMOD_Asp__Arg = 300001074,

            /// <summary>Asp-&gt;Ser substitution.</summary>
            UNIMOD_Asp__Ser = 300001075,

            /// <summary>Asp-&gt;Thr substitution.</summary>
            UNIMOD_Asp__Thr = 300001076,

            /// <summary>Asp-&gt;Trp substitution.</summary>
            UNIMOD_Asp__Trp = 300001077,

            /// <summary>Glu-&gt;Cys substitution.</summary>
            UNIMOD_Glu__Cys = 300001078,

            /// <summary>Glu-&gt;Phe substitution.</summary>
            UNIMOD_Glu__Phe = 300001079,

            /// <summary>Glu-&gt;His substitution.</summary>
            UNIMOD_Glu__His = 300001080,

            /// <summary>Glu-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Glu__Xle = 300001081,

            /// <summary>Glu-&gt;Met substitution.</summary>
            UNIMOD_Glu__Met = 300001082,

            /// <summary>Glu-&gt;Asn substitution.</summary>
            UNIMOD_Glu__Asn = 300001083,

            /// <summary>Glu-&gt;Pro substitution.</summary>
            UNIMOD_Glu__Pro = 300001084,

            /// <summary>Glu-&gt;Arg substitution.</summary>
            UNIMOD_Glu__Arg = 300001085,

            /// <summary>Glu-&gt;Ser substitution.</summary>
            UNIMOD_Glu__Ser = 300001086,

            /// <summary>Glu-&gt;Thr substitution.</summary>
            UNIMOD_Glu__Thr = 300001087,

            /// <summary>Glu-&gt;Trp substitution.</summary>
            UNIMOD_Glu__Trp = 300001088,

            /// <summary>Glu-&gt;Tyr substitution.</summary>
            UNIMOD_Glu__Tyr = 300001089,

            /// <summary>Phe-&gt;Ala substitution.</summary>
            UNIMOD_Phe__Ala = 300001090,

            /// <summary>Phe-&gt;Asp substitution.</summary>
            UNIMOD_Phe__Asp = 300001091,

            /// <summary>Phe-&gt;Glu substitution.</summary>
            UNIMOD_Phe__Glu = 300001092,

            /// <summary>Phe-&gt;Gly substitution.</summary>
            UNIMOD_Phe__Gly = 300001093,

            /// <summary>Phe-&gt;His substitution.</summary>
            UNIMOD_Phe__His = 300001094,

            /// <summary>Phe-&gt;Lys substitution.</summary>
            UNIMOD_Phe__Lys = 300001095,

            /// <summary>Phe-&gt;Met substitution.</summary>
            UNIMOD_Phe__Met = 300001096,

            /// <summary>Phe-&gt;Asn substitution.</summary>
            UNIMOD_Phe__Asn = 300001097,

            /// <summary>Phe-&gt;Pro substitution.</summary>
            UNIMOD_Phe__Pro = 300001098,

            /// <summary>Phe-&gt;Gln substitution.</summary>
            UNIMOD_Phe__Gln = 300001099,

            /// <summary>Phe-&gt;Arg substitution.</summary>
            UNIMOD_Phe__Arg = 300001100,

            /// <summary>Phe-&gt;Thr substitution.</summary>
            UNIMOD_Phe__Thr = 300001101,

            /// <summary>Phe-&gt;Trp substitution.</summary>
            UNIMOD_Phe__Trp = 300001102,

            /// <summary>Gly-&gt;Phe substitution.</summary>
            UNIMOD_Gly__Phe = 300001103,

            /// <summary>Gly-&gt;His substitution.</summary>
            UNIMOD_Gly__His = 300001104,

            /// <summary>Gly-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Gly__Xle = 300001105,

            /// <summary>Gly-&gt;Lys substitution.</summary>
            UNIMOD_Gly__Lys = 300001106,

            /// <summary>Gly-&gt;Met substitution.</summary>
            UNIMOD_Gly__Met = 300001107,

            /// <summary>Gly-&gt;Asn substitution.</summary>
            UNIMOD_Gly__Asn = 300001108,

            /// <summary>Gly-&gt;Pro substitution.</summary>
            UNIMOD_Gly__Pro = 300001109,

            /// <summary>Gly-&gt;Gln substitution.</summary>
            UNIMOD_Gly__Gln = 300001110,

            /// <summary>Gly-&gt;Thr substitution.</summary>
            UNIMOD_Gly__Thr = 300001111,

            /// <summary>Gly-&gt;Tyr substitution.</summary>
            UNIMOD_Gly__Tyr = 300001112,

            /// <summary>His-&gt;Ala substitution.</summary>
            UNIMOD_His__Ala = 300001113,

            /// <summary>His-&gt;Cys substitution.</summary>
            UNIMOD_His__Cys = 300001114,

            /// <summary>His-&gt;Glu substitution.</summary>
            UNIMOD_His__Glu = 300001115,

            /// <summary>His-&gt;Phe substitution.</summary>
            UNIMOD_His__Phe = 300001116,

            /// <summary>His-&gt;Gly substitution.</summary>
            UNIMOD_His__Gly = 300001117,

            /// <summary>His-&gt;Lys substitution.</summary>
            UNIMOD_His__Lys = 300001119,

            /// <summary>His-&gt;Met substitution.</summary>
            UNIMOD_His__Met = 300001120,

            /// <summary>His-&gt;Ser substitution.</summary>
            UNIMOD_His__Ser = 300001121,

            /// <summary>His-&gt;Thr substitution.</summary>
            UNIMOD_His__Thr = 300001122,

            /// <summary>His-&gt;Val substitution.</summary>
            UNIMOD_His__Val = 300001123,

            /// <summary>His-&gt;Trp substitution.</summary>
            UNIMOD_His__Trp = 300001124,

            /// <summary>Leu/Ile-&gt;Ala substitution.</summary>
            UNIMOD_Xle__Ala = 300001125,

            /// <summary>Leu/Ile-&gt;Cys substitution.</summary>
            UNIMOD_Xle__Cys = 300001126,

            /// <summary>Leu/Ile-&gt;Asp substitution.</summary>
            UNIMOD_Xle__Asp = 300001127,

            /// <summary>Leu/Ile-&gt;Glu substitution.</summary>
            UNIMOD_Xle__Glu = 300001128,

            /// <summary>Leu/Ile-&gt;Gly substitution.</summary>
            UNIMOD_Xle__Gly = 300001129,

            /// <summary>Leu/Ile-&gt;Tyr substitution.</summary>
            UNIMOD_Xle__Tyr = 300001130,

            /// <summary>Lys-&gt;Ala substitution.</summary>
            UNIMOD_Lys__Ala = 300001131,

            /// <summary>Lys-&gt;Cys substitution.</summary>
            UNIMOD_Lys__Cys = 300001132,

            /// <summary>Lys-&gt;Asp substitution.</summary>
            UNIMOD_Lys__Asp = 300001133,

            /// <summary>Lys-&gt;Phe substitution.</summary>
            UNIMOD_Lys__Phe = 300001134,

            /// <summary>Lys-&gt;Gly substitution.</summary>
            UNIMOD_Lys__Gly = 300001135,

            /// <summary>Lys-&gt;His substitution.</summary>
            UNIMOD_Lys__His = 300001136,

            /// <summary>Lys-&gt;Pro substitution.</summary>
            UNIMOD_Lys__Pro = 300001137,

            /// <summary>Lys-&gt;Ser substitution.</summary>
            UNIMOD_Lys__Ser = 300001138,

            /// <summary>Lys-&gt;Val substitution.</summary>
            UNIMOD_Lys__Val = 300001139,

            /// <summary>Lys-&gt;Trp substitution.</summary>
            UNIMOD_Lys__Trp = 300001140,

            /// <summary>Lys-&gt;Tyr substitution.</summary>
            UNIMOD_Lys__Tyr = 300001141,

            /// <summary>Met-&gt;Ala substitution.</summary>
            UNIMOD_Met__Ala = 300001142,

            /// <summary>Met-&gt;Cys substitution.</summary>
            UNIMOD_Met__Cys = 300001143,

            /// <summary>Met-&gt;Asp substitution.</summary>
            UNIMOD_Met__Asp = 300001144,

            /// <summary>Met-&gt;Glu substitution.</summary>
            UNIMOD_Met__Glu = 300001145,

            /// <summary>Met-&gt;Phe substitution.</summary>
            UNIMOD_Met__Phe = 300001146,

            /// <summary>Met-&gt;Gly substitution.</summary>
            UNIMOD_Met__Gly = 300001147,

            /// <summary>Met-&gt;His substitution.</summary>
            UNIMOD_Met__His = 300001148,

            /// <summary>Met-&gt;Asn substitution.</summary>
            UNIMOD_Met__Asn = 300001149,

            /// <summary>Met-&gt;Pro substitution.</summary>
            UNIMOD_Met__Pro = 300001150,

            /// <summary>Met-&gt;Gln substitution.</summary>
            UNIMOD_Met__Gln = 300001151,

            /// <summary>Met-&gt;Ser substitution.</summary>
            UNIMOD_Met__Ser = 300001152,

            /// <summary>Met-&gt;Trp substitution.</summary>
            UNIMOD_Met__Trp = 300001153,

            /// <summary>Met-&gt;Tyr substitution.</summary>
            UNIMOD_Met__Tyr = 300001154,

            /// <summary>Asn-&gt;Ala substitution.</summary>
            UNIMOD_Asn__Ala = 300001155,

            /// <summary>Asn-&gt;Cys substitution.</summary>
            UNIMOD_Asn__Cys = 300001156,

            /// <summary>Asn-&gt;Glu substitution.</summary>
            UNIMOD_Asn__Glu = 300001157,

            /// <summary>Asn-&gt;Phe substitution.</summary>
            UNIMOD_Asn__Phe = 300001158,

            /// <summary>Asn-&gt;Gly substitution.</summary>
            UNIMOD_Asn__Gly = 300001159,

            /// <summary>Asn-&gt;Met substitution.</summary>
            UNIMOD_Asn__Met = 300001160,

            /// <summary>Asn-&gt;Pro substitution.</summary>
            UNIMOD_Asn__Pro = 300001161,

            /// <summary>Asn-&gt;Gln substitution.</summary>
            UNIMOD_Asn__Gln = 300001162,

            /// <summary>Asn-&gt;Arg substitution.</summary>
            UNIMOD_Asn__Arg = 300001163,

            /// <summary>Asn-&gt;Val substitution.</summary>
            UNIMOD_Asn__Val = 300001164,

            /// <summary>Asn-&gt;Trp substitution.</summary>
            UNIMOD_Asn__Trp = 300001165,

            /// <summary>Pro-&gt;Cys substitution.</summary>
            UNIMOD_Pro__Cys = 300001166,

            /// <summary>Pro-&gt;Asp substitution.</summary>
            UNIMOD_Pro__Asp = 300001167,

            /// <summary>Pro-&gt;Glu substitution.</summary>
            UNIMOD_Pro__Glu = 300001168,

            /// <summary>Pro-&gt;Phe substitution.</summary>
            UNIMOD_Pro__Phe = 300001169,

            /// <summary>Pro-&gt;Gly substitution.</summary>
            UNIMOD_Pro__Gly = 300001170,

            /// <summary>Pro-&gt;Lys substitution.</summary>
            UNIMOD_Pro__Lys = 300001171,

            /// <summary>Pro-&gt;Met substitution.</summary>
            UNIMOD_Pro__Met = 300001172,

            /// <summary>Pro-&gt;Asn substitution.</summary>
            UNIMOD_Pro__Asn = 300001173,

            /// <summary>Pro-&gt;Val substitution.</summary>
            UNIMOD_Pro__Val = 300001174,

            /// <summary>Pro-&gt;Trp substitution.</summary>
            UNIMOD_Pro__Trp = 300001175,

            /// <summary>Pro-&gt;Tyr substitution.</summary>
            UNIMOD_Pro__Tyr = 300001176,

            /// <summary>Gln-&gt;Ala substitution.</summary>
            UNIMOD_Gln__Ala = 300001177,

            /// <summary>Gln-&gt;Cys substitution.</summary>
            UNIMOD_Gln__Cys = 300001178,

            /// <summary>Gln-&gt;Asp substitution.</summary>
            UNIMOD_Gln__Asp = 300001179,

            /// <summary>Gln-&gt;Phe substitution.</summary>
            UNIMOD_Gln__Phe = 300001180,

            /// <summary>Gln-&gt;Gly substitution.</summary>
            UNIMOD_Gln__Gly = 300001181,

            /// <summary>Gln-&gt;Met substitution.</summary>
            UNIMOD_Gln__Met = 300001182,

            /// <summary>Gln-&gt;Asn substitution.</summary>
            UNIMOD_Gln__Asn = 300001183,

            /// <summary>Gln-&gt;Ser substitution.</summary>
            UNIMOD_Gln__Ser = 300001184,

            /// <summary>Gln-&gt;Thr substitution.</summary>
            UNIMOD_Gln__Thr = 300001185,

            /// <summary>Gln-&gt;Val substitution.</summary>
            UNIMOD_Gln__Val = 300001186,

            /// <summary>Gln-&gt;Trp substitution.</summary>
            UNIMOD_Gln__Trp = 300001187,

            /// <summary>Gln-&gt;Tyr substitution.</summary>
            UNIMOD_Gln__Tyr = 300001188,

            /// <summary>Arg-&gt;Ala substitution.</summary>
            UNIMOD_Arg__Ala = 300001189,

            /// <summary>Arg-&gt;Asp substitution.</summary>
            UNIMOD_Arg__Asp = 300001190,

            /// <summary>Arg-&gt;Glu substitution.</summary>
            UNIMOD_Arg__Glu = 300001191,

            /// <summary>Arg-&gt;Asn substitution.</summary>
            UNIMOD_Arg__Asn = 300001192,

            /// <summary>Arg-&gt;Val substitution.</summary>
            UNIMOD_Arg__Val = 300001193,

            /// <summary>Arg-&gt;Tyr substitution.</summary>
            UNIMOD_Arg__Tyr = 300001194,

            /// <summary>Arg-&gt;Phe substitution.</summary>
            UNIMOD_Arg__Phe = 300001195,

            /// <summary>Ser-&gt;Asp substitution.</summary>
            UNIMOD_Ser__Asp = 300001196,

            /// <summary>Ser-&gt;Glu substitution.</summary>
            UNIMOD_Ser__Glu = 300001197,

            /// <summary>Ser-&gt;His substitution.</summary>
            UNIMOD_Ser__His = 300001198,

            /// <summary>Ser-&gt;Lys substitution.</summary>
            UNIMOD_Ser__Lys = 300001199,

            /// <summary>Ser-&gt;Met substitution.</summary>
            UNIMOD_Ser__Met = 300001200,

            /// <summary>Ser-&gt;Gln substitution.</summary>
            UNIMOD_Ser__Gln = 300001201,

            /// <summary>Ser-&gt;Val substitution.</summary>
            UNIMOD_Ser__Val = 300001202,

            /// <summary>Thr-&gt;Cys substitution.</summary>
            UNIMOD_Thr__Cys = 300001203,

            /// <summary>Thr-&gt;Asp substitution.</summary>
            UNIMOD_Thr__Asp = 300001204,

            /// <summary>Thr-&gt;Glu substitution.</summary>
            UNIMOD_Thr__Glu = 300001205,

            /// <summary>Thr-&gt;Phe substitution.</summary>
            UNIMOD_Thr__Phe = 300001206,

            /// <summary>Thr-&gt;Gly substitution.</summary>
            UNIMOD_Thr__Gly = 300001207,

            /// <summary>Thr-&gt;His substitution.</summary>
            UNIMOD_Thr__His = 300001208,

            /// <summary>Thr-&gt;Gln substitution.</summary>
            UNIMOD_Thr__Gln = 300001209,

            /// <summary>Thr-&gt;Val substitution.</summary>
            UNIMOD_Thr__Val = 300001210,

            /// <summary>Thr-&gt;Trp substitution.</summary>
            UNIMOD_Thr__Trp = 300001211,

            /// <summary>Thr-&gt;Tyr substitution.</summary>
            UNIMOD_Thr__Tyr = 300001212,

            /// <summary>Val-&gt;Cys substitution.</summary>
            UNIMOD_Val__Cys = 300001213,

            /// <summary>Val-&gt;His substitution.</summary>
            UNIMOD_Val__His = 300001214,

            /// <summary>Val-&gt;Lys substitution.</summary>
            UNIMOD_Val__Lys = 300001215,

            /// <summary>Val-&gt;Asn substitution.</summary>
            UNIMOD_Val__Asn = 300001216,

            /// <summary>Val-&gt;Pro substitution.</summary>
            UNIMOD_Val__Pro = 300001217,

            /// <summary>Val-&gt;Gln substitution.</summary>
            UNIMOD_Val__Gln = 300001218,

            /// <summary>Val-&gt;Arg substitution.</summary>
            UNIMOD_Val__Arg = 300001219,

            /// <summary>Val-&gt;Ser substitution.</summary>
            UNIMOD_Val__Ser = 300001220,

            /// <summary>Val-&gt;Thr substitution.</summary>
            UNIMOD_Val__Thr = 300001221,

            /// <summary>Val-&gt;Trp substitution.</summary>
            UNIMOD_Val__Trp = 300001222,

            /// <summary>Val-&gt;Tyr substitution.</summary>
            UNIMOD_Val__Tyr = 300001223,

            /// <summary>Trp-&gt;Ala substitution.</summary>
            UNIMOD_Trp__Ala = 300001224,

            /// <summary>Trp-&gt;Asp substitution.</summary>
            UNIMOD_Trp__Asp = 300001225,

            /// <summary>Trp-&gt;Glu substitution.</summary>
            UNIMOD_Trp__Glu = 300001226,

            /// <summary>Trp-&gt;Phe substitution.</summary>
            UNIMOD_Trp__Phe = 300001227,

            /// <summary>Trp-&gt;His substitution.</summary>
            UNIMOD_Trp__His = 300001228,

            /// <summary>Trp-&gt;Lys substitution.</summary>
            UNIMOD_Trp__Lys = 300001229,

            /// <summary>Trp-&gt;Met substitution.</summary>
            UNIMOD_Trp__Met = 300001230,

            /// <summary>Trp-&gt;Asn substitution.</summary>
            UNIMOD_Trp__Asn = 300001231,

            /// <summary>Trp-&gt;Pro substitution.</summary>
            UNIMOD_Trp__Pro = 300001232,

            /// <summary>Trp-&gt;Gln substitution.</summary>
            UNIMOD_Trp__Gln = 300001233,

            /// <summary>Trp-&gt;Thr substitution.</summary>
            UNIMOD_Trp__Thr = 300001234,

            /// <summary>Trp-&gt;Val substitution.</summary>
            UNIMOD_Trp__Val = 300001235,

            /// <summary>Trp-&gt;Tyr substitution.</summary>
            UNIMOD_Trp__Tyr = 300001236,

            /// <summary>Tyr-&gt;Ala substitution.</summary>
            UNIMOD_Tyr__Ala = 300001237,

            /// <summary>Tyr-&gt;Glu substitution.</summary>
            UNIMOD_Tyr__Glu = 300001238,

            /// <summary>Tyr-&gt;Gly substitution.</summary>
            UNIMOD_Tyr__Gly = 300001239,

            /// <summary>Tyr-&gt;Lys substitution.</summary>
            UNIMOD_Tyr__Lys = 300001240,

            /// <summary>Tyr-&gt;Met substitution.</summary>
            UNIMOD_Tyr__Met = 300001241,

            /// <summary>Tyr-&gt;Pro substitution.</summary>
            UNIMOD_Tyr__Pro = 300001242,

            /// <summary>Tyr-&gt;Gln substitution.</summary>
            UNIMOD_Tyr__Gln = 300001243,

            /// <summary>Tyr-&gt;Arg substitution.</summary>
            UNIMOD_Tyr__Arg = 300001244,

            /// <summary>Tyr-&gt;Thr substitution.</summary>
            UNIMOD_Tyr__Thr = 300001245,

            /// <summary>Tyr-&gt;Val substitution.</summary>
            UNIMOD_Tyr__Val = 300001246,

            /// <summary>Tyr-&gt;Trp substitution.</summary>
            UNIMOD_Tyr__Trp = 300001247,

            /// <summary>Tyr-&gt;Leu/Ile substitution.</summary>
            UNIMOD_Tyr__Xle = 300001248,

            /// <summary>Azidohomoalanine coupled to reductively cleaved tag.</summary>
            UNIMOD_AHA_SS = 300001249,

            /// <summary>Carbamidomethylated form of reductively cleaved tag coupled to azidohomoalanine.</summary>
            UNIMOD_AHA_SS_CAM = 300001250,

            /// <summary>Sulfo-SBED Label Photoreactive Biotin Crosslinker.</summary>
            UNIMOD_Biotin_Thermo_33033 = 300001251,

            /// <summary>Sulfo-SBED Label Photoreactive Biotin Crosslinker minus Hydrogen.</summary>
            UNIMOD_Biotin_Thermo_33033_H = 300001252,

            /// <summary>S-(2-monomethylsuccinyl) cysteine.</summary>
            UNIMOD_2_monomethylsuccinyl = 300001253,

            /// <summary>O-toluene.</summary>
            UNIMOD_Saligenin = 300001254,

            /// <summary>O-toluyl-phosphorylation.</summary>
            UNIMOD_Cresylphosphate = 300001255,

            /// <summary>Cresyl-Saligenin-phosphorylation.</summary>
            UNIMOD_CresylSaligeninPhosphate = 300001256,

            /// <summary>Ub Bromide probe addition.</summary>
            UNIMOD_Ub_Br2 = 300001257,

            /// <summary>Ubiquitin vinylmethylester.</summary>
            UNIMOD_Ub_VME = 300001258,

            /// <summary>Ub Fluorescein probe addition.</summary>
            UNIMOD_Ub_fluorescein = 300001261,

            /// <summary>S-(2-dimethylsuccinyl) cysteine.</summary>
            UNIMOD_2_dimethylsuccinyl = 300001262,

            /// <summary>Addition of Glycine.</summary>
            UNIMOD_Gly = 300001263,

            /// <summary>Addition of GGE.</summary>
            UNIMOD_pupylation = 300001264,

            /// <summary>13C4 Methionine label.</summary>
            UNIMOD_Label_13C_4_ = 300001266,

            /// <summary>Oxidised 13C4 labelled Methionine.</summary>
            UNIMOD_Label_13C_4__Oxidation = 300001267,

            /// <summary>N-Homocysteine thiolactone.</summary>
            UNIMOD_HCysThiolactone = 300001270,

            /// <summary>S-homocysteinylation.</summary>
            UNIMOD_HCysteinyl = 300001271,

            /// <summary>Side reaction of HisTag.</summary>
            UNIMOD_UgiJoullie = 300001276,

            /// <summary>Cys modified with dipy ligand.</summary>
            UNIMOD_Dipyridyl = 300001277,

            /// <summary>Chemical modification of the iodinated sites of thyroglobulin by Suzuki reaction.</summary>
            UNIMOD_Furan = 300001278,

            /// <summary>Chemical modification of the diiodinated sites of thyroglobulin by Suzuki reaction.</summary>
            UNIMOD_Difuran = 300001279,

            /// <summary>1-methyl-3-benzoyl-4-hydroxy-4-phenylpiperidine.</summary>
            UNIMOD_BMP_piperidinol = 300001281,

            /// <summary>Side reaction of PG with Side chain of aspartic or glutamic acid.</summary>
            UNIMOD_UgiJoullieProGly = 300001282,

            /// <summary>Side reaction of PGPG with Side chain of aspartic or glutamic acid.</summary>
            UNIMOD_UgiJoullieProGlyProGly = 300001283,

            /// <summary>Glycosylation with IME linked Hex(2) NeuAc.</summary>
            UNIMOD_IMEHex_2_NeuAc_1_ = 300001286,

            /// <summary>Loss of arginine due to transpeptidation.</summary>
            UNIMOD_Arg_loss = 300001287,

            /// <summary>Addition of arginine due to transpeptidation.</summary>
            UNIMOD_Arg = 300001288,

            /// <summary>Butyryl.</summary>
            UNIMOD_Butyryl = 300001289,

            /// <summary>Double Carbamidomethylation.</summary>
            UNIMOD_Dicarbamidomethyl = 300001290,

            /// <summary>Dimethyl-Medium.</summary>
            UNIMOD_Dimethyl_2H_6_ = 300001291,

            /// <summary>SUMOylation leaving GlyGlyGln.</summary>
            UNIMOD_GGQ = 300001292,

            /// <summary>SUMOylation leaving GlnThrGlyGly.</summary>
            UNIMOD_QTGG = 300001293,

            /// <summary>13C3 label for SILAC.</summary>
            UNIMOD_Label_13C_3_ = 300001296,

            /// <summary>SILAC or AQUA label.</summary>
            UNIMOD_Label_13C_3_15N_1_ = 300001297,

            /// <summary>13C4 15N1 label for SILAC.</summary>
            UNIMOD_Label_13C_4_15N_1_ = 300001298,

            /// <summary>2H(10) label.</summary>
            UNIMOD_Label_2H_10_ = 300001299,

            /// <summary>Label:2H(4)13C(1).</summary>
            UNIMOD_Label_2H_4_13C_1_ = 300001300,

            /// <summary>Addition of lysine due to transpeptidation.</summary>
            UNIMOD_Lys = 300001301,

            /// <summary>MTRAQ heavy.</summary>
            UNIMOD_mTRAQ_13C_6_15N_2_ = 300001302,

            /// <summary>N-acetyl neuraminic acid.</summary>
            UNIMOD_NeuAc = 300001303,

            /// <summary>N-glycoyl neuraminic acid.</summary>
            UNIMOD_NeuGc = 300001304,

            /// <summary>Propyl.</summary>
            UNIMOD_Propyl = 300001305,

            /// <summary>Propyl:2H(6).</summary>
            UNIMOD_Propyl_2H_6_ = 300001306,

            /// <summary>Propiophenone.</summary>
            UNIMOD_Propiophenone = 300001310,

            /// <summary>Reduced acrolein addition +58.</summary>
            UNIMOD_Delta_H_6_C_3_O_1_ = 300001312,

            /// <summary>Reduced acrolein addition +96.</summary>
            UNIMOD_Delta_H_8_C_6_O_1_ = 300001313,

            /// <summary>Biotin hydrazide labeled acrolein addition +298.</summary>
            UNIMOD_biotinAcrolein298 = 300001314,

            /// <summary>3-methyl-5-(methylamino)-1,3-diphenylpentan-1-one.</summary>
            UNIMOD_MM_diphenylpentanone = 300001315,

            /// <summary>2-ethyl-3-hydroxy-1,3-diphenylpentan-1-one.</summary>
            UNIMOD_EHD_diphenylpentanone = 300001317,

            /// <summary>Maleimide-Biotin + 2Water.</summary>
            UNIMOD_Biotin_Thermo_21901_2H2O = 300001320,

            /// <summary>Accurate mass for DiLeu 115 isobaric tag.</summary>
            UNIMOD_DiLeu4plex115 = 300001321,

            /// <summary>Accurate mass for DiLeu 116 isobaric tag.</summary>
            UNIMOD_DiLeu4plex = 300001322,

            /// <summary>Accurate mass for DiLeu 117 isobaric tag.</summary>
            UNIMOD_DiLeu4plex117 = 300001323,

            /// <summary>Accurate mass for DiLeu 118 isobaric tag.</summary>
            UNIMOD_DiLeu4plex118 = 300001324,

            /// <summary>N-ethylmaleimideSulfur.</summary>
            UNIMOD_NEMsulfur = 300001326,

            /// <summary>SulfurDioxide.</summary>
            UNIMOD_SulfurDioxide = 300001327,

            /// <summary>N-ethylmaleimideSulfurWater.</summary>
            UNIMOD_NEMsulfurWater = 300001328,

            /// <summary>BisANS with loss of both sulfonates.</summary>
            UNIMOD_bisANS_sulfonates = 300001330,

            /// <summary>Chemical reaction with 2,4-dinitro-1-chloro benzene (DNCB).</summary>
            UNIMOD_DNCB_hapten = 300001331,

            /// <summary>Biotin-PEG11-maleimide.</summary>
            UNIMOD_Biotin_Thermo_21911 = 300001340,

            /// <summary>Native iodoacetyl Tandem Mass Tag.</summary>
            UNIMOD_iodoTMT = 300001341,

            /// <summary>Sixplex iodoacetyl Tandem Mass Tag.</summary>
            UNIMOD_iodoTMT6plex = 300001342,

            /// <summary>Phosphogluconoylation.</summary>
            UNIMOD_Phosphogluconoylation = 300001344,

            /// <summary>Reaction with phenyl salicylate (PS).</summary>
            UNIMOD_PS_Hapten = 300001345,

            /// <summary>Cy3 Maleimide mono-Reactive dye.</summary>
            UNIMOD_Cy3_maleimide = 300001348,

            /// <summary>Modification of the lysine side chain from NH2 to guanidine with a H removed in favor of a benzyl group.</summary>
            UNIMOD_benzylguanidine = 300001349,

            /// <summary>A fixed +1 charge tag attached to the N-terminus of peptides.</summary>
            UNIMOD_CarboxymethylDMAP = 300001350,

            /// <summary>Formation of five membered aromatic heterocycle.</summary>
            UNIMOD_azole = 300001355,

            /// <summary>Phosphate-ribosylation.</summary>
            UNIMOD_phosphoRibosyl = 300001356,

            /// <summary>D5 N-ethylmaleimide+water on cysteines.</summary>
            UNIMOD_NEM_2H_5__H2O = 300001358,

            /// <summary>Crotonylation.</summary>
            UNIMOD_Crotonyl = 300001363,

            /// <summary>O-ethyl, N-dimethyl phosphate.</summary>
            UNIMOD_O_Et_N_diMePhospho = 300001364,

            /// <summary>N-dimethylphosphate.</summary>
            UNIMOD_N_dimethylphosphate = 300001365,

            /// <summary>Hex1dHex1.</summary>
            UNIMOD_dHex_1_Hex_1_ = 300001367,

            /// <summary>3-fold methylated lysine labelled with Acetyl_heavy.</summary>
            UNIMOD_Methyl_2H_3__Acetyl_2H_3_ = 300001368,

            /// <summary>Oxidised 2H(3) labelled Methionine.</summary>
            UNIMOD_Label_2H_3__Oxidation = 300001370,

            /// <summary>3-fold methylation with deuterated methyl groups.</summary>
            UNIMOD_Trimethyl_2H_9_ = 300001371,

            /// <summary>Heavy acetylation.</summary>
            UNIMOD_Acetyl_13C_2_ = 300001372,

            /// <summary>Hex2dHex1.</summary>
            UNIMOD_dHex_1_Hex_2_ = 300001375,

            /// <summary>Hex3dHex1.</summary>
            UNIMOD_dHex_1_Hex_3_ = 300001376,

            /// <summary>Hex4dHex1.</summary>
            UNIMOD_dHex_1_Hex_4_ = 300001377,

            /// <summary>Hex5dHex1.</summary>
            UNIMOD_dHex_1_Hex_5_ = 300001378,

            /// <summary>Hex6dHex1.</summary>
            UNIMOD_dHex_1_Hex_6_ = 300001379,

            /// <summary>Reaction with methyl vinyl sulfone.</summary>
            UNIMOD_methylsulfonylethyl = 300001380,

            /// <summary>Reaction with ethyl vinyl sulfone.</summary>
            UNIMOD_ethylsulfonylethyl = 300001381,

            /// <summary>Reaction with phenyl vinyl sulfone.</summary>
            UNIMOD_phenylsulfonylethyl = 300001382,

            /// <summary>PLP bound to lysine reduced by sodium borohydride (NaBH4) to create amine linkage.</summary>
            UNIMOD_PyridoxalPhosphateH2 = 300001383,

            /// <summary>Methionine oxidation to homocysteic acid.</summary>
            UNIMOD_Homocysteic_acid = 300001384,

            /// <summary>ADP-ribosylation followed by conversion to hydroxamic acid via hydroxylamine.</summary>
            UNIMOD_Hydroxamic_acid = 300001385,

            /// <summary>3-phosphoglyceryl.</summary>
            UNIMOD_3_phosphoglyceryl = 300001387,

            /// <summary>Modification by hydroxylated mechloroethamine (HN-2).</summary>
            UNIMOD_HN2_mustard = 300001388,

            /// <summary>Modification by hydroxylated tris-(2-chloroethyl)amine (HN-3).</summary>
            UNIMOD_HN3_mustard = 300001389,

            /// <summary>N-ethylmaleimide on cysteine sulfenic acid.</summary>
            UNIMOD_Oxidation_NEM = 300001390,

            /// <summary>Fluorescein-hexanoate-NHS hydrolysis.</summary>
            UNIMOD_NHS_fluorescein = 300001391,

            /// <summary>Representative mass and accurate mass for 114.</summary>
            UNIMOD_DiART6plex = 300001392,

            /// <summary>Accurate mass for DiART6plex 115.</summary>
            UNIMOD_DiART6plex115 = 300001393,

            /// <summary>Accurate mass for DiART6plex 116 and 119.</summary>
            UNIMOD_DiART6plex116_119 = 300001394,

            /// <summary>Accurate mass for DiART6plex 117.</summary>
            UNIMOD_DiART6plex117 = 300001395,

            /// <summary>Accurate mass for DiART6plex 118.</summary>
            UNIMOD_DiART6plex118 = 300001396,

            /// <summary>Iodoacetanilide derivative.</summary>
            UNIMOD_Iodoacetanilide = 300001397,

            /// <summary>13C labelled iodoacetanilide derivative.</summary>
            UNIMOD_Iodoacetanilide_13C_6_ = 300001398,

            /// <summary>Diaminopimelic acid-DSP monolinked.</summary>
            UNIMOD_Dap_DSP = 300001399,

            /// <summary>N-Acetylmuramic acid.</summary>
            UNIMOD_MurNAc = 300001400,

            /// <summary>Label:2H(7)15N(4).</summary>
            UNIMOD_Label_2H_7_15N_4_ = 300001402,

            /// <summary>Label:2H(6)15N(1).</summary>
            UNIMOD_Label_2H_6_15N_1_ = 300001403,

            /// <summary>Sumoylation by SUMO-1 after Cyanogen bromide (CNBr) cleavage.</summary>
            UNIMOD_EEEDVIEVYQEQTGG = 300001405,

            /// <summary>Sumoylation by SUMO-2/3 after Cyanogen bromide (CNBr) cleavage.</summary>
            UNIMOD_EDEDTIDVFQQQTGG = 300001406,

            /// <summary>A2G2S2/G2S2.</summary>
            UNIMOD_Hex_5_HexNAc_4_NeuAc_2_ = 300001408,

            /// <summary>A2G2S1/G2S1.</summary>
            UNIMOD_Hex_5_HexNAc_4_NeuAc_1_ = 300001409,

            /// <summary>FA2G2S1/G2FS1.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_4_NeuAc_1_ = 300001410,

            /// <summary>FA2G2S2/G2FS2.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_4_NeuAc_2_ = 300001411,

            /// <summary>O3S1HexNAc1.</summary>
            UNIMOD_s_GlcNAc = 300001412,

            /// <summary>H1O3P1Hex2.</summary>
            UNIMOD_PhosphoHex_2_ = 300001413,

            /// <summary>3-fold methylation with fully labelled methyl groups.</summary>
            UNIMOD_Trimethyl_13C_3_2H_9_ = 300001414,

            /// <summary>Loss of ammonia (15N).</summary>
            UNIMOD_15N_oxobutanoic = 300001419,

            /// <summary>Spermine adduct.</summary>
            UNIMOD_spermine = 300001420,

            /// <summary>Spermidine adduct.</summary>
            UNIMOD_spermidine = 300001421,

            /// <summary>Biotin_PEG4.</summary>
            UNIMOD_Biotin_Thermo_21330 = 300001423,

            /// <summary>Pentose.</summary>
            UNIMOD_Pentose = 300001425,

            /// <summary>Hex Pent.</summary>
            UNIMOD_Hex_1_Pent_1_ = 300001426,

            /// <summary>Hex HexA.</summary>
            UNIMOD_Hex_1_HexA_1_ = 300001427,

            /// <summary>Hex Pent(2).</summary>
            UNIMOD_Hex_1_Pent_2_ = 300001428,

            /// <summary>Hex HexNAc Phos.</summary>
            UNIMOD_Hex_1_HexNAc_1_Phos_1_ = 300001429,

            /// <summary>Hex HexNAc Sulf.</summary>
            UNIMOD_Hex_1_HexNAc_1_Sulf_1_ = 300001430,

            /// <summary>Hex NeuAc ---OR--- HexNAc Kdn.</summary>
            UNIMOD_Hex_1_NeuAc_1_ = 300001431,

            /// <summary>Hex NeuGc.</summary>
            UNIMOD_Hex_1_NeuGc_1_ = 300001432,

            /// <summary>HexNAc(3).</summary>
            UNIMOD_HexNAc_3_ = 300001433,

            /// <summary>HexNAc NeuAc.</summary>
            UNIMOD_HexNAc_1_NeuAc_1_ = 300001434,

            /// <summary>HexNAc NeuGc.</summary>
            UNIMOD_HexNAc_1_NeuGc_1_ = 300001435,

            /// <summary>Hex HexNAc dHex Me.</summary>
            UNIMOD_Hex_1_HexNAc_1_dHex_1_Me_1_ = 300001436,

            /// <summary>Hex HexNAc dHex Me(2).</summary>
            UNIMOD_Hex_1_HexNAc_1_dHex_1_Me_2_ = 300001437,

            /// <summary>Hex(2) HexNAc.</summary>
            UNIMOD_Hex_2_HexNAc_1_ = 300001438,

            /// <summary>Hex HexA HexNAc.</summary>
            UNIMOD_Hex_1_HexA_1_HexNAc_1_ = 300001439,

            /// <summary>Hex(2) HexNAc Me.</summary>
            UNIMOD_Hex_2_HexNAc_1_Me_1_ = 300001440,

            /// <summary>Hex Pent(3).</summary>
            UNIMOD_Hex_1_Pent_3_ = 300001441,

            /// <summary>Hex NeuAc Pent.</summary>
            UNIMOD_Hex_1_NeuAc_1_Pent_1_ = 300001442,

            /// <summary>Hex(2) HexNAc Sulf.</summary>
            UNIMOD_Hex_2_HexNAc_1_Sulf_1_ = 300001443,

            /// <summary>Hex(2) NeuAc ---OR--- Hex HexNAc Kdn.</summary>
            UNIMOD_Hex_2_NeuAc_1_ = 300001444,

            /// <summary>Hex2 dHex2.</summary>
            UNIMOD_dHex_2_Hex_2_ = 300001445,

            /// <summary>DHex Hex(2) HexA.</summary>
            UNIMOD_dHex_1_Hex_2_HexA_1_ = 300001446,

            /// <summary>Hex HexNAc(2) Sulf.</summary>
            UNIMOD_Hex_1_HexNAc_2_Sulf_1_ = 300001447,

            /// <summary>Hex(4).</summary>
            UNIMOD_Hex_4_ = 300001448,

            /// <summary>DHex Hex(2) HexNAc(2) Pent.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_2_Pent_1_ = 300001449,

            /// <summary>Hex(2) HexNAc(2) NeuAc ---OR--- dHex Hex HexNAc(2) NeuGc.</summary>
            UNIMOD_Hex_2_HexNAc_2_NeuAc_1_ = 300001450,

            /// <summary>Hex(3) HexNAc(2) Pent.</summary>
            UNIMOD_Hex_3_HexNAc_2_Pent_1_ = 300001451,

            /// <summary>Hex(4) HexNAc(2).</summary>
            UNIMOD_Hex_4_HexNAc_2_ = 300001452,

            /// <summary>DHex Hex(4) HexNAc Pent.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_1_Pent_1_ = 300001453,

            /// <summary>DHex Hex(3) HexNAc(2) Pent.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_2_Pent_1_ = 300001454,

            /// <summary>Hex(3) HexNAc(2) NeuAc.</summary>
            UNIMOD_Hex_3_HexNAc_2_NeuAc_1_ = 300001455,

            /// <summary>Hex(4) HexNAc(2) Pent.</summary>
            UNIMOD_Hex_4_HexNAc_2_Pent_1_ = 300001456,

            /// <summary>Hex(3) HexNAc(3) Pent.</summary>
            UNIMOD_Hex_3_HexNAc_3_Pent_1_ = 300001457,

            /// <summary>Hex(5) HexNAc(2) Phos.</summary>
            UNIMOD_Hex_5_HexNAc_2_Phos_1_ = 300001458,

            /// <summary>DHex Hex(4) HexNAc(2) Pent.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_2_Pent_1_ = 300001459,

            /// <summary>Hex(7) HexNAc.</summary>
            UNIMOD_Hex_7_HexNAc_1_ = 300001460,

            /// <summary>Hex(4) HexNAc(2) NeuAc ---OR--- Hex(3) HexNAc(2) dHex NeuGc.</summary>
            UNIMOD_Hex_4_HexNAc_2_NeuAc_1_ = 300001461,

            /// <summary>DHex Hex(5) HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_2_ = 300001462,

            /// <summary>DHex Hex(3) HexNAc(3) Pent.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_3_Pent_1_ = 300001463,

            /// <summary>Hex(3) HexNAc(4) Sulf.</summary>
            UNIMOD_Hex_3_HexNAc_4_Sulf_1_ = 300001464,

            /// <summary>M6/Man6.</summary>
            UNIMOD_Hex_6_HexNAc_2_ = 300001465,

            /// <summary>Hex(4) HexNAc(3) Pent.</summary>
            UNIMOD_Hex_4_HexNAc_3_Pent_1_ = 300001466,

            /// <summary>DHex Hex(4) HexNAc(3).</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_3_ = 300001467,

            /// <summary>Hex(5) HexNAc(3).</summary>
            UNIMOD_Hex_5_HexNAc_3_ = 300001468,

            /// <summary>Hex(3) HexNAc(4) Pent.</summary>
            UNIMOD_Hex_3_HexNAc_4_Pent_1_ = 300001469,

            /// <summary>Hex(6) HexNAc(2) Phos.</summary>
            UNIMOD_Hex_6_HexNAc_2_Phos_1_ = 300001470,

            /// <summary>DHex Hex(4) HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_3_Sulf_1_ = 300001471,

            /// <summary>DHex Hex(5) HexNAc(2) Pent.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_2_Pent_1_ = 300001472,

            /// <summary>Hex(8) HexNAc.</summary>
            UNIMOD_Hex_8_HexNAc_1_ = 300001473,

            /// <summary>DHex Hex(3) HexNAc(3) Pent(2).</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_3_Pent_2_ = 300001474,

            /// <summary>DHex(2) Hex(3) HexNAc(3) Pent.</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_3_Pent_1_ = 300001475,

            /// <summary>DHex Hex(3) HexNAc(4) Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_4_Sulf_1_ = 300001476,

            /// <summary>DHex Hex(6) HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_6_HexNAc_2_ = 300001477,

            /// <summary>DHex Hex(4) HexNAc(3) Pent.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_3_Pent_1_ = 300001478,

            /// <summary>Hex(4) HexNAc(4) Sulf.</summary>
            UNIMOD_Hex_4_HexNAc_4_Sulf_1_ = 300001479,

            /// <summary>M7/Man7.</summary>
            UNIMOD_Hex_7_HexNAc_2_ = 300001480,

            /// <summary>DHex(2) Hex(4) HexNAc(3).</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_3_ = 300001481,

            /// <summary>Hex(5) HexNAc(3) Pent.</summary>
            UNIMOD_Hex_5_HexNAc_3_Pent_1_ = 300001482,

            /// <summary>Hex(4) HexNAc(3) NeuGc.</summary>
            UNIMOD_Hex_4_HexNAc_3_NeuGc_1_ = 300001483,

            /// <summary>DHex Hex(5) HexNAc(3).</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_3_ = 300001484,

            /// <summary>DHex Hex(3) HexNAc(4) Pent.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_4_Pent_1_ = 300001485,

            /// <summary>Hex(3) HexNAc(5) Sulf.</summary>
            UNIMOD_Hex_3_HexNAc_5_Sulf_1_ = 300001486,

            /// <summary>Hex(6) HexNAc(3).</summary>
            UNIMOD_Hex_6_HexNAc_3_ = 300001487,

            /// <summary>Hex(3) HexNAc(4) NeuAc ---OR--- Hex(2) HexNAc(4) dHex NeuGc.</summary>
            UNIMOD_Hex_3_HexNAc_4_NeuAc_1_ = 300001488,

            /// <summary>Hex(4) HexNAc(4) Pent.</summary>
            UNIMOD_Hex_4_HexNAc_4_Pent_1_ = 300001489,

            /// <summary>Hex(7) HexNAc(2) Phos.</summary>
            UNIMOD_Hex_7_HexNAc_2_Phos_1_ = 300001490,

            /// <summary>Hex(4) HexNAc(4) Me(2) Pent.</summary>
            UNIMOD_Hex_4_HexNAc_4_Me_2_Pent_1_ = 300001491,

            /// <summary>DHex Hex(3) HexNAc(3) Pent(3) ---OR--- Hex(4) HexNAc(2) dHex(2) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_3_Pent_3_ = 300001492,

            /// <summary>DHex Hex(5) HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_3_Sulf_1_ = 300001493,

            /// <summary>DHex(2) Hex(3) HexNAc(3) Pent(2).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_3_Pent_2_ = 300001494,

            /// <summary>Hex(6) HexNAc(3) Phos.</summary>
            UNIMOD_Hex_6_HexNAc_3_Phos_1_ = 300001495,

            /// <summary>Hex(4) HexNAc(5).</summary>
            UNIMOD_Hex_4_HexNAc_5_ = 300001496,

            /// <summary>DHex(3) Hex(3) HexNAc(3) Pent.</summary>
            UNIMOD_dHex_3_Hex_3_HexNAc_3_Pent_1_ = 300001497,

            /// <summary>DHex(2) Hex(4) HexNAc(3) Pent.</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_3_Pent_1_ = 300001498,

            /// <summary>DHex Hex(4) HexNAc(4) Sulf.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_4_Sulf_1_ = 300001499,

            /// <summary>DHex Hex(7) HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_7_HexNAc_2_ = 300001500,

            /// <summary>DHex Hex(4) HexNAc(3) NeuAc ---OR--- dHex(2) Hex(3) HexNAc(3) NeuGc.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_3_NeuAc_1_ = 300001501,

            /// <summary>Hex(7) HexNAc(2) Phos(2).</summary>
            UNIMOD_Hex_7_HexNAc_2_Phos_2_ = 300001502,

            /// <summary>Hex(5) HexNAc(4) Sulf.</summary>
            UNIMOD_Hex_5_HexNAc_4_Sulf_1_ = 300001503,

            /// <summary>M8/Man8.</summary>
            UNIMOD_Hex_8_HexNAc_2_ = 300001504,

            /// <summary>DHex Hex(3) HexNAc(4) Pent(2).</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_4_Pent_2_ = 300001505,

            /// <summary>DHex Hex(4) HexNAc(3) NeuGc ---OR--- Hex(5) HexNAc(3) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_3_NeuGc_1_ = 300001506,

            /// <summary>DHex(2) Hex(3) HexNAc(4) Pent.</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_4_Pent_1_ = 300001507,

            /// <summary>DHex Hex(3) HexNAc(5) Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_5_Sulf_1_ = 300001508,

            /// <summary>DHex Hex(6) HexNAc(3).</summary>
            UNIMOD_dHex_1_Hex_6_HexNAc_3_ = 300001509,

            /// <summary>DHex Hex(3) HexNAc(4) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_4_NeuAc_1_ = 300001510,

            /// <summary>DHex(3) Hex(3) HexNAc(4).</summary>
            UNIMOD_dHex_3_Hex_3_HexNAc_4_ = 300001511,

            /// <summary>DHex Hex(4) HexNAc(4) Pent.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_4_Pent_1_ = 300001512,

            /// <summary>Hex(4) HexNAc(5) Sulf.</summary>
            UNIMOD_Hex_4_HexNAc_5_Sulf_1_ = 300001513,

            /// <summary>Hex(7) HexNAc(3).</summary>
            UNIMOD_Hex_7_HexNAc_3_ = 300001514,

            /// <summary>DHex Hex(4) HexNAc(3) NeuAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_3_NeuAc_1_Sulf_1_ = 300001515,

            /// <summary>Hex(5) HexNAc(4) Me(2) Pent.</summary>
            UNIMOD_Hex_5_HexNAc_4_Me_2_Pent_1_ = 300001516,

            /// <summary>Hex(3) HexNAc(6) Sulf.</summary>
            UNIMOD_Hex_3_HexNAc_6_Sulf_1_ = 300001517,

            /// <summary>DHex Hex(6) HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_6_HexNAc_3_Sulf_1_ = 300001518,

            /// <summary>DHex Hex(4) HexNAc(5).</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_5_ = 300001519,

            /// <summary>DHex Hex(5) HexA HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_5_HexA_1_HexNAc_3_Sulf_1_ = 300001520,

            /// <summary>Hex(7) HexNAc(3) Phos.</summary>
            UNIMOD_Hex_7_HexNAc_3_Phos_1_ = 300001521,

            /// <summary>Hex(6) HexNAc(4) Me(3).</summary>
            UNIMOD_Hex_6_HexNAc_4_Me_3_ = 300001522,

            /// <summary>DHex(2) Hex(4) HexNAc(4) Sulf.</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_4_Sulf_1_ = 300001523,

            /// <summary>Hex(4) HexNAc(3) NeuAc(2).</summary>
            UNIMOD_Hex_4_HexNAc_3_NeuAc_2_ = 300001524,

            /// <summary>DHex Hex(3) HexNAc(4) Pent(3).</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_4_Pent_3_ = 300001525,

            /// <summary>DHex(2) Hex(5) HexNAc(3) Pent.</summary>
            UNIMOD_dHex_2_Hex_5_HexNAc_3_Pent_1_ = 300001526,

            /// <summary>DHex Hex(5) HexNAc(4) Sulf.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_4_Sulf_1_ = 300001527,

            /// <summary>DHex(2) Hex(3) HexNAc(4) Pent(2).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_4_Pent_2_ = 300001528,

            /// <summary>DHex Hex(5) HexNAc(3) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_3_NeuAc_1_ = 300001529,

            /// <summary>Hex(3) HexNAc(6) Sulf(2).</summary>
            UNIMOD_Hex_3_HexNAc_6_Sulf_2_ = 300001530,

            /// <summary>M9/Man9.</summary>
            UNIMOD_Hex_9_HexNAc_2_ = 300001531,

            /// <summary>Hex(4) HexNAc(6).</summary>
            UNIMOD_Hex_4_HexNAc_6_ = 300001532,

            /// <summary>DHex(3) Hex(3) HexNAc(4) Pent.</summary>
            UNIMOD_dHex_3_Hex_3_HexNAc_4_Pent_1_ = 300001533,

            /// <summary>DHex Hex(5) HexNAc(3) NeuGc ---OR--- Hex(6) HexNAc(3) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_3_NeuGc_1_ = 300001534,

            /// <summary>DHex(2) Hex(4) HexNAc(4) Pent.</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_4_Pent_1_ = 300001535,

            /// <summary>DHex Hex(4) HexNAc(5) Sulf.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_5_Sulf_1_ = 300001536,

            /// <summary>DHex Hex(7) HexNAc(3).</summary>
            UNIMOD_dHex_1_Hex_7_HexNAc_3_ = 300001537,

            /// <summary>DHex Hex(5) HexNAc(4) Pent.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_4_Pent_1_ = 300001538,

            /// <summary>DHex Hex(5) HexA HexNAc(3) Sulf(2).</summary>
            UNIMOD_dHex_1_Hex_5_HexA_1_HexNAc_3_Sulf_2_ = 300001539,

            /// <summary>Hex(3) HexNAc(7).</summary>
            UNIMOD_Hex_3_HexNAc_7_ = 300001540,

            /// <summary>DHex(2) Hex(5) HexNAc(4).</summary>
            UNIMOD_dHex_2_Hex_5_HexNAc_4_ = 300001541,

            /// <summary>DHex(2) Hex(4) HexNAc(3) NeuAc Sulf.</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_3_NeuAc_1_Sulf_1_ = 300001542,

            /// <summary>DHex Hex(5) HexNAc(4) Sulf(2).</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_4_Sulf_2_ = 300001543,

            /// <summary>DHex Hex(5) HexNAc(4) Me(2) Pent.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_4_Me_2_Pent_1_ = 300001544,

            /// <summary>Hex(5) HexNAc(4) NeuGc.</summary>
            UNIMOD_Hex_5_HexNAc_4_NeuGc_1_ = 300001545,

            /// <summary>DHex Hex(3) HexNAc(6) Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_6_Sulf_1_ = 300001546,

            /// <summary>DHex Hex(6) HexNAc(4).</summary>
            UNIMOD_dHex_1_Hex_6_HexNAc_4_ = 300001547,

            /// <summary>DHex Hex(5) HexNAc(3) NeuAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_3_NeuAc_1_Sulf_1_ = 300001548,

            /// <summary>Hex(7) HexNAc(4).</summary>
            UNIMOD_Hex_7_HexNAc_4_ = 300001549,

            /// <summary>DHex Hex(5) HexNAc(3) NeuGc Sulf.</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_3_NeuGc_1_Sulf_1_ = 300001550,

            /// <summary>Hex(4) HexNAc(5) NeuAc.</summary>
            UNIMOD_Hex_4_HexNAc_5_NeuAc_1_ = 300001551,

            /// <summary>Hex(6) HexNAc(4) Me(3) Pent.</summary>
            UNIMOD_Hex_6_HexNAc_4_Me_3_Pent_1_ = 300001552,

            /// <summary>DHex Hex(7) HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_7_HexNAc_3_Sulf_1_ = 300001553,

            /// <summary>DHex Hex(7) HexNAc(3) Phos.</summary>
            UNIMOD_dHex_1_Hex_7_HexNAc_3_Phos_1_ = 300001554,

            /// <summary>DHex Hex(5) HexNAc(5).</summary>
            UNIMOD_dHex_1_Hex_5_HexNAc_5_ = 300001555,

            /// <summary>DHex Hex(4) HexNAc(4) NeuAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_4_NeuAc_1_Sulf_1_ = 300001556,

            /// <summary>DHex(3) Hex(4) HexNAc(4) Sulf.</summary>
            UNIMOD_dHex_3_Hex_4_HexNAc_4_Sulf_1_ = 300001557,

            /// <summary>Hex(3) HexNAc(7) Sulf.</summary>
            UNIMOD_Hex_3_HexNAc_7_Sulf_1_ = 300001558,

            /// <summary>A3G3.</summary>
            UNIMOD_Hex_6_HexNAc_5_ = 300001559,

            /// <summary>Hex(5) HexNAc(4) NeuAc Sulf.</summary>
            UNIMOD_Hex_5_HexNAc_4_NeuAc_1_Sulf_1_ = 300001560,

            /// <summary>Hex(3) HexNAc(6) NeuAc.</summary>
            UNIMOD_Hex_3_HexNAc_6_NeuAc_1_ = 300001561,

            /// <summary>DHex(2) Hex(3) HexNAc(6).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_6_ = 300001562,

            /// <summary>Hex HexNAc NeuGc.</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuGc_1_ = 300001563,

            /// <summary>DHex Hex(2) HexNAc.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_1_ = 300001564,

            /// <summary>HexNAc(3) Sulf.</summary>
            UNIMOD_HexNAc_3_Sulf_1_ = 300001565,

            /// <summary>Hex(3) HexNAc.</summary>
            UNIMOD_Hex_3_HexNAc_1_ = 300001566,

            /// <summary>Hex HexNAc Kdn Sulf.</summary>
            UNIMOD_Hex_1_HexNAc_1_Kdn_1_Sulf_1_ = 300001567,

            /// <summary>HexNAc(2) NeuAc.</summary>
            UNIMOD_HexNAc_2_NeuAc_1_ = 300001568,

            /// <summary>HexNAc Kdn(2) ---OR--- Hex(2) HexNAc HexA.</summary>
            UNIMOD_HexNAc_1_Kdn_2_ = 300001570,

            /// <summary>Hex(3) HexNAc Me.</summary>
            UNIMOD_Hex_3_HexNAc_1_Me_1_ = 300001571,

            /// <summary>Hex(2) HexA Pent Sulf.</summary>
            UNIMOD_Hex_2_HexA_1_Pent_1_Sulf_1_ = 300001572,

            /// <summary>HexNAc(2) NeuGc.</summary>
            UNIMOD_HexNAc_2_NeuGc_1_ = 300001573,

            /// <summary>Hex(4) Phos.</summary>
            UNIMOD_Hex_4_Phos_1_ = 300001575,

            /// <summary>Hex HexNAc NeuAc Sulf.</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuAc_1_Sulf_1_ = 300001577,

            /// <summary>Hex HexA HexNAc(2).</summary>
            UNIMOD_Hex_1_HexA_1_HexNAc_2_ = 300001578,

            /// <summary>DHex Hex(2) HexNAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_1_Sulf_1_ = 300001579,

            /// <summary>DHex HexNAc(3).</summary>
            UNIMOD_dHex_1_HexNAc_3_ = 300001580,

            /// <summary>DHex Hex HexNAc Kdn ---OR--- Hex(2) dHex NeuAc.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_1_Kdn_1_ = 300001581,

            /// <summary>Hex HexNAc(3).</summary>
            UNIMOD_Hex_1_HexNAc_3_ = 300001582,

            /// <summary>HexNAc(2) NeuAc Sulf.</summary>
            UNIMOD_HexNAc_2_NeuAc_1_Sulf_1_ = 300001583,

            /// <summary>DHex(2) Hex(3).</summary>
            UNIMOD_dHex_2_Hex_3_ = 300001584,

            /// <summary>Hex(2) HexA HexNAc Sulf.</summary>
            UNIMOD_Hex_2_HexA_1_HexNAc_1_Sulf_1_ = 300001585,

            /// <summary>DHex(2) Hex(2) HexA.</summary>
            UNIMOD_dHex_2_Hex_2_HexA_1_ = 300001586,

            /// <summary>DHex Hex HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_2_Sulf_1_ = 300001587,

            /// <summary>DHex Hex HexNAc NeuAc.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_1_NeuAc_1_ = 300001588,

            /// <summary>Hex(2) HexNAc(2) Sulf.</summary>
            UNIMOD_Hex_2_HexNAc_2_Sulf_1_ = 300001589,

            /// <summary>Hex(5).</summary>
            UNIMOD_Hex_5_ = 300001590,

            /// <summary>HexNAc(4).</summary>
            UNIMOD_HexNAc_4_ = 300001591,

            /// <summary>HexNAc NeuGc(2).</summary>
            UNIMOD_HexNAc_1_NeuGc_2_ = 300001592,

            /// <summary>DHex Hex HexNAc NeuGc ---OR--- Hex(2) HexNAc NeuAc.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_1_NeuGc_1_ = 300001593,

            /// <summary>DHex(2) Hex(2) HexNAc.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_1_ = 300001594,

            /// <summary>Hex(2) HexNAc NeuGc.</summary>
            UNIMOD_Hex_2_HexNAc_1_NeuGc_1_ = 300001595,

            /// <summary>DHex Hex(3) HexNAc.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_1_ = 300001596,

            /// <summary>DHex Hex(2) HexA HexNAc.</summary>
            UNIMOD_dHex_1_Hex_2_HexA_1_HexNAc_1_ = 300001597,

            /// <summary>Hex HexNAc(3) Sulf.</summary>
            UNIMOD_Hex_1_HexNAc_3_Sulf_1_ = 300001598,

            /// <summary>Hex(4) HexNAc.</summary>
            UNIMOD_Hex_4_HexNAc_1_ = 300001599,

            /// <summary>Hex HexNAc(2) NeuAc.</summary>
            UNIMOD_Hex_1_HexNAc_2_NeuAc_1_ = 300001600,

            /// <summary>Hex HexNAc(2) NeuGc.</summary>
            UNIMOD_Hex_1_HexNAc_2_NeuGc_1_ = 300001602,

            /// <summary>Hex(5) Phos.</summary>
            UNIMOD_Hex_5_Phos_1_ = 300001604,

            /// <summary>DHex(2) Hex HexNAc Kdn.</summary>
            UNIMOD_dHex_2_Hex_1_HexNAc_1_Kdn_1_ = 300001606,

            /// <summary>DHex Hex(3) HexNAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_1_Sulf_1_ = 300001607,

            /// <summary>DHex Hex HexNAc(3).</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_3_ = 300001608,

            /// <summary>DHex Hex(2) HexA HexNAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_2_HexA_1_HexNAc_1_Sulf_1_ = 300001609,

            /// <summary>Hex(2) HexNAc(3).</summary>
            UNIMOD_Hex_2_HexNAc_3_ = 300001610,

            /// <summary>Hex HexNAc(2) NeuAc Sulf.</summary>
            UNIMOD_Hex_1_HexNAc_2_NeuAc_1_Sulf_1_ = 300001611,

            /// <summary>DHex(2) Hex(4).</summary>
            UNIMOD_dHex_2_Hex_4_ = 300001612,

            /// <summary>DHex(2) HexNAc(2) Kdn.</summary>
            UNIMOD_dHex_2_HexNAc_2_Kdn_1_ = 300001614,

            /// <summary>DHex Hex(2) HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_2_Sulf_1_ = 300001615,

            /// <summary>DHex HexNAc(4).</summary>
            UNIMOD_dHex_1_HexNAc_4_ = 300001616,

            /// <summary>Hex HexNAc NeuAc NeuGc.</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuAc_1_NeuGc_1_ = 300001617,

            /// <summary>DHex Hex HexNAc(2) Kdn ---OR--- Hex(2) HexNAc dHex NeuAc.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_2_Kdn_1_ = 300001618,

            /// <summary>Hex HexNAc NeuGc(2).</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuGc_2_ = 300001619,

            /// <summary>Ac Hex HexNAc NeuAc(2).</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuAc_2_Ac_1_ = 300001620,

            /// <summary>DHex(2) Hex(2) HexA HexNAc.</summary>
            UNIMOD_dHex_2_Hex_2_HexA_1_HexNAc_1_ = 300001621,

            /// <summary>DHex Hex HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_3_Sulf_1_ = 300001622,

            /// <summary>Hex(2) HexA NeuAc Pent Sulf.</summary>
            UNIMOD_Hex_2_HexA_1_NeuAc_1_Pent_1_Sulf_1_ = 300001623,

            /// <summary>DHex Hex HexNAc(2) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_2_NeuAc_1_ = 300001624,

            /// <summary>DHex Hex(3) HexA HexNAc.</summary>
            UNIMOD_dHex_1_Hex_3_HexA_1_HexNAc_1_ = 300001625,

            /// <summary>Hex(2) HexNAc(3) Sulf.</summary>
            UNIMOD_Hex_2_HexNAc_3_Sulf_1_ = 300001626,

            /// <summary>Hex(5) HexNAc.</summary>
            UNIMOD_Hex_5_HexNAc_1_ = 300001627,

            /// <summary>HexNAc(5).</summary>
            UNIMOD_HexNAc_5_ = 300001628,

            /// <summary>Ac(2) Hex HexNAc NeuAc(2).</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuAc_2_Ac_2_ = 300001630,

            /// <summary>Hex(2) HexNAc(2) NeuGc.</summary>
            UNIMOD_Hex_2_HexNAc_2_NeuGc_1_ = 300001631,

            /// <summary>Hex(5) Phos(3).</summary>
            UNIMOD_Hex_5_Phos_3_ = 300001632,

            /// <summary>Hex(6) Phos.</summary>
            UNIMOD_Hex_6_Phos_1_ = 300001633,

            /// <summary>DHex Hex(2) HexA HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_2_HexA_1_HexNAc_2_ = 300001634,

            /// <summary>DHex(2) Hex(3) HexNAc Sulf.</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_1_Sulf_1_ = 300001635,

            /// <summary>Hex HexNAc(3) NeuAc.</summary>
            UNIMOD_Hex_1_HexNAc_3_NeuAc_1_ = 300001636,

            /// <summary>DHex(2) Hex HexNAc(3).</summary>
            UNIMOD_dHex_2_Hex_1_HexNAc_3_ = 300001637,

            /// <summary>Hex HexNAc(3) NeuGc.</summary>
            UNIMOD_Hex_1_HexNAc_3_NeuGc_1_ = 300001638,

            /// <summary>DHex Hex HexNAc(2) NeuAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_2_NeuAc_1_Sulf_1_ = 300001639,

            /// <summary>DHex Hex(3) HexA HexNAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexA_1_HexNAc_1_Sulf_1_ = 300001640,

            /// <summary>DHex Hex HexA HexNAc(3).</summary>
            UNIMOD_dHex_1_Hex_1_HexA_1_HexNAc_3_ = 300001641,

            /// <summary>Hex(2) HexNAc(2) NeuAc Sulf.</summary>
            UNIMOD_Hex_2_HexNAc_2_NeuAc_1_Sulf_1_ = 300001642,

            /// <summary>DHex(2) Hex(2) HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_2_Sulf_1_ = 300001643,

            /// <summary>DHex(2) Hex HexNAc(2) Kdn ---OR--- Hex(2) HexNAc dHex(2) NeuAc.</summary>
            UNIMOD_dHex_2_Hex_1_HexNAc_2_Kdn_1_ = 300001644,

            /// <summary>DHex Hex HexNAc(4).</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_4_ = 300001645,

            /// <summary>Hex(2) HexNAc(4).</summary>
            UNIMOD_Hex_2_HexNAc_4_ = 300001646,

            /// <summary>Hex(2) HexNAc NeuGc(2).</summary>
            UNIMOD_Hex_2_HexNAc_1_NeuGc_2_ = 300001647,

            /// <summary>DHex(2) Hex(4) HexNAc.</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_1_ = 300001648,

            /// <summary>Hex HexNAc(2) NeuAc(2).</summary>
            UNIMOD_Hex_1_HexNAc_2_NeuAc_2_ = 300001649,

            /// <summary>DHex(2) Hex HexNAc(2) NeuAc.</summary>
            UNIMOD_dHex_2_Hex_1_HexNAc_2_NeuAc_1_ = 300001650,

            /// <summary>DHex Hex(2) HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_3_Sulf_1_ = 300001651,

            /// <summary>DHex HexNAc(5).</summary>
            UNIMOD_dHex_1_HexNAc_5_ = 300001652,

            /// <summary>DHex(2) Hex HexNAc(2) NeuGc ---OR--- Hex(2) HexNAc(2) dHex NeuAc ---OR--- Hex HexNAc(3) dHex Kdn.</summary>
            UNIMOD_dHex_2_Hex_1_HexNAc_2_NeuGc_1_ = 300001653,

            /// <summary>DHex(3) Hex(2) HexNAc(2).</summary>
            UNIMOD_dHex_3_Hex_2_HexNAc_2_ = 300001654,

            /// <summary>Hex(3) HexNAc(3) Sulf.</summary>
            UNIMOD_Hex_3_HexNAc_3_Sulf_1_ = 300001655,

            /// <summary>DHex(2) Hex(2) HexNAc(2) Sulf(2).</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_2_Sulf_2_ = 300001656,

            /// <summary>DHex Hex(2) HexNAc(2) NeuGc ---OR--- Hex(3) HexNAc(2) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_2_NeuGc_1_ = 300001657,

            /// <summary>DHex Hex HexNAc(3) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_3_NeuAc_1_ = 300001658,

            /// <summary>Hex(6) Phos(3).</summary>
            UNIMOD_Hex_6_Phos_3_ = 300001659,

            /// <summary>DHex Hex(3) HexA HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_3_HexA_1_HexNAc_2_ = 300001660,

            /// <summary>DHex Hex HexNAc(3) NeuGc ---OR--- Hex(2) HexNAc(3) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_3_NeuGc_1_ = 300001661,

            /// <summary>Hex HexNAc(2) NeuAc(2) Sulf.</summary>
            UNIMOD_Hex_1_HexNAc_2_NeuAc_2_Sulf_1_ = 300001662,

            /// <summary>DHex(2) Hex(3) HexA HexNAc Sulf.</summary>
            UNIMOD_dHex_2_Hex_3_HexA_1_HexNAc_1_Sulf_1_ = 300001663,

            /// <summary>Hex HexNAc NeuAc(3).</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuAc_3_ = 300001664,

            /// <summary>Hex(2) HexNAc(3) NeuGc.</summary>
            UNIMOD_Hex_2_HexNAc_3_NeuGc_1_ = 300001665,

            /// <summary>DHex Hex(2) HexNAc(2) NeuAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_2_NeuAc_1_Sulf_1_ = 300001666,

            /// <summary>DHex(3) Hex HexNAc(2) Kdn.</summary>
            UNIMOD_dHex_3_Hex_1_HexNAc_2_Kdn_1_ = 300001667,

            /// <summary>DHex(2) Hex(3) HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_2_Sulf_1_ = 300001668,

            /// <summary>DHex(2) Hex(2) HexNAc(2) Kdn.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_2_Kdn_1_ = 300001669,

            /// <summary>DHex(2) Hex(2) HexA HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_2_Hex_2_HexA_1_HexNAc_2_Sulf_1_ = 300001670,

            /// <summary>DHex Hex(2) HexNAc(4).</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_4_ = 300001671,

            /// <summary>Hex HexNAc NeuGc(3).</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuGc_3_ = 300001672,

            /// <summary>DHex Hex HexNAc(3) NeuAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_3_NeuAc_1_Sulf_1_ = 300001673,

            /// <summary>DHex Hex(3) HexA HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexA_1_HexNAc_2_Sulf_1_ = 300001674,

            /// <summary>DHex Hex HexNAc(2) NeuAc(2).</summary>
            UNIMOD_dHex_1_Hex_1_HexNAc_2_NeuAc_2_ = 300001675,

            /// <summary>DHex(3) HexNAc(3) Kdn.</summary>
            UNIMOD_dHex_3_HexNAc_3_Kdn_1_ = 300001676,

            /// <summary>Hex(2) HexNAc(3) NeuAc Sulf.</summary>
            UNIMOD_Hex_2_HexNAc_3_NeuAc_1_Sulf_1_ = 300001678,

            /// <summary>DHex(2) Hex(2) HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_3_Sulf_1_ = 300001679,

            /// <summary>DHex(2) HexNAc(5).</summary>
            UNIMOD_dHex_2_HexNAc_5_ = 300001680,

            /// <summary>Hex(2) HexNAc(2) NeuAc(2).</summary>
            UNIMOD_Hex_2_HexNAc_2_NeuAc_2_ = 300001681,

            /// <summary>DHex(2) Hex(2) HexNAc(2) NeuAc ---OR--- Hex HexNAc(3) dHex(2) Kdn.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_2_NeuAc_1_ = 300001682,

            /// <summary>DHex Hex(3) HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_3_Sulf_1_ = 300001683,

            /// <summary>DHex(2) Hex(2) HexNAc(2) NeuGc ---OR--- Hex(3) HexNAc(2) dHex NeuAc ---OR--- Hex(2) HexNAc(3) dHex Kdn.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_2_NeuGc_1_ = 300001684,

            /// <summary>Hex(2) HexNAc(5).</summary>
            UNIMOD_Hex_2_HexNAc_5_ = 300001685,

            /// <summary>DHex Hex(3) HexNAc(2) NeuGc.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_2_NeuGc_1_ = 300001686,

            /// <summary>Hex HexNAc(3) NeuAc(2).</summary>
            UNIMOD_Hex_1_HexNAc_3_NeuAc_2_ = 300001687,

            /// <summary>DHex Hex(2) HexNAc(3) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_3_NeuAc_1_ = 300001688,

            /// <summary>DHex(3) Hex(2) HexNAc(3).</summary>
            UNIMOD_dHex_3_Hex_2_HexNAc_3_ = 300001689,

            /// <summary>Hex(7) Phos(3).</summary>
            UNIMOD_Hex_7_Phos_3_ = 300001690,

            /// <summary>DHex Hex(4) HexA HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_4_HexA_1_HexNAc_2_ = 300001691,

            /// <summary>Hex(3) HexNAc(3) NeuAc ---OR--- Hex(2) HexNAc(3) dHex NeuGc ---OR--- Hex(2) HexNAc(4) Kdn.</summary>
            UNIMOD_Hex_3_HexNAc_3_NeuAc_1_ = 300001692,

            /// <summary>DHex Hex(3) HexA(2) HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_3_HexA_2_HexNAc_2_ = 300001693,

            /// <summary>Hex(2) HexNAc(2) NeuAc(2) Sulf.</summary>
            UNIMOD_Hex_2_HexNAc_2_NeuAc_2_Sulf_1_ = 300001694,

            /// <summary>DHex(2) Hex(2) HexNAc(2) NeuAc Sulf.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_2_NeuAc_1_Sulf_1_ = 300001695,

            /// <summary>Hex(3) HexNAc(3) NeuGc.</summary>
            UNIMOD_Hex_3_HexNAc_3_NeuGc_1_ = 300001696,

            /// <summary>DHex(4) Hex HexNAc(2) Kdn.</summary>
            UNIMOD_dHex_4_Hex_1_HexNAc_2_Kdn_1_ = 300001697,

            /// <summary>DHex(3) Hex(2) HexNAc(2) Kdn.</summary>
            UNIMOD_dHex_3_Hex_2_HexNAc_2_Kdn_1_ = 300001698,

            /// <summary>DHex(3) Hex(2) HexA HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_3_Hex_2_HexA_1_HexNAc_2_Sulf_1_ = 300001699,

            /// <summary>Hex(2) HexNAc(4) NeuAc.</summary>
            UNIMOD_Hex_2_HexNAc_4_NeuAc_1_ = 300001700,

            /// <summary>DHex(2) Hex(2) HexNAc(4).</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_4_ = 300001701,

            /// <summary>DHex(2) Hex(3) HexA HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_2_Hex_3_HexA_1_HexNAc_2_Sulf_1_ = 300001702,

            /// <summary>DHex(4) HexNAc(3) Kdn.</summary>
            UNIMOD_dHex_4_HexNAc_3_Kdn_1_ = 300001703,

            /// <summary>Hex(2) HexNAc NeuGc(3).</summary>
            UNIMOD_Hex_2_HexNAc_1_NeuGc_3_ = 300001705,

            /// <summary>DHex(4) Hex HexNAc Kdn(2).</summary>
            UNIMOD_dHex_4_Hex_1_HexNAc_1_Kdn_2_ = 300001706,

            /// <summary>DHex Hex(2) HexNAc(3) NeuAc Sulf.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_3_NeuAc_1_Sulf_1_ = 300001707,

            /// <summary>DHex Hex(2) HexNAc(2) NeuAc(2).</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_2_NeuAc_2_ = 300001708,

            /// <summary>DHex(3) Hex HexNAc(3) Kdn.</summary>
            UNIMOD_dHex_3_Hex_1_HexNAc_3_Kdn_1_ = 300001709,

            /// <summary>Hex(3) HexNAc(3) NeuAc Sulf.</summary>
            UNIMOD_Hex_3_HexNAc_3_NeuAc_1_Sulf_1_ = 300001711,

            /// <summary>Hex(3) HexNAc(2) NeuAc(2).</summary>
            UNIMOD_Hex_3_HexNAc_2_NeuAc_2_ = 300001712,

            /// <summary>Hex(3) HexNAc(3) NeuGc Sulf.</summary>
            UNIMOD_Hex_3_HexNAc_3_NeuGc_1_Sulf_1_ = 300001713,

            /// <summary>DHex Hex(2) HexNAc(2) NeuGc(2).</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_2_NeuGc_2_ = 300001714,

            /// <summary>DHex(2) Hex(3) HexNAc(2) NeuGc ---OR--- Hex(4) HexNAc(2) dHex NeuAc.</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_2_NeuGc_1_ = 300001715,

            /// <summary>DHex Hex(3) HexA HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexA_1_HexNAc_3_Sulf_1_ = 300001716,

            /// <summary>Hex(2) HexNAc(3) NeuAc(2).</summary>
            UNIMOD_Hex_2_HexNAc_3_NeuAc_2_ = 300001717,

            /// <summary>DHex(2) Hex(2) HexNAc(3) NeuAc.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_3_NeuAc_1_ = 300001718,

            /// <summary>DHex(4) Hex(2) HexNAc(3).</summary>
            UNIMOD_dHex_4_Hex_2_HexNAc_3_ = 300001719,

            /// <summary>Hex(2) HexNAc(3) NeuAc NeuGc.</summary>
            UNIMOD_Hex_2_HexNAc_3_NeuAc_1_NeuGc_1_ = 300001720,

            /// <summary>DHex(2) Hex(2) HexNAc(3) NeuGc ---OR--- Hex(3) HexNAc(3) dHex NeuAc ---OR--- Hex(2) HexNAc(4) dHex Kdn.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_3_NeuGc_1_ = 300001721,

            /// <summary>DHex(3) Hex(3) HexNAc(3).</summary>
            UNIMOD_dHex_3_Hex_3_HexNAc_3_ = 300001722,

            /// <summary>Hex(8) Phos(3).</summary>
            UNIMOD_Hex_8_Phos_3_ = 300001723,

            /// <summary>DHex Hex(2) HexNAc(2) NeuAc(2) Sulf.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_2_NeuAc_2_Sulf_1_ = 300001724,

            /// <summary>Hex(2) HexNAc(3) NeuGc(2).</summary>
            UNIMOD_Hex_2_HexNAc_3_NeuGc_2_ = 300001725,

            /// <summary>DHex(4) Hex(2) HexNAc(2) Kdn.</summary>
            UNIMOD_dHex_4_Hex_2_HexNAc_2_Kdn_1_ = 300001726,

            /// <summary>DHex Hex(2) HexNAc(4) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_4_NeuAc_1_ = 300001727,

            /// <summary>DHex(3) Hex(2) HexNAc(4).</summary>
            UNIMOD_dHex_3_Hex_2_HexNAc_4_ = 300001728,

            /// <summary>Hex HexNAc NeuGc(4).</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuGc_4_ = 300001729,

            /// <summary>DHex(4) Hex HexNAc(3) Kdn.</summary>
            UNIMOD_dHex_4_Hex_1_HexNAc_3_Kdn_1_ = 300001730,

            /// <summary>Hex(4) HexNAc(4) Sulf(2).</summary>
            UNIMOD_Hex_4_HexNAc_4_Sulf_2_ = 300001732,

            /// <summary>DHex(3) Hex(2) HexNAc(3) Kdn ---OR--- Hex(3) HexNAc(2) dHex(3) NeuAc.</summary>
            UNIMOD_dHex_3_Hex_2_HexNAc_3_Kdn_1_ = 300001733,

            /// <summary>DHex(2) Hex(2) HexNAc(5).</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_5_ = 300001735,

            /// <summary>DHex(2) Hex(3) HexA HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_2_Hex_3_HexA_1_HexNAc_3_Sulf_1_ = 300001736,

            /// <summary>DHex Hex(4) HexA HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_1_Hex_4_HexA_1_HexNAc_3_Sulf_1_ = 300001737,

            /// <summary>Hex(3) HexNAc(3) NeuAc(2).</summary>
            UNIMOD_Hex_3_HexNAc_3_NeuAc_2_ = 300001738,

            /// <summary>DHex(2) Hex(3) HexNAc(3) NeuAc ---OR--- Hex(2) HexNAc(4) dHex(2) Kdn.</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_3_NeuAc_1_ = 300001739,

            /// <summary>DHex(4) Hex(3) HexNAc(3).</summary>
            UNIMOD_dHex_4_Hex_3_HexNAc_3_ = 300001740,

            /// <summary>Hex(9) Phos(3).</summary>
            UNIMOD_Hex_9_Phos_3_ = 300001742,

            /// <summary>DHex(2) HexNAc(7).</summary>
            UNIMOD_dHex_2_HexNAc_7_ = 300001743,

            /// <summary>Hex(2) HexNAc NeuGc(4).</summary>
            UNIMOD_Hex_2_HexNAc_1_NeuGc_4_ = 300001744,

            /// <summary>Hex(3) HexNAc(3) NeuAc(2) Sulf.</summary>
            UNIMOD_Hex_3_HexNAc_3_NeuAc_2_Sulf_1_ = 300001745,

            /// <summary>DHex(2) Hex(3) HexNAc(5).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_5_ = 300001746,

            /// <summary>DHex Hex(2) HexNAc(2) NeuGc(3).</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_2_NeuGc_3_ = 300001747,

            /// <summary>DHex(2) Hex(4) HexA HexNAc(3) Sulf.</summary>
            UNIMOD_dHex_2_Hex_4_HexA_1_HexNAc_3_Sulf_1_ = 300001748,

            /// <summary>Hex(2) HexNAc(3) NeuAc(3).</summary>
            UNIMOD_Hex_2_HexNAc_3_NeuAc_3_ = 300001749,

            /// <summary>DHex Hex(3) HexNAc(3) NeuAc(2).</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_3_NeuAc_2_ = 300001750,

            /// <summary>DHex(3) Hex(3) HexNAc(3) NeuAc.</summary>
            UNIMOD_dHex_3_Hex_3_HexNAc_3_NeuAc_1_ = 300001751,

            /// <summary>Hex(2) HexNAc(3) NeuGc(3).</summary>
            UNIMOD_Hex_2_HexNAc_3_NeuGc_3_ = 300001752,

            /// <summary>Hex(10) Phos(3).</summary>
            UNIMOD_Hex_10_Phos_3_ = 300001753,

            /// <summary>DHex Hex(2) HexNAc(4) NeuAc(2).</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_4_NeuAc_2_ = 300001754,

            /// <summary>Hex HexNAc NeuGc(5).</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuGc_5_ = 300001755,

            /// <summary>Hex(4) HexNAc(4) NeuAc Sulf(2).</summary>
            UNIMOD_Hex_4_HexNAc_4_NeuAc_1_Sulf_2_ = 300001756,

            /// <summary>Hex(4) HexNAc(4) NeuGc Sulf(2).</summary>
            UNIMOD_Hex_4_HexNAc_4_NeuGc_1_Sulf_2_ = 300001757,

            /// <summary>DHex(2) Hex(3) HexNAc(3) NeuAc(2).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_3_NeuAc_2_ = 300001758,

            /// <summary>Hex(4) HexNAc(4) NeuAc Sulf(3).</summary>
            UNIMOD_Hex_4_HexNAc_4_NeuAc_1_Sulf_3_ = 300001759,

            /// <summary>DHex(2) Hex(2) HexNAc(2).</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_2_ = 300001760,

            /// <summary>DHex Hex(3) HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_2_ = 300001761,

            /// <summary>DHex Hex(2) HexNAc(3).</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_3_ = 300001762,

            /// <summary>Hex(3) HexNAc(3).</summary>
            UNIMOD_Hex_3_HexNAc_3_ = 300001763,

            /// <summary>DHex Hex(3) HexNAc(2) Sulf.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_2_Sulf_1_ = 300001764,

            /// <summary>DHex(2) Hex(3) HexNAc(2).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_2_ = 300001765,

            /// <summary>DHex Hex(4) HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_2_ = 300001766,

            /// <summary>DHex(2) Hex(2) HexNAc(3).</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_3_ = 300001767,

            /// <summary>DHex Hex(3) HexNAc(3).</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_3_ = 300001768,

            /// <summary>Hex(4) HexNAc(3).</summary>
            UNIMOD_Hex_4_HexNAc_3_ = 300001769,

            /// <summary>DHex(2) Hex(4) HexNAc(2).</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_2_ = 300001770,

            /// <summary>DHex(2) Hex(3) HexNAc(3).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_3_ = 300001771,

            /// <summary>A3.</summary>
            UNIMOD_Hex_3_HexNAc_5_ = 300001772,

            /// <summary>Hex(4) HexNAc(3) NeuAc ---OR--- Hex(3) HexNAc(4) Kdn.</summary>
            UNIMOD_Hex_4_HexNAc_3_NeuAc_1_ = 300001773,

            /// <summary>DHex(2) Hex(3) HexNAc(4).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_4_ = 300001774,

            /// <summary>DHex Hex(3) HexNAc(5).</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_5_ = 300001775,

            /// <summary>A4.</summary>
            UNIMOD_Hex_3_HexNAc_6_ = 300001776,

            /// <summary>Hex(4) HexNAc(4) NeuAc.</summary>
            UNIMOD_Hex_4_HexNAc_4_NeuAc_1_ = 300001777,

            /// <summary>DHex(2) Hex(4) HexNAc(4) ---OR--- Hex(4) HexNAc(4) dHex Pent Me.</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_4_ = 300001778,

            /// <summary>Hex(6) HexNAc(4).</summary>
            UNIMOD_Hex_6_HexNAc_4_ = 300001779,

            /// <summary>Hex(5) HexNAc(5).</summary>
            UNIMOD_Hex_5_HexNAc_5_ = 300001780,

            /// <summary>DHex Hex(3) HexNAc(6).</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_6_ = 300001781,

            /// <summary>DHex Hex(4) HexNAc(4) NeuAc ---OR--- Hex(3) HexNAc(5) dHex Kdn.</summary>
            UNIMOD_dHex_1_Hex_4_HexNAc_4_NeuAc_1_ = 300001782,

            /// <summary>DHex(3) Hex(4) HexNAc(4).</summary>
            UNIMOD_dHex_3_Hex_4_HexNAc_4_ = 300001783,

            /// <summary>DHex Hex(3) HexNAc(5) NeuAc.</summary>
            UNIMOD_dHex_1_Hex_3_HexNAc_5_NeuAc_1_ = 300001784,

            /// <summary>DHex(2) Hex(4) HexNAc(5).</summary>
            UNIMOD_dHex_2_Hex_4_HexNAc_5_ = 300001785,

            /// <summary>Ac Hex HexNAc NeuAc.</summary>
            UNIMOD_Hex_1_HexNAc_1_NeuAc_1_Ac_1_ = 300001786,

            /// <summary>13C(2) 15N(2).</summary>
            UNIMOD_Label_13C_2_15N_2_ = 300001787,

            /// <summary>Ammonium-quenched monolink of DSS/BS3 crosslinker.</summary>
            UNIMOD_Xlink_DSS_155_ = 300001789,

            /// <summary>SUMOylation by Giardia lamblia.</summary>
            UNIMOD_NQIGG = 300001799,

            /// <summary>Carboxyethylpyrrole.</summary>
            UNIMOD_Carboxyethylpyrrole = 300001800,

            /// <summary>Fluorescein-tyramine adduct by peroxidase activity.</summary>
            UNIMOD_Fluorescein_tyramine = 300001801,

            /// <summary>Transamidation of glycine ethyl ester to glutamine.</summary>
            UNIMOD_GEE = 300001824,

            /// <summary>Simulate peptide-RNA conjugates.</summary>
            UNIMOD_RNPXL = 300001825,

            /// <summary>Pyro-Glu from E + Methylation.</summary>
            UNIMOD_Glu__pyro_Glu_Methyl = 300001826,

            /// <summary>Pyro-Glu from E + Methylation Medium.</summary>
            UNIMOD_Glu__pyro_Glu_Methyl_2H_2_13C_1_ = 300001827,

            /// <summary>LeumethylArgGlyGly.</summary>
            UNIMOD_LRGG_methyl = 300001828,

            /// <summary>LeudimethylArgGlyGly.</summary>
            UNIMOD_LRGG_dimethyl = 300001829,

            /// <summary>Biotin-Phenol.</summary>
            UNIMOD_Biotin_tyramide = 300001830,

            /// <summary>Tris adduct causes 104 Da addition at asparagine-succinimide intermediate.</summary>
            UNIMOD_Tris = 300001831,

            /// <summary>Iodoacetamide derivative of stilbene (reaction product with thiol).</summary>
            UNIMOD_IASD = 300001832,

            /// <summary>NP-40 synthetic polymer terminus.</summary>
            UNIMOD_NP40 = 300001833,

            /// <summary>Tween 20 synthetic polymer terminus.</summary>
            UNIMOD_Tween20 = 300001834,

            /// <summary>Tween 80 synthetic polymer terminus.</summary>
            UNIMOD_Tween80 = 300001835,

            /// <summary>Triton synthetic polymer terminus.</summary>
            UNIMOD_Triton = 300001836,

            /// <summary>Brij 35 synthetic polymer terminus.</summary>
            UNIMOD_Brij35 = 300001837,

            /// <summary>Brij 58 synthetic polymer terminus.</summary>
            UNIMOD_Brij58 = 300001838,

            /// <summary>Beta-Funaltrexamine.</summary>
            UNIMOD_betaFNA = 300001839,

            /// <summary>Fucosylated biantennary + 2 alphaGal.</summary>
            UNIMOD_dHex_1_Hex_7_HexNAc_4_ = 300001840,

            /// <summary>EZ-Link Sulfo-NHS-SS-Biotin.</summary>
            UNIMOD_Biotin_Thermo_21328 = 300001841,

            /// <summary>Cytidine monophosphate.</summary>
            UNIMOD_PhosphoCytidine = 300001843,

            /// <summary>Azidophenylalanine.</summary>
            UNIMOD_AzidoF = 300001845,

            /// <summary>Cys alkylation by dimethylaminoethyl halide.</summary>
            UNIMOD_Dimethylaminoethyl = 300001846,

            /// <summary>Glutarylation.</summary>
            UNIMOD_Gluratylation = 300001848,

            /// <summary>2-hydroxyisobutyrylation.</summary>
            UNIMOD_hydroxyisobutyryl = 300001849,

            /// <summary>S-Methyl Methyl phosphorothioate.</summary>
            UNIMOD_MeMePhosphorothioate = 300001868,

            /// <summary>Replacement of 3 protons by iron.</summary>
            UNIMOD_Cation_Fe_III_ = 300001870,

            /// <summary>DTT adduct of cysteine.</summary>
            UNIMOD_DTT = 300001871,

            /// <summary>Sulfenic Acid specific probe.</summary>
            UNIMOD_DYn_2 = 300001872,

            /// <summary>Acetone chemical artifact.</summary>
            UNIMOD_MesitylOxide = 300001873,

            /// <summary>Formaldehyde induced modifications.</summary>
            UNIMOD_methylol = 300001875,

            /// <summary>Tris-quenched monolink of DSS/BS3 crosslinker.</summary>
            UNIMOD_Xlink_DSS_259_ = 300001877,

            /// <summary>Water-quenched monolink of DSSO crosslinker.</summary>
            UNIMOD_Xlink_DSSO_176_ = 300001878,

            /// <summary>Ammonia-quenched monolink of DSSO crosslinker.</summary>
            UNIMOD_Xlink_DSSO_175_ = 300001879,

            /// <summary>Tris-quenched monolink of DSSO crosslinker.</summary>
            UNIMOD_Xlink_DSSO_279_ = 300001880,

            /// <summary>Alkene fragment of DSSO crosslinker.</summary>
            UNIMOD_Xlink_DSSO_54_ = 300001881,

            /// <summary>Thiol fragment of DSSO crosslinker.</summary>
            UNIMOD_Xlink_DSSO_86_ = 300001882,

            /// <summary>Sulfenic acid fragment of DSSO crosslinker.</summary>
            UNIMOD_Xlink_DSSO_104_ = 300001883,

            /// <summary>BuUr fragment of BuUrBu crosslinker.</summary>
            UNIMOD_Xlink_BuUrBu_111_ = 300001885,

            /// <summary>Bu fragment of BuUrBu crosslinker.</summary>
            UNIMOD_Xlink_BuUrBu_85_ = 300001886,

            /// <summary>Ammonia quenched monolink of BuUrBu crosslinker.</summary>
            UNIMOD_Xlink_BuUrBu_213_ = 300001887,

            /// <summary>Water quenched monolink of BuUrBu crosslinker.</summary>
            UNIMOD_Xlink_BuUrBu_214_ = 300001888,

            /// <summary>Tris quenched monolink of BuUrBu crosslinker.</summary>
            UNIMOD_Xlink_BuUrBu_317_ = 300001889,

            /// <summary>Intact DSSO crosslinker.</summary>
            UNIMOD_Xlink_DSSO_158_ = 300001896,

            /// <summary>Intact EGS cross-linker.</summary>
            UNIMOD_Xlink_EGS_226_ = 300001897,

            /// <summary>Intact DSS/BS3 crosslinker.</summary>
            UNIMOD_Xlink_DSS_138_ = 300001898,

            /// <summary>Intact BuUrBu crosslinker.</summary>
            UNIMOD_Xlink_BuUrBu_196_ = 300001899,

            /// <summary>Intact DTBP crosslinker.</summary>
            UNIMOD_Xlink_DTBP_172_ = 300001900,

            /// <summary>Intact DST crosslinker.</summary>
            UNIMOD_Xlink_DST_114_ = 300001901,

            /// <summary>Intact DSP/DTSSP crosslinker.</summary>
            UNIMOD_Xlink_DTSSP_174_ = 300001902,

            /// <summary>Intact SMCC cross-link.</summary>
            UNIMOD_Xlink_SMCC_219_ = 300001903,

            /// <summary>Intact BS2-G crosslinker.</summary>
            UNIMOD_Xlink_BS2G_96_ = 300001905,

            /// <summary>Ammonium-quenched monolink of BS2-G crosslinker.</summary>
            UNIMOD_Xlink_BS2G_113_ = 300001906,

            /// <summary>Water-quenched monolink of BS2-G crosslinker.</summary>
            UNIMOD_Xlink_BS2G_114_ = 300001907,

            /// <summary>Tris-quenched monolink of BS2-G crosslinker.</summary>
            UNIMOD_Xlink_BS2G_217_ = 300001908,

            /// <summary>Replacement of 3 protons by aluminium.</summary>
            UNIMOD_Cation_Al_III_ = 300001910,

            /// <summary>Ammonia quenched monolink of DMP crosslinker.</summary>
            UNIMOD_Xlink_DMP_139_ = 300001911,

            /// <summary>Intact DMP crosslinker.</summary>
            UNIMOD_Xlink_DMP_122_ = 300001912,

            /// <summary>Glyoxal-derived AGE.</summary>
            UNIMOD_glyoxalAGE = 300001913,

            /// <summary>Methionine oxidation to aspartic semialdehyde.</summary>
            UNIMOD_Met__AspSA = 300001914,

            /// <summary>Decarboxylation.</summary>
            UNIMOD_Decarboxylation = 300001915,

            /// <summary>Aspartylurea.</summary>
            UNIMOD_Aspartylurea = 300001916,

            /// <summary>In Bachi as Formylaspargine (typo?).</summary>
            UNIMOD_Formylasparagine = 300001917,

            /// <summary>Aldehyde and ketone modifications.</summary>
            UNIMOD_Carbonyl = 300001918,

            /// <summary>Adduction of aflatoxin B1 Dialdehyde to lysine.</summary>
            UNIMOD_AFB1_Dialdehyde = 300001920,

            /// <summary>Proline oxidation to 5-hydroxy-2-aminovaleric acid.</summary>
            UNIMOD_Pro__HAVA = 300001922,

            /// <summary>Tryptophan oxidation to beta-unsaturated-2,4-bis-tryptophandione.</summary>
            UNIMOD_Delta_H__4_O_2_ = 300001923,

            /// <summary>Tryptophan oxidation to hydroxy-bis-tryptophandione.</summary>
            UNIMOD_Delta_H__4_O_3_ = 300001924,

            /// <summary>Tryptophan oxidation to dihydroxy-N-formaylkynurenine.</summary>
            UNIMOD_Delta_O_4_ = 300001925,

            /// <summary>Methylglyoxal-derived carboxyethyllysine.</summary>
            UNIMOD_Delta_H_4_C_3_O_2_ = 300001926,

            /// <summary>Methylglyoxal-derived argpyrimidine.</summary>
            UNIMOD_Delta_H_4_C_5_O_1_ = 300001927,

            /// <summary>Crotonaldehyde-derived dimethyl-FDP-lysine.</summary>
            UNIMOD_Delta_H_10_C_8_O_1_ = 300001928,

            /// <summary>Methylglyoxal-derived tetrahydropyrimidine.</summary>
            UNIMOD_Delta_H_6_C_7_O_4_ = 300001929,

            /// <summary>Pent(2).</summary>
            UNIMOD_Pent_2_ = 300001930,

            /// <summary>Pent HexNAc.</summary>
            UNIMOD_Pent_1_HexNAc_1_ = 300001931,

            /// <summary>Hex(2) O(3) S.</summary>
            UNIMOD_Hex_2_Sulf_1_ = 300001932,

            /// <summary>Hex:1 Pent:2 Me:1.</summary>
            UNIMOD_Hex_1_Pent_2_Me_1_ = 300001933,

            /// <summary>HexNAc(2) Sulf.</summary>
            UNIMOD_HexNAc_2_Sulf_1_ = 300001934,

            /// <summary>Hex Pent(3) Me.</summary>
            UNIMOD_Hex_1_Pent_3_Me_1_ = 300001935,

            /// <summary>Hex(2) Pent(2).</summary>
            UNIMOD_Hex_2_Pent_2_ = 300001936,

            /// <summary>Hex(2) Pent(2) Me.</summary>
            UNIMOD_Hex_2_Pent_2_Me_1_ = 300001937,

            /// <summary>Hex(4) HexA.</summary>
            UNIMOD_Hex_4_HexA_1_ = 300001938,

            /// <summary>Hex(2) HexNAc Pent HexA.</summary>
            UNIMOD_Hex_2_HexNAc_1_Pent_1_HexA_1_ = 300001939,

            /// <summary>Hex(3) HexNAc HexA.</summary>
            UNIMOD_Hex_3_HexNAc_1_HexA_1_ = 300001940,

            /// <summary>Hex HexNAc(2) dHex(2) Sulf.</summary>
            UNIMOD_Hex_1_HexNAc_2_dHex_2_Sulf_1_ = 300001941,

            /// <summary>HexA(2) HexNAc(3).</summary>
            UNIMOD_HexA_2_HexNAc_3_ = 300001942,

            /// <summary>DHex Hex(4) HexA.</summary>
            UNIMOD_dHex_1_Hex_4_HexA_1_ = 300001943,

            /// <summary>Hex(5) HexA.</summary>
            UNIMOD_Hex_5_HexA_1_ = 300001944,

            /// <summary>Hex(4) HexA HexNAc.</summary>
            UNIMOD_Hex_4_HexA_1_HexNAc_1_ = 300001945,

            /// <summary>DHex(3) Hex(3) HexNAc.</summary>
            UNIMOD_dHex_3_Hex_3_HexNAc_1_ = 300001946,

            /// <summary>Hex(6) HexNAc.</summary>
            UNIMOD_Hex_6_HexNAc_1_ = 300001947,

            /// <summary>Sulf dHex Hex HexNAc(4).</summary>
            UNIMOD_Hex_1_HexNAc_4_dHex_1_Sulf_1_ = 300001948,

            /// <summary>DHex Hex(2) HexNAc NeuAc(2).</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_1_NeuAc_2_ = 300001949,

            /// <summary>DHex(3) Hex(3) HexNAc(2).</summary>
            UNIMOD_dHex_3_Hex_3_HexNAc_2_ = 300001950,

            /// <summary>DHex(2) Hex HexNAc(4) Sulf.</summary>
            UNIMOD_dHex_2_Hex_1_HexNAc_4_Sulf_1_ = 300001951,

            /// <summary>DHex Hex(2) HexNAc(4) Sulf(2).</summary>
            UNIMOD_dHex_1_Hex_2_HexNAc_4_Sulf_2_ = 300001952,

            /// <summary>Hex(9).</summary>
            UNIMOD_Hex_9_ = 300001953,

            /// <summary>Sulf dHex(2) Hex(3) HexNAc(3).</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_3_Sulf_1_ = 300001954,

            /// <summary>Me dHex(2) Hex(5) HexNAc(2).</summary>
            UNIMOD_dHex_2_Hex_5_HexNAc_2_Me_1_ = 300001955,

            /// <summary>Sulf(2) dHex(2) Hex(2) HexNAc(4).</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_4_Sulf_2_ = 300001956,

            /// <summary>Hex(9) HexNAc.</summary>
            UNIMOD_Hex_9_HexNAc_1_ = 300001957,

            /// <summary>DHex(3) Hex(2) HexNAc(4) Sulf(2).</summary>
            UNIMOD_dHex_3_Hex_2_HexNAc_4_Sulf_2_ = 300001958,

            /// <summary>Hex(4) HexNAc(4) NeuGc.</summary>
            UNIMOD_Hex_4_HexNAc_4_NeuGc_1_ = 300001959,

            /// <summary>DHex(4) Hex(3) HexNAc(2) NeuAc(1).</summary>
            UNIMOD_dHex_4_Hex_3_HexNAc_2_NeuAc_1_ = 300001960,

            /// <summary>Hex(3) HexNAc(5) NeuAc(1).</summary>
            UNIMOD_Hex_3_HexNAc_5_NeuAc_1_ = 300001961,

            /// <summary>Hex(10) HexNAc(1).</summary>
            UNIMOD_Hex_10_HexNAc_1_ = 300001962,

            /// <summary>DHex Hex(8) HexNAc(2).</summary>
            UNIMOD_dHex_1_Hex_8_HexNAc_2_ = 300001963,

            /// <summary>Hex(3) HexNAc(4) NeuAc(2).</summary>
            UNIMOD_Hex_3_HexNAc_4_NeuAc_2_ = 300001964,

            /// <summary>DHex(2) Hex(3) HexNAc(4) NeuAc.</summary>
            UNIMOD_dHex_2_Hex_3_HexNAc_4_NeuAc_1_ = 300001965,

            /// <summary>DHex(2) Hex(2) HexNAc(6) Sulf.</summary>
            UNIMOD_dHex_2_Hex_2_HexNAc_6_Sulf_1_ = 300001966,

            /// <summary>Hex(5) HexNAc(4) NeuAc Ac.</summary>
            UNIMOD_Hex_5_HexNAc_4_NeuAc_1_Ac_1_ = 300001967,

            /// <summary>Hex(3) HexNAc(3) NeuAc(3).</summary>
            UNIMOD_Hex_3_HexNAc_3_NeuAc_3_ = 300001968,

            /// <summary>Hex(5) HexNAc(4) NeuAc Ac(2).</summary>
            UNIMOD_Hex_5_HexNAc_4_NeuAc_1_Ac_2_ = 300001969,

            /// <summary>Unidentified modification of 162.1258 found in open search.</summary>
            UNIMOD_Unknown_162 = 300001970,

            /// <summary>Unidentified modification of 176.7462 found in open search.</summary>
            UNIMOD_Unknown_177 = 300001971,

            /// <summary>Unidentified modification of 210.1616 found in open search.</summary>
            UNIMOD_Unknown_210 = 300001972,

            /// <summary>Unidentified modification of 216.1002 found in open search.</summary>
            UNIMOD_Unknown_216 = 300001973,

            /// <summary>Unidentified modification of 234.0742 found in open search.</summary>
            UNIMOD_Unknown_234 = 300001974,

            /// <summary>Unidentified modification of 248.1986 found in open search.</summary>
            UNIMOD_Unknown_248 = 300001975,

            /// <summary>Unidentified modification of 249.981 found in open search.</summary>
            UNIMOD_Unknown_250 = 300001976,

            /// <summary>Unidentified modification of 301.9864 found in open search.</summary>
            UNIMOD_Unknown_302 = 300001977,

            /// <summary>Unidentified modification of 306.0952 found in open search.</summary>
            UNIMOD_Unknown_306 = 300001978,

            /// <summary>Unidentified modification of 420.0506 found in open search.</summary>
            UNIMOD_Unknown_420 = 300001979,

            /// <summary>O-diethylphosphothione.</summary>
            UNIMOD_Diethylphosphothione = 300001986,

            /// <summary>O-dimethylphosphothione.</summary>
            UNIMOD_Dimethylphosphothione = 300001987,

            /// <summary>O-methylphosphothione.</summary>
            UNIMOD_monomethylphosphothione = 300001989,

            /// <summary>Ubiquitin D (FAT10) leaving after chymotrypsin digestion Cys-Ile-Gly-Gly.</summary>
            UNIMOD_CIGG = 300001990,

            /// <summary>Ubiquitin D (FAT10) leaving after trypsin digestion Gly-Asn-Leu-Leu-Phe-Leu-Ala-Cys-Tyr-Cys-Ile-Gly-Gly.</summary>
            UNIMOD_GNLLFLACYCIGG = 300001991,

            /// <summary>5-glutamyl serotonin.</summary>
            UNIMOD_serotonylation = 300001992,

            /// <summary>Heavy tris(2,4,6-trimethoxyphenyl)phosphonium acetic acid N-hydroxysuccinimide ester derivative.</summary>
            UNIMOD_TMPP_Ac_13C_9_ = 300001993,

            /// <summary>DST crosslinker cleaved by sodium periodate.</summary>
            UNIMOD_Xlink_DST_56_ = 300001999,

            /// <summary>Carbobenzoxy-L-glutaminyl-glycine.</summary>
            UNIMOD_ZQG = 300002001,

            /// <summary>O-Dichloroethylphosphate.</summary>
            UNIMOD_Haloxon = 300002006,

            /// <summary>S-methyl amino phosphinate.</summary>
            UNIMOD_Methamidophos_S = 300002007,

            /// <summary>O-methyl amino phosphinate.</summary>
            UNIMOD_Methamidophos_O = 300002008,

            /// <summary>Loss of O2; nitro photochemical decomposition.</summary>
            UNIMOD_Nitrene = 300002014,

            /// <summary>Super Heavy Tandem Mass Tag.</summary>
            UNIMOD_shTMT = 300002015,

            /// <summary>TMTpro 16plex Tandem Mass Tag.</summary>
            UNIMOD_TMTpro = 300002016,

            /// <summary>Native TMTpro Tandem Mass Tag.</summary>
            UNIMOD_TMTpro_zero = 300002017,

            /// <summary>Glycosylation of Serine/Threonine residues with KDO.</summary>
            UNIMOD_Ser_Thr_KDO = 300002022,

            /// <summary>Andrographolide with the loss of H2O.</summary>
            UNIMOD_Andro_H2O = 300002025,

            /// <summary>Photo-induced histidine adduct.</summary>
            UNIMOD_His_O_2_ = 300002027,

            /// <summary>A3G3S3.</summary>
            UNIMOD_Hex_6_HexNAc_5_NeuAc_3_ = 300002028,

            /// <summary>A4G4.</summary>
            UNIMOD_Hex_7_HexNAc_6_ = 300002029,

            /// <summary>Photo-induced Methionine Adduct.</summary>
            UNIMOD_Met_O_2_ = 300002033,

            /// <summary>Photo-induced Glycine Adduct.</summary>
            UNIMOD_Gly_O_2_ = 300002034,

            /// <summary>Photo-induced Proline adduct.</summary>
            UNIMOD_Pro_O_2_ = 300002035,

            /// <summary>Photo-induced Lysine adduct.</summary>
            UNIMOD_Lys_O_2_ = 300002036,

            /// <summary>Photo-induced Glutamate adduct.</summary>
            UNIMOD_Glu_O_2_ = 300002037,

            /// <summary>Addition of lophotoxin to tyrosine.</summary>
            UNIMOD_LTX_Lophotoxin = 300002039,

            /// <summary>MBS_233p24 plus peptide 1250p53.</summary>
            UNIMOD_MBS_peptide = 300002040,

            /// <summary>3-hydroxybenzyl phosphate.</summary>
            UNIMOD_3_hydroxybenzyl_phosphate = 300002041,

            /// <summary>Phenyl phosphate.</summary>
            UNIMOD_phenyl_phosphate = 300002042,

            /// <summary>RNA-protein UVC-crosslinked, hydrofluoride-digested uridine adduct.</summary>
            UNIMOD_RBS_ID_Uridine = 300002044,

            /// <summary>Super Heavy TMTpro.</summary>
            UNIMOD_shTMTpro = 300002050,

            /// <summary>Intact DADPS Biotin Alkyne tag.</summary>
            UNIMOD_Biotin_Aha_DADPS = 300002052,

            /// <summary>Intact PC Biotin Alkyne tag.</summary>
            UNIMOD_Biotin_Aha_PC = 300002053,

            /// <summary>RNA-protein UVA-crosslinked, hydrofluoride-digested 4-thiouridine adduct.</summary>
            UNIMOD_pRBS_ID_4_thiouridine = 300002054,

            /// <summary>RNA-protein UVA-crosslinked, hydrofluoride-digested 6-thioguanosine adduct.</summary>
            UNIMOD_pRBS_ID_6_thioguanosine = 300002055,

            /// <summary>Iodoacetamido-LC-Phosphonic Acid derivative.</summary>
            UNIMOD_6C_CysPAT = 300002057,

            /// <summary>Intact DSPP/TBDSPP crosslinker.</summary>
            UNIMOD_Xlink_DSPP_210_ = 300002058,

            /// <summary>Water-quenched monolink of DSPP/TBDSPP crosslinker.</summary>
            UNIMOD_Xlink_DSPP_228_ = 300002059,

            /// <summary>Tris-quenched monolink of DSPP/TBDSPP crosslinker.</summary>
            UNIMOD_Xlink_DSPP_331_ = 300002060,

            /// <summary>Ammonia-quenched monolink of DSPP/TBDSPP crosslinker.</summary>
            UNIMOD_Xlink_DSPP_226_ = 300002061,

            /// <summary>Desthiobiotinylation of cysteine with DBIA probe.</summary>
            UNIMOD_DBIA = 300002062,

            /// <summary>Monomodification of N-propargyl-L-Gln probe with clicked desthiobiotin-azide.</summary>
            UNIMOD_Mono_N_gamma_propargyl_L_Gln_desthiobiotin = 300002067,

            /// <summary>Dimodification of L-Glu and N-propargyl-L-Gln probe with clicked desthiobiotin-azide.</summary>
            UNIMOD_Di_L_Glu_N_gamma_propargyl_L_Gln_desthiobiotin = 300002068,

            /// <summary>Dimodification of L-Gln and N-propargyl-L-Gln probe with clicked desthiobiotin-azide.</summary>
            UNIMOD_Di_L_Gln_N_gamma_propargyl_L_Gln_desthiobiotin = 300002069,

            /// <summary>Monomodification with glutamine.</summary>
            UNIMOD_L_Gln = 300002070,

            /// <summary>Glyceroylation.</summary>
            UNIMOD_Glyceroyl = 300002072,

            /// <summary>A unit of measurement is a standardized quantity of a physical quality.</summary>
            UO_unit = 400000000,

            /// <summary>A unit which is a standard measure of the distance between two points.</summary>
            UO_length_unit = 400000001,

            /// <summary>A unit which is a standard measure of the amount of matter/energy of a physical object.</summary>
            UO_mass_unit = 400000002,

            /// <summary>A unit which is a standard measure of the dimension in which events occur in sequence.</summary>
            UO_time_unit = 400000003,

            /// <summary>A unit which is a standard measure of the flow of electric charge.</summary>
            UO_electric_current_unit = 400000004,

            /// <summary>A unit which is a standard measure of the average kinetic energy of the particles in a sample of matter.</summary>
            UO_temperature_unit = 400000005,

            /// <summary>A unit which is a standardised quantity of an element or compound with uniform composition.</summary>
            UO_substance_unit = 400000006,

            /// <summary>A unit which is a standard measure of the wavelength-weighted power emitted by a light source in a particular direction.</summary>
            UO_luminous_intensity_unit = 400000007,

            /// <summary>A length unit which is equal to the length of the path traveled by light in vacuum during a time interval of 1/299 792 458 of a second.</summary>
            UO_meter = 400000008,

            /// <summary>A mass unit which is equal to the mass of the International Prototype Kilogram kept by the BIPM at Svres, France.</summary>
            UO_kilogram = 400000009,

            /// <summary>A time unit which is equal to the duration of 9 192 631 770 periods of the radiation corresponding to the transition between the two hyperfine levels of the ground state of the caesium 133 atom.</summary>
            UO_second = 400000010,

            /// <summary>An electric current unit which is equal to the constant current which, if maintained in two straight parallel conductors of infinite length, of negligible circular cross-section, and placed 1 m apart in vacuum, would produce between these conductors a force equal to 2 x 10^</summary>
            UO_ampere = 400000011,

            /// <summary>A thermodynamic temperature unit which is equal to the fraction 1/273.16 of the thermodynamic temperature of the triple point of water.</summary>
            UO_kelvin = 400000012,

            /// <summary>A substance unit which is equal to the amount of substance of a molecular system which contains as many elementary entities as there are atoms in 0.012 kilogram of carbon 12.</summary>
            UO_mole = 400000013,

            /// <summary>A luminous intensity unit which equal to the luminous intensity, in a given direction, of a source that emits monochromatic radiation of frequency 540 x 1012 hertz and that has a radiant intensity in that direction of 1/683 watt per steradian.</summary>
            UO_candela = 400000014,

            /// <summary>A length unit which is equal to one hundredth of a meter or 10^</summary>
            UO_centimeter = 400000015,

            /// <summary>A length unit which is equal to one thousandth of a meter or 10^</summary>
            UO_millimeter = 400000016,

            /// <summary>A length unit which is equal to one millionth of a meter or 10^</summary>
            UO_micrometer = 400000017,

            /// <summary>A length unit which is equal to one thousandth of one millionth of a meter or 10^</summary>
            UO_nanometer = 400000018,

            /// <summary>A length unit which is equal to 10</summary>
            UO_angstrom = 400000019,

            /// <summary>A length unit which is equal to 10^</summary>
            UO_picometer = 400000020,

            /// <summary>A mass unit which is equal to one thousandth of a kilogram or 10^</summary>
            UO_gram = 400000021,

            /// <summary>A mass unit which is equal to one thousandth of a gram or 10^</summary>
            UO_milligram = 400000022,

            /// <summary>A mass unit which is equal to one millionth of a gram or 10^</summary>
            UO_microgram = 400000023,

            /// <summary>A mass unit which is equal to one thousandth of one millionth of a gram or 10^</summary>
            UO_nanogram = 400000024,

            /// <summary>A mass unit which is equal to 10^</summary>
            UO_picogram = 400000025,

            /// <summary>A mass unit which is equal to 10^</summary>
            UO_femtogram = 400000026,

            /// <summary>A temperature unit which is equal to one kelvin degree. However, they have their zeros at different points. The centigrade scale has its zero at 273.15 K.</summary>
            UO_degree_Celsius = 400000027,

            /// <summary>A time unit which is equal to one thousandth of a second or 10^</summary>
            UO_millisecond = 400000028,

            /// <summary>A time unit which is equal to one millionth of a second or 10^</summary>
            UO_microsecond = 400000029,

            /// <summary>A time unit which is equal to 10^</summary>
            UO_picosecond = 400000030,

            /// <summary>A time unit which is equal to 60 seconds.</summary>
            UO_minute = 400000031,

            /// <summary>A time unit which is equal to 3600 seconds or 60 minutes.</summary>
            UO_hour = 400000032,

            /// <summary>A time unit which is equal to 24 hours.</summary>
            UO_day = 400000033,

            /// <summary>A time unit which is equal to 7 days.</summary>
            UO_week = 400000034,

            /// <summary>A time unit which is approximately equal to the length of time of one of cycle of the moon's phases which in science is taken to be equal to 30 days.</summary>
            UO_month = 400000035,

            /// <summary>A time unit which is equal to 12 months which in science is taken to be equal to 365.25 days.</summary>
            UO_year = 400000036,

            /// <summary>An electric current unit current which is equal to one thousandth of an ampere or 10^</summary>
            UO_milliampere = 400000037,

            /// <summary>An electric current unit current which is equal to one millionth of an ampere or 10^</summary>
            UO_microampere = 400000038,

            /// <summary>A substance unit equal to a millionth of a mol or 10^</summary>
            UO_micromole = 400000039,

            /// <summary>A substance unit equal to a thousandth of a mol or 10^</summary>
            UO_millimole = 400000040,

            /// <summary>A substance unit equal to one thousandth of one millionth of a mole or 10^</summary>
            UO_nanomole = 400000041,

            /// <summary>A substance unit equal to 10^</summary>
            UO_picomole = 400000042,

            /// <summary>A substance unit equal to 10^</summary>
            UO_femtomole = 400000043,

            /// <summary>A substance unit equal to 10^</summary>
            UO_attomole = 400000044,

            /// <summary>A unit which is one of a particular measure to which all measures of that type can be related.</summary>
            UO_base_unit = 400000045,

            /// <summary>Description not provided</summary>
            UO_prefix = 400000046,

            /// <summary>A unit which is a standard measure of the amount of a 2-dimensional flat surface.</summary>
            UO_area_unit = 400000047,

            /// <summary>A unit which is a standard measure of the rate of change of velocity in either speed or direction.</summary>
            UO_acceleration_unit = 400000048,

            /// <summary>A unit which is a standard measure of the rate of angular movement about an axis; the angle rotated in a given time.</summary>
            UO_angular_velocity_unit = 400000049,

            /// <summary>A unit which is a standard measure of the rate of change of angular velocity.</summary>
            UO_angular_acceleration_unit = 400000050,

            /// <summary>A unit which represents a standard measurement of how much of a given substance there is mixed with another substance.</summary>
            UO_concentration_unit = 400000051,

            /// <summary>A density unit which is a standard measure of the mass of a substance in a given volume.</summary>
            UO_mass_density_unit = 400000052,

            /// <summary>A unit which is a standard measure of the luminous intensity impinging on a given area.</summary>
            UO_luminance_unit = 400000053,

            /// <summary>A density unit which is a standard measure of the mass exerting an influence on a given area.</summary>
            UO_area_density_unit = 400000054,

            /// <summary>A unit which is a standard measure of the mass of a homogeneous substance containing 6.02 x 1023 atoms or molecules.</summary>
            UO_molar_mass_unit = 400000055,

            /// <summary>A unit which is a standard measure of the volume of a homogeneous substance containing 6.02 x 1023 atoms or molecules.</summary>
            UO_molar_volume_unit = 400000056,

            /// <summary>A unit which is a standard measure of the quantity of motion measured by the product of mass and velocity.</summary>
            UO_momentum_unit = 400000057,

            /// <summary>A unit which is a standard measure of the number of rotations in a given time.</summary>
            UO_rotational_frequency_unit = 400000058,

            /// <summary>A unit which is a standard measure of the volume of a given mass of substance (the reciprocal of density).</summary>
            UO_specific_volume_unit = 400000059,

            /// <summary>A unit which is a standard measure of the rate of movement. Speed is measured in the same physical units of measurement as velocity, but does not contain the element of direction that velocity has. Speed is thus the magnitude component of velocity.</summary>
            UO_speed_velocity_unit = 400000060,

            /// <summary>A concentration unit which is a standard measure of the number of moles of a given substance per liter of solution.</summary>
            UO_unit_of_molarity = 400000061,

            /// <summary>A unit of concentration which expresses a concentration of 1 mole of solute per liter of solution (mol/L).</summary>
            UO_molar = 400000062,

            /// <summary>A unit of molarity which is equal to one thousandth of a molar or 10^</summary>
            UO_millimolar = 400000063,

            /// <summary>A unit of molarity which is equal to one millionth of a molar or 10^</summary>
            UO_micromolar = 400000064,

            /// <summary>A unit of molarity which is equal to one thousandth of one millionth of a molar or 10^</summary>
            UO_nanomolar = 400000065,

            /// <summary>A unit of molarity which is equal to 10^</summary>
            UO_picomolar = 400000066,

            /// <summary>A concentration unit which is a standard measure of the number of moles of a given substance per kilogram of solvent.</summary>
            UO_unit_of_molality = 400000067,

            /// <summary>A unit of concentration which expresses a concentration of a solution of 1 mole per kilogram of solvent (mol/kg).</summary>
            UO_molal = 400000068,

            /// <summary>A molality unit which is equal to one thousandth of a molal or 10^</summary>
            UO_millimolal = 400000069,

            /// <summary>A molality unit which is equal to one millionth of a molal or 10^</summary>
            UO_micromolal = 400000070,

            /// <summary>A molality unit which is equal to one thousandth of one millionth of a molal or 10^</summary>
            UO_nanomolal = 400000071,

            /// <summary>A molality unit which is equal to 10^</summary>
            UO_picomolal = 400000072,

            /// <summary>A unit of molarity which is equal to 10^</summary>
            UO_femtomolar = 400000073,

            /// <summary>A unit of concentration which highlights the chemical nature of salts.</summary>
            UO_unit_of_normality = 400000074,

            /// <summary>A unit of concentration which is one gram equivalent of a solute per liter of solution. A gram equivalent weight or equivalent is a measure of the reactive capacity of a given molecule.</summary>
            UO_normal = 400000075,

            /// <summary>A concentration unit which denotes the number of moles of solute as a proportion of the total number of moles in a solution.</summary>
            UO_mole_fraction = 400000076,

            /// <summary>An acceleration unit which is equal to the acceleration an object changing its velocity by 1meter/s over a time period that equals one second.</summary>
            UO_meter_per_second_per_second = 400000077,

            /// <summary>An angular unit acceleration which is equal to the angular acceleration of an object changing its angular velocity by 1rad/s over a time period that equals one second.</summary>
            UO_radian_per_second_per_second = 400000078,

            /// <summary>An angular unit velocity which is equal to about 9.54930 rpm (revolutions per minute).</summary>
            UO_radian_per_second = 400000079,

            /// <summary>An area unit which is equal to an area enclosed by a square with sides each 1 meter long.</summary>
            UO_square_meter = 400000080,

            /// <summary>An area unit which is equal to one ten thousandth of a square meter or 10^</summary>
            UO_square_centimeter = 400000081,

            /// <summary>An area unit which is equal to one millionth of a square meter or 10^</summary>
            UO_square_millimeter = 400000082,

            /// <summary>A mass unit density which is equal to mass of an object in kilograms divided by the volume in cubic meters.</summary>
            UO_kilogram_per_cubic_meter = 400000083,

            /// <summary>A mass unit density which is equal to mass of an object in grams divided by the volume in cubic centimeters.</summary>
            UO_gram_per_cubic_centimeter = 400000084,

            /// <summary>A luminance unit which is equal to a luminous intensity of one candela radiating from a surface whose area is one square meter.</summary>
            UO_candela_per_square_meter = 400000085,

            /// <summary>An area density unit which is equal to the mass of an object in kilograms divided by the surface area in meters squared.</summary>
            UO_kilogram_per_square_meter = 400000086,

            /// <summary>A molar mass unit which is equal to one kilogram of mass of one mole of chemical element or chemical compound.</summary>
            UO_kilogram_per_mole = 400000087,

            /// <summary>A molar mass unit which is equal to one gram of mass of one mole of chemical element or chemical compound.</summary>
            UO_gram_per_mole = 400000088,

            /// <summary>A molar volume unit which is equal to 1 cubic meter occupied by one mole of a substance in the form of a solid, liquid, or gas.</summary>
            UO_cubic_meter_per_mole = 400000089,

            /// <summary>A molar volume unit which is equal to 1 cubic centimeter occupied by one mole of a substance in the form of a solid, liquid, or gas.</summary>
            UO_cubic_centimeter_per_mole = 400000090,

            /// <summary>A momentum unit which is equal to the momentum of a one kilogram mass object with a speed of one meter per second.</summary>
            UO_kilogram_meter_per_second = 400000091,

            /// <summary>A rotational frequency unit which is equal to the number complete turn in a period of time that equals to 1 second.</summary>
            UO_turns_per_second = 400000092,

            /// <summary>A specific volume unit which is equal to one cubic meter volume occupied by one kilogram of a particular substance.</summary>
            UO_cubic_meter_per_kilogram = 400000093,

            /// <summary>A speed/velocity unit which is equal to the speed of an object traveling 1 meter distance in one second.</summary>
            UO_meter_per_second = 400000094,

            /// <summary>A unit which is a standard measure of the amount of space occupied by any substance, whether solid, liquid, or gas.</summary>
            UO_volume_unit = 400000095,

            /// <summary>A volume unit which is equal to the volume of a cube with edges one meter in length. One cubic meter equals to 1000 liters.</summary>
            UO_cubic_meter = 400000096,

            /// <summary>A volume unit which is equal to one millionth of a cubic meter or 10^</summary>
            UO_cubic_centimeter = 400000097,

            /// <summary>A volume unit which is equal to one thousandth of a liter or 10^</summary>
            UO_milliliter = 400000098,

            /// <summary>A volume unit which is equal to one thousandth of a cubic meter or 10^</summary>
            UO_liter = 400000099,

            /// <summary>A volume unit which is equal to one thousand of a cubic meter or 10^</summary>
            UO_cubic_decimeter = 400000100,

            /// <summary>A volume unit which is equal to one millionth of a liter or 10^</summary>
            UO_microliter = 400000101,

            /// <summary>A volume unit which is equal to one thousandth of one millionth of a liter or 10^</summary>
            UO_nanoliter = 400000102,

            /// <summary>A volume unit which is equal to 10^</summary>
            UO_picoliter = 400000103,

            /// <summary>A volume unit which is equal to 10^</summary>
            UO_femtoliter = 400000104,

            /// <summary>A unit which is a standard measure of the number of repetitive actions in a particular time.</summary>
            UO_frequency_unit = 400000105,

            /// <summary>A frequency unit which is equal to 1 complete cycle of a recurring phenomenon in 1 second.</summary>
            UO_hertz = 400000106,

            /// <summary>A unit which is a standard measure of the force is applied when a mass is accelerated.</summary>
            UO_force_unit = 400000107,

            /// <summary>A force unit which is equal to the force required to cause an acceleration of 1m/s2 of a mass of 1 Kg in the direction of the force.</summary>
            UO_newton = 400000108,

            /// <summary>A unit which is a standard measure of the force applied to a given area.</summary>
            UO_pressure_unit = 400000109,

            /// <summary>A pressure unit which is equal to the pressure or stress on a surface caused by a force of 1 newton spread over a surface of 1 m^</summary>
            UO_pascal = 400000110,

            /// <summary>A unit which is a standard measure of the work done by a certain force (gravitational, electric, magnetic, force of inertia, etc).</summary>
            UO_energy_unit = 400000111,

            /// <summary>An energy unit which is equal to the energy required when a force of 1 newton moves an object 1 meter in the direction of the force.</summary>
            UO_joule = 400000112,

            /// <summary>A unit which is a standard measure power or the rate of doing work.</summary>
            UO_power_unit = 400000113,

            /// <summary>A power unit which is equal to the power used when work is done at the rate of 1 joule per second.</summary>
            UO_watt = 400000114,

            /// <summary>A unit which is a standard measure of the luminous flux incident on a unit area.</summary>
            UO_illuminance_unit = 400000115,

            /// <summary>An illuminance unit which is equal to the illuminance produced by 1 lumen evenly spread over an area 1 m^</summary>
            UO_lux = 400000116,

            /// <summary>A unit which is a standard measure of the flow of radiant energy.</summary>
            UO_luminous_flux_unit = 400000117,

            /// <summary>A luminous flux unit which is equal to the luminous flux emitted into 1 steradian by a point source of 1 candela.</summary>
            UO_lumen = 400000118,

            /// <summary>A unit which is a standard measure of the amount of the action of a catalyst.</summary>
            UO_catalytic_activity_unit = 400000119,

            /// <summary>A catalytic unit activity which is equal to the activity of a catalyst in moles per second, such as the amount of an enzyme needed to transform one mole of substrate per second.</summary>
            UO_katal = 400000120,

            /// <summary>A unit which is a standard measure of the figure or space formed by the junction of two lines or planes.</summary>
            UO_angle_unit = 400000121,

            /// <summary>A unit which is a standard measure of the angle formed by two straight lines in the same plane.</summary>
            UO_plane_angle_unit = 400000122,

            /// <summary>A plane angle unit which is equal to the angle subtended at the center of a circle by an arc equal in length to the radius of the circle, approximately 57 degrees 17 minutes and 44.6 seconds.</summary>
            UO_radian = 400000123,

            /// <summary>A unit which is a standard measure of the angle formed by three or more planes intersecting at a common point.</summary>
            UO_solid_angle_unit = 400000124,

            /// <summary>A solid angle unit which is equal to the solid angle subtended at the center of a sphere by an area on the surface of the sphere that is equal to the radius squared.</summary>
            UO_steradian = 400000125,

            /// <summary>A unit which is a standard measure of the amount of radiation emitted by a given radiation source as well as the amount of radiation absorbed or deposited in a specific material by a radiation source.</summary>
            UO_radiation_unit = 400000127,

            /// <summary>A unit which is a standard measure of the transformation (disintegration) rate of a radioactive substance.</summary>
            UO_activity__of_a_radionuclide__unit = 400000128,

            /// <summary>A unit which is a standard measure of the energy imparted by ionizing radiation to unit mass of matter such as tissue.</summary>
            UO_absorbed_dose_unit = 400000129,

            /// <summary>A unit which is a standard measure of the expression of dose in terms of its biological effect.</summary>
            UO_dose_equivalent_unit = 400000130,

            /// <summary>A unit which is a standard measure of the quantity that expresses the ability of radiation to ionize air and thereby create electric charges which can be collected and measured.</summary>
            UO_exposure_unit = 400000131,

            /// <summary>An activity (of a radionuclide) unit which is equal to the activity of a quantity of radioactive material in which one nucleus decays per second or there is one atom disintegration per second (dps).</summary>
            UO_becquerel = 400000132,

            /// <summary>An activity (of a radionuclide) unit which is equal to the activity of a quantity of radioactive material in which there are 3.7 x 10^</summary>
            UO_curie = 400000133,

            /// <summary>An absorbed dose unit which is equal to the absorption of one joule of radiation energy by one kilogram of matter.</summary>
            UO_gray = 400000134,

            /// <summary>An absorbed dose unit which is equal to 0.01 gray (Gy).</summary>
            UO_rad = 400000135,

            /// <summary>An exposure unit which is equal to the amount of radiation required to liberate positive and negative charges of one electrostatic unit of charge in 1 cm^</summary>
            UO_roentgen = 400000136,

            /// <summary>A dose equivalent unit which is equal to the absorption of one joule of radiation energy by one kilogram of matter.</summary>
            UO_sievert = 400000137,

            /// <summary>A dose equivalent unit which is equal to one thousandth of a sievert or 10^</summary>
            UO_millisievert = 400000138,

            /// <summary>A dose equivalent unit which is equal to one millionth of a sievert or 10^</summary>
            UO_microsievert = 400000139,

            /// <summary>A dose equivalent unit which when multiplied by hundred is equal to one sievert or 1 Sv. 1 Sv is equal to 100 rem.</summary>
            UO_Roentgen_equivalent_man = 400000140,

            /// <summary>An absorbed dose unit which is equal to one millionth of a gray or 10^</summary>
            UO_microgray = 400000141,

            /// <summary>An absorbed dose unit which is equal to one thousandth of a gray or 10^</summary>
            UO_milligray = 400000142,

            /// <summary>An absorbed dose unit which is equal to one thousandth of a millionth of a gray or 10^</summary>
            UO_nanogray = 400000143,

            /// <summary>A dose equivalent unit which is equal to one thousandth of a millionth of a sievert or 10^</summary>
            UO_nanosievert = 400000144,

            /// <summary>An activity (of a radionuclide) unit which is equal to one thousandth of a curie or 10^</summary>
            UO_millicurie = 400000145,

            /// <summary>An activity (of a radionuclide) unit which is equal to one millionth of a curie or 10^</summary>
            UO_microcurie = 400000146,

            /// <summary>An activity (of a radionuclide) unit which is equal to the activity of a quantity of radioactive material in which one nucleus decays per minute or there is one atom disintegration per minute.</summary>
            UO_disintegrations_per_minute = 400000147,

            /// <summary>An activity (of a radionuclide) unit which is equal to the number of light emissions produced by ionizing radiation in one minute.</summary>
            UO_counts_per_minute = 400000148,

            /// <summary>A time unit which is equal to one thousandth of one millionth of a second or 10^</summary>
            UO_nanosecond = 400000150,

            /// <summary>A time unit which is equal to 100 years.</summary>
            UO_century = 400000151,

            /// <summary>A time unit which represents the period over which the activity or concentration of a specified chemical or element falls to half its original activity or concentration.</summary>
            UO_half_life = 400000152,

            /// <summary>An illuminance unit which is equal to the illuminance produced by 1 lumen evenly spread over an area 1 foot^</summary>
            UO_foot_candle = 400000153,

            /// <summary>A unit which is a standard measure of the power of electromagnetic radiation at a surface, per unit area.</summary>
            UO_irradiance_unit = 400000154,

            /// <summary>An irradiance unit which is equal to 1 watt of radiant power incident per one square meter surface area.</summary>
            UO_watt_per_square_meter = 400000155,

            /// <summary>An irradiance unit which is equal to one einstein per square meter per second. One einstein is one mole of photons, regardless of their frequency. Therefore, the number of photons in an einstein is Avogadro's number.</summary>
            UO_einstein_per_square_meter_per_second = 400000156,

            /// <summary>A unit which is a standard measure of the intensity of light.</summary>
            UO_light_unit = 400000157,

            /// <summary>A radiance unit which is equal to one watt of radiant power incident per steradian solid angle per one square meter projected area of the source, as viewed from the given direction.</summary>
            UO_watt_per_steradian_per_square_meter = 400000158,

            /// <summary>A unit which is a standard measure of the intensity of electromagnetic radiation.</summary>
            UO_radiant_intensity_unit = 400000159,

            /// <summary>An irradiance unit which is equal to one microeinstein per square meter per second or 10^</summary>
            UO_microeinstein_per_square_meter_per_second = 400000160,

            /// <summary>A unit which is a standard measure of the power of electromagnetic radiation through space or through a material medium in the form of electromagnetic waves.</summary>
            UO_radiance_unit = 400000161,

            /// <summary>A radiant intensity unit which is equal to one kilogram meter squared per second cubed per steradian.</summary>
            UO_watt_per_steradian = 400000162,

            /// <summary>A dimensionless concentration unit which denotes the mass of a substance in a mixture as a percentage of the mass of the entire mixture.</summary>
            UO_mass_percentage = 400000163,

            /// <summary>A dimensionless concentration unit which denotes the mass of the substance in a mixture as a percentage of the volume of the entire mixture.</summary>
            UO_mass_volume_percentage = 400000164,

            /// <summary>A dimensionless concentration unit which denotes the volume of the solute in mL per 100 mL of the resulting solution.</summary>
            UO_volume_percentage = 400000165,

            /// <summary>A dimensionless concentration notation which describes the amount of one substance in another. It is the ratio of the amount of the substance of interest to the amount of that substance plus the amount of the substance.</summary>
            UO_parts_per_notation_unit = 400000166,

            /// <summary>A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 100 regardless of the units of measure as long as they are the same.</summary>
            UO_parts_per_hundred = 400000167,

            /// <summary>A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1000 regardless of the units of measure as long as they are the same.</summary>
            UO_parts_per_thousand = 400000168,

            /// <summary>A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1,000,000 regardless of the units of measure used as long as they are the same or 1 part in 10^</summary>
            UO_parts_per_million = 400000169,

            /// <summary>A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1,000,000,000 regardless of the units of measure as long as they are the same or 1 part in 10^</summary>
            UO_parts_per_billion = 400000170,

            /// <summary>A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1,000,000,000 regardless of the units of measure used as long as they are the same or 1 part in 10^</summary>
            UO_parts_per_trillion = 400000171,

            /// <summary>A dimensionless concentration notation which denotes the amount of a given substance in a total amount of 1,000,000,000,000 regardless of the units of measure used as long as they are the same or 1 part in 10^</summary>
            UO_parts_per_quadrillion = 400000172,

            /// <summary>A mass unit density which is equal to mass of an object in grams divided by the volume in milliliter.</summary>
            UO_gram_per_milliliter = 400000173,

            /// <summary>A mass unit density which is equal to mass of an object in kilograms divided by the volume in liters.</summary>
            UO_kilogram_per_liter = 400000174,

            /// <summary>A mass unit density which is equal to mass of an object in grams divided by the volume in liters.</summary>
            UO_gram_per_liter = 400000175,

            /// <summary>A mass unit density which is equal to mass of an object in milligrams divided by the volume in milliliters.</summary>
            UO_milligram_per_milliliter = 400000176,

            /// <summary>A concentration unit which is a standard measure of the number of units, as an agreed arbitrary amount, of a given substance per a specific volume of solution.</summary>
            UO_unit_per_volume_unit = 400000177,

            /// <summary>A unit per milliliter unit which is equal to one unit of an agreed arbitrary amount per one milliliter.</summary>
            UO_unit_per_milliliter = 400000178,

            /// <summary>A unit per milliliter unit which is equal to one unit of an agreed arbitrary amount per one liter.</summary>
            UO_unit_per_liter = 400000179,

            /// <summary>A concentration unit which is a standard measure of the mass of a substance in a given volume (density).</summary>
            UO_mass_per_unit_volume = 400000180,

            /// <summary>A catalytic unit activity which is equal to the amount of the enzyme that catalyzes the conversion of 1 micro mole of substrate per minute.</summary>
            UO_enzyme_unit = 400000181,

            /// <summary>A unit which is a standard measure of the influence exerted by some mass.</summary>
            UO_density_unit = 400000182,

            /// <summary>A density unit which is a standard measure of the mass exerting an influence on a one-dimensional object.</summary>
            UO_linear_density_unit = 400000183,

            /// <summary>An area density unit which is equal to the mass of an object in kilograms divided by one meter.</summary>
            UO_kilogram_per_meter = 400000184,

            /// <summary>A plane angle unit which is equal to 1/360 of a full rotation or 1.7453310^</summary>
            UO_degree = 400000185,

            /// <summary>A unit which is a standard measure of physical quantity consisting of only a numerical number without any units.</summary>
            UO_dimensionless_unit = 400000186,

            /// <summary>A dimensionless ratio unit which denotes numbers as fractions of 100.</summary>
            UO_percent = 400000187,

            /// <summary>A dimensionless unit which denoted an irrational real number, approximately equal to 3.14159 which is the ratio of a circle's circumference to its diameter in Euclidean geometry.</summary>
            UO_pi = 400000188,

            /// <summary>A dimensionless unit which denotes a simple count of things.</summary>
            UO_count_unit = 400000189,

            /// <summary>A dimensionless unit which denotes an amount or magnitude of one quantity relative to another.</summary>
            UO_ratio = 400000190,

            /// <summary>A dimensionless ratio unit which relates the part (the numerator) to the whole (the denominator).</summary>
            UO_fraction = 400000191,

            /// <summary>A dimensionless count unit which denotes the number of molecules.</summary>
            UO_molecule_count = 400000192,

            /// <summary>A dimensionless percent unit which denotes the homogeneity of a biomaterial.</summary>
            UO_purity_percentage = 400000193,

            /// <summary>A dimensionless percent unit which denotes the density of an attached or monolayer culture (e.g., cell culture).</summary>
            UO_confluence_percentage = 400000194,

            /// <summary>A temperature unit which is equal to 5/9ths of a kelvin. Negative 40 degrees Fahrenheit is equal to negative 40 degrees Celsius.</summary>
            UO_degree_Fahrenheit = 400000195,

            /// <summary>A dimensionless concentration notation which denotes the acidity of a solution in terms of activity of hydrogen ions (H+).</summary>
            UO_pH = 400000196,

            /// <summary>A specific volume unit which is equal to one liter volume occupied by one kilogram of a particular substance.</summary>
            UO_liter_per_kilogram = 400000197,

            /// <summary>A specific volume unit which is equal to a thousandth of a liter per kilogram or 10^</summary>
            UO_milliliter_per_kilogram = 400000198,

            /// <summary>A specific volume unit which is equal to one millionth of a liter per kilogram or 10^</summary>
            UO_microliter_per_kilogram = 400000199,

            /// <summary>A concentration unit which denotes the average cell number in a given volume.</summary>
            UO_cell_concentration_unit = 400000200,

            /// <summary>A unit of cell concentration which is equal to one cell in a volume of 1 milliliter.</summary>
            UO_cells_per_milliliter = 400000201,

            /// <summary>A concentration unit which is a standard measure of the amount of the action of a catalyst in a given volume.</summary>
            UO_catalytic__activity__concentration_unit = 400000202,

            /// <summary>A catalytic (activity) concentration unit which is equal to 1 katal activity of a catalyst in a given volume of one cubic meter.</summary>
            UO_katal_per_cubic_meter = 400000203,

            /// <summary>A catalytic (activity) concentration unit which is equal to 1 katal activity of a catalyst in a given volume of one thousandth of a cubic meter.</summary>
            UO_katal_per_liter = 400000204,

            /// <summary>A dimensionless concentration unit which denotes the given volume of the solute in the total volume of the resulting solution.</summary>
            UO_volume_per_unit_volume = 400000205,

            /// <summary>A volume per unit volume unit which is equal to one millionth of a liter of solute in one cubic meter of solution.</summary>
            UO_milliliter_per_cubic_meter = 400000206,

            /// <summary>A volume per unit volume unit which is equal to one millionth of a liter of solute in one liter of solution.</summary>
            UO_milliliter_per_liter = 400000207,

            /// <summary>A mass density unit which is equal to mass of an object in grams divided by the volume in deciliters.</summary>
            UO_gram_per_deciliter = 400000208,

            /// <summary>A volume unit which is equal to one tenth of a liter or 10^</summary>
            UO_deciliter = 400000209,

            /// <summary>A dimensionless count unit which a measure of viable bacterial numbers.</summary>
            UO_colony_forming_unit = 400000210,

            /// <summary>A dimensionless count unit which a measure of plague forming units in a given volume.</summary>
            UO_plaque_forming_unit = 400000211,

            /// <summary>A concentration unit which a measure of viable bacterial numbers in a given volume.</summary>
            UO_colony_forming_unit_per_volume = 400000212,

            /// <summary>A colony forming unit which a measure of viable bacterial numbers in one milliliter.</summary>
            UO_colony_forming_unit_per_milliliter = 400000213,

            /// <summary>A concentration unit which a measure of plague forming units in a given volume.</summary>
            UO_plaque_forming_unit_per_volume = 400000214,

            /// <summary>A concentration unit which a measure of plague forming units in one milliliter.</summary>
            UO_plaque_forming_unit_per_milliliter = 400000215,

            /// <summary>An activity (of a radionuclide) unit which is equal to the activity of a quantity of radioactive material in which one nucleus decays per second or there is one atom disintegration per second.</summary>
            UO_disintegrations_per_second = 400000216,

            /// <summary>A unit which is a standard measure of the work done per unit charge as a charge is moved between two points in an electric field.</summary>
            UO_electric_potential_difference_unit = 400000217,

            /// <summary>An electric potential difference unit which is equal to the work per unit charge. One volt is the potential difference required to move one coulomb of charge between two points in a circuit while using one joule of energy.</summary>
            UO_volt = 400000218,

            /// <summary>A unit which is a standard measure of the quantity of unbalanced electricity in a body (either positive or negative) and construed as an excess or deficiency of electrons.</summary>
            UO_electric_charge = 400000219,

            /// <summary>An electrical charge unit which is equal to the amount of charge transferred by a current of 1 ampere in 1 second.</summary>
            UO_coulomb = 400000220,

            /// <summary>An independently to the base SI units defined mass unit which is equal to one twelfth of the mass of an unbound atom of the carbon-12 nuclide, at rest and in its ground state.</summary>
            UO_dalton = 400000221,

            /// <summary>A mass unit which is equal to one thousand daltons.</summary>
            UO_kilodalton = 400000222,

            /// <summary>An energy unit which is equal to the amount of electrical energy equivalent to a one-watt load drawing power for one hour.</summary>
            UO_watt_hour = 400000223,

            /// <summary>An energy unit which is equal to 1,000 watt-hours.</summary>
            UO_kilowatt_hour = 400000224,

            /// <summary>A unit which is a standard measure of quantity of magnetism, taking account of the strength and the extent of a magnetic field.</summary>
            UO_magnetic_flux_unit = 400000225,

            /// <summary>A magnetic flux unit which is equal to the amount of flux that when linked with a single turn of wire for an interval of one second will induce an electromotive force of one volt.</summary>
            UO_weber = 400000226,

            /// <summary>A unit which is a standard measure of the strength of a magnetic field.</summary>
            UO_magnetic_flux_density_unit = 400000227,

            /// <summary>A magnetic flux density unit which is equal to one weber per square meter.</summary>
            UO_tesla = 400000228,

            /// <summary>A magnetic flux unit which is equal to 3600 Wb.</summary>
            UO_volt_hour = 400000229,

            /// <summary>A magnetic flux unit which is equal to one thousand volt-hours.</summary>
            UO_kilovolt_hour = 400000230,

            /// <summary>A unit which is a standard measure of the amount of information.</summary>
            UO_information_unit = 400000231,

            /// <summary>An information unit which refers to a digit in the binary numeral system, which consists of base 2 digits (ie there are only 2 possible values: 0 or 1).</summary>
            UO_bit = 400000232,

            /// <summary>An information unit which is equal to 8 bits.</summary>
            UO_byte = 400000233,

            /// <summary>An information unit which is equal to 1000 bytes.</summary>
            UO_kilobyte = 400000234,

            /// <summary>An information unit which is equal to 1000 kB.</summary>
            UO_megabyte = 400000235,

            /// <summary>An information unit which is a standard measure of the detail an image holds.</summary>
            UO_image_resolution_unit = 400000236,

            /// <summary>An image resolution unit which is a standard measure of the amount of spatial detail in an image.</summary>
            UO_chroma_sampling_unit = 400000237,

            /// <summary>An image resolution unit which is a standard measure of the amount of contrast available in a pixel.</summary>
            UO_dynamic_range_unit = 400000238,

            /// <summary>An image resolution unit which is a standard measure of the way luminance and chrominance may be sampled at different levels.</summary>
            UO_spatial_resolution_unit = 400000239,

            /// <summary>A spatial resolution unit which is a standard measure of the printing resolution, in particular the number of individual dots of ink a printer or toner can produce within a linear one-inch space.</summary>
            UO_dots_per_inch = 400000240,

            /// <summary>A spatial resolution unit which is equal to a pixel size of one micrometer.</summary>
            UO_micron_pixel = 400000241,

            /// <summary>A spatial resolution unit which is a standard measure of the resolution of a computer display, related to the size of the display in inches and the total number of pixels in the horizontal and vertical directions.</summary>
            UO_pixels_per_inch = 400000242,

            /// <summary>A spatial resolution unit which is a standard measure of the number of pixels in one millimeter length or width of a digital image divided by the physical length or width of a printed image.</summary>
            UO_pixels_per_millimeter = 400000243,

            /// <summary>A count unit which contains one nucleotide.</summary>
            UO_base_pair = 400000244,

            /// <summary>An information unit which is equal to 1024 B.</summary>
            UO_kibibyte = 400000245,

            /// <summary>An information unit which is equal to 1024 KiB.</summary>
            UO_mebibyte = 400000246,

            /// <summary>An electric potential difference unit which is equal to one thousandth of a volt or 10^</summary>
            UO_millivolt = 400000247,

            /// <summary>An electric potential difference unit which is equal to one thousand volts or 10^</summary>
            UO_kilovolt = 400000248,

            /// <summary>An electric potential difference unit which is equal to one millionth of a volt or 10^</summary>
            UO_microvolt = 400000249,

            /// <summary>An electric potential difference unit which is equal to one billionth of a volt or 10^</summary>
            UO_nanovolt = 400000250,

            /// <summary>An electric potential difference unit which is equal to one trillionth of a volt or 10^</summary>
            UO_picovolt = 400000251,

            /// <summary>An electric potential difference unit which is equal to one million volts or 10^</summary>
            UO_megavolt = 400000252,

            /// <summary>A unit which is a standard measure of the ability of a liguid to attraction of molecules at its surface as a result of unbalanced molecular cohesive forces.</summary>
            UO_surface_tension_unit = 400000253,

            /// <summary>A surface tension unit which is equal to one newton per meter.</summary>
            UO_newton_per_meter = 400000254,

            /// <summary>A surface tension unit which is equal to one dyne per centimeter.</summary>
            UO_dyne_per_cm = 400000255,

            /// <summary>A unit which is a standard measure of the internal resistance of fluids to flow.</summary>
            UO_viscosity_unit = 400000256,

            /// <summary>A viscosity unit which is equal to one pascale per second.</summary>
            UO_pascal_second = 400000257,

            /// <summary>A viscosity unit which is equal to one dyne second per square centimeter.</summary>
            UO_poise = 400000258,

            /// <summary>A ratio unit which is an indicator of sound power per unit area.</summary>
            UO_decibel = 400000259,

            /// <summary>A unit which is a standard measure of the estimate of the stochastic effect that a non-uniform radiation dose has on a human.</summary>
            UO_effective_dose_unit = 400000260,

            /// <summary>A unit which represents a standard measurement of the transmission of an entity through a medium.</summary>
            UO_conduction_unit = 400000261,

            /// <summary>A unit which represents a standard measurement of the movement of electrically charged particles through a transmission medium (electrical conductor).</summary>
            UO_electrical_conduction_unit = 400000262,

            /// <summary>A unit which represents a standard measurement of the spontaneous transfer of thermal energy through matter, from a region of higher temperature to a region of lower temperature.</summary>
            UO_heat_conduction_unit = 400000263,

            /// <summary>An electrical conduction unit which is equal to A/V.</summary>
            UO_siemens = 400000264,

            /// <summary>An heat conduction unit which is equal to one watt divided by meter kelvin.</summary>
            UO_watt_per_meter_kelvin = 400000265,

            /// <summary>A non-SI unit of energy (eV) defined as the energy acquired by a single unbound electron when it passes through an electrostatic potential difference of one volt. An electronvolt is equal to 1.602 176 53(14) x 10^-19 J.</summary>
            UO_electronvolt = 400000266,

            /// <summary>The electric field strength is a unit which is a measure of the potential difference between two points some distance apart.</summary>
            UO_electric_field_strength_unit = 400000267,

            /// <summary>The volt per meter is a unit of electric field strength equal to the a potential difference of 1 volt existing between two points that are 1 meter apart.</summary>
            UO_volt_per_meter = 400000268,

            /// <summary>A dimensionless logarithmic unit assigned to a measure of absorbance of light through a partially absorbing substance, defined as -log10(I/I_0) where I = transmitted light and I_0 = incident light.</summary>
            UO_absorbance_unit = 400000269,

            /// <summary>A unit which is a standard measure of the volume of fluid which passes through a given surface per unit time .</summary>
            UO_volumetric_flow_rate_unit = 400000270,

            /// <summary>A volumetric flow rate unit which is equal to one microliter volume through a given surface in one minute.</summary>
            UO_microliters_per_minute = 400000271,

            /// <summary>A unit of pressure equal to the amount of fluid pressure one millimeter deep in mercury at zero degrees centigrade on Earth.</summary>
            UO_millimetres_of_mercury = 400000272,

            /// <summary>A mass unit density which is equal to mass of an object in milligrams divided by the volume in liters.</summary>
            UO_milligram_per_liter = 400000273,

            /// <summary>A mass unit density which is equal to mass of an object in micrograms divided by the volume in millliters.</summary>
            UO_microgram_per_milliliter = 400000274,

            /// <summary>A mass unit density which is equal to mass of an object in nanograms divided by the volume in milliliters.</summary>
            UO_nanogram_per_milliliter = 400000275,

            /// <summary>A concentration unit which is a standard measure of the amount of a substance in a given container.</summary>
            UO_amount_per_container = 400000276,

            /// <summary>A unit which is equal to one microgram per disk, where a disk is some physical surface/container upon which the substance is deposited.</summary>
            UO_ug_disk = 400000277,

            /// <summary>A unit which is equal to one nanomole per disk, where a disk is some physical surface/container upon which the substance is deposited.</summary>
            UO_nmole_disk = 400000278,

            /// <summary>A unit per milliliter unit which is equal to one thousandth of a unit of an agreed arbitrary amount per one milliliter.</summary>
            UO_milliunits_per_milliliter = 400000279,

            /// <summary>A unit which represents a standard measurement occurrence of a process per unit time.</summary>
            UO_rate_unit = 400000280,

            /// <summary>A rate unit which is equal to one over one nanomolar second.</summary>
            UO_count_per_nanomolar_second = 400000281,

            /// <summary>A rate unit which is equal to one over one molar second.</summary>
            UO_count_per_molar_second = 400000282,

            /// <summary>An area density unit which is equal to the mass of an object in kilograms divided by the surface area in hectares.</summary>
            UO_kilogram_per_hectare = 400000283,

            /// <summary>A rate unit which is equal to one over one nanomolar.</summary>
            UO_count_per_nanomolar = 400000284,

            /// <summary>A rate unit which is equal to one over one molar.</summary>
            UO_count_per_molar = 400000285,

            /// <summary>A prefix in the metric system denoting a factor of ten to the power of 24.</summary>
            UO_yotta = 400000286,

            /// <summary>A prefix in the metric system denoting a factor of one hundred.</summary>
            UO_hecto = 400000287,

            /// <summary>A prefix in the metric system denoting a factor of ten to the power of 21.</summary>
            UO_zetta = 400000288,

            /// <summary>A prefix in the metric system denoting a factor of ten to the power of 18.</summary>
            UO_exa = 400000289,

            /// <summary>A prefix in the metric system denoting a factor of ten to the power of 15.</summary>
            UO_peta = 400000290,

            /// <summary>A prefix in the metric system denoting a factor of ten to the power of 12.</summary>
            UO_tera = 400000291,

            /// <summary>A prefix in the metric system denoting a factor of ten to the power of 9.</summary>
            UO_giga = 400000292,

            /// <summary>A prefix in the metric system denoting a factor of million.</summary>
            UO_mega = 400000293,

            /// <summary>A prefix in the metric system denoting a factor of one thousand.</summary>
            UO_kilo = 400000294,

            /// <summary>A prefix in the metric system denoting a factor of ten.</summary>
            UO_deca = 400000295,

            /// <summary>A prefix in the metric system denoting a factor of one tenth.</summary>
            UO_deci = 400000296,

            /// <summary>A prefix in the metric system denoting a factor of one thousand.</summary>
            UO_milli1 = 400000297,

            /// <summary>A prefix in the metric system denoting a factor of one hundred.</summary>
            UO_centi = 400000298,

            /// <summary>A prefix in the metric system denoting a factor of 10 to the power of -6.</summary>
            UO_micro = 400000299,

            /// <summary>A prefix in the metric system denoting a factor of 10 to the power of -9.</summary>
            UO_nano = 400000300,

            /// <summary>A mass unit density which is equal to mass of an object in micrograms divided by the volume in liters.</summary>
            UO_microgram_per_liter = 400000301,

            /// <summary>A prefix in the metric system denoting a factor of 10 to the power of -12.</summary>
            UO_pico = 400000302,

            /// <summary>A prefix in the metric system denoting a factor of 10 to the power of -15.</summary>
            UO_femto = 400000303,

            /// <summary>A prefix in the metric system denoting a factor of 10 to the power of -18.</summary>
            UO_atto = 400000304,

            /// <summary>A prefix in the metric system denoting a factor of 10 to the power of -21.</summary>
            UO_zepto = 400000305,

            /// <summary>A prefix in the metric system denoting a factor of 10 to the power of -24.</summary>
            UO_yocto = 400000306,

            /// <summary>A concentration unit which is a standard measure of the amount of a toxic or pharmaceutical substance administered to a recipient subject, expressed in terms of the size of the subject.</summary>
            UO_dose_unit = 400000307,

            /// <summary>A dose unit which is equal to 1 milligram of a toxic or pharmaceutical substance per kilogram body weight of the recipient subject.</summary>
            UO_milligram_per_kilogram = 400000308,

            /// <summary>A dose unit which is equal to 1 milligram of a toxic or pharmaceutical substance per square meter of surface area of the recipient subject.</summary>
            UO_milligram_per_square_meter = 400000309,

            /// <summary>A concentration unit which is a standard measure of the amount of a toxic or pharmaceutical substance administered over time to a recipient subject, expressed in terms of the size of the subject.</summary>
            UO_dosage_unit = 400000310,

            /// <summary>A dosage unit which is equal to 1 milligram per day of a toxic or pharmaceutical substance per kilogram body weight of the recipient subject.</summary>
            UO_milligram_per_kilogram_per_day = 400000311,

            /// <summary>A derived unit which is a measure of relative light intensity, as typically measured by a luminometer, spectrophotometer, or fluorimeter in biological research applications.</summary>
            UO_relative_light_unit = 400000312,

            /// <summary>A relative light unit which is a measure of relative luminescence intensity.</summary>
            UO_relative_luminescence_unit = 400000313,

            /// <summary>A relative light unit which is a measure of relative fluorescence intensity.</summary>
            UO_relative_fluorescence_unit = 400000314,

            /// <summary>A unit used to indicate the clarity of water or other solutions or suspensions, as measured by the ability of the solution or suspension to scatter light of a defined wavelength range.</summary>
            UO_turbidity_unit = 400000315,

            /// <summary>A unit of cell concentration which is equal to one cell in a volume of 1 microliter.</summary>
            UO_cells_per_microliter = 400000316,

            /// <summary>A unit of cell concentration which is equal to 1 cell in a well or discrete container of arbitrary volume.</summary>
            UO_cells_per_well = 400000317,

            /// <summary>1000 formazin turbidity units (FNU) on the empirical formazin turbidity scale represents reflectance of insol. reaction products of 0.0725 g hydrazine sulfate with 0.7250 g hexamethylenetetramine diluted to 1 L.</summary>
            UO_formazin_nephelometric_unit = 400000318,

            /// <summary>A concentration unit which is a standard measure of the amount of radioactivity in a given volume.</summary>
            UO_radioactivity_concentration = 400000319,

            /// <summary>A unit of radioactivity concentration which is equal to one curie in a volume of 1 liter.</summary>
            UO_curie_per_liter = 400000320,

            /// <summary>A unit of radioactivity concentration which is equal to one micro curie in a volume of 1 liter.</summary>
            UO_microcurie_per_milliliter = 400000321,

            /// <summary>A unit that is the ratio of concentration of two solutions of interest, typically with one solution derived from the other by the addition of solvent.</summary>
            UO_fold_dilution = 400000322,

            /// <summary>An area density unit which is equal to the mass of an object in tons divided by the surface area in hectares.</summary>
            UO_ton_per_hectare = 400000323,

            /// <summary>An area unit which is equal to an area enclosed by a square with sides each 1 angstrom long.</summary>
            UO_square_angstrom = 400000324,

            /// <summary>A frequency unit which is equal to one million hertz or 10^</summary>
            UO_megaHertz = 400000325,

            /// <summary>A unit used to express distances on a genetic map. In genetic mapping, distances between markers are determined by measuring the rate of meoitic recombination between them, which increases proportionately with the distance separating them. A cM is defined as the length of an interval in which there is a 1% probability of recombination. On the average, 1 cM is roughly equivalent to 1 megabase (Mb) of DNA, although this can vary widely due to hot and cold spots of recombination.</summary>
            UO_centiMorgan = 400000326,

            /// <summary>A unit of genetic map distance defined corresponding to an interval in which there is a 1% probability of X-irradiation induced breakage. To be completely specified, the unit must be qualified by the radiation in dosage in rads (e.g. cR8000), because this determines the actual breakage probability.</summary>
            UO_centiRay = 400000327,

            /// <summary>A unit equal to one thousand base pairs.</summary>
            UO_kilo_base_pair = 400000328,

            /// <summary>A unit equal to one million base pairs</summary>
            UO_mega_base_pair = 400000329,

            /// <summary>A unit equal to one billion base pairs.</summary>
            UO_giga_base_pair = 400000330,

            /// <summary>An information unit which is equal to 1000000000 bytes.</summary>
            UO_gigabyte = 400000331,

            /// <summary>An information unit which is equal to 1000 gigabytes.</summary>
            UO_terabyte = 400000332,

            /// <summary>An area unit which is equal to an area enclosed by a square with sides each 1 micrometer long.</summary>
            UO_square_micrometer = 400010001,

            /// <summary>An electrical conduction unit which is equal to one thousandth of a siemen or 10^</summary>
            UO_millisiemens = 400010002,

            /// <summary>A specific concentration unit which is equal to 1 micromole in a given volume of one thousandth of a cubic meter.</summary>
            UO_micromole_per_litre = 400010003,

            /// <summary>A specific concentration unit which is equal to 1 micromole of a given substance per kilogram of solvent.</summary>
            UO_micromole_per_kilogram = 400010004,

            /// <summary>A speed/velocity unit which is equal to the speed of an object traveling 1 millimeter distance in one day.</summary>
            UO_millimeters_per_day_ = 400010005,

            /// <summary>A dimensionless ratio unit which, given a pair of quantities a and b, for which b is a multiple of a, denotes b by giving the multiplier (coefficient) c for a to result in b.</summary>
            UO_ratio1 = 400010006,

            /// <summary>An electrical mobility unit which is equal to one volt second per square centimeter.</summary>
            UO_volt_second_per_square_centimeter = 400010007,

            /// <summary>A speed/velocity unit which is equal to the speed of an object traveling 1 kilometer distance in one hour.</summary>
            UO_kilometer_per_hour = 400010008,

            /// <summary>A mass unit which is equal to 1/12 the mass of 12C</summary>
            UO_milli = 400010009,

            /// <summary>An area unit which is equal to an area of 10,000 square meters. Equivalent to 2.471 acres.</summary>
            UO_hectare = 400010010,

            /// <summary>A length unit which is equal to 0.0254 metres.</summary>
            UO_inch = 400010011,

            /// <summary>A length unit which is equal to 0.0254 millimetres.</summary>
            UO_thou = 400010012,

            /// <summary>A length unit which is equal to 0.3048 metres, or 12 inches.</summary>
            UO_foot = 400010013,

            /// <summary>A length unit which is equal to 0.9144 metres, or 3 feet.</summary>
            UO_yard = 400010014,

            /// <summary>A length unit which is equal to 20.1168 metres, 66 feet, or 22 yards.</summary>
            UO_chain = 400010015,

            /// <summary>A length unit which is equal to 20,116.8 metres, 660 feet, or 10 chains.</summary>
            UO_furlong = 400010016,

            /// <summary>A length unit which is equal to 1,609.344 metres, or 8 furlongs.</summary>
            UO_mile = 400010017,

            /// <summary>A length unit which is equal to 3 miles, or 4,828.032 metres</summary>
            UO_league = 400010018,

            /// <summary>A maritime length unit is one used primarily at sea.</summary>
            UO_maritime_length_unit = 400010019,

            /// <summary>A maritime length unit which is equal to 6.08 feet, or 1.853184 metres</summary>
            UO_fathom = 400010020,

            /// <summary>A maritime length unit which is equal to 608 feet, 100 fathoms, or 185.3184 metres</summary>
            UO_cable = 400010021,

            /// <summary>A maritime length unit which is equal to 6,080 feet, 10 cables, or 1,853.184 metres</summary>
            UO_nautical_mile = 400010022,

            /// <summary>An area unit which is equal to an area of 25.292,852,64 square meters, or 1 square rod.</summary>
            UO_perch = 400010023,

            /// <summary>An area unit which is equivalent to 1 furlong x 1 rod. This is equal to an area of 1,011.714,1056 square meters, or 40 square rods.</summary>
            UO_rood = 400010024,

            /// <summary>An area unit which is equivalent to 1 furlong x 1 chain. This is equal to an area of 4,046.856,4224 square meters, or 43,500 square feet.</summary>
            UO_acre = 400010025,

            /// <summary>An imperial volume unit which is equivalent to 28.413,0625 millilitres.</summary>
            UO_fluid_ounce = 400010026,

            /// <summary>An imperial volume unit which is equivalent to 142.065,3125 millilitres.</summary>
            UO_gill = 400010027,

            /// <summary>An imperial volume unit which is equivalent to 568.261,25 millilitres.</summary>
            UO_pint = 400010028,

            /// <summary>An imperial volume unit which is equivalent to 1,136.5225 millilitres, or two pints.</summary>
            UO_quart = 400010029,

            /// <summary>An imperial volume unit which is equivalent to 4,546.09 millilitres, or 8 pints.</summary>
            UO_gallon = 400010030,

            /// <summary>An imperial mass unit which is equivalent to 64.798,91 milligrams.</summary>
            UO_grain = 400010031,

            /// <summary>An imperial mass unit which is equivalent to 1.771,845,195,3125 grams, or 1/256 of 1 pound.</summary>
            UO_drachm = 400010032,

            /// <summary>An imperial mass unit which is equivalent to 28.349,523,125 grams, or 1/16 of 1 pound.</summary>
            UO_ounce = 400010033,

            /// <summary>An imperial mass unit which is equivalent to 453.592,37 grams.</summary>
            UO_pound = 400010034,

            /// <summary>An imperial mass unit which is equivalent to 6,350.293,18 grams, or 14 pounds.</summary>
            UO_stone = 400010035,

            /// <summary>An imperial mass unit which is equivalent to 12.700,586,36 kilograms, or 28 pounds.</summary>
            UO_quarter = 400010036,

            /// <summary>An imperial mass unit which is equivalent to 50.802,345,44 kilograms, 112 pounds, or 8 stone.</summary>
            UO_hundredweight = 400010037,

            /// <summary>An imperial mass unit which is equivalent to 1,016.046,9088 kilograms, or 2,240 pounds.</summary>
            UO_ton = 400010038,

            /// <summary>An imperial gravitational unit which is equivalent to a mass that accelerates by 1ft/s when a force of one pound (lbf) is exerted on it.</summary>
            UO_slug = 400010039,

            /// <summary>A metric teaspoon is a unit of measurement of volume widely used in cooking recipes and pharmaceutic prescriptions. It equals a 5mL volume.</summary>
            UO_teaspoon = 400010040,

            /// <summary>A United States customary units teaspoon is a unit of measurement of volume widely used in cooking recipes and pharmaceutic prescriptions in America. It equals a 4.93 mL volume.</summary>
            UO_united_states_customary_teaspoon = 400010041,

            /// <summary>A metric tablespoon is a unit of measurement of volume widely used in cooking recipes and pharmaceutic prescriptions. It equals a 15mL volume.</summary>
            UO_tablespoon = 400010042,

            /// <summary>An Australian metric tablespoon is a unit of measurement of volume used in Australia for cooking recipes and pharmaceutic prescriptions. It equals a 20mL volume.</summary>
            UO_australian_metric_tablespoon = 400010043,

            /// <summary>A United States customary units tablespoon is a unit of measurement of volume widely used in cooking recipes and pharmaceutic prescriptions in America. It equals a 14.79 mL volume.</summary>
            UO_united_states_customary_tablespoon_ = 400010044,

            /// <summary>A metric cup is a unit of measurement of volume widely used in cooking recipes and pharmaceutic prescriptions. It equals a 250mL volume.</summary>
            UO_metric_cup = 400010045,

            /// <summary>A United States customary units cup is a unit of measurement of volume widely used in cooking recipes and pharmaceutic prescriptions in America. It equals a 236.59 mL volume.</summary>
            UO_united_states_customary_cup = 400010046,

            /// <summary>A United States FDA cup is a unit of measurement of volume used by the US Federal Department of Agriculture as a nutritional serving measure. It equals a 240 mL volume.</summary>
            UO_united_states_fda_cup = 400010047,

            /// <summary>A substance unit which is equal to one millionth of a mole.</summary>
            UO_micromole1 = 400010048,

            /// <summary>An area density unit which is equal to the mass of an object in grams divided by the surface area in meters squared.</summary>
            UO_gram_per_square_meter = 400010049,

            /// <summary>A mass unit density which is equal to mass of an object in nanograms divided by the volume in microliters.</summary>
            UO_nanogram_per_microliter_ = 400010050,

            /// <summary>A unit of energy widely used in nutrition, equivalent to the amount of heat needed to cause one kilogram of water to rise in temperature by one degree Celsius.</summary>
            UO_large_calorie = 400010051,

            /// <summary>A pressure unit which is equal to 6894.757 pascal.</summary>
            UO_pounds_per_square_inch = 400010052,

            /// <summary>A unit of mass concentration defined as the number of micrograms of a substance per 250 ml Wheaton bottle.</summary>
            UO_micrograms_per_wheaton_bottle = 400010053,

            /// <summary>An electric current unit current which is equal to one trillionth of an ampere or 10^</summary>
            UO_picoampere = 400010054,

            /// <summary>An absorbed dose unit which is equal to 0.01 gray (Gy)</summary>
            UO_centigray = 400010055,

            /// <summary>A unit of energy that is equal to one thousand million electronvolts.</summary>
            UO_gigaelectronvolt = 400010056,

            /// <summary>Derived unit which is equal to the total kinetic energy of an atomic nucleus in GeV divided by the number of nucleons in the nucleus</summary>
            UO_gigaelectronvolt_per_nucleon = 400010057,

            /// <summary>Derived unit which is equal to the total kinetic energy of an atomic nucleus in MeV divided by the number of nucleons in the nucleus</summary>
            UO_megaelectronvolt_per_nucleon = 400010058,

            /// <summary>A derived unit equal to the energy deposited by ionizing radiation per micron of path length in matter</summary>
            UO_kiloelectronvolt_per_micron = 400010059,

            /// <summary>A derived absorbed dose unit which is equal to one gray absorbed per minute.</summary>
            UO_gray_per_minute = 400010060,

            /// <summary>A derived absorbed dose unit which is equal to one centigray absorbed per minute.</summary>
            UO_centigray_per_minute = 400010061,

            /// <summary>A derived absorbed dose unit which is equal to one milligray absorbed per minute.</summary>
            UO_milligray_per_minute = 400010062,

            /// <summary>A derived absorbed dose unit which is equal to one milligray absorbed per day.</summary>
            UO_milligray_per_day = 400010063,

            /// <summary>A derived absorbed dose unit which is equal to one milligray absorbed per hour.</summary>
            UO_milligray_per_hour = 400010064,

            /// <summary>A derived absorbed dose unit which is equal to one milligray absorbed per second.</summary>
            UO_milligray_per_second = 400010065,

        }
    }
}
